{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "## importing requried libraries\n",
    "import os\n",
    "import glob\n",
    "import fitz  # PyMuPDF for PDF processing\n",
    "import numpy as np\n",
    "import faiss\n",
    "\n",
    "from PyPDF2 import PdfReader\n",
    "\n",
    "# from langchain.llms import Groq\n",
    "from langchain_groq import ChatGroq\n",
    "from langchain.embeddings import HuggingFaceEmbeddings,OllamaEmbeddings\n",
    "from langchain.vectorstores.cassandra import Cassandra\n",
    "import cassio\n",
    "\n",
    "from dotenv import load_dotenv\n",
    "load_dotenv()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "## calling api keys\n",
    "GROQ_API_KEY = os.environ['GROQ_API_KEY']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Initialize LLM\n",
    "groq_llm = ChatGroq(api_key=GROQ_API_KEY)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract text from PDFs\n",
    "data_folder = \"data/\"\n",
    "\n",
    "### Load the documents from the data folder\n",
    "pdf_files = glob.glob(os.path.join(data_folder, \"*.pdf\"))\n",
    "documents = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['data\\\\paper1.pdf',\n",
       " 'data\\\\paper2.pdf',\n",
       " 'data\\\\paper3.pdf',\n",
       " 'data\\\\paper4.pdf',\n",
       " 'data\\\\paper5.pdf',\n",
       " 'data\\\\paper6.pdf']"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pdf_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pymupdf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [],
   "source": [
    "for file in pdf_files:\n",
    "        with pymupdf.open(file) as doc:\n",
    "            text = \"\\n\".join([page.get_text(\"text\") for page in doc])\n",
    "            documents.append(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Chunking the text\n",
    "chunk_size=500\n",
    "for i in range(0, len(text), chunk_size):\n",
    "    documents.append(text[i:i+chunk_size])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "163"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(documents)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Expert Opinion on Drug Discovery\\nISSN: (Print) (Online) Journal homepage: www.tandfonline.com/journals/iedc20\\nInduﬆrializing AI-powered drug discovery: lessons\\nlearned from the Patrimony computing platform\\nMickaël Guedj, Jack Swindle, Antoine Hamon, Sandra Hubert, Emiko\\nDesvaux, Jessica Laplume, Laura Xuereb, Céline Lefebvre, Yannick Haudry,\\nChristine Gabarroca, Audrey Aussy, Laurence Laigle, Isabelle Dupin-Roger &\\nPhilippe Moingeon\\nTo cite this article: Mickaël Guedj, Jack Swindle, Antoine Hamon, Sandra Hubert, Emiko\\nDesvaux, Jessica Laplume, Laura Xuereb, Céline Lefebvre, Yannick Haudry, Christine Gabarroca,\\nAudrey Aussy, Laurence Laigle, Isabelle Dupin-Roger & Philippe Moingeon (2022) Industrializing\\nAI-powered drug discovery: lessons learned from the Patrimony computing platform, Expert\\nOpinion on Drug Discovery, 17:8, 815-824, DOI: 10.1080/17460441.2022.2095368\\nTo link to this article:  https://doi.org/10.1080/17460441.2022.2095368\\n© 2022 The Author(s). Published by Informa\\nUK Limited, trading as Taylor & Francis\\nGroup.\\nView supplementary material \\nPublished online: 10 Jul 2022.\\nSubmit your article to this journal \\nArticle views: 3907\\nView related articles \\nView Crossmark data\\nCiting articles: 11 View citing articles \\nFull Terms & Conditions of access and use can be found at\\nhttps://www.tandfonline.com/action/journalInformation?journalCode=iedc20\\n\\nTECHNOLOGY EVALUATION\\nIndustrializing AI-powered drug discovery: lessons learned from the Patrimony \\ncomputing platform\\nMickaël Guedj\\na, Jack Swindleb, Antoine Hamonb, Sandra Huberta, Emiko Desvauxa, Jessica Laplumea, \\nLaura Xuereba, Céline Lefebvrea, Yannick Haudrya, Christine Gabarrocaa, Audrey Aussya, Laurence Laiglea, \\nIsabelle Dupin-Rogera and Philippe Moingeon\\na\\naServier, Research & Development, Suresnes, France; bLincoln, Research & Development, Boulogne-Billancourt, France\\nABSTRACT\\nIntroduction: As a mid-size international pharmaceutical company, we initiated 4 years ago the launch \\nof a dedicated high-throughput computing platform supporting drug discovery. The platform named \\n‘Patrimony’ was built up on the initial predicate to capitalize on our proprietary data while leveraging \\npublic data sources in order to foster a Computational Precision Medicine approach with the power of \\nartificial intelligence.\\nAreas covered: Specifically, Patrimony is designed to identify novel therapeutic target candidates. With \\nseveral successful use cases in immuno-inflammatory diseases, and current ongoing extension to \\napplications to oncology and neurology, we document how this industrial computational platform \\nhas had a transformational impact on our R&D, making it more competitive, as well time and cost \\neffective through a model-based educated selection of therapeutic targets and drug candidates.\\nExpert opinion: We report our achievements, but also our challenges in implementing data access and \\ngovernance processes, building up hardware and user interfaces, and acculturing scientists to use \\npredictive models to inform decisions.\\nARTICLE HISTORY\\nReceived 19 April 2022  \\nAccepted 24 June 2022  \\nKEYWORDS\\nDrug discovery; target \\nidentification; data \\nintegration; artificial \\nintelligence; multi-omics; \\ncomputing platform; \\nComputational Precision \\nMedicine\\n1. Introduction\\nIn the last decades, the pharmaceutical industry has faced \\na continuous decrease in productivity. R&D efficiency, mea\\xad\\nsured by the number of new drugs brought to patients per \\ndollar spent, has halved approximately every 10 years since \\n1950. This trend is often referred to as the Eroom’s Law, i.e. \\na reverse of the well-known Moore’s Law reflecting the expo\\xad\\nnential growth observed over time for numbers of transistors \\non a microchip [1]. As of today, it takes on average 12 years \\nand 2.6 billion U.S. dollars to bring a new drug to patients, \\nwith a probability of success of around 5–10%[2]. Root causes \\nencompass an increase in regulatory requirements, the lack of \\nsufficient validation in the selection of both therapeutic tar\\xad\\ngets and drug candidates prior to setting up costly and time- \\nconsuming clinical trials, and a need to optimize organiza\\xad\\ntional processes to better integrate scientific knowledge \\nwithin R&D [3–5]. Arguably, the most actionable lever for the \\npharmaceutical industry in order to increase the probability of \\nsuccess during drug development is to strengthen the ratio\\xad\\nnale behind decision-making, most particularly as it relates to \\nthe choice of the therapeutic target and the selection of the \\ndrug candidate.\\nTo this aim, recent breakthroughs in both biomedical and \\ncomputational sciences create new opportunities to inform \\ndrug development through computer-based approaches [6– \\n13]. Rapid advances in omics, imaging, and electronic capture \\ntechnologies make it now possible to characterize individuals \\nat both molecular, cellular, and clinical levels in a cost and \\ntime effective way [14]. Those advances occur in parallel with \\nthe exponential accumulation of information and knowledge \\naccessible through hundreds of structured biomedical data\\xad\\nbases, such as those managed by the European Bioinformatics \\nInstitute (EBI) or the US National Center for Biotechnology \\nInformation (NCBI). An effective use of these massive amounts \\nof data is facilitated by new computational approaches includ\\xad\\ning artificial intelligence (AI) and machine-learning (ML), thus \\ncreating an unprecedented opportunity to better inform deci\\xad\\nsion-making and decrease both costs and attrition rates at all \\nstages of drug discovery and development [5]. This ongoing \\nrevolution toward AI-powered drug discovery already trans\\xad\\nlates into concrete successes, with machine-designed anti- \\ncancer molecules reported to reach Phase 1 in less than \\n2 years in contrast to the 5–7 years commonly needed for \\nthe discovery phase [15].\\nAn emerging strategy for the pharmaceutical industry to \\ncapitalize on those new approaches is to build-an internal \\ncomputing platform to support the identification of disease \\ntargets as well as the repurposing of existing drugs in \\na systematic and efficient way. To this aim, a technical chal\\xad\\nlenge is to integrate the ever-increasing number of data \\nsources compiling massive and multidimensional information \\ncoming from genetics, multi-omics, molecular interactions, \\nCONTACT Mickaël Guedj \\nmickael.guedj@gmail.com \\nServier, Research & Development, Suresnes Cedex, France\\nSupplemental data for this article can be accessed online at https://doi.org/10.1080/17460441.2022.2095368\\nEXPERT OPINION ON DRUG DISCOVERY                                                                                                                        \\n2022, VOL. 17, NO. 8, 815–824\\nhttps://doi.org/10.1080/17460441.2022.2095368\\n© 2022 The Author(s). Published by Informa UK Limited, trading as Taylor & Francis Group. \\nThis is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives License (http://creativecommons.org/licenses/by-nc-nd/4.0/), \\nwhich permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited, and is not altered, transformed, or built upon in any way.\\n\\npreclinical experiments, clinical as well as real-life evidence \\ndata. A dedicated and specifically designed computational \\nframework is necessary to capture the value of all these infor\\xad\\nmation in order to guide decision-making and increase the \\nprobability of success during drug development.\\nIn light of the rapidly evolving environment driven by AI \\nand digital technologies, a decision was made in early 2018 \\nby the pharmaceutical group Servier to implement within the \\nR&D new data processes and computational methods \\nthrough a dedicated high-throughput computing platform. \\nGiven that a primary objective of this initiative was to valorize \\nexisting data, knowledge, and assets produced by the com\\xad\\npany during previous or ongoing R&D projects, the platform \\nwas named ‘Patrimony.’ The latter was built up with the intent \\nto capitalize on both proprietary and public data to drive \\ninnovation. After 4 years of implementation, the Patrimony \\nplatform has transformed very significantly Servier’s approach \\nto drug discovery and development. Herein, we share the \\nmain lessons learned from this initiative during its conception \\nand implementation, as well as the challenges that were \\nfaced to make it impactful on our drug development \\nprocesses.\\n2. Overview of the market\\nSeveral public or public–private initiatives have emerged with \\nthis orientation such as Open Targets [16,17]. In parallel, the \\nperspective of substantial cost savings offered to pharmaceuti\\xad\\ncal industries by substituting computerized modeling systems to \\ntraditional wet-bench biology, prompted the emergence of \\nnumerous start-ups aiming to reinvent drug discovery (Supp. \\nTable 1). In this context, in parallel to initiating this project, we \\nperformed a benchmark analysis of external solutions. We con\\xad\\ncluded that given its foreseen strategic relevance for the com\\xad\\npany, there was a strong added value to implement the project \\ninternally for gaining reactivity, flexibility, as well as for a better \\nintegration of internal and external data sources.\\n3. How the Patrimony computing platform works\\n3.1. General framework\\nA computing platform comprises a set of hardware infrastruc\\xad\\ntures, software, and user interface components that allow stor\\xad\\ning data and running algorithms in order to achieve a set of \\nwell-defined tasks within a specific field of application. The \\nmethodology we applied to build up such a computing platform \\nfor drug discovery is summarized in Figure 1, combining a range \\nof computational approaches in workflows to integrate, analyze, \\nand interpret data. A first step was to identify all relevant exist\\xad\\ning biomedical databases and knowledge sources, both struc\\xad\\ntured and unstructured, public or internal. A second step was to \\ncurate and integrate data sources into a knowledge graph (or \\nnetwork) as defined below. Lastly, algorithms were developed to \\nmine this graph for a specific application (related to a given \\ndisease or a therapeutic area) in order to generate and prioritize \\nhypotheses regarding new therapeutic targets or opportunities \\nfor drug repurposing. Analyzing such a volume and diversity of \\ndata turned out to be challenging, thus requiring an adapted \\ncomputational framework to ensure both a good integration, \\nappropriate use, and traceability. This methodology was con\\xad\\ncretized into a high-throughput scalable process encompassing \\nall steps from data acquisition, hypothesis generation, prioritiza\\xad\\ntion of outcomes, and experimental validation.\\nWe separated data sources based upon whether they were \\neither in-house or public, as well as application-agnostic versus \\napplication-related. From a core set of in-house/public sources \\nshared for all applications, we subsequently added a data pack\\xad\\nage specific to each single application of interest, making the \\nPatrimony framework both robust and highly flexible. In-house \\nsources broadly used to support Servier’s R&D projects com\\xad\\nprised data related to both therapeutic targets being investi\\xad\\ngated, potentially relevant proprietary drug(s), the phase within \\nresearch or development as well as the therapeutic area. Core \\npublic sources used such as DrugBank or UniProt listed in Supp. \\nTable 2 encompass the existing public knowledge on multiple \\ndomains of interest (e.g. biomolecular, pharmacological, clinical) \\n[16–52]. The application-related data package further assembled \\nfocused on the disease or set of diseases of interest, which for \\nServier relates to immuno-inflammatory, oncological, and neu\\xad\\nrological disorders. The level of implication of various genes and \\nproteins in pathophysiological processes was obtained from \\nmulti-omics patient profiling data, by comparing cases and \\ncontrols in different relevant conditions; either from aggregated \\nstatistics or derived from sample-level molecular data (subse\\xad\\nquently turned internally into aggregated statistics) retrieved \\nfrom both public repositories such as GEO or UK Biobank listed \\nSupp. Table 2, partnerships such as public–private IMI projects \\nas well as proprietary experimental data [53–57].\\n3.2. Building up a proprietary and adaptable knowledge \\ngraph\\nKnowledge graphs are network-like digital structures repre\\xad\\nsenting knowledge as a set of concepts and their relationships. \\nAs such, they facilitate the interface between humans and \\nmachines to analyze their content and support complex \\nArticle highlights\\n●Patrimony is a computing platform implemented by the pharmaceu\\xad\\ntical company Servier to capture the value of massive biomedical \\ndata (proprietary and public) with machine-learning techniques to \\nsupport drug discovery.\\n●The platform is based on a knowledge-graph connecting biomolecu\\xad\\nlar, pharmacological, and clinical domains, which can be mined to \\ngenerate new therapeutic hypotheses.\\n●Hypotheses are assessed and prioritized by aggregating information \\naround five summary criteria (Biological Relevance, Causality, \\nTractability, Safety, and Innovativeness).\\n●The platform was implemented following three iterations (proof-of- \\nconcept, structuration, and industrialization), while relying upon an \\nAgile operating model and FAIR guiding principles for data \\nmanagement.\\n●Reported applications encompass immuno-inflammatory diseases \\nand COVID-19; from these successes, the use of the platform is \\nbeing extended to oncology and neurology.\\n●Patrimony fosters an open innovation mindset, requiring both agility \\nand transversality across a broad range of existing and emerging \\nexpertises associated with AI.\\n816\\nM. GUEDJ ET AL.\\n\\ndecision-making. They are well adapted to model the inter\\xad\\nconnectivity of biomedical systems. Numerous detailed and \\nformal introductions to their application in Medicine have \\nbeen published elsewhere [58–60].\\nFrom the data sources described above, we built \\na knowledge graph connecting an unbiased mapping of all \\nknown molecular interactions (i.e. the Human Interactome) to \\npharmacological and clinical domains (Figure 2a). The result\\xad\\ning graph was made of overall 50k+ nodes and 200k+ relation\\xad\\nships, combined with all information we could gather or \\ngenerate as attributes on each node or interaction, and further \\nenriched with semantics or ontologies to help navigating \\nthrough the concepts (e.g. GO, ChEBI, EFO, MedDRA) [29– \\n31,36,48]. For each application, specific related data sources \\nwere used to complement the core knowledge graph with \\na set of additional nodes, interactions and attributes. \\nAggregate statistics resulting from multi-omics analyses (e.g. \\nfold-changes, p-values, etc.) were mapped to the knowledge \\ngraph as attributes of gene or protein nodes. Interactions \\ninferred from patient-level data such as gene–gene co- \\nexpression values were used to weight or complete the \\nknown interactions between them.\\nAt the heart of Patrimony is the relationship between the \\nbiomolecular, pharmacological, and clinical spaces in order to \\nidentify the most relevant therapeutic targets with regard to \\nthe measured pathophysiological manifestations of a disease \\nor a set of diseases (Figure 2a). The resulting knowledge graph \\nis unique to the owner and proprietary in the sense that it \\nintegrates public data with internal inputs. Consequently, it \\nrepresents a very strategic entry point to capture the value of \\nthe scientific knowledge accumulated over time by the \\ncompany.\\nIn order to mine this knowledge graph and extract the \\nmost relevant information, some specific methodological \\nexpertise has been developed (Figure 2b). As main examples, \\nthe identification of nodes exhibiting more frequent interac\\xad\\ntions with other nodes (i.e. hubs), or nodes having frequent \\ninteractions with each other (i.e. clusters) were deemed of \\nFigure 1. Patrimony general framework. Data sources are curated and integrated into a knowledge graph. For each application starting from one disease or a set of \\ndiseases, the knowledge graph is mined in order to evaluate the putative targets, generate hypotheses, and assist the consolidation.\\na\\nb\\nFigure 2. Patrimony knowledge graph. (a) Data sources are integrated through a relational diagram to form the Patrimony knowledge graph covering and \\ninterconnecting biomolecular, pharmacological, disease, and clinical domains. (b) Different metrics and approaches are used to mine the knowledge graph.\\nEXPERT OPINION ON DRUG DISCOVERY\\n817\\n\\nparticular interest as they tend to correspond to molecules \\npredicted to play a key role in biological or pathological \\nprocesses. The concept of distance within networks was also \\nfound to be critical, albeit not easy to quantify. Whereas the \\ndistance between two nodes can appear trivial to determine, \\nassessing the distance between two sets of nodes raises many \\nquestions depending upon the use case. We generally use \\na topological distance corresponding to the shortest path \\nlength in the graph between the nodes of interest. As the \\nsignificance of the distance depends on the density of the \\ngraph, we generate a distribution from bootstrapping similar \\nnodes defined by same degree in the graph; from this given \\ndistribution, \\nwe \\nderive \\na \\nstrandardized \\nz-score \\nand \\na corresponding p-value. Diffusion/propagation algorithms \\nwere selected as particularly useful because – as their name \\ninfer – they diffuse or propagate the information along the \\nlinks of the graph. It assesses the impact of a perturbation \\nstarting from a given node on the network by use of radom- \\nwalk probabilities. As such, they allow to capture the informa\\xad\\ntion on both the nodes and their neighborhood [61]. \\nFurthermore, the graph structure turned out to be very \\nadapted to facilitate the direct application of graph-based \\ndeep learning approaches such as Graph Convolutional \\nNetworks (GCNs) that aggregate features of the different \\ntypes of nodes and their relationships in order to predict \\nnew associations between drugs, targets, and diseases \\n[60,62]. The general principle is to learn how to represent \\nthe graph and map its nodes into a compact embedding \\nspace. As a result, it embeds diseases associated with similar \\ngenes or drugs whose target proteins have similar local neigh\\xad\\nborhoods close together in the embedding space. Altogether, \\none advantage of the knowledge graph supporting the \\nPatrimony computing platform turned out to allow initiating \\ninvestigations from any entry point among genes, diseases, or \\ndrugs depending on the data sources already integrated \\nwithin the platform. Potential queries related for instance to \\nthe most relevant targets for a given disease, or to potential \\nindications in which a drug could be repurposed. As such, the \\nPatrimony computing platform has now been well established \\nwithin Servier as a versatile tool to create and maintain \\na competitive advantage when developing drugs against dis\\xad\\neases of interest.\\n3.3. Target hypothesis assessment and prioritization\\nA central application of Patrimony is to identify novel thera\\xad\\npeutic target candidates from the modeling of a disease of \\ninterest. To capitalize on the high quantity of information \\ncontained in its knowledge graph, some metrics have been \\nestablished in order to rationalize the assessment and prior\\xad\\nitization of actionable therapeutic targets. Inspired by princi\\xad\\nples originally found in Open Targets, we eventually selected \\nstrategic summary criteria along five distinct dimensions \\n(Figure 3a).\\nThe first and most important criterion relates to Biological \\nRelevance. For a given use, it summarizes all the activities \\ncontributing to the understanding of the pathophysiology of \\na disease from multi-omics data. A score is generated to \\nquantify the level of cumulated evidence predicting a gene \\nor a protein to be a highly relevant target because of biomo\\xad\\nlecular associations or dysregulations, by itself and/or when \\nconsidering its neighborhood within the graph. Genes with \\na high Biological Relevance are referred to as disease-related \\ngenes, in that they are likely to contribute to the pathophy\\xad\\nsiology as a cause or a consequence. Also, they tend to cluster \\nand form identifiable disease modules within the knowledge \\na\\nc\\nb\\nFigure 3. Target assessment and prioritization. (a) Targets are assessed based on strategic summary criteria along five distinct dimensions. (b)The five criteria are \\nindividually quantified and a global scoring is then computed in order to prioritize the top targets for which the overall rationale is the highest. (c) Individual target \\nID cards are generated to represent the global assessment in a visual and easily interpretable format to end users.\\n818\\nM. GUEDJ ET AL.\\n\\ngraph. Computing techniques described previously including \\nAI/ML support here the prediction of new disease–gene asso\\xad\\nciations as well as disease modules. A second criterion, related \\nto biological relevance but nevertheless assessed indepen\\xad\\ndently, is Causality, which we consider as a highly critical \\ndimension. The latter is being assessed to discriminate \\nbetween causes and consequences in the observed pathophy\\xad\\nsiology. As such and depending on the information available, \\nit can be derived from (i) the force of genetic associations \\naround the target (taking into account that genetic signals are \\nrather causal compared to transcriptomic profiles which often \\nresult from downstream regulations), (ii) the expression of the \\ntarget in cells or tissues relevant to the disease of interest \\n(documented for instance from GTeX), and (iii) AI/ML predic\\xad\\ntions of what is likely to be a real target based on graph \\nfeatures and known approved targets as training. On \\na concrete basis, Causality may for instance refer to master \\nregulators or driver mutations [32,33]. A third criterion relates \\nto Tractability (also referred to as Druggability), which repre\\xad\\nsents the likelihood to modulate the function of a target with \\neither small synthetic or biological drugs. To date, we have \\nbeen using the measure proposed by Open Targets based on \\nboth precedence of the target in clinical trials, discovery \\nexperiments and computational predictions. In the future, \\nthis assessment will be supplemented with our own measures \\nof Tractability to include additional modalities of interest, e.g. \\nRNA-based antisense oligonucleotides. The fourth criterion is \\nlinked to potential Safety implications when interfering with \\na given target. It is assessed by the number of safety events \\nassociated with drugs known to bind the target reported in \\ndatabases such as SIDER [44]. In subsequent developments, we \\nwill consider not only the number but also the severity of \\nadverse events. The last criterion considers Innovativeness in \\nrelationship with the application on a disease of interest, \\ndocumented from either clinical trials, as well as patent or \\nliterature-mining by using Natural Language Processing (NLP) \\n[63]. Given the flexibility of the Patrimony computational fra\\xad\\nmework, any other relevant summary criterion could be easily \\nincluded in the future. For instance, assessing the Feasibility of \\ndrug development could be added to support target ranking, \\nas suggested by others [64].\\nThe above mentioned five criteria are individually quanti\\xad\\nfied and subsequently computed in a global scoring in order \\nto prioritize the top targets for which the overall rationale is \\nthe highest (Figure 3b). For any given target, individual ‘target \\nID cards’ are generated to represent the overall assessment in \\na visual and easily interpretable manner to end users \\n(Figure 3c).\\n3.4. From targets to drugs\\nFollowing the prioritization of top hypotheses on candidate \\ntargets with the Patrimony scoring system, the rationale is \\nsubsequently consolidated through a deep-dive investigation \\nby biologists and pharmacologists. During this consolidation \\nphase, both an extensive literature review and in house trans\\xad\\nlational data analyses are performed to confirm that identified \\ntarget candidates are involved in specific disease pathways \\nand are druggable with a specific compound modality.\\nResearchers can then validate target hypotheses, through \\nan experimental confirmation that disease activity is impacted \\nfollowing perturbation of the target of interest with a drug or \\na tool compound. Conducting wet-lab gene inhibition (e.g. via \\nCRISPR-Cas9 deletion or RNA silencing) or preclinical experi\\xad\\nments by using cellular assays or animal models are com\\xad\\nmonly implemented to corroborate the hypothesis that \\ndrugs interacting with the target exhibit the anticipated phar\\xad\\nmacological activity. Once a therapeutic target has been \\nselected, multiple processes streamlined by the pharmaceuti\\xad\\ncal industry can be used to identify small molecules or biolo\\xad\\ngicals interacting with it. For instance, High-Throughput \\nScreening (HTS) can be implemented to test the company’s \\nproprietary compound library in various molecular or cell- \\nbased assay systems in order to identify drug candidates \\n[65]. As of today, another strategy relies upon dedicated com\\xad\\nputational methods to select in silico drugs predicted to \\nengage the target of interest [15].\\nOne specific use of the Patrimony platform consists in \\nidentifying existing drugs modulating the target, e.g. within \\nthe R&D pipeline of the company or among marketed com\\xad\\npounds. Referred to as drug repurposing, drug rescuing or \\nindication extension, this approach has generated a growing \\ninterest during the last decade, with evidence that mining \\navailable biomedical data with proper algorithms can gener\\xad\\nate fast and valid innovative hypotheses [41,66–68]. The \\nknowledge graph used in Patrimony has thus also been \\napplied to such drug repurposing approaches, by using both \\ndistances between drug targets and disease-related proteins \\n[69], connectivity maps [23,24], and deep-learning methodol\\xad\\nogy to identify in silico new targets for known drugs [70].\\n3.5. Implementation\\nWe defined three iterations to implement the Patrimony com\\xad\\nputing platform: a proof-of-concept, a structuration, and an \\nindustrialization step. The proof-of-concept was pilot initiative \\nperformed within 3 months aiming for quick-wins. Based on \\na minimal set of datasets and algorithms, the aim was to \\nposition a given set of targets of interest into one disease \\n(namely Sjögren syndrome). Then, the structuration step \\naimed to list, retrieve, and implement all the necessary data\\xad\\nsets and algorithms along with conceiving a first dedicated \\nand adapted computational infrastructure. This step was per\\xad\\nformed around a well-defined application focused on \\nimmuno-inflammatory diseases. Finally, the industrialization \\nstep aimed to transfer the existing Patrimony platform into \\na more scalable architecture in order to pave the ground for \\na subsequent application to all therapeutic areas of interest for \\nthe company. Within each iteration, we adopted an Agile \\noperating model for software development with alternated \\nsequences of brainstorming, implementation (sometimes in \\nprecisely defined sprints), generation of results, consolidation, \\nand feedback [71]. Technical choices for IT infrastructure, soft\\xad\\nware and algorithms have proven to be challenging to ensure \\na robust and flexible solution in a fast-evolving field. Codes \\nwere developed in Python and R. For the two first iterations, \\nwe put in place a Microsoft Azure cloud-based sandbox with \\nMongoDB for database management. We subsequently moved \\nEXPERT OPINION ON DRUG DISCOVERY\\n819\\n\\nto Google Cloud Platform (GCP) and BigQuery for scalability. We \\nalso used a mixture of Neo4j, Cytoscape, Python graph-tool, and \\nR igraph to support graph storage, mining, and visualization \\n[72]. We followed FAIR guiding principles to enable findability, \\naccessibility, interoperability, and reusability of the data when \\nbuilding up our new data governance. Integrating large-scale \\nand multidimensional data generated from multiple technol\\xad\\nogies with proper quality attributes in terms of consistency \\nand reliability remained a significant difficulty throughout data \\nlife-cycle management. Assessing the right to use data from \\npublic sources also turned out to be complex. Whereas some \\ndata sources apply a clear ‘no restriction’ policy for any use of \\nthe data, e.g. under a Creative Common license for sharing, \\nothers make a distinction between the type of requesting \\ninstitution (profit versus nonprofit) or intended use (research \\nversus commercial purposes). Of note, it has been decided not \\nto include any patient-level data in order to overcome the \\nregulatory complexity linked to their use. Also, the possibility \\nto include data from partnerships such as IMI projects needs \\nto be evaluated case by case.\\n4. Applications\\nThe two first iterations (proof-of-concept and structuration) of \\nthe Patrimony initiative were built while focusing on immuno- \\ninflammatory diseases as selected indications. They were spe\\xad\\ncifically designed to evaluate the capacity of the Patrimony \\nplatform to support two direct applications, i.e. identifying \\ntherapeutic targets and generating hypotheses for drug \\nrepurposing.\\nAs a first run, we mapped multi-omics profiling data from \\nthe PRECISESADS cohort of patients with various autoim\\xad\\nmune diseases into the knowledge graph to support drug \\ndevelopment against primary Sjögren’s syndrome [73]. \\nA particular challenge in this indication was to rationally \\ndesign immunotherapeutic approaches acting at a systemic \\nlevel and/or target organs (i.e. salivary and lachrymal glands). \\nAs concrete outputs, Patrimony helped to identify and prior\\xad\\nitize several innovative therapeutic targets, supported by \\na robust and multidimensional set of evidence. Most of the \\ntargets identified as of interest in primary Sjogren Syndrome \\nwere confirmed to be valid as well in several autoimmune \\ndiseases sharing common pathophysiological mechanisms, \\ne.g. Systemic Lupus Erythematosus. Furthermore, Patrimony \\nwas very useful not only to identify new therapeutic targets \\nbut also to validate other ones for which our company had \\nalready initiated the development of drug candidates. \\nSpecifically, two monoclonal antibodies at an early clinical \\ndevelopment stage for autoimmune diseases, including anti- \\ntype 1 interferon and anti-IL7R antibodies (ClinicalTrial.gov \\nNCT04605978) were confirmed as valid therapeutic options \\nin Sjogren Syndrome and Systemic Lupus Erythematosus. \\nLastly, the availability of disease models providing emphasis \\non specific therapeutic targets as being relevant in various \\nautoimmune diseases has been a powerful tool to support \\nServier’s assessment of external licensing opportunities for \\ndrug candidates.\\nAs an effort to contribute to the global fight against the \\nCOVID-19 pandemics, a second application of the Patrimony \\nplatform aimed at identifying existing drugs that could be \\nrepurposed to treat those patients infected by the SARS-CoV \\n-2 virus who develop severe forms of the disease requiring \\nhospitalization. Specifically, we modeled the severe lung \\ninflammation associated with the life-threatening acute \\nrespiratory distress syndrome, which affects up to 75% of \\nCOVID-19 patients transferred to intensive care units [74]. \\nFrom data available in the scientific literature documenting \\ndifferences at a molecular level in both immune responses and \\ntissue inflammation between patients with either mild or very \\nsevere forms of the disease, an interactome of proteins pre\\xad\\ndicted to contribute to lung inflammation in severe COVID-19 \\nwas produced. The latter was used to confirm the interest of \\nseveral drugs already used in this indication such as dexa\\xad\\nmethasone, anti-IL6R antibodies or JAK2 inhibitors. It further \\nidentified additional drugs, either available in other indications \\nor in development, as being relevant for repurposing in severe \\nCOVID-19, such as inhibitors of alarmins and their recep\\xad\\ntors [75].\\nBased on these promising pilot applications to support our \\ndrug development in immuno-inflammation, we have now \\ninitiated an industrialization phase to extend its application \\nto oncology and neurology. Whereas the methodology and \\nprocesses behind were broadly applicable in those additional \\nindications, new challenges emerged reflecting the specifici\\xad\\nties in terms of categories of data predominantly used in \\ndistinct therapeutic areas. For instance, a complexity in neu\\xad\\nrology is to get access to data related to biological processes \\noccurring in the brain beyond solely postmortem samples. \\nThis hurdle induces as a consequence an emphasis in this \\nfield in genetics and animal data when compared with other \\ntherapeutic areas. In contrast, disease modeling in oncology \\ncan benefit from an abundance of molecular data from a great \\nvariety of sources, encompassing both constitutional and \\ntumor-specific molecular alterations. In the latter field, docu\\xad\\nmenting gene essentiality through CRISPR-Cas9 deletion \\nexperiments, e.g. as the ones centralized in the Depmap pro\\xad\\nject, are critical to prioritize among a wealth of hypoth\\xad\\neses [54].\\nVarious additional applications could be considered in the \\nfuture, as an extension to further support the identification \\nand optimization of drug candidates, beyond the repurposing \\nof existing drugs. The current models produced by Patrimony \\nleading to therapeutic target identification could be nicely \\nextended by machine-learning approaches to perform multi\\xad\\ntask parallel prediction of drug candidate characteristics. The \\nlatter include training artificial neural networks to select sui\\xad\\ntable therapeutic modalities for engaging a given target, pre\\xad\\ndicting both binding characteristics as well as pharmacological \\nand ADMET properties of virtual compounds, and even creat\\xad\\ning new molecules by using generative adversarial networks \\n[6]. Other applications requiring specific computational meth\\xad\\nods include generating hypotheses on potential combination \\ntherapies to address complex diseases [76]. Whereas the inter\\xad\\nest of the latter approach is well identified in oncology, we \\nhave as well recently documented how it could be \\n820\\nM. GUEDJ ET AL.\\n\\nimplemented in autoimmune diseases, and arguably in many \\nother therapeutic areas [77].\\n5. Challenges\\nImplementing ex nihilo in a pharmaceutical environment at an \\nindustrial scale, an initiative such as Patrimony has been dis\\xad\\nruptive. A major challenge was that it required a high level of \\ntransversality between multidisciplinary teams assembling \\nnumerous expertises encompassing computational hardware, \\ncloud computing, network computing, machine-learning and \\nAI, statistics, bioinformatics, large-scale biology, pharmacol\\xad\\nogy, clinical knowledge, but also legal skills to assess data \\naccessibility and use. Aligning those very diverse human \\nexpertises necessitated continuous training and internal com\\xad\\nmunication to foster acculturation to computational modeling \\nas it applies to drug discovery and development. Another \\nimportant challenge was the consolidation step needed to \\nvalidate \\nthe \\nscientific \\nrationale \\nof \\npredicted \\ntargets. \\nAlgorithms can generate many hypotheses in a short time \\nframe. The extensive literature search by human experts \\nremained time consuming in order to corroborate or refute \\nthe hypotheses generated. In this exercise, we found it critical \\nfor each given application to a disease of interest to evaluate \\nthe outputs with known targets and disease-modifying drugs. \\nTo that end, clinically approved drugs and their respective \\ntargets with established proof of efficacy have been assessed \\non the platform and their relevance verified. Altogether, dur\\xad\\ning the consolidation step, we consistently observed that the \\nrelevance of the hypotheses generated depends on the qual\\xad\\nity, completeness and continuous updates of data sources. \\nThe interpretation of model outputs by human experts was \\nalso difficult, owing to the inherent complexity of the analysis \\nof biomedical data, but also in light of the numerous existing \\ngaps in the thesaurus of human knowledge. The current \\nHuman Interactome is estimated to cover only circa 25% of \\nall possible molecular interactions [78]. As a consequence, \\nmany genes related to a given disease may appear to be \\ndispersed in the interactomes making the identification of \\ncoherent disease modules challenging. In our experience, we \\nfound that protein expression, documented for example by \\nproteomics or flow cytometry, was more translatable to dis\\xad\\nease pathological mechanisms than gene expression solely \\nassessed by transcriptomics.\\n6. Conclusions\\nThe knowledge-graph represented within Patrimony is \\na sophisticated and evolving way to represent a disease within \\na computer system. Disease modeling has recently emerged \\nas a powerful mean to educate the design and development \\nof drug candidates, in the context of Precision Medicine \\napproaches aiming to offer therapies better tailored to the \\npatients’ specificities. Such strategies are thus being estab\\xad\\nlished to capitalize on a comprehensive knowledge of the \\ndisease and of patient population heterogeneity to select \\ntherapeutic targets and drug candidates predicted to be \\nmost suitable in this indication. This trend that we termed \\nComputational Precision Medicine has become \\nhighly \\nstrategic for pharmaceutical industries to differentiate from \\nthe competition [6]. We thus emphasize the tremendous \\nvalue of developing a proprietary computational platform to \\ncreate a major competitive advantage for drug developers in \\ntheir disease areas of interest.\\nWithin Servier, we designed the Patrimony platform by \\ncombining innovative concepts, methodologies and suppor\\xad\\ntive infrastructure to grow significantly our ability to integrate \\nlarge-scale biomedical data. As of today, Patrimony allows to \\ngenerate multidimensional models relating to diseases, ther\\xad\\napeutic targets, and to some level drug candidates. Based on \\nour positive experience in exploring pilot applications in \\nimmuno-inflammatory diseases, the use of this computational \\nplatform is now being deployed throughout all therapeutic \\nareas of interest for the company.\\n7. Expert opinion\\nAs AI has the potential to transform drug discovery and sev\\xad\\neral AI-driven molecules have progressed into clinical trials \\n(most often with accelerated timelines and reduced costs), its \\nimpact and remaining challenges need to be globally \\naddressed [79,80]. As explainabilty is often invoked, we recom\\xad\\nmend paying attention to generating results that are both \\nrobust (e.g. to small changes in parameters and data), inter\\xad\\npretable (e.g. avoiding applying transformation to variables \\nthat would disconnect them to their biological or clinical \\nmeaning), and reproducible (e.g. by applying good coding \\nand data practices). Another important hurdle to implement \\ndisease modeling in support of drug discovery is the difficulty \\nto distinguish causal from incidental genes or proteins in the \\npathophysiology. In future analyses, computational inferences \\nof causality based for example on Bayesian networks represent \\nan interesting option to shed light on disease-related master \\nregulators or driver mutations [81,82]. The converging \\nadvances in high-throughput technologies such as single-cell \\nRNA sequencing or deep immunophenotyping along with \\nprotein structure prediction with the AlphaFold algorithms \\nwill contribute to expand continuously our knowledge space \\n[83,84]. Integrating this flow of new data implies to regularly \\nupdate the knowledge graph, thus raising concerns in terms \\nof trustworthiness of data sources and quality control. The \\nanalysis of disease-related processes requires complementing \\ntopology with dynamic properties at large scale, which \\nremains a major challenge [85,86]. Also, as computational \\nmodels are accumulating, there is an increasing need for \\nrapid validation and consolidation with data generated on \\npurpose or well-established external reference datasets such \\nas those proposed in the frame of data challenges (e.g. Kaggle, \\nDream, or GeneDisco). Eventually, computational innovative \\ndevelopments should also encompass other therapeutic mod\\xad\\nalities beyond small molecules, such as biologics, antisense \\noligonucleotides, protein degradation targeting and nanopar\\xad\\nticles as well as potential combination therapies.\\nImportantly, the Patrimony platform has become a highly \\ntransforming asset within our company’s R&D. It was seminal \\nto initiate a transition from classical drug development relying \\nupon life sciences, chemical expertise, and biotechnologies to \\nan educated computer-based approach powered by AI. As \\nEXPERT OPINION ON DRUG DISCOVERY\\n821\\n\\nsuch, it still challenges traditional organizational structures for \\ndrug discovery and imposes a close cooperation between \\nmultidisciplinary teams. It progressively fosters an open inno\\xad\\nvation mindset within the company, requiring both agility and \\ntransversality across a broad range of existing and emerging \\nexpertises. In light of this very significant cultural change, \\nacculturation to help acceptance by human experts of com\\xad\\nputational approaches to drug R&D is presently one of the \\nmost critical issues to solve in order to capture their full \\nbenefit.\\nAcknowledgments\\nThe authors would like to thank all members of Servier R&D who sup\\xad\\nported the initiation and implementation of Patrimony. In particular \\nM Coste, S Ollivier, V Robert, C Hébert, A Saugeot, A Bril, R Jeggo, \\nF Schmidlin, and C Bertrand. The authors also thank B Theofilopoulou \\nand T Bolba for legal assistance.\\nFunding\\nThis work was funded by Servier.\\nDeclaration of Interest\\nAll authors are employees at Servier, an international pharmaceutical \\ncompany governed by a non-profit foundation. The authors have no \\nother relevant affiliations or financial involvement with any organization \\nor entity with a financial interest in or financial conflict with the subject \\nmatter or materials discussed in the manuscript apart from those \\ndisclosed.\\nAuthor contributions\\nConceptualization and supervision: M Guedj, S Hubert, J Laplume, \\nI Dupin-Roger, and P Moingeon\\nImplementation: M Guedj, J Swindle, A Hamon, J Laplume, L Xuereb, \\nC Lefebvre, Y Haudry, and C Gabarroca\\nData access, curation, and integration: J Swindle, A Hamon, and \\nJ Laplume\\nDesign of applications: S Hubert, E Desvaux, A Aussy, L Laigle, I Dupin- \\nRoger, and P Moingeon\\nWriting of the manuscript: M Guedj and P Moingeon\\nReview and editing: all the authors\\nURLs\\nPublic sources\\nAlphaFold https://alphafold.ebi.ac.uk\\nBindingDB https://www.bindingdb.org\\nBlueprint https://www.blueprint-epigenome.eu\\nChEBI https://www.ebi.ac.uk/chebi\\nClinicalTrials.gov https://clinicaltrials.gov\\nCMAP https://clue.io/cmap\\nDepmap https://depmap.org\\nDisGenet https://www.disgenet.org\\nDrugBank https://www.drugbank.ca\\nEBI https://www.ebi.ac.uk\\nEFO http://www.ebi.ac.uk/efo\\nEnsembl https://www.ensembl.org\\nGEO https://www.ncbi.nlm.nih.gov/geo\\nGO http://geneontology.org\\nGTeX https://gtexportal.org\\nGWAS Catalog https://www.ebi.ac.uk/gwas\\nIMI https://www.imi.europa.eu\\nMedDRA https://www.meddra.org\\nMSigDB https://www.gsea-msigdb.org\\nNCBI https://www.ncbi.nlm.nih.gov\\nNCBI Genes (ex Entrez) https://www.ncbi.nlm.nih.gov/gene\\nOmicSoft DiseaseLand https://digitalinsights.qiagen.com\\nOpen Targets https://www.opentargets.org\\nPharmGKB https://www.pharmgkb.org\\nReactome https://reactome.org\\nSIDER http://sideeffects.embl.de\\nSTRING https://string-db.org\\nTTD http://db.idrblab.net/ttd\\nUK Biobank https://www.ukbiobank.ac.uk\\nUniProt https://www.uniprot.org\\nImplementation\\nBigQuery https://cloud.google.com/bigquery\\nCreative Common https://creativecommons.org\\nCytoscape https://cytoscape.org\\nFAIR https://www.go-fair.org/fair-principles\\nGCP https://cloud.google.com\\nGraph-tool https://graph-tool.skewed.de\\nIgraph https://igraph.org\\nMicrosoft Azure https://azure.microsoft.com\\nMongoDB https://www.mongodb.com\\nNeo4j https://neo4j.com\\nPython https://www.python.org\\nR https://cran.r-project.org\\nData challenges\\nDream http://dreamchallenges.org\\nGeneDisco https://www.gsk.ai/genedisco-challenge\\nKaggle https://www.kaggle.com\\nORCID\\nMickaël Guedj \\nhttp://orcid.org/0000-0002-9742-957X\\nPhilippe Moingeon \\nhttp://orcid.org/0000-0002-2380-9983\\nReferences\\nPapers of special note have been highlighted as either of interest (•) or of \\nconsiderable interest (••) to readers.\\n1. Ringel MS, Scannell JW, Baedeker M, et al. Breaking Eroom’s law. \\nNat Rev Drug Discov. 2020;19(12):833–834.\\n2. Waring MJ, Arrowsmith J, Leach AR, et al. An analysis of the \\nattrition of drug candidates from four major pharmaceutical \\ncompanies. Nat Rev Drug Discov. 2015;14(7):475–486.\\n3. Paul SM, Mytelka DS, Dunwiddie CT, et al., How to improve R&D \\nproductivity: the pharmaceutical industry’s grand challenge. Nat \\nRev Drug Discov. 2010;9(3):203–214. \\n•• A key paper describing the challenges of pharma R&D.\\n4. Morgan P, Brown DG, Lennard S, et al. Impact of a five-dimensional \\nframework on R&D productivity at AstraZeneca. Nat Rev Drug \\nDiscov. 2018;17(3):167–181.\\n5. Pammolli F, Righetto L, Abrignani S, et al. The endless frontier? The \\nrecent increase of R&D productivity in pharmaceuticals. J Transl \\nMed. 2020;18(1):162.\\n6. Moingeon \\nP, \\nKuenemann \\nM, \\nGuedj \\nM. \\nArtificial \\nintelligence-enhanced drug design and development: toward \\na computational precision medicine. Drug Discov Today. 2022;27 \\n(1):215–222. \\n•• An introduction to the concept Computational Precision \\nMedicine.\\n7. Rajpurkar P, Chen E, Banerjee O, et al. AI in health and medicine. \\nNat Med. 2022;28(1):31–38. \\n822\\nM. GUEDJ ET AL.\\n\\n•• A key introduction to AI in Medicine.\\n8. Chen H, Engkvist O, Wang Y, et al. The rise of deep learning in drug \\ndiscovery. Drug Discov Today. 2018;23(6):1241–1250.\\n9. Ekins S, Puhl AC, Zorn KM, et al. Exploiting machine learning for \\nend-to-end drug discovery and development. Nat Mater. 2019;18 \\n(5):435–441.\\n10. Greener JG, Kandathil SM, Moffat L, et al. A guide to machine \\nlearning for biologists. Nat Rev Mol Cell Biol. 2022;23(1):40–55. \\n•• A didactic introduction to machine learning applied to biome\\xad\\ndical data.\\n11. Topol EJ. High-performance medicine: the convergence of human \\nand artificial intelligence. Nat Med. 2019;25(1):44–56.\\n12. Vamathevan J, Clark D, Czodrowski P, et al. Applications of machine \\nlearning in drug discovery and development. Nat Rev Drug Discov. \\n2019;18(6):463–477.\\n13. Apweiler R, Beissbarth T, Berthold MR, et al., Whither systems \\nmedicine? Exp Mol Med. 2018;50(3):e453–e453. \\n• An introduction to the concept of Systems Medicine.\\n14. Yu MK, Ma J, Fisher J, et al. Visible machine learning for \\nbiomedicine. Cell. 2018;173(7):1562–1565.\\n15. Savage N. Tapping into the drug discovery potential of AI. \\nBiopharm Deal. 2021:d43747-021-00045–7\\n16. Koscielny G, An P, Carvalho-Silva D, et al. Open targets: a platform \\nfor therapeutic target identification and validation. Nucleic Acids \\nRes. 2017;45(D1):D985–D994.\\n17. Carvalho-Silva D, Pierleoni A, Pignatelli M, et al. Open targets plat\\xad\\nform: new developments and updates two years on. Nucleic Acids \\nRes. 2019;47(D1):D1056–D1065.\\n18. Wishart DS, Knox C, Guo AC, et al. DrugBank: a knowledgebase for \\ndrugs, drug actions and drug targets. Nucleic Acids Res. 2008;36 \\n(suppl_1):D901–D906.\\n19. Apweiler R. UniProt: the universal protein knowledgebase. Nucleic \\nAcids Res. 2004;32(90001):115D–119 .\\n20. The UniProt Consortium, Bateman A, Martin M-J, Orchard S, et al. \\nUniProt: the universal protein knowledgebase in 2021. Nucleic \\nAcids Res. 2021;49(D1):D480–D489.\\n21. Wishart DS, Feunang YD, Guo AC, et al. DrugBank 5.0: a major \\nupdate to the DrugBank database for 2018. Nucleic Acids Res. \\n2018;46(D1):D1074–D1082.\\n22. Liu T, Lin Y, Wen X, et al. BindingDB: a web-accessible database of \\nexperimentally \\ndetermined \\nprotein-ligand \\nbinding \\naffinities. \\nNucleic Acids Res. 2007;35(Database):D198–D201.\\n23. Lamb J. The connectivity map: a new tool for biomedical research. \\nNat Rev Cancer. 2007;7(1):54–60.\\n24. Subramanian A, Narayan R, Corsello SM, et al. A next generation \\nconnectivity map: L1000 platform and the first 1,000,000 profiles. \\nCell. 2017;171(6):1437–1452.e17.\\n25. Pinero J, Queralt-Rosinach N, Bravo A, et al. DisGeNET: a discovery \\nplatform for the dynamical exploration of human diseases and \\ntheir genes. Database. 2015;2015:bav028–bav028.\\n26. Piñero J, Ramírez-Anguita JM, Saüch-Pitarch J, et al. The DisGeNET \\nknowledge platform for disease genomics: 2019 update. Nucleic \\nAcids Res. 2019;gkz1021. 10.1093/nar/gkz1021\\n27. Hubbard T. The Ensembl genome database project. Nucleic Acids \\nRes. 2002;30(1):38–41.\\n28. Howe KL, Achuthan P, Allen J, et al. Ensembl 2021. Nucleic Acids \\nRes. 2021;49(D1):D884–D891.\\n29. Malone J, Holloway E, Adamusiak T, et al. Modeling sample vari\\xad\\nables with an experimental factor ontology. Bioinformatics. 2010;26 \\n(8):1112–1118.\\n30. Ontology Consortium G, Gene Ontology Consortium. The gene \\nontology (GO) database and informatics resource. Nucleic Acids \\nRes. 2004;32(90001):258D–261 .\\n31. The Gene Ontology Consortium. The gene ontology resource: 20 \\nyears and still GOing strong. Nucleic Acids Res. 2019;47(D1):D330– \\nD338.\\n32. Lonsdale J, Thomas J, Salvatore M, et al. The genotype-tissue \\nexpression (GTEx) project. Nat Genet. 2013;45(6):580–585.\\n33. The GTEx Consortium, Ardlie KG, Deluca DS, Segrè AV, et al. The \\ngenotype-tissue expression (GTEx) pilot analysis: multitissue gene \\nregulation in humans. Science. 2015;348(6235):648–660.\\n34. Welter D, MacArthur J, Morales J, et al. The NHGRI GWAS Catalog, \\na curated resource of SNP-trait associations. Nucleic Acids Res. \\n2014;42(D1):D1001–D1006.\\n35. MacArthur J, Bowler E, Cerezo M, et al. The new NHGRI-EBI Catalog \\nof published genome-wide association studies (GWAS Catalog). \\nNucleic Acids Res. 2017;45(D1):D896–D901.\\n36. Brown EG, Wood L, Wood S. The medical dictionary for regulatory \\nactivities (MedDRA). Drug Saf. 1999;20(2):109–117.\\n37. Liberzon A, Subramanian A, Pinchback R, et al. Molecular signa\\xad\\ntures \\ndatabase \\n(MSigDB) \\n3.0. \\nBioinformatics. \\n2011;27(12): \\n1739–1740.\\n38. Maglott D. Entrez gene: gene-centered information at NCBI. Nucleic \\nAcids Res. 2004;33:D54–D58.\\n39. Maglott D, Ostell J, Pruitt KD, et al. Entrez gene: gene-centered \\ninformation at NCBI. Nucleic Acids Res. 2011;39(Database):D52– \\nD57.\\n40. Hewett M. PharmGKB: the pharmacogenetics knowledge base. \\nNucleic Acids Res. 2002;30(1):163–165.\\n41. Cheng F, Desai RJ, Handy DE, et al. Network-based approach to \\nprediction and population-based validation of in silico drug \\nrepurposing. Nat Commun. 2018;9(1):2691.\\n42. Joshi-Tope G. Reactome: a knowledgebase of biological pathways. \\nNucleic Acids Res. 2004;33:D428–D432.\\n43. Fabregat A, Jupe S, Matthews L, et al. The reactome pathway \\nknowledgebase. Nucleic Acids Res. 2018;46(D1):D649–D655.\\n44. Kuhn M, Letunic I, Jensen LJ, et al. The SIDER database of drugs and \\nside effects. Nucleic Acids Res. 2016;44(D1):D1075–D1079.\\n45. Snel B. STRING: a web-server to retrieve and display the repeatedly \\noccurring neighbourhood of a gene. Nucleic Acids Res. 2000;28 \\n(18):3442–3444.\\n46. Szklarczyk D, Gable AL, Nastou KC, et al. The STRING database in \\n2021: customizable protein–protein networks, and functional char\\xad\\nacterization of user-uploaded gene/measurement sets. Nucleic \\nAcids Res. 2021;49(D1):D605–D612.\\n47. Chen X. TTD: therapeutic target database. Nucleic Acids Res. \\n2002;30(1):412–415.\\n48. Degtyarenko K, de Matos P, Ennis M, et al. ChEBI: a database and \\nontology for chemical entities of biological interest. Nucleic Acids \\nRes. 2007;36(Database):D344–D350.\\n49. Ross JS, Mulvey GK, Hines EM, et al. Trial publication after registra\\xad\\ntion in clinicalTrials.gov: a cross-sectional analysis. PLoS Med. \\n2009;6(9):e1000144.\\n50. Zarin DA, Tse T, Williams RJ, et al. Trial reporting in ClinicalTrials. \\ngov — the final rule. N Engl J Med. 2016;375(20):1998–2004.\\n51. Wang Y, Zhang S, Li F, et al. Therapeutic target database 2020: \\nenriched resource for facilitating research and early development \\nof targeted therapeutics. Nucleic Acids Res. 2019;gkz981. 10.1093/ \\nnar/gkz981.\\n52. Menche J, Sharma A, Kitsak M, et al. Uncovering disease-disease \\nrelationships through the incomplete interactome. Science. \\n2015;347(6224):1257601.\\n53. Adams D, Altucci L, Antonarakis SE, et al. BLUEPRINT to decode the \\nepigenetic signature written in blood. Nat Biotechnol. 2012;30 \\n(3):224–226.\\n54. Tsherniak A, Vazquez F, Montgomery PG, et al. Defining a cancer \\ndependency map. Cell. 2017;170(3):564–576.e16.\\n55. Sudlow C, Gallacher J, Allen N, et al. UK biobank: an open access \\nresource for identifying the causes of a wide range of complex \\ndiseases of middle and old age. PLOS Med. 2015;12(3):e1001779.\\n56. Bycroft C, Freeman C, Petkova D, et al. The UK biobank resource \\nwith deep phenotyping and genomic data. Nature. 2018;562 \\n(7726):203–209.\\n57. Edgar R. Gene expression omnibus: NCBI gene expression and \\nhybridization array data repository. Nucleic Acids Res. 2002;30 \\n(1):207–210.\\nEXPERT OPINION ON DRUG DISCOVERY\\n823\\n\\n58. Barabási A-L, Gulbahce N, Loscalzo J. Network medicine: a \\nnetwork-based approach to human disease. Nat Rev Genet. \\n2011;12(1):56–68. \\n•• A key introduction to networks in Medicine.\\n59. Lee LY-H, Loscalzo J. Network medicine in pathobiology. Am \\nJ Pathol. 2019;189(7):1311–1326. \\n•• A key introduction to networks in Medicine.\\n60. Gaudelet T, Day B, Jamasb AR, et al. Utilizing graph machine \\nlearning within drug discovery and development. Brief Bioinform. \\n2021;22(6):bbab159. \\n•• A comprehensive review of the application of graph machine- \\nlearning in drug discovery and development.\\n61. Cowen L, Ideker T, Raphael BJ, et al. Network propagation: a universal \\namplifier of genetic associations. Nat Rev Genet. 2017;18(9):551–562.\\n62. Sun M, Zhao S, Gilvary C, et al. Graph convolutional networks for \\ncomputational drug development and discovery. Brief Bioinform. \\n2021;21(3):919–935.\\n63. Jensen LJ, Saric J, Bork P. Literature mining for the biologist: from \\ninformation retrieval to biological discovery. Nat Rev Genet. 2006;7 \\n(2):119–129.\\n64. Vergetis V, Liaropoulos G, and Georganaki M, et al. A machine \\nlearning approach for assessing drug development risk. bioRxiv. \\n2020.\\n65. Hughes J, Rees S, Kalindjian S, et al. Principles of early drug dis\\xad\\ncovery: principles of early drug discovery. BrJ Pharmacol. 2011;162 \\n(6):1239–1249.\\n66. Ashburn TT, Thor KB. Drug repositioning: identifying and developing \\nnew uses for existing drugs. Nat Rev Drug Discov. 2004;3(8):673–683. \\n• A key introduction to the concept of drug repurposing.\\n67. Nelson MR, Tipney H, Painter JL, et al. The support of human \\ngenetic evidence for approved drug indications. Nat Genet. \\n2015;47(8):856–860.\\n68. Pushpakom S, Iorio F, Eyers PA, et al., Drug repurposing: progress, \\nchallenges and recommendations. Nat Rev Drug Discov. 2019;18 \\n(1):41–58. \\n• A review of systematic drug repurposing approaches.\\n69. Fiscon G, Conte F, Farina L, et al. SAveRUNNER: a network-based \\nalgorithm for drug repurposing and its application to COVID-19. \\nPLOS Comput Biol. 2021;17(2):e1008686.\\n70. Zeng X, Zhu S, Liu X, et al. deepDR: a network-based deep learning \\napproach to in silico drug repositioning. Bioinformatics. 2019;35 \\n(24):5191–5198.\\n71. Steinberg D, Horwitz G, Zohar D. Building a business model in \\ndigital medicine. Nat Biotechnol. 2015;33(9):910–920.\\n72. Shannon P, Markiel A, Ozier O, et al. Cytoscape: a software envir\\xad\\nonment for integrated models of biomolecular interaction \\nnetworks. Genome Res. 2003;13(11):2498–2504.\\n73. Soret P, Le Dantec C, Desvaux E, et al. A new molecular classifica\\xad\\ntion to drive precision treatment strategies in primary Sjögren’s \\nsyndrome. Nat Commun. 2021;12(1):3523.\\n74. Tzotzos SJ, Fischer B, Fischer H, et al. Incidence of ARDS and out\\xad\\ncomes in hospitalized patients with COVID-19: a global literature \\nsurvey. Crit Care. 2020;24(1):516.\\n75. Desvaux E, Hamon A, and Hubert S, et al. Network-based \\nrepurposing identifies anti-alarmins as drug candidates to con\\xad\\ntrol severe lung inflammation in COVID-19. PLOS ONE. 2021;16 \\n(7):e0254374. \\n• Application of the Patrimony platform to COVID-19.\\n76. Cheng F, Kovács IA, Barabási A-L. Network-based prediction of drug \\ncombinations. Nat Commun. 2019;10(1):1197.\\n77. Desvaux E, Aussy A, and Hubert S, et al. Model-based computa\\xad\\ntional precision medicine to develop combination therapies for \\nautoimmune diseases. Expert Rev Clin Immunol. 2021;18 \\n(1):47–56.\\n78. Rolland T, Taşan M, Charloteaux B, et al. A proteome-scale map of \\nthe human interactome network. Cell. 2014;159(5):1212–1226.\\n79. Jayatunga MKP, Xie W, Ruder L, et al. AI in small-molecule drug \\ndiscovery: a coming wave? Nat Rev Drug Discov. 2022;21 \\n(3):175–176.\\n80. Aittokallio T. What are the current challenges for machine learning \\nin drug discovery and repurposing? Expert Opin Drug Discov. \\n2022;17(5):423–425. \\n•• Introducing some current challenges for AI/MAL in drug \\ndiscovery.\\n81. Needham CJ, Bradford JR, Bulpitt AJ, et al. Inference in Bayesian \\nnetworks. Nat Biotechnol. 2006;24(1):51–53.\\n82. Luo Y, Peng J, and Ma J. When causal inference meets deep \\nlearning. Nat Mach Intell. 2020;2(8):426–427.\\n83. Heath JR, Ribas A, Mischel PS. Single-cell analysis tools for drug dis\\xad\\ncovery and development. Nat Rev Drug Discov. 2016;15(3):204–216.\\n84. Jumper J, Evans R, Pritzel A, et al. Highly accurate protein structure \\nprediction with AlphaFold. Nature. 2021;596(7873):583–589.\\n85. Liang Y, Kelemen A. Dynamic modeling and network approaches \\nfor omics time course data: overview of computational approaches \\nand applications. Brief Bioinform. 2018;19(5):1051–1068.\\n86. Selevsek N, Caiment F, Nudischer R, et al. Network integration and \\nmodelling of dynamic drug responses at multi-omics levels. \\nCommun Biol. 2020;3(1):573.\\n824\\nM. GUEDJ ET AL.\\n',\n",
       " \"Opinion\\nAI-powered therapeutic target discovery\\nFrank W. Pun,1 Ivan V. Ozerov,1 and Alex Zhavoronkov\\n1,2,3,*\\nDisease modeling and target identiﬁcation are the most crucial initial steps in\\ndrug discovery, and inﬂuence the probability of success at every step of drug\\ndevelopment. Traditional target identiﬁcation is a time-consuming process\\nthat takes years to decades and usually starts in an academic setting.\\nGiven its advantages of analyzing large datasets and intricate biological net-\\nworks, artiﬁcial intelligence (AI) is playing a growing role in modern drug tar-\\nget identiﬁcation. We review recent advances in target discovery, focusing on\\nbreakthroughs in AI-driven therapeutic target exploration. We also discuss\\nthe importance of striking a balance between novelty and conﬁdence in\\ntarget selection. An increasing number of AI-identiﬁed targets are being vali-\\ndated through experiments and several AI-derived drugs are entering clinical\\ntrials; we highlight current limitations and potential pathways for moving\\nforward.\\nOverview of target identiﬁcation\\nThe drug discovery pipeline is widely recognized to be a time-consuming, expensive, and\\nrisk-laden process that typically requires around 10 years and $2 billion to bring a novel\\ndrug to market [1]. By 2022 fewer than 500 successful drug targets had been identiﬁed\\n[2], representing a tiny fraction of the estimated druggable targets in humans [3,4]. Al-\\nthough numerous drug candidates undergo extensive optimization during preclinical\\nstages, the average failure rate in clinical trials from 2009 to 2018 reached 84.6%i. The\\nlack of clinical efﬁcacy remains the key factor contributing to the failure of both Phase 2\\nand 3 trials [5], leading to substantial ﬁnancial losses and resource wastage. Identifying\\nthe right drug targets is crucial for increasing the likelihood of developing clinically effective\\ntherapies.\\nTarget identiﬁcation, the process of identifying the right biological molecules or cellular path-\\nways that can be modulated by drugs to achieve therapeutic beneﬁts, is increasingly important\\nin modern drug discovery. Although innovations in experimental and omic technologies have\\nbeen growing over the past few decades (Figure 1), identifying actionable therapeutic targets\\nremains challenging. The integration of multiomic data with AI (see Glossary) algorithms has re-\\ncently emerged as a promising approach for target identiﬁcationii,iii. We discuss here the con-\\nventional target identiﬁcation approaches with a focus on the application of AI algorithms to\\ntarget identiﬁcation. This paper aims to offer a progressive outlook on the emergence of the\\nAI-driven drug discovery era and encourage the integration of AI technologies into drug discov-\\nery pipelines.\\nStrategies in target identiﬁcation: from experiments to machine learning\\nTarget identiﬁcation can be classiﬁed into three distinct strategies – experimental, multiomic, and\\ncomputational approaches (Figure 2). Using these methods collaboratively can generate novel\\ntherapeutic hypotheses in exploratory target identiﬁcation, thus signiﬁcantly enhancing our under-\\nstanding of complex diseases.\\nHighlights\\nDisease modeling and target discovery\\nare crucial initial steps in the drug discov-\\nery process and signiﬁcantly impact on\\nthe success of drug development.\\nGiven the advantages of analyzing large\\ndatasets and complex biological net-\\nworks, artiﬁcial intelligence (AI) is playing\\na growing role in modern drug target\\nidentiﬁcation.\\nWe discuss the use of deep learning\\nmodels\\nfor\\ntarget\\ndiscovery,\\nAI-\\nidentiﬁed targets validated through ex-\\nperiments, and the use of synthetic\\ndata produced using generative AI for\\ntarget identiﬁcation.\\nNovelty, in addition to druggability and\\ntoxicity, is a crucial factor in target selec-\\ntion. There is a trade-off between choos-\\ning high-conﬁdence and novel targets.\\nOver the past few years several AI-\\nderived drugs have entered clinical trials,\\nsignaling the dawn of a new era in AI-\\ndriven drug discovery.\\n1Insilico Medicine Hong Kong Ltd., Hong\\nKong Science and Technology Park,\\nNew Territories, Hong Kong\\n2Insilico Medicine MENA, 6F IRENA\\nBuilding, Abu Dhabi, United Arab Emirates\\n3Buck Institute for Research on Aging,\\nNovato, CA, USA\\n*Correspondence:\\nalex@insilico.com (A. Zhavoronkov).\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\nhttps://doi.org/10.1016/j.tips.2023.06.010\\n561\\n© 2023 The Author(s). Published by Elsevier Ltd. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).\\nTrends in\\nPharmacological Sciences\\nOPEN ACCESS\\n\\nExperimental approaches\\nExperimental approaches, including afﬁnity-based biochemical, comparative proﬁling, and\\nchemical/genetic screening, have demonstrated their striking contributions to target identiﬁcation\\nsince the 1960s. The use of small-molecule afﬁnity probes, which allow traceless protein labeling\\nupon ligand–protein interaction [6], is the most straightforward method among the three experi-\\nmental approaches. The selection of probes is highly dependent on the identity of the starting mol-\\necule [7]. Stable isotope labeling by amino acids in cell culture (SILAC), an example of comparative\\nproﬁling, is a popular quantitative proteomics tool that uses stable isotope-labeled amino acids to\\naccurately differentiate cellular proteomes [8]. Studies conducted in multiple cancer types such as\\nhepatocellular carcinoma (HCC) [9], multiple myeloma [10,11], endometrial cancer [12], and colo-\\nrectal cancer [13,14] have clearly exempliﬁed the effectiveness of SILAC in identifying pivotal\\nplayers in disease pathogenesis. Chemical/genetic screening, implemented by RNA interference\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 1. The emergence of artiﬁcial intelligence (AI) in early drug development. (Upper panel) Key technological\\nadvances in the history of target identiﬁcation are classiﬁed into three types: experiment-based (red), multiomic (blue), and\\ncomputational (green) approaches. Traditionally, experiment-based methods have been the go-to approach for discovering\\ntherapeutic targets. However, with the rise of big data, integrated analysis of multiomic data has become a more efﬁcient\\nstrategy for target identiﬁcation. In addition, recent advances in AI-driven biological analysis have identiﬁed novel targets and\\nAI-designed drugs are now entering clinical trials. (Lower panel) AI applications in the early stages of drug discovery.\\nAbbreviations: AGC chemistry, afﬁnity-guided catalyst chemistry; ALS, amyotrophic lateral sclerosis; DL, deep learning;\\nEGFR, epidermal growth factor receptor; GAN, generative adversarial network; GWAS, genome-wide association study; LD\\nchemistry, ligand-directed chemistry; MTOR, mammalian target of rapamycin; NSCLC, non-small cell lung cancer; SILAC,\\nstable isotope labeling with amino acids in cell culture; TID, target identiﬁcation. Figure created with BioRender.com.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n562\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\nGlossary\\nArtificial intelligence (AI): the ability of\\na computer or computer-controlled\\nmachine to perform problem-solving\\nand decision-making tasks that are\\ncommonly associated with intelligent\\nbeings.\\nBiomarker: a biological molecule in any\\ntype of body ﬂuid or tissue that serves as\\na sign of a biological state.\\nDrug repurposing: the process of\\nidentifying a novel therapeutic\\napplication for existing drugs that have\\nbeen FDA-approved or clinically\\ninvestigated for speciﬁc medical\\nindications.\\nDrug–target interaction: an important\\nstep in drug discovery that recognizes\\nhow a chemical compound and a\\nprotein target interact in the human\\nbody.\\nGenerative adversarial networks\\n(GANs): a class of machine learning\\nframeworks that consists of two neural\\nnetworks that compete against each\\nother during the training process and\\nimprove their functionalities to generate\\nsamples indistinguishable from the real\\ndata.\\nGenome-wide association study\\n(GWAS): a method to identify genomic\\nvariants that are statistically associated\\nwith a risk for a disease or a trait by\\ncomparing the frequencies of genomic\\nvariants between people with and\\nwithout that speciﬁc disease or trait.\\nIndication prioritization: the process\\nof prioritizing the potential indications of\\na drug based on the expected relevancy\\nof the drug and a speciﬁc indication\\nusing AI.\\nInduced pluripotent stem cells\\n(iPSCs): artiﬁcial stem cells generated\\nfrom an adult somatic cell through the\\ncoexpression of speciﬁc pluripotency-\\nassociated genes, namely c-Myc, Oct3/\\n4, Sox2, and Klf4.\\nMachine learning: a branch of artiﬁcial\\nintelligence that focuses on mimicking\\nhuman learning processes via the use of\\ndata and algorithms to gradually improve\\nits accuracy.\\nNatural language processing: a ﬁeld\\nof AI that processes and analyzes large\\namounts of natural language data with a\\ngoal to enable computers to understand,\\ninterpret, generate human language, and\\nextract information from documents.\\nPharmacokinetics: the study of the\\nfate of an administered substance in an\\norganism, namely absorption,\\ndistribution, metabolism, and excretion.\\n\\n(RNAi) or CRISPR-Cas9 gene editing, has been of great interest to biologists for decades. Owing to\\nits high speciﬁcity and efﬁciency [15], CRISPR has dramatically expanded our knowledge of the\\nmechanistic and pharmacological aspects of human diseases. For example, BRD2 was identiﬁed\\nas an essential regulator of the host response to SARS-CoV-2 infection by a targeted CRISPR in-\\nterference screen [16]. Making use of the CRISPR interference- and CRISPR activation-based\\nfunctional genomics platform, Ramkumar et al. identiﬁed the determining roles of HDAC7 and\\nthe Sec61 complex in modulating the immunotherapy response in multiple myeloma [17]. Although\\nit has been 10 years since its introduction, CRISPR technology continues to evolve to further en-\\nhance its ﬂexibility, simplicity, and efﬁciency, thus offering a great beneﬁt to the research community\\nnot only for target identiﬁcation but also as a gene therapy and diagnostic tool.\\nMultiomic approaches\\nMultiomic data provide researchers with interconnected molecular information from different per-\\nspectives, including static genomic data and spatiotemporally dynamic expression and metabolic\\nTarget identification\\nT\\nMultiomics\\nPromoter Target gene\\nMethyl \\ngroup\\nExon Intron Exon \\nGenomics\\nEpigenomics\\nProteomics\\nMetabolomics\\nTranscriptomics\\nComputational\\nStructure similarity\\nAI Machine learning\\nReverse docking\\nPharmacophore screening\\nHO\\nN\\nO\\nN\\nQuery\\nligand\\ndatabase\\nProtein\\nDocking\\nOH\\nN\\nO\\nH\\nO\\nHO\\nOH\\nH\\nOC\\nO\\nHO\\nH\\nO\\nOH\\nOH\\nHO\\nHO\\nO\\nH\\nH\\nH\\nH\\nH\\nO\\nO\\nOH\\nO\\nO\\nH\\nH\\nH\\nO\\nO\\nOH\\nOH\\nO\\nO\\nOH\\nOH\\nA\\nB\\nQuery ligand\\nLigand\\nSimilarity\\n#1\\n0.99\\n#2\\n0.87\\nPharmacophore \\nmodels\\nLigands\\nLigand-\\npharmacophore \\nmapping\\nExperimental\\nChemical-genetic\\nm/z\\nRelative \\nintensity\\n‘Heavy’ group\\n‘Light’ group\\nMix cell \\nlysates\\nLC-MS/\\nMS\\nStable-isotope labeling by \\namino acids in cell culture \\nCRISPR/Cas9\\nsiRNA\\nRISC\\nRNAi\\nLD Chemistry\\nAGC Chemistry\\nO\\nS\\nO\\nO\\nO\\nProbe\\nLDT reagent\\nO\\nN\\nN\\nLg\\nAGD \\ncatalyst\\nProbe\\nStructure-based\\nDeep learning\\nATP\\nA\\nHeat\\nCO2 Water\\nChemical \\nenergy\\nMetabolism\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 2. Three exploratory strategies for target identiﬁcation. Exploratory techniques for target identiﬁcation can be\\nclassiﬁed into three strategies: experimental, multiomic, and computational approaches. The experimental approach involves\\nconducting wet-lab experiments to identify targets based on afﬁnity, genetic modiﬁcation screening, and comparative\\nproﬁling. The multiomic approach predicts gene–disease associations by analyzing various omic datasets such as\\ngenomics, transcriptomics, proteomics, epigenomics, and metabolomics. Lastly, the computational discovery approach\\nefﬁciently identiﬁes potential targets by using machine learning or structure-based methods including reverse docking,\\npharmacophore screening, and structure similarity analysis. Abbreviations: AGC chemistry, afﬁnity-guided catalyst\\nchemistry; AGD, afﬁnity-guided DMAP (4-dimethylaminopyridine); AI, artiﬁcial intelligence; LC, liquid chromatography; LD\\nchemistry, ligand-directed chemistry; LDT, ligand-directed tosyl; MS, mass spectrometry; RISC, RNA-induced silencing\\ncomplex; RNAi, RNA interference; siRNA, short interfering RNA. Figure created with BioRender.com.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n563\\nRecurrent neural networks: a class\\nof artiﬁcial neural networks with\\nfeedback connections that are designed\\nto learn sequential or time-varying data.\\nTransfer learning: a machine learning\\nmethod where a pretrained model is\\nreused as the starting point for a model\\non another related task; this approach is\\ncommonly used as an optimization\\ntechnique to save time and increase\\nperformance.\\nTherapeutic modality: the type of\\ntherapy used to treat a disease or\\nmedical condition, including small-\\nmolecule drugs, protein-based\\ntherapies, advanced therapies (such as\\ncell and gene therapies), and\\nmicroorganism-based therapies.\\n\\nproﬁles [18]. As the ﬁrst established and most mature omics discipline [19], genomics focuses on\\ngenetic variants in the DNA sequence. Large-scale genome-wide association study (GWAS)\\nanalysis powered by next-generation sequencing has yielded hundreds of thousands of associ-\\nations between genetic variants and complex diseases or traits [20], leading to the development\\nof breakthrough therapies such as the cystic ﬁbrosis modulator drugs targeting CFTR mutations\\n[21], and novel drugs for the treatment of inﬂammatory bowel disease targeting the disease-\\nassociated gene IL23A [22]. More recently, meta-analyses of published GWAS data have re-\\nvealed novel genetic loci attributable to different diseases, thus opening up drug repurposing\\nopportunities [23,24]. Although genomic evidence has been one of the indispensable factors in\\ntarget identiﬁcation, distinguishing the causative genetic variants that lead to a given disease re-\\nmains challenging. In this regard, integrating multiple omic lines of evidence can be useful. Tran-\\nscriptomic and proteomic data can be used to identify causal genetic loci that regulate gene and\\nprotein levels and facilitate the discovery of genes and pathways underlying disease pathogenesis\\n[25–27]. Likewise, epigenomic and metabolomic data can also serve as functional evidence for\\nGWAS-identiﬁed variants to support their disease associations and clinical applications [28–30].\\nAs compared to single omic approaches, integrated multiomic analysis can provide a more com-\\nprehensive view of disease mechanisms and is therefore increasingly used to facilitate biomarker\\nand therapeutic target discoveries, treatment response, and patient prognosis predictions [31–34].\\nComputational approaches\\nBecause typical experiment-based target identiﬁcation is laborious and resource-intensive,\\ncomputational approaches have emerged as promising alternatives for achieving efﬁcient tar-\\nget screening. Depending on the availability of protein structure and the chemical structure of\\nthe compound of interest, pharmacophore screening [35], reverse docking [36], and structure\\nsimilarity assessment [37,38] have been used to predict novel biological targets for small mol-\\necules. On the other hand, AI is a growing discipline in computational science for target discov-\\nery. Machine learning is an indispensable component of AI that can be applied either with or\\nwithout supervision. Supervised learning utilizes labeled datasets to train models for data clas-\\nsiﬁcation and reliable outcome prediction. By contrast, unsupervised learning explores the hid-\\nden structure of unlabeled data without human intervention [39]. The application of machine\\nlearning is not limited to predicting biological targets of the existing drugs or compounds,\\nand can also identify novel therapeutic targets for any disease of interest. The details of how\\nmachine learning facilitates target discovery for disease treatment will be elaborated upon in\\nthe following AI sections.\\nAI-driven target identiﬁcation\\nIn recent years we have witnessed an explosion of biomedical data ranging from basic research\\non disease mechanisms to clinical investigation in patients. Although large amounts of information\\nhave been generated, the growth of data also poses challenges for data analysis. This is where\\nthe emerging role of AI comes into play. Given the advantage of AI in processing and tackling\\ncomplex biomedical networks of data, using AI algorithms can reveal patterns and relationships\\nwithin the data that may not be apparent to humans, and may possibly lead to better understand-\\ning and treatment of diseases. AI has made notable contributions that facilitate biomarker and tar-\\nget identiﬁcation [40–42], indication prioritization [43], drug-like molecule design [44,45],\\npharmacokinetics prediction [46], drug–target interaction [47,48], and clinical trial design\\n[49] (Figure 1, lower panel). Although still in the early stages of clinical trials, AI-derived drugs\\nare increasingly emerging in clinical studies (Table 1), as exempliﬁed by GS-0976 for the treatment\\nof non-alcoholic steatohepatitis, EXS-21546 for solid tumors, and INS018_055 for idiopathic pul-\\nmonary ﬁbrosis, which is the ﬁrst-ever AI-derived drug with positive topline results in a Phase 1\\nclinical trial.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n564\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nApplication of deep learning models in target discovery\\nIn recent years machine learning-based algorithms, particularly deep learning methodologies,\\nhave drawn signiﬁcant attention and have achieved excellent results in pharmaceutical areas.\\nDeep learning, also known as deep neural networks, consists of multiple hidden layers of\\nnodes through which data processing and feature extraction are conducted successively in a\\ncascade manner [50]. Compared to traditional machine learning methods, more recent deep\\nlearning-based architectures, such as generative adversarial networks (GANs), recurrent\\nTable 1. AI-derived drugs in clinical trials\\nCompany\\nTarget\\nIndicationa\\nCompound\\nDevelopment status\\nTrial numberb\\nBenevolentAI\\nTrk\\nAtopic dermatitis\\nBEN-2293\\nPhase 2\\nNCT04737304\\nExscientia\\nA2AR\\nSolid tumors\\nEXS-21546\\nPhase 1\\nNCT04727138\\n5-HT1A\\nObsessive compulsive\\ndisorder\\nDSP-1181\\nPhase 1\\nUndisclosedvi\\n5-HT1A/2A\\nAlzheimer's disease\\npsychosis\\nDSP-0038\\nPhase 1\\nUndisclosedvii\\nPKC-θ\\nInﬂammatory diseases\\nEXS4318\\nPhase 1/2\\nUndisclosedviii\\nInsilico Medicine\\nTarget X\\nIdiopathic pulmonary ﬁbrosis\\nINS018_055\\nPhase 2\\nNCT05938920,\\nCTR20230776\\n3CLPro\\nCOVID-19\\nISM3312\\nPhase 1\\nCTR20230768\\nUSP1\\nBRCA-mutant cancer\\nISM3091\\nPhase 1\\nNCT05932862\\nNimbus Therapeutics\\nACC\\nNonalcoholic steatohepatitis\\nNDI-010976/GS-0976\\nPhase 2\\nNCT02856555,\\nNCT03987074,\\nNCT02891408,\\nNCT02876796\\nPharos iBio\\nFLT3\\nAcute myeloid leukemia\\nOvarian cancer\\nTriple-negative breast cancer\\nRadiation sensitizer\\nPHI-101\\nPhase 1\\nNCT04842370\\nNCT04678102\\nRecursion\\nPharmaceuticals\\nCCM2\\nCerebral cavernous\\nmalformation\\nREC-994\\nPhase 2\\nNCT05085561\\nHDAC\\nNeuroﬁbromatosis type 2\\nREC-2282\\nPhase 2/3\\nNCT05130866\\nMEK1/2\\nFamilial adenomatous\\npolyposis\\nREC-4881\\nPhase 2\\nNCT05552755\\nRelay Therapeutics\\nSHP2\\nSolid tumors\\nRLY-1971/RG-6433\\nPhase 1\\nNCT04252339\\nFGFR2\\nFGFR2-driven cancers\\nIntrahepatic\\ncholangiocarcinoma\\nAdvanced solid tumors\\nRLY-4008\\nPhase 1/2\\nNCT04526106\\nPI3Kα\\nSolid tumors\\nRLY-2608\\nPhase 1\\nNCT05216432\\nSchrödinger\\nMALT1\\nNon-Hodgkin's lymphoma\\nSGR-1505\\nPhase 1\\nNCT05544019\\nStructure Therapeutics\\nGLP1R\\nType 2 diabetes\\nObesity\\nGSBR-1290\\nPhase 1\\nNCT05762471\\nAPLNR\\nPulmonary arterial\\nhypertension\\nIdiopathic pulmonary ﬁbrosis\\nANPA-0073\\nPhase 1\\nACTRN12621000644864\\nValo Health\\nS1P1\\nPost-myocardial infarction\\nAcute kidney injury\\nOPL-0301\\nPhase 2\\nNCT05327855\\nROCK1/2\\nDiabetic retinopathy\\nDiabetic complications\\nOPL-0401\\nPhase 2\\nNCT05393284\\naIndications retrieved from the company pipeline.\\nbFor undisclosed trial numbers, press releases are provided as the source of reference.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n565\\n\\nneural networks, and transfer learning techniques, have attracted increasing attention and\\nhave been applied to various aspects of healthcare, such as de novo small-molecule design\\n[51], aging research [44], and pharmacological prediction of drugs based on transcriptional\\ndata of drug-perturbed cell lines [52]. Using publicly available multiomic data and text mining\\n(Figure 3, Key ﬁgure), deep learning has recently been used in studies of fatal disorders with ur-\\ngent and unmet clinical needs. To identify actionable therapeutic targets in amyotrophic lateral\\nsclerosis (ALS), Pun et al. combined a variety of bioinformatic- and deep learning-based models\\nthat were trained using disease-speciﬁc multiomic and text-based data to prioritize druggable\\ngenes, revealing 18 potential targets for ALS treatment [53]. In addition, Fabris et al. established\\na deep learning-based method with a novel modular architecture to identify human genes asso-\\nciated with multiple age-related diseases by learning patterns retrieved from gene or protein fea-\\ntures such as Gene Ontology terms, protein–protein interactions, and biological pathways [54].\\nWest et al. developed a deep learning ensemble trained using the transcriptomic proﬁles of\\n>12 000 embryonic and adult cells [55]. A novel target (COX7A1) for controlling the embryonic–\\nfetal transition was revealed, which could facilitate our understanding of normal development,\\nepimorphic tissue regeneration, and cancer.\\nFurthermore, large language models also aid therapeutic target discovery via rapid biomedical\\ntext mining. Pretrained on a vast amount of text data extracted from millions of publications,\\nlarge language model-based Chat functionalities, such as BioGPT from Microsoft [56] and\\nChatPandaGPT from Insilico Medicineiv, can connect diseases, genes, and biological processes\\nto allow rapid identiﬁcation of the biological mechanisms involved in disease development and\\nprogression, as well as the identiﬁcation of potential drug targets and biomarkers. The ability of\\nKey figure\\nWorkﬂow of artiﬁcial intelligence (AI)-driven target discovery\\nTarget\\nidentification\\nText Data\\nOmic data models\\n●\\n●Metabolomics\\n●\\nTranscriptomics\\n●\\nSignaling pathways\\nMolecular interactions\\n●\\n●Patents\\nDevelopment status\\n●Clinical trial phase\\n●Ligand/compound availability\\nDruggability\\n●\\nTherapeutic modality\\n●Protein localization\\n●\\nAvailability of protein structure\\nDisease causality\\nPublications\\n●\\n●Clinical trials\\n●Financial reports \\n●Protein-protein interaction\\nTarget selection criteria\\nNovelty\\nToxicity\\n●Gene essentiality\\n●Tissue specificity\\n●Cellular process involved\\n●Proteomics\\nGrants\\n●Genomics\\n●Epigenomics\\nDruggable classes \\n●\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 3. AI prioritizes targets for speciﬁc indications by using multi-models that utilize a diverse range of publicly available\\nomic and text data. Omic data encompass genomics, transcriptomics, proteomics, epigenomics, and metabolomics.\\nThese data provide information about altered signaling pathways, molecular interactions, and protein–protein interactions\\nthat can serve as additional inputs for target prioritization. Text-based data are retrieved from funding reports, patents,\\npublications, and clinical trials. During target prioritization, multiple target selection criteria such as protein family class,\\ndevelopment status, druggability, toxicity, and novelty can be applied to reﬁne the list of AI-driven targets to align with\\nspeciﬁc research objectives.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n566\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nthe large language models to understand natural language and interpret complex scientiﬁc con-\\ncepts could make them valuable tools in accelerating disease hypothesis generation. Neverthe-\\nless, large language models, which are typically trained on human-generated text, may not\\nhave the ability to determine the accuracy and appropriateness of the input data. As a result,\\nthey could inadvertently perpetuate human biases and preconceived notions. Moreover, given\\nthat these models rely heavily on published data, they may have limited potential to identify gen-\\nuinely novel targets. Therefore, it is important to acknowledge these limitations and to comple-\\nment their use with other models to ensure the discovery of truly novel and pertinent targets.\\nThe use of AI-generated synthetic data for target identiﬁcation\\n'Synthetic data' refers to artiﬁcially generated data that mimic real-world patterns and charac-\\nteristics. By leveraging AI algorithms, synthetic data can be created to simulate various biolog-\\nical scenarios, thus enabling researchers to explore and analyze a broader range of possibilities\\n[57–59]. This approach can be particularly valuable in therapeutic areas where experimental\\ndata are scarce or difﬁcult to obtain. For example, in rare diseases or conditions where patient\\ndata are limited, AI can generate synthetic data based on existing knowledge and patterns.\\nThese synthetic data can then be used to train AI models and identify potential therapeutic\\ntargets that may have been overlooked [60]. Synthetic data can also be used to validate predic-\\ntions made by AI algorithms, thus providing an additional layer of conﬁdence in the target\\ndiscovery process.\\nFurthermore, AI-generated synthetic data can help to address data imbalance or bias issues. In\\nsome therapeutic areas, particular patient populations may be under-represented in the available\\ndatasets, leading to challenges in target identiﬁcation. AI can generate synthetic data representing\\nthese under-represented populations, allowing more comprehensive and inclusive analysis [61].\\nAlthough AI-generated synthetic data can offer advantages in exploring a broader range of pos-\\nsibilities and addressing data scarcity, it is essential to recognize its limitations. A model cannot\\nsimulate data containing complexities that the model is unaware of, and this limitation should\\nbe fully acknowledged [62]. Simulating under-represented populations, although tempting due\\nto data scarcity, raises ethical concerns because collecting relevant data should be pursued\\nwhenever possible rather than relying solely on synthetic data [63,64]. Moreover, ensuring that\\nthe synthetic data accurately capture the intricate and nuanced aspects of real-world biological\\nsystems presents a signiﬁcant challenge. Therefore, implementing robust validation and quality\\ncontrol measures becomes crucial to establish the reliability and relevance of the generated\\ndata [65].\\nTo responsibly validate and control the quality of synthetic omic data, several options can be con-\\nsidered. First, comparative analyses can be performed to assess the similarity between the syn-\\nthetic data and real-world data. This can involve statistical measures, such as comparing\\ndistributional characteristics, correlation patterns, or feature-level comparisons. In addition,\\nbenchmarking against known ground-truth data, where available, can help to evaluate the accu-\\nracy and performance of the synthetic data [66]. Another approach involves conducting func-\\ntional analyses, such as focusing on the representation of particular cellular types in the\\nsynthetic dataset in the case of single-cell data, to determine whether the synthetic data captures\\nbiological knowledge and exhibits coherent functional relationships [67]. Finally, involving domain\\nexperts and conducting rigorous peer review can provide valuable insights and ensure the appro-\\npriateness and relevance of the synthetic data for target identiﬁcation [59]. These validation and\\nquality control measures, although challenging, can contribute to establishing conﬁdence in the\\nuse of synthetic omic data in research and drug target discovery.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n567\\n\\nTarget selection criteria\\nThe criteria used to select drug targets can greatly impact on the success of drug development\\n(Figure 3). Causality represents a crucial criterion for selecting drug targets. Understanding the\\ncausal mechanisms behind a disease can help researchers to identify driver genes and key path-\\nways that have the greatest potential for effective disease treatment [68]. Apart from experimental\\nmethods, a common computational approach to infer causal relationships between targets and\\ndiseases is network-based analysis, which involves the construction of biological networks that\\ncapture the relationships between different genes, proteins, drugs, and other molecular entities\\n[69]. These networks can be used to identify potential targets that might have a causal involve-\\nment in a disease based on their centrality and connectivity within the network. The growing in-\\nterest in AI and computational biology has led to a need for the development of machine\\nlearning methods that can be utilized for causal inference in biological networks [70]. In this re-\\ngard, the adaptation of classiﬁcation algorithms for causal discovery marks the emergence of\\ncausal inference models in biomedical research [71–73].\\nAnother important consideration is the druggability of a target – the ability of a target to be mod-\\nulated by a drug molecule. Factors that affect druggability include therapeutic modality, protein\\nlocalization, class, and structure availability. For instance, small-molecule drugs are typically used\\nfor targets with well-deﬁned binding pockets (e.g., kinases), whereas protein-based therapies are\\nmore suitable for targets that are difﬁcult to tackle with small molecules. Structural information on\\ndrug targets is helpful for drug design and optimization with AI-based predictions, such as\\nAlphaFold [74], thus expanding protein structure coverage. Target toxicity must also be consid-\\nered by assessing the cellular processes, gene essentiality, and tissue speciﬁcity involved.\\nTrade-off between high-conﬁdence and novel targets\\nNovelty is another crucial factor in target selection in addition to causality, druggability, and toxic-\\nity. Text-based evidence can be used to assess novelty and conﬁdence of a given target. Through\\nscrutinizing the relationship between approved drugs, molecular targets, and therapeutic indica-\\ntions, Santos et al. revealed that high-conﬁdence targets (or 'privileged' target families) accounted\\nfor the majority of approved drugs, whereas drugs tackling novel ﬁrst-in-class targets repre-\\nsented only a small proportion, although this is increasing, especially in the ﬁeld of oncology\\n[75]. Striking a balance between novelty and conﬁdence is essential for target selection. AI-\\npowered natural language processing methodologies can aid this target selection process\\nby extracting supporting evidence connecting a potential target to an indication based on huge\\namounts of data involving scientiﬁc publications, grants, and clinical trials, and this can provide\\na quantiﬁable scale for the novelty and conﬁdence of targets in the context of the disease and en-\\nable ﬂexible target-hunting workﬂows [76]. In addition, tools have been developed to quantify tar-\\nget novelty and conﬁdence. TIN-X is an example that uses text-mining data processed from the\\nscientiﬁc literature to quantify target novelty and conﬁdence by providing two bibliometric indices,\\nnamely the 'novelty index' that represents the scarcity of target-associated publications, and the\\n'importance index' that assesses the strength of the association between a given target–disorder\\npair [77]. Furthermore, AI could facilitate drug repurposing by connecting a high-conﬁdence tar-\\nget with known drugs to new disorders where the drugs have not been investigated, enabling\\ncost-effective and time-saving drug discovery for both common and rare diseases [78].\\nAI-identiﬁed targets validated in experiments\\nTarget validation using cell and animal models is a crucial step in target discovery to reduce the\\nproject attrition rate and the cost of drug development in the pharmaceutical industry (Box 1).\\nAn increasing number of AI-identiﬁed targets are being successfully validated. For example, 28\\nAI-proposed targets for ALS treatment were validated in an ALS-mimicking Drosophila model,\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n568\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nrevealing eight unreported targets whose suppression strongly rescues eye neurodegeneration\\n[53]. In addition, in the same therapeutic area, Zhang et al. developed a machine learning-\\nbased method to identify KANK1 as a novel gene linked to ALS and validated the neurotoxic\\neffects of KANK1 mutations reproduced by CRISPR–Cas9 in human neurons [79]. Inhibition of\\nHDAC6 was identiﬁed as a cardioprotective strategy by deep learning, and was validated via a\\nBAG3 cardiomyocyte-knockout mouse model of dilated cardiomyopathy [80]. CDK20 was\\nidentiﬁed as a target for the treatment of HCC using deep learning-based methods, and a\\nhighly potent small-molecule inhibitor designed by generative AI demonstrated selective\\nantiproliferation activity in an HCC cell line [81]. Furthermore, Zeng et al. developed deepDTnet\\nbased on 15 heterogeneous types of chemical, genomic, phenotypic, and cellular networks to\\nfacilitate in silico identiﬁcation of molecular targets for known drugs [82]. One of the identiﬁed\\ndrugs speciﬁcally targeting human ROR-γt shows therapeutic effects in a mouse model of\\nmultiple sclerosis.\\nConcluding remarks and future perspectives\\nTarget discovery is a crucial initial step in the modern drug discovery pipeline. Given that only a small\\nproportion of the potentially druggable targets in humans have been identiﬁed, there is a pressing\\nneed for effective target discovery methods. The growing number of AI-identiﬁed targets being val-\\nidated in experiments highlights the beneﬁts of incorporating AI algorithms into target identiﬁcation to\\nenhance the efﬁciency of novel target discovery and the development of new therapeutics.\\nOne area where AI is expected to make signiﬁcant contributions is in tackling complex diseases.\\nDiseases such as cancer, neurodegenerative disorders, and autoimmune conditions often in-\\nvolve intricate molecular mechanisms that are challenging to unravel. AI-driven target discovery\\nmethods can help to uncover novel targets and pathways underlying these diseases, paving\\nthe way for the development of more effective treatments.\\nMoreover, unexpected infectious disease outbreaks pose a constant threat to global health. The\\nrapid identiﬁcation of potential drug targets and the development of antiviral therapies are crucial\\nfor combating emerging pathogensv. By analyzing genomic data, AI algorithms can aid the iden-\\ntiﬁcation of essential viral proteins or host factors that can be targeted to inhibit viral replication,\\nthus providing valuable insights for the development of antiviral drugs [83].\\nBox 1. Advances in target validation\\nTarget validation using both cell and animal models is crucial to conﬁrm the modulatory effects of the proposed target on\\ndisease development. Although 2D cell culture and rodent models are the prevailing tools for target validation, the difﬁculty\\nof system establishment and the lack of complexity or recapitulation of human development limit their power as highly\\nrepresentative models. Organoids – 3D cell models derived from either induced pluripotent stem cells (iPSCs) or adult\\nstem cells (ASCs) – have arisen as a promising technique for both disease research and drug testing by allowing the cap-\\nture of tissue architecture and cellular microenvironment in vitro [84]. Taking advantage of their self-organizing ability,\\norganoids are able to mimic actual organ development, and have been successfully established for multiple human organs\\n(e.g., intestine, stomach, lung, liver, kidney, and brain) to explore the pathogenic mechanisms of various diseases [85–87].\\nFurthermore, because patient-derived organoids can retain the genetic, histopathological, and therapeutic response\\nphenotypes of the primary disease tissue, these models have made their way into identifying personalized therapeutic\\nregimens and drug efﬁcacy testing [88,89]. In colorectal cancer, patient-derived colon organoids served as an effective tool\\nto evaluate the efﬁcacy of CAR-T cell therapy [90].\\nIn both industrial and clinical laboratories there is a tendency to adopt automation to streamline experiments, data collec-\\ntion, and data analysis. With recent breakthroughs in bioengineering and machine learning, laboratory automation can\\ngreatly improve work efﬁciency and reproducibility by increasing data generation rate, reducing human technical variation,\\nand avoiding contaminant exposure [91,92]. The development and commercialization rate of novel therapeutic interven-\\ntions can also be enhanced by automation. For example, Insilico Medicine have launched an AI-driven robotic laboratory\\nthat is an interconnected expansion of their end-to-end AI drug discovery platformix. Despite several remaining obstacles,\\nthe progressive integration of automation will revolutionize the laboratory environment to maximize research success.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n569\\nOutstanding questions\\nCan AI algorithms accurately predict\\ntarget validation results and adverse\\neffects, as well as druggability, speci-\\nﬁcity, off-target effects, and potential\\ninteractions with other drugs, for po-\\ntential targets across different test sys-\\ntems (cell lines, animals, and humans)?\\nHow can AI-driven target discovery ap-\\nproaches be validated, benchmarked\\nagainst traditional experimental methods,\\nand also effectively incorporate domain\\nknowledge and expert insights to ensure\\nreliability, reproducibility, and enhanced\\ntarget identiﬁcation and validation?\\nHow can AI algorithms uncover the full\\nmechanism of action at selected tar-\\ngets, consider the heterogeneity and\\nvariability of diseases including individ-\\nual variations, and leverage this under-\\nstanding\\nto\\noptimize\\ncombination\\ntherapies, leading to the identiﬁcation\\nof synergistic drug–target combinations\\nfor improved treatment outcomes?\\nHow can we validate the reliability and\\nrobustness\\nof\\npredictions\\nand\\ndiscoveries based on synthetic AI-\\ngenerated data, and how does it\\ncompare to experimental validation\\nusing real-world data?\\n\\nAI also has the potential to revolutionize the discovery of efﬁcient combinations of therapeutic tar-\\ngets and mechanisms. Complex diseases often involve multiple molecular pathways and inter-\\nplay among various biological factors. AI algorithms can analyze diverse datasets, including\\ngenomic data, patient records, and synthetic lethality, to identify synergistic combinations of tar-\\ngets and mechanisms that may offer enhanced therapeutic effects. This approach can potentially\\ntransform treatment strategies, particularly in diseases where monotherapies have shown limited\\neffectiveness.\\nFurthermore, the integration of AI with fully automated robotic laboratories offers the potential for\\nhigh-throughput target validation and screening. Automated experiments, coupled with AI-driven\\ndata analysis, can expedite the validation of predicted targets, enabling researchers to assess\\ntheir therapeutic potential quickly. This combination of AI and automation has the potential to\\nrevolutionize the drug discovery process and signiﬁcantly reduce the time and cost required for\\ntarget identiﬁcation and validation.\\nDespite the tremendous progress made in AI-driven target discovery, several outstanding ques-\\ntions and challenges remain (see Outstanding questions). Ethical considerations, data privacy,\\nand regulatory frameworks are crucial aspects that must be addressed to ensure responsible\\nand ethical deployment of AI in drug development. Furthermore, the interpretability and\\nexplainability of AI algorithms are essential for gaining trust and acceptance from the scientiﬁc\\nand medical communities. It is pertinent to note that, although AI has demonstrated potential in ex-\\npediting the early stages of drug discovery such as target identiﬁcation and lead optimization, it\\ncannot signiﬁcantly shorten the time required for clinical trials during drug development. This is be-\\ncause of the long period of time spent on ethical and regulatory approval, patient recruitment, du-\\nration of treatment, and data analysis, irrespective of whether the drug was developed by AI or not.\\nIn summary, AI has emerged as a powerful tool in target discovery and drug development, and\\nis revolutionizing how we identify novel drug targets and repurpose existing drugs. With the\\ncontinued advancements in AI technology and the collaborative efforts of researchers, we\\ncan look forward to a future where AI plays an indispensable role in accelerating the develop-\\nment of safe and effective therapeutics for a wide range of diseases, ultimately improving\\nhuman health and well-being.\\nAcknowledgments\\nWe thank Drs Feng Ren and Alex Aliper for expert advice, and Drs Xi Long and Bonnie Hei Man Liu for graphical illustrations\\nand literature review.\\nDeclaration of interests\\nF.W.P., I.V.O., and A.Z. are employees of Insilico Medicine Hong Kong Ltd.\\nResources\\nihttps://ftloscience.com/process-costs-drug-development/\\niiwww.ﬁercebiotech.com/medtech/breaking-big-pharma-s-ai-barrier-insilico-medicine-uncovers-novel-target-new-drug-for\\niiiwww.nature.com/articles/d43747-021-00045-7\\nivwww.eurekalert.org/news-releases/982543\\nvwww.prnewswire.com/news-releases/insilico-medicine-announces-novel-3cl-protease-inhibitor-preclinical-candidate-for-\\ncovid-19-treatment-301553766.html\\nviwww.exscientia.ai/dsp-1181\\nviihttps://investors.exscientia.ai/press-releases/press-release-details/2021/exscientia-announces-second-molecule-\\ncreated-using-ai-from-sumitomo-dainippon-pharma-collaboration-to-enter-phase-1-clinical-trial/Default.aspx\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n570\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nviiihttps://investors.exscientia.ai/press-releases/press-release-details/2023/Exscientia-Announces-First-in-Human-Study-\\nfor-Bristol-Myers-Squibb-In-Licensed-PKC-Theta-Inhibitor-EXS4318/default.aspx\\nixwww.globenewswire.com/news-release/2023/01/05/2583816/0/en/Insilico-Medicine-launches-6th-generation-\\nIntelligent-Robotics-Lab-to-further-accelerate-its-AI-driven-drug-discovery.html\\nReferences\\n1. Hinkson, I.V. et al. (2020) Accelerating therapeutics for opportu-\\nnities in medicine: a paradigm shift in drug discovery. Front.\\nPharmacol. 11, 770\\n2. Zhou, Y. et al. (2022) Therapeutic target database update 2022:\\nfacilitating drug discovery with enriched comparative data of\\ntargeted agents. Nucleic Acids Res. 50, D1398–D1407\\n3. Kana, O. and Brylinski, M. (2019) Elucidating the druggability of\\nthe human proteome with eFindSite. J. Comput. Aided Mol.\\nDes. 33, 509–519\\n4. Finan, C. et al. (2017) The druggable genome and support for\\ntarget identiﬁcation and validation in drug development. Sci.\\nTransl. Med. 9, eaag1166\\n5. Sun, D. et al. (2022) Why 90% of clinical drug development fails\\nand how to improve it? Acta Pharm. Sin. B 12, 3049–3062\\n6. Shiraiwa, K. et al. (2020) Chemical tools for endogenous protein\\nlabeling and proﬁling. Cell Chem. Biol. 27, 970–985\\n7. van der Zouwen, A.J. and Witte, M.D. (2021) Modular ap-\\nproaches to synthesize activity- and afﬁnity-based chemical\\nprobes. Front. Chem. 9, 644811\\n8. Ong, S.E. and Mann, M. (2006) A practical recipe for stable iso-\\ntope labeling by amino acids in cell culture (SILAC). Nat. Protoc.\\n1, 2650–2660\\n9. Jin, J. et al. (2023) SIRT3-dependent delactylation of cyclin E2\\nprevents hepatocellular carcinoma growth. EMBO Rep. 24,\\ne56052\\n10. Li, X. et al. (2023) Deacetylation induced nuclear condensation of\\nHP1gamma promotes multiple myeloma drug resistance. Nat.\\nCommun. 14, 1290\\n11. Wang, Y. et al. (2022) DUT enhances drug resistance to protea-\\nsome inhibitors via promoting mitochondrial function in multiple\\nmyeloma. Carcinogenesis 43, 1030–1038\\n12. Montero-Calle, A. et al. (2023) In-depth quantitative proteomics\\nanalysis revealed C1GALT1 depletion in ECC-1 cells mimics an\\naggressive endometrial cancer phenotype observed in cancer\\npatients with low C1GALT1 expression. Cell Oncol. (Dordr) 46,\\n697–715\\n13. Kortum, B. et al. (2022) Combinatorial treatment with statins and\\nniclosamide prevents CRC dissemination by unhinging the\\nMACC1–beta-catenin–S100A4 axis of metastasis. Oncogene\\n41, 4446–4458\\n14. Qi, T.F. et al. (2023) Parallel-reaction monitoring revealed altered\\nexpression of a number of epitranscriptomic reader, writer, and\\neraser proteins accompanied with colorectal cancer metastasis.\\nProteomics 23, e2200059\\n15. Nidhi, S. et al. (2021) Novel CRISPR–Cas systems: an updated\\nreview of the current achievements, applications, and future re-\\nsearch perspectives. Int. J. Mol. Sci. 22, 3327\\n16. Samelson, A.J. et al. (2022) BRD2 inhibition blocks SARS-CoV-2\\ninfection by reducing transcription of the host cell receptor ACE2.\\nNat. Cell Biol. 24, 24–34\\n17. Ramkumar, P. et al. (2020) CRISPR-based screens uncover de-\\nterminants of immunotherapy response in multiple myeloma.\\nBlood Adv. 4, 2899–2911\\n18. Chakraborty, S. et al. (2018) Onco-Multi-OMICS approach: a\\nnew frontier in cancer research. Biomed. Res. Int. 2018,\\n9836256\\n19. Nurk, S. et al. (2022) The complete sequence of a human\\ngenome. Science 376, 44–53\\n20. Buniello, A. et al. (2019) The NHGRI–EBI GWAS catalog of\\npublished genome-wide association studies, targeted arrays\\nand summary statistics 2019. Nucleic Acids Res. 47,\\nD1005–D1012\\n21. Einarsson, G.G. et al. (2021) Extended-culture and culture-\\nindependent molecular analysis of the airway microbiota in cystic\\nﬁbrosis following CFTR modulation with ivacaftor. J. Cyst. Fibros.\\n20, 747–753\\n22. Sewell, G.W. and Kaser, A. (2022) Interleukin-23 in the patho-\\ngenesis of inﬂammatory bowel disease and implications for ther-\\napeutic intervention. J. Crohns Colitis 16, ii3–ii19\\n23. Deelen, J. et al. (2019) Publisher correction: a meta-analysis of\\ngenome-wide association studies identiﬁes multiple longevity\\ngenes. Nat. Commun. 10, 3669\\n24. Namba, S. et al. (2022) A practical guideline of genomics-driven\\ndrug discovery in the era of global biobank meta-analysis. Cell\\nGenom. 2, 100190\\n25. Abell, N.S. et al. (2022) Multiple causal variants underlie genetic\\nassociations in humans. Science 375, 1247–1254\\n26. Assum, I. et al. (2022) Tissue-speciﬁc multi-omics analysis of\\natrial ﬁbrillation. Nat. Commun. 13, 441\\n27. Suhre, K. et al. (2017) Erratum: connecting genetic risk to dis-\\nease end points through the human blood plasma proteome.\\nNat. Commun. 8, 14357\\n28. Yin, X. et al. (2022) Integrating transcriptomics, metabolomics,\\nand GWAS helps reveal molecular mechanisms for metabolite\\nlevels and disease risk. Am. J. Hum. Genet. 109, 1727–1741\\n29. Mountjoy, E. et al. (2021) An open approach to systematically pri-\\noritize causal variants and genes at all published human GWAS\\ntrait-associated loci. Nat. Genet. 53, 1527–1533\\n30. Na, F. et al. (2022) KMT2C deﬁciency promotes small cell\\nlung cancer metastasis through DNMT3A-mediated epigenetic\\nreprogramming. Nat. Can. 3, 753–767\\n31. Gulﬁdan, G. et al. (2022) Systems biomarkers for papillary thyroid\\ncancer prognosis and treatment through multi-omics networks.\\nArch. Biochem. Biophys. 715, 109085\\n32. Lu, J. et al. (2021) Multi-omics analysis of fatty acid metabolism in\\nthyroid carcinoma. Front. Oncol. 11, 737127\\n33. Raivola, J. et al. (2022) Multiomics characterization implicates\\nPTK7 in ovarian cancer EMT and cell plasticity and offers strate-\\ngies for therapeutic intervention. Cell Death Dis. 13, 714\\n34. Pinero, J. et al. (2018) Network, transcriptomic and genomic fea-\\ntures differentiate genes relevant for drug response. Front.\\nGenet. 9, 412\\n35. Wolber,\\nG.\\net\\nal.\\n(2008)\\nMolecule–pharmacophore\\nsuperpositioning and pattern matching in computational drug\\ndesign. Drug Discov. Today 13, 23–29\\n36. Lee, A. et al. (2016) Using reverse docking for target identiﬁcation\\nand its applications for drug discovery. Expert Opin. Drug Discov.\\n11, 707–715\\n37. Nettles, J.H. et al. (2006) Bridging chemical and biological space:\\n'target ﬁshing' using 2D and 3D molecular descriptors. J. Med.\\nChem. 49, 6802–6810\\n38. Lo, Y.C. et al. (2016) 3D chemical similarity networks for\\nstructure-based target prediction and scaffold hopping. ACS\\nChem. Biol. 11, 2244–2253\\n39. Vamathevan, J. et al. (2019) Applications of machine learning in\\ndrug discovery and development. Nat. Rev. Drug Discov. 18,\\n463–477\\n40. Mamoshina, P. et al. (2018) Machine learning on human muscle\\ntranscriptomic data for biomarker discovery and tissue-speciﬁc\\ndrug target identiﬁcation. Front. Genet. 9, 242\\n41. Zhavoronkov, A. et al. (2019) Deep biomarkers of aging and\\nlongevity: from research to applications. Aging (Albany NY) 11,\\n10771–10780\\n42. Muslu, O. et al. (2022) GuiltyTargets: prioritization of novel thera-\\npeutic targets with network representation learning. IEEE/ACM\\nTrans. Comput. Biol. Bioinform. 19, 491–500\\n43. Liu, R. et al. (2021) A deep learning framework for drug\\nrepurposing via emulating clinical trials on real-world patient\\ndata. Nat. Mach. Intell. 3, 68–75\\n44. Zhavoronkov, A. et al. (2019) Deep learning enables rapid identi-\\nﬁcation of potent DDR1 kinase inhibitors. Nat. Biotechnol. 37,\\n1038–1040\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n571\\n\\n45. Ivanenkov, Y.A. et al. (2023) Chemistry42: an AI-driven platform\\nfor molecular design and optimization. J. Chem. Inf. Model. 63,\\n695–701\\n46. Obrezanova, O. (2023) Artiﬁcial intelligence for compound phar-\\nmacokinetics prediction. Curr. Opin. Struct. Biol. 79, 102546\\n47. Chen, R. et al. (2018) Machine learning for drug–target interac-\\ntion prediction. Molecules 23, 2208\\n48. Ye, Q. et al. (2021) A uniﬁed drug–target interaction prediction\\nframework based on knowledge graph and recommendation\\nsystem. Nat. Commun. 12, 6775\\n49. Kavalci, E. and Hartshorn, A. (2023) Improving clinical trial design\\nusing interpretable machine learning based prediction of early\\ntrial termination. Sci. Rep. 13, 121\\n50. McCulloch, W.S. and Pitts, W. (1943) A logical calculus of the\\nideas immanent in nervous activity. Bull. Math. Biophys. 5,\\n115–133\\n51. Zhavoronkov, A. et al. (2019) Artiﬁcial intelligence for aging and\\nlongevity research: Recent advances and perspectives. Ageing\\nRes. Rev. 49, 49–66\\n52. Aliper, A. et al. (2016) Deep learning applications for predicting\\npharmacological properties of drugs and drug repurposing\\nusing transcriptomic data. Mol. Pharm. 13, 2524–2530\\n53. Pun, F.W. et al. (2022) Identiﬁcation of therapeutic targets for\\namyotrophic lateral sclerosis using pandaomics – an AI-\\nenabled biological target discovery platform. Front. Aging\\nNeurosci. 14, 914017\\n54. Fabris, F. et al. (2020) Using deep learning to associate human\\ngenes with age-related diseases. Bioinformatics 36, 2202–2208\\n55. West, M.D. et al. (2018) Use of deep neural network ensembles\\nto identify embryonic-fetal transition markers: repression of\\nCOX7A1 in embryonic and cancer cells. Oncotarget 9,\\n7796–7811\\n56. Luo, R. et al. (2022) BioGPT: generative pre-trained transformer\\nfor biomedical text generation and mining. Brief. Bioinform. 23,\\nbbac409\\n57. Shayakhmetov, R. et al. (2020) Molecular generation for desired\\ntranscriptome changes with adversarial autoencoders. Front.\\nPharmacol. 11, 269\\n58. Vinas, R. et al. (2022) Adversarial generation of gene expression\\ndata. Bioinformatics 38, 730–737\\n59. Beaulieu-Jones, B.K. et al. (2019) Privacy-preserving generative\\ndeep neural networks support clinical data sharing. Circ.\\nCardiovasc. Qual. Outcomes 12, e005122\\n60. Song, J. et al. (2021) The discovery of new drug–target interac-\\ntions for breast cancer treatment. Molecules 26, 7474\\n61. Chawla, N.V. et al. (2002) SMOTE: synthetic minority over-\\nsampling technique. J. Artif. Intell. Res. 16, 321–357\\n62. Achuthan, S. et al. (2022) Leveraging deep learning algorithms\\nfor synthetic data generation to design and analyze biological\\nnetworks. J. Biosci. 47, 43\\n63. Howe, E.G., III and Elenberg, F. (2020) Ethical challenges posed\\nby big data. Innov. Clin. Neurosci. 17, 24–30\\n64. Bhanot, K. et al. (2021) The problem of fairness in synthetic\\nhealthcare data. Entropy (Basel) 23, 1165\\n65. Rajotte, J.F. et al. (2022) Synthetic data as an enabler for ma-\\nchine learning applications in medicine. iScience 25, 105331\\n66. El Emam, K. et al. (2022) Utility metrics for evaluating synthetic\\nhealth data generation methods: validation study. JMIR Med.\\nInform. 10, e35734\\n67. Treppner, M. et al. (2021) Synthetic single cell RNA sequencing\\ndata from small pilot studies using deep generative models.\\nSci. Rep. 11, 9403\\n68. Nogales, C. et al. (2022) Network pharmacology: curing causal\\nmechanisms instead of treating symptoms. Trends Pharmacol.\\nSci. 43, 136–150\\n69. Buphamalai, P. et al. (2021) Network analysis reveals rare dis-\\nease signatures across multiple levels of biological organization.\\nNat. Commun. 12, 6306\\n70. Lecca, P. (2021) Machine learning for causal inference in biolog-\\nical networks: perspectives of this challenge. Front. Bioinform. 1,\\n746712\\n71. Cassan, O. et al. (2021) Inferring and analyzing gene regulatory\\nnetworks from multi-factorial expression data: a complete and\\ninteractive suite. BMC Genomics 22, 387\\n72. Gillani, Z. et al. (2014) CompareSVM: supervised, support vec-\\ntor machine (SVM) inference of gene regularity networks. BMC\\nBioinformatics 15, 395\\n73. Zhou, X. and Kosorok, M.R. (2017) Causal nearest neighbor\\nrules for optimal treatment regimes. ArXiv Published online\\nNovember 22, 2017. https://arxiv.org/abs/1711.08451\\n74. Varadi, M. et al. (2022) AlphaFold Protein Structure Database:\\nmassively expanding the structural coverage of protein-\\nsequence space with high-accuracy models. Nucleic Acids\\nRes. 50, D439–D444\\n75. Santos, R. et al. (2017) A comprehensive map of molecular drug\\ntargets. Nat. Rev. Drug Discov. 16, 19–34\\n76. Vera, C.D. et al. (2022) Treating Duchenne muscular dystrophy:\\nthe promise of stem cells, artiﬁcial intelligence, and multi-omics.\\nFront. Cardiovasc. Med. 9, 851491\\n77. Cannon, D.C. et al. (2017) TIN-X: target importance and novelty\\nexplorer. Bioinformatics 33, 2601–2603\\n78. Pushpakom, S. et al. (2019) Drug repurposing: progress, chal-\\nlenges and recommendations. Nat. Rev. Drug Discov. 18, 41–58\\n79. Zhang, S. et al. (2022) Genome-wide identiﬁcation of the genetic\\nbasis of amyotrophic lateral sclerosis. Neuron 110, 992–1008\\n80. Yang, J. et al. (2022) Phenotypic screening with deep learning\\nidentiﬁes HDAC6 inhibitors as cardioprotective in a BAG3\\nmouse model of dilated cardiomyopathy. Sci. Transl. Med. 14,\\neabl5654\\n81. Ren, F. et al. (2023) AlphaFold accelerates artiﬁcial intelligence\\npowered drug discovery: efﬁcient discovery of a novel CDK20\\nsmall molecule inhibitor. Chem. Sci. 14, 1443–1452\\n82. Zeng, X. et al. (2020) Target identiﬁcation among known drugs by\\ndeep learning from heterogeneous networks. Chem. Sci. 11,\\n1775–1797\\n83. Ong, E. et al. (2020) COVID-19 coronavirus vaccine design using\\nreverse vaccinology and machine learning. Front. Immunol. 11,\\n1581\\n84. Jensen, C. and Teng, Y. (2020) Is it time to start transitioning\\nfrom 2D to 3D cell culture? Front. Mol. Biosci. 7, 33\\n85. Fan, W. et al. (2022) Applications of brain organoids for infectious\\ndiseases. J. Mol. Biol. 434, 167243\\n86. Sidhaye, J. and Knoblich, J.A. (2021) Brain organoids: an en-\\nsemble of bioassays to investigate human neurodevelopment\\nand disease. Cell Death Differ. 28, 52–67\\n87. Angus, H.C.K. et al. (2019) Intestinal organoids as a tool for inﬂam-\\nmatory bowel disease research. Front. Med. (Lausanne) 6, 334\\n88. Wensink, G.E. et al. (2021) Patient-derived organoids as a pre-\\ndictive biomarker for treatment response in cancer patients.\\nNPJ Precis. Oncol. 5, 30\\n89. Berkers, G. et al. (2019) Rectal organoids enable personalized\\ntreatment of cystic ﬁbrosis. Cell Rep. 26, 1701–1708\\n90. Schnalzger, T.E. et al. (2019) 3D model for CAR-mediated cyto-\\ntoxicity using patient-derived colorectal cancer organoids. EMBO\\nJ. 38, e100928\\n91. Burger, B. et al. (2020) A mobile robotic chemist. Nature 583,\\n237–241\\n92. Crone, M.A. et al. (2020) A role for biofoundries in rapid development\\nand validation of automated SARS-CoV-2 clinical diagnostics. Nat.\\nCommun. 11, 4464\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n572\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\",\n",
       " 'Zhichao Liu a,⇑, Ruth A. Roberts a,b,c,\\nMadhu Lal-Nag d, Xi Chen a, Ruili Huang e,\\nWeida Tong a,⇑\\na National Center for Toxicological Research, US Food and Drug Administration,\\nJefferson, AR 72079, USA\\nb ApconiX, BioHub at Alderley Park, Alderley Edge SK10 4TG, UK\\nc University of Birmingham, Edgbaston, Birmingham B15 2TT, UK\\nd Ofﬁce of Translational Sciences, Center for Drug Evaluation and Research,\\nUS FDA, Silver Spring, MD 20993, USA\\ne National Center for Advancing Translational Sciences, National Institutes of Health,\\nRockville, MD 20850, USA\\nThe discovery and development of new medicines is\\nexpensive, time-consuming, and often inefﬁcient, with\\nmany failures along the way. Powered by artiﬁcial intelli-\\ngence (AI), language models (LMs) have changed the\\nlandscape of natural language processing (NLP), offering\\npossibilities to transform treatment development more\\neffectively. Here, we summarize advances in AI-powered\\nLMs and their potential to aid drug discovery and devel-\\nopment. We highlight opportunities for AI-powered LMs in\\ntarget identiﬁcation, clinical design, regulatory decision-\\nmaking, and pharmacovigilance. We speciﬁcally empha-\\nsize the potential role of AI-powered LMs for developing\\nnew treatments for Coronavirus 2019 (COVID-19) strate-\\ngies, including drug repurposing, which can be extrapo-\\nlated to other infectious diseases that have the potential to\\ncause pandemics. Finally, we set out the remaining chal-\\nlenges and propose possible solutions for improvement.\\nKeywords: Artiﬁcial intelligence; Language models;\\nNatural language processing; Drug discovery;\\nDrug development; COVID-19\\nZhichao Liu Zhichao Liu is a technical lea-\\nder in Artiﬁcial Intelligence Research Force\\n(AIRForce), Division of Bioinformatics &\\nBiostatistics,\\nFDA/NCTR.\\nDr\\nLiu’s\\nback-\\nground spans the ﬁelds of chemistry,\\nbiology, and computer science. He has led\\nmany cutting-edge projects over the past\\ndecade by designing, implementing, and\\ndeploying\\nartiﬁcial\\nintelligence\\n(AI)/\\nmachine-learning\\n(ML)\\nsolutions\\nfor\\nadvanced regulatory sciences. Speciﬁcally,\\nDr Liu developed a standard pipeline for\\nAI-powered drug repositioning to help the industry seek the optimal\\nroute to accelerate drug-development efﬁcacy from an advanced\\nregulatory-sciences perspective. Furthermore, Dr Liu developed AI/ML\\nsolutions for promoting predictive toxicology, with successful models\\nadopted by the industry and regulators. His achievements have been\\nreﬂected by Dr Liu being awarded ﬁve FDA-wide Awards, nine NCTR-level\\nAwards, two scientiﬁc community-level awards, and by more than 100\\npeer-reviewed publications.\\nRuth A. Roberts Ruth A. Roberts is Chair\\nand Director of Drug Discovery at Birm-\\ningham University, UK and cofounder of\\nApconiX, an integrated toxicology and ion\\nchannel company. She is a former presi-\\ndent of EUROTOX and of the British Toxi-\\ncology Society (BTS), a Fellow and past\\npresident of the Academy of Toxicological\\nSciences (ATS), a fellow of the Royal Col-\\nlege of Pathologists and of the Royal\\nSociety of Biology, and Vice Chair of the\\nHealth and Environmental Sciences Insti-\\ntute (HESI) Board. Ruth was the recipient of the SOT Achievement award\\nin 2002, the EUROTOX Bo Holmstedt Award in 2009 and the SOT 2018\\nFounders award, given in recognition of outstanding leadership in fos-\\ntering the role of toxicological sciences in safety decision making. With\\nmore than 150 publications in peer-reviewed journals, she is interested in\\ndeveloping and implementing innovative models in drug discovery and\\ndevelopment.\\nAI-based language models powering drug\\ndiscovery and development\\n⇑Corresponding authors. Liu, Z. (zhichao.liu@fda.hhs.gov), Tong, W. (weida.tong@fda.hhs.gov).\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nREVIEWS\\nKEYNOTE (GREEN)\\n1359-6446/\\x01 2021 The Authors. Published by Elsevier Ltd. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).\\nhttps://doi.org/10.1016/j.drudis.2021.06.009\\nwww.drugdiscoverytoday.com\\n2593\\n\\nMadhu Lal-Nag Madhu Lal-Nag was awar-\\nded a PhD in molecular and cellular\\noncology from The George Washington\\nUniversity and an MSc in bioscience busi-\\nness from The Keck Graduate Institute of\\nApplied Biosciences, Claremont, CA. She\\nwas also awarded an MSc in biochemistry\\nfrom the University of Mumbai, India. At\\nNCATS, Dr Lal-Nag served as the Director\\nof the Trans NIH RNAi Facility, which runs\\nhigh-throughput\\nfunctional\\ngenomics\\nscreens for the entire NIH intramural pro-\\ngram, serving 21 institutes. Dr Lal-Nag moved to the Center for Drug\\nEvaluation and Research at the FDA in December 2018, where she is the\\nProgram Director for the Research Governance Council, an advisory\\ncouncil overseeing CDER research. She is active in the microphysiological\\nsystems community, continuing to teach workshops and give scientiﬁc\\ntalks about the role and intersection of alternative animal models and\\nmicrophysiological systems in evaluating the efﬁcacy and safety of drugs\\nin therapeutic development.\\nWeida Tong Weida Tong is the Director of\\nthe Division of Bioinformatics and Bio-\\nstatistics at FDA/NCTR. He has published\\nover 300 peer-reviewed papers from his\\nroles in supervising and leading the FDA-\\nled\\ncommunity-wide\\nMicroArray\\nand\\nSequencing Quality Control (MAQC/SEQC)\\nconsortium to analyze technical perfor-\\nmance and practical utility of emerging\\ngenomic technologies with emphasis on\\nregulatory application and precision med-\\nicine; addressing drug safety concerns\\nrelated to drug-induced liver injury (DILI); developing machine learning\\n(ML) and AI for digital health and drug repositioning; and conducting\\nmolecular modeling and QSARs on various toxicological endpoints, such\\nas carcinogenicity.\\nKEYNOTE (GREEN)\\nIntroduction\\nA LM aims to determine the probability of a given sequence of words occurring\\nin a sentence by using different statistical and probabilistic techniques. Powered\\nby AI, a LM acts as a human-like learning process, not only to predict words, but\\nalso to understand languages. Moreover, the knowledge gained by the LM can\\nbe transferred to other tasks, as humans do when they routinely learn from\\none task and transfer the knowledge to another. This innovative revolution\\nhas massively empowered NLP. Consequently, AI-based LMs have proved their\\nuse in a variety of real-world applications, such as chatbots, automated transla-\\ntions, customer experience, sentiment-based news aggregation, and language\\nidentiﬁcation.1 Here, the AI-powered LMs we describe are mainly focused on\\nLM based on a neural architecture.\\nInnovations in emerging biological technologies have made enormous\\nimprovements to our understanding of disease etiology and pathogenesis.2,3\\nHowever, drug discovery and development remain a time-consuming and costly\\nprocess, beset by high failure rates and uncertainty.4 Signiﬁcant efforts are being\\ninvested in reﬁning, revising, and reforming the de novo drug discovery and\\ndevelopment process, with particular emphasis on data-driven approaches to\\nnew treatments, improved patient outcomes, and lower costs.5 With a rapid\\nincrease in the quantity of biomedical data, a better understanding of the char-\\nacteristics of the data generated and the types of analysis that can be carried out\\nwill be valuable in understanding the potential of data resources.6\\nAlthough attention tends to focus on ‘omics data and bioassays generated\\nfrom large technology platforms, text-based data remains a valuable information\\nresource in the drug discovery and development process. Thus, conventional\\nNLP-based approaches and tools have been developed to uncover hidden infor-\\nmation embedded in such documents.7 However, more advanced strategies are\\nurgently needed to harness a growing wealth of available data and to stay abreast\\nof the latest accumulated text-based documents. Notably, AI-powered LMs have\\nthe potential to unlock new possibilities for drug development and usher in an\\nera of faster, cheaper, and more effective drug discovery and development.8\\nSeveral text-based documents and applications of AI-powered LMs have been\\nshown to be useful in different stages of drug discovery and development\\n(Fig. 1). During the preclinical stages, the incomplete understanding of the\\npathophysiology of complex diseases is one of the signiﬁcant hurdles for target\\nidentiﬁcation. Furthermore, animal models might not reﬂect the underlying\\nmechanisms of human disorders. During the clinical phases, patient selection,\\nrecruitment, and monitoring pose a strategic challenge. During the post-\\nmarketing phase, there are shortfalls in the ability of the current system to effec-\\ntively and efﬁciently detect, interpret, and analyze safety signals. In addition,\\ncomplexity in the regulatory submission process could hinder harmonized com-\\nmunication between pharmaceutical companies and regulatory agencies. In this\\nreview, we provide an overview of tangible opportunities for AI-powered LMs in\\ndrug discovery and development, and offer potential solutions for key remain-\\ning challenges.\\nAI-powered language models\\nRapidly evolving LMs have enormously increased our ability to uncover action-\\nable insights from text (Box 1). Powered by AI, many intriguing LM infrastruc-\\ntures, particularly transformer-based LMs, have been developed and have\\nshown potential in information retrieval, text classiﬁcation, text summariza-\\ntion, and sentiment analysis. The heart of transformer-based LMs is sequence-\\nto-sequence learning (Seq2Seq) through self-attention and positional encoding,\\nwhich has changed the way we work with text data, from processing language to\\nlearning language9 (Box 2).\\nA Seq2seq model comprises a combination of an Encoder and a Decoder, aim-\\ning to convert sequences from one domain (e.g., a sentence in English) to\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2594\\nwww.drugdiscoverytoday.com\\n\\nanother (e.g., the exact meaning of the sentence in French).10\\nDeep-learning model architectures, such as recurring neural net-\\nwork (RNN) or long short-term memory (LSTM), could be used\\nfor Encoder and Decoder development. The Encoder takes the\\nsequence as an input and maps the sequence into internal state\\nvectors or context vectors, where the output of the Encoder is\\nthen discarded. The generated context vector could encapsulate\\ninput sequence information to facilitate the prediction of the\\nDecoder. The training process of the Decoder is referred to as\\n‘teach forcing’. Speciﬁcally, the Decoder takes the extracted con-\\ntext vector of the Encoder as the initial state to generate the out-\\nput sequence. These outputs are also taken into consideration for\\nfuture outputs. Seq2seq models have been used to solve complex\\nNLP tasks, such as Machine Translation, Question & Answering\\n(Q&A), Chatbots, Text Summarization, and so on. LSTMs func-\\ntion through a cell, an input gate, an output gate, and a forget\\ngate, avoiding the problem of a vanishing gradient seen, for\\nexample, with RNN.11,12 The main revolutionary part of trans-\\nformer models is the possibility of directly accessing all positions\\nin the sequence, equivalent to having full random-access mem-\\nory of the sequence during encoding/decoding.\\nTransformer-based LMs can mimic some human-like charac-\\nteristics of constant acquisition, ﬁne-tuning, and transfer of\\nknowledge and skills (Fig. 2). First, transformer-based LMs can\\noffer a transfer learning framework.13 For this, the learned knowl-\\nedge is stored in a pretrained model, allowing users to be further\\ntrained with incrementally available information or domain-\\nspeciﬁc knowledge. One example is BioBERT, which is a pre-\\ntrained language representation model derived by training the\\noriginal BERT model with biomedical corpora in PubMed.14 Bio-\\nBERT outperformed the original BERT base model in most\\nbiomedical text-mining tasks, including biomedical named\\nentity recognition, biomedical relation extraction, and biomedi-\\ncal question answering. The same learning strategy was also\\nadopted by ClinicalBERT, which trained the BERT-based models\\nwith electronic medical records (EHRs) data to enhance its clini-\\ncal application.15 Second, transformer-based LMs could be ﬁne-\\ntuned based on the downstream task. A human can apply the\\nright knowledge to solve related questions; this function also\\nappears to be possible with transformer-based LMs. For different\\ntasks, the pretrained language models could be ﬁne-tuned with\\njust one additional output layer to create state-of-the-art models\\nfor a range of NLP tasks. For example, the same pretrained BERT\\nmodel,\\nwith\\none\\nﬁne-tuning\\nlayer,\\nyielded\\nbetter\\nmodel\\nperformance for 11 state-of-the-art NLP tasks with large margins\\ncompared with XXXX. These NLP tasks fall primarily into three\\ncategories: text classiﬁcation, textual entailment, and question\\nanswering. More encouragingly, BERT is the ﬁrst to outperform\\nthe human-level performance for two tasks: the Stanford Ques-\\ntion Answering Dataset (SQuAD), and the Situations With Adver-\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today\\nFIGURE 1\\nArtificial intelligence (AI)-powered language models in the context of drug discovery and development. The overall stages of the development process are\\nillustrated in the top layer (green), and the objectives from this process are captured in the layer below (blue). The text documents related to each stage are\\nlisted, and the opportunities of AI-powered language models are summarized in the following two layers (yellow and pink). Abbreviations: PD,\\npharmacodynamics; PK, pharmacokinetics.\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2595\\n\\nsarial Generations (SWAG).16 Third, transformer-based LMs are\\ncapable of summarizing the knowledge embedded in different\\ndocuments. One of the human learning abilities is to accurately\\nsum up information in documents and turn it into useful knowl-\\nedge. The long sequence-based transformer model has been\\ndemonstrated to generate ﬂuent, coherent multisentence para-\\ngraphs; even whole Wikipedia articles could be created this\\nway as a multidocument summarization of source documents.17\\nHowever, the vast computational memory requirement of long\\nsequence summarization models has limited their applications.\\nThe newly proposed Reformer model of Google shows a tremen-\\ndously increased capability for handling long sequences gained\\nby adopting a locality-sensitive hashing technique, which will\\nsigniﬁcantly\\nexpand\\nthe\\nhorizon\\nof\\nmultidocument\\nsummarization.18\\nSelection of ‘ﬁt-for-purpose’ AI-powered language\\nmodels\\nThe diversity of transformer-based LMs has massively enhanced\\ncapabilities in tackling unstructured text for various real-world\\napplications. However, it is challenging to select and reposition\\nthe transformer-based LMs in the context of biomedical applica-\\ntions. The implementation of a ‘ﬁt-for-purpose’ transformer-\\nbased LM in drug discovery and drug development is multifacto-\\nrial; signiﬁcant factors are the availability of domain-speciﬁc\\ntraining data sets, downstream NLP tasks, and computational\\npower. Crucial steps essential for selecting a ‘ﬁt-for-purpose’ AI-\\nbased LM in drug discovery and development are ‘deﬁne a pur-\\npose’, ‘manage data availability’, and ‘measure scalability’.\\nDeﬁne a purpose\\nAI-powered LMs have potential at every stage of drug discovery\\nand development, but it is essential to deﬁne the purpose before\\nseeking the right AI solution. For example, a scientist at a phar-\\nmaceutical company might need to understand the biological\\nrole of a protein target and then collate the patents on therapeu-\\ntic categories to support target identiﬁcation and validation. For\\nthis, an AI-powered Q&A system that can aggregate publicly\\navailable literature and medical patent databases might be the\\nright solution. Patient recruiters might be more interested in\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nBox 1 Evolution of language models.\\nLMs enable a computer to make sense of the human language by\\nestimating the probability distribution of various linguistic units\\n(e.g., words, sentences, paragraphs, etc.). LMs are mainly divided\\ninto two categories: count-based and continuous-space LMs. The\\nclassic example of count-based LM is n-gram, which aims to con-\\nstruct the joint probability distribution of sentences to predict\\nthe words.103 However, several drawbacks of n-gram LM have\\nlimited its real-world applications: (i) the n-gram LM is not\\ncapable of inferring new word sequence combinations that\\nwere not encountered in the training corpus; and (ii) the n-\\ngram model is not able to take into consideration the semantic\\nrelationship among words. The shortcoming of count-based\\nLMs led to the idea of continuous-space LM by applying\\nneural network or dimension reduction techniques to extract\\nsyntactic\\nand\\nsemantic\\nfeatures\\nof\\nlanguages.104\\nMost\\ncontinuous-space LMs are neural probabilistic-based models.105\\nMikolov\\net\\nal.106\\nproposed\\nWord2Vec\\nto\\ngenerate\\nvector\\nrepresentation of words (i.e., word embeddings) by training a\\nshallow neural\\nnetwork\\nto\\nlearn\\nthe\\nsimilarities\\nbetween\\nwords. Word2Vec is one of the most widely used neural-based\\nLM and takes breakthroughs to the whole ﬁeld. Along with\\nother\\nword\\nrepresentation\\nmodels,\\n(e.g.,\\nGloVe107\\nand\\nFastText108), these word-embedding techniques require less\\nmemory and decreased compute time, and have been shown\\nto\\nimprove\\ndownstream\\nmodel\\nperformance\\ndrastically.\\nHowever, word embeddings provide a one-to-one relationship\\nbetween word and vector representation that does not solve\\nthe problem of polysemous words. Subsequently, RNNs and\\nLSTM were proposed to handle the processing of textual\\nsequence data.12 However, the two algorithms suffer from a\\nvanishing gradient problem and have a difﬁculty in dealing\\nwith\\nlong\\nsequence\\nsentences.98\\nThe\\nmost\\ninnovative\\ngroundbreaking NLP framework is transform-based LMs, and\\nthe BERT model from Google is an outstanding example of\\nsuch an approach.16\\nBox 2 Architectures of transformer-based language models.\\nTwo kinds of transformer developed with distinct learning sce-\\nnarios currently dominate the ﬁeld: BERT and Generative Pre-\\ntrained Transformer (GPT) models.BERT and its derivatives are\\nbuilt using a complete encoder-decoder transformer, which is\\nﬁne-tuned for downstream NLP tasks.16 The BERT-based model\\narchitecture is task agnostic. There is a need for task-speciﬁc\\ndata sets and task-speciﬁc ﬁne-tuning to achieve optimized\\nmodel performance. However, it could be challenging to\\ncollect task-speciﬁc labeled data, especially in the biomedical\\ndomain. Furthermore, the generalization made under this\\nparadigm can be inadequate because the model is overly\\nspeciﬁc to the training distribution and does not generalize\\nwell outside it. Two kinds of strategy were developed to\\novercome these shortcomings. First, the training material was\\nincreased\\nin\\nsize\\nand\\ndiversity\\nto\\nenhance\\nmodel\\ngeneralizability for different NLP tasks. One example is the\\nRobustly Optimized BERT Pretraining Approach (RoBERTa).109\\nBesides the original BERT training corpus, RoBERTa extends\\nthe\\ntraining\\ncorpus with\\nnews\\nand\\nstories\\ncorpora from\\nCommon Crawl.110 Subsequently, the model outperformed by\\n2–20% both BERT and XLNet on GLUE benchmark results\\nwith a dynamic masking training strategy. Second, adjusted\\ntraining\\nstrategies\\nwere\\ndeveloped\\nto\\nimprove\\nmodel\\nperformance. Examples, including ARBERT,27 ELECTRA,26 and\\nDistillBERT,25 provided lite model architectures without losing\\npredictive model performances.GPT models are autoregressive\\ntransformer-based\\nLMs.\\nThe\\nmodels\\nhave\\na\\ntask-speciﬁc\\nlearning architecture without an intensive ﬁne-tuning process.\\nGPT models used the ‘in-context learning’ concept, in which\\nmodels develop a broad set of skills and pattern recognition\\nabilities at training time and then use those abilities at\\ninference time to adapt to or recognize the desired task rapidly.\\nGPT-3 was released recently, comprising a huge transformer\\nmodel with 175 billion parameters trained with 45 Tb of\\ncompressed plaintext from Common Crawl, plus high-quality\\nreference corpora, such as Wikipedia.111 The GPT-3 model was\\ndemonstrated\\nto\\nachieve\\nbetter\\nstate-of-the-art\\nresults\\nfor\\nvarious NLP tasks with few-shot learners in task-speciﬁc data\\nsets.\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2596\\nwww.drugdiscoverytoday.com\\n\\nlooking for an automated route to prioritize clinical sites and\\npatients. Accordingly, an AI-based contextual-based patient-\\nmatching system might be useful. Drug dossier reviewers might\\nbe more interested in a powerful tool to detect safety signals from\\nclinical documents. For this, AI-powered biomedical named-\\nentity\\nrecognition\\n(NER)\\nand\\nentity\\nrelation\\nextraction\\napproaches could be options. Given the diversity of data and\\n‘needs’, deﬁning a purpose for any data-driven hypothesis\\nbecomes a priority.\\nManage data availability\\nA large body of the text is required to train AI-based language\\nmodels. The current publically available pretrained LMs are\\nmainly trained based on general knowledge, such as books, news,\\nwebpages, social media, and Wikipedia. A few domain-speciﬁc\\nLMs, such as BioBERT14 and ClinicalBERT,19 have been proposed\\nto enhance the clinical application by using publicly available\\nbiomedical literature or deidentiﬁed EHRs. However, labeled data\\nare still required for the ﬁne-tuning process and to enable the\\nmodel ﬁt for downstream tasks. However, the curation of labeled\\ndata is a challenging and time-consuming process, in which sig-\\nniﬁcant domain expertise and knowledge are necessary. Further-\\nmore, data generated across drug discovery and development\\nmight be company sensitive, posing a challenge around data\\nsharing in LM development.20,21 Consequently, it is recom-\\nmended to have a clear view of data availability and the effort\\nrequired to curate labeled data before selecting a suitable AI-\\npowered LM.\\nAlthough data annotation is still the bottleneck in AI-powered\\nLM development, several successful examples exist that can stim-\\nulate interest in the community to accelerate further and pro-\\nmote ‘labeled data’ development in the biomedical ﬁeld. First,\\ncrowd-sourced biomedical labeling could be an effective way to\\ncurate domain-speciﬁc labeling data. The concept of crowd-\\nsourced biomedical labeling aims to outsource biomedical data\\nannotation to a distributed ‘crowd’ of experts worldwide. Some\\nbusiness models, such as Amazon Mechanical Turk, have been\\ndeveloped for this purpose. We recommend establishing a\\nvoluntary-based biomedical labeling consortium to facilitate\\nbiomedical data annotation. Second, a reorganization of publicly\\navailable biomedical corpus would be useful for addressing speci-\\nﬁc BioNLP tasks. For example, the combined different domain-\\nspeciﬁc corpus could be a practical approach to creating anno-\\ntated bioconcept ambiguity data.22 Third, labeling tools could\\nbe a solution to facilitate the manual data curation and annota-\\ntion process. The most common starting point is an Excel/Goo-\\ngle spreadsheet to handle common labeling tasks, such as part-\\nof-speech and named entity recognition labeling. However, this\\ncould be error-prone because typographic errors in transcription\\nare common and the cells and columns are not the most intu-\\nitive way to read a text document. For example, gene name errors\\nare common in scientiﬁc literature, with up to 30% of gene\\nnames misrepresented.23 Some standard labeling tools, such as\\nProdigy, LightTag, TagTog, and Datasaur.ai, provide more stan-\\ndardized solutions for offering customizability and handling\\nadvanced NLP tasks.24\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today\\nFIGURE 2\\nComparison of artificial intelligence (AI)-powered language models and human intelligence: (1) Transfer learning (green); (2) Apply knowledge (blue); and (3)\\nsummarize knowledge (yellow).\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2597\\n\\nKEYNOTE (GREEN)\\nTABLE 1\\nSelected examples of transformer-based language models.\\nArchitectures\\nBERT\\nOpenAI GPT\\nXLNet\\nALBERT\\nRoBERTa\\nELECTRA\\nDistillBERT\\nPre-training\\ncorpus\\nBooksCorpus and\\nEnglish Wikipedia\\nSize: 16 Gb\\n8 million web pages\\nfrom Common Crawl\\nSize: unknown\\nBase: BooksCorpus\\nand English\\nWikipedia\\nSize: 16 Gb\\nLarge: BooksCorpus\\n+ English Wikipedia\\n+ Giga5 + ClueWeb\\n2012-B + Common\\nCrawl\\nSize: 113 Gb\\nBooksCorpus and\\nEnglish Wikipedia\\nSize: 16 Gb\\nBooksCorpus + English\\nWikipedia + Common\\nCrawl news dataset + Web\\ntext corpus + stories from\\nCommon Crawl\\nSize: 160 Gb\\nBooksCorpus and English\\nWikipedia\\nSize: 16 Gb\\nBooksCorpus and English\\nWikipedia\\nSize: 16 Gb\\nModel\\nParameters\\nBase: 110M\\nLarge: 340M\\nGPT-2: 1.5 billion\\nparameters\\nGPT-3: 1.7 billion\\nparameters\\nBase: 110M\\nLarge: 340M\\nBase: 12M\\nLarge: 18M\\nBase: 125M\\nLarge: 355M\\nSmall: 14M\\nBase: 110M\\nLarge: 335M\\nBase: 66M\\nTraining\\nstrategies\\nMasked Language\\nModel (MLM) and\\nNext Sentence\\nPrediction (NSP)\\nprocess the input\\ntext left-to-right,\\npredicting the next\\nword given the\\nprevious context\\nPermutation-based\\nmodeling\\nBERT with reduced\\nparameters for\\nsentence order\\nprediction\\nBERT with a dynamic\\nmasking strategy. Without\\nNext Sentence Prediction\\n(NSP)\\nELECTRA models are trained\\nto distinguish “real” input\\ntokens vs. “fake” input tokens\\ngenerated by another neural\\nnetwork.\\nBERT base model with a\\ndistillation loss function\\nTraining time\\nBase: 12 days with\\n8 Nvidia Tesla V100\\nGPUs\\nLarge: 4 days with\\n64 TPUs or 1 day\\nwith 280 Nvidia\\nTesla V100 GPUs\\nUnknown\\nLarge: 2.5 days with\\n512 TPUs\\nLarge: 1.7 faster\\nthan BERT\\n1 day with 1024 Nvidia\\nTesla V100 GPUs\\nSmall: \\x014 days on a v100\\nGPU\\n3.5 days with 8 Nvidia\\nTesla V100 GPUs\\nFeature\\ndimension\\nBase: up to 768\\nLarge: up to 1024\\nGPT-2: Up to 1024\\nGPT-3: Up to 2048\\nUp to 4096\\nBase: up to 768\\nLarge: up to 1024\\nBase: up to 768\\nLarge: up to 1024\\nSmall: up to 256\\nBase: up to 768\\nLarge: up to 1024\\nBase: up to 768\\nPerformance\\nOutperformed\\nstate-of-the-art in\\n11 NLP tasks\\nGP7-2 Achieves\\nstate-of-the-art\\nresults on 7 out of 8\\ntested language\\nmodeling datasets\\n2%\\x0115%\\nimprovement over\\nBERT\\nLarge: 3%\\nimprovement over\\nBERT\\noutperforms 2%-20% both\\nBERT and XLNet on GLUE\\nbenchmark results\\nSmall: perform roughly in\\nbetween GPT and BERT-Base\\nin terms of GLUE\\nperformance\\nLarge: ELECTRA achieves\\nstate-of-the-art results on the\\nSQuAD dataset\\nretaining 97%\\nperformance of BERT base\\nmodel on GLUE\\nbenchmark results\\nWeblink\\nhttps://\\ngithub.com/google-\\nresearch/bert\\nGPT-2: https://\\ngithub.com/\\nminimaxir/gpt-2-\\nsimple\\nGPT_3: https://\\ngithub.com/openai/\\ngpt-3\\nhttps://github.com/\\nzihangdai/xlnet\\nhttps://\\ngithub.com/google-\\nresearch/ALBERT\\nhttps://github.com/\\npytorch/fairseq/\\ntree/master/examples/\\nroberta\\nhttps://github.com/google-\\nresearch/electra\\nhttps://github.com/\\nhuggingface/transformers\\nReferences\\n13\\n90–91\\n92\\n21\\n89\\n20\\n19\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2598\\nwww.drugdiscoverytoday.com\\n\\nMeasure scalability\\nThe performance improvement of transformer-based LMs results\\nfrom increased data and model size, computational power, or\\ntraining procedures. Comparing similarities and differences\\namong popular AI-powered LMs frameworks (Table 1) is useful\\nto support model selection. First, speed matters in applying AI-\\npowered LMs in different tasks. For example, suppose the AI-\\npowered LM is targeted toward the patient-monitoring process.\\nIn that case, a faster inference speed is set as the highest priority\\nto meet real-time data collection and analysis requirements.\\nThus, the distilled model architecture, such as DistilBERT,25\\nELECTRA,26 and ALBERT,27 might be a reasonable starting point,\\nalthough a few percentage points might compromise the predic-\\ntion\\nperformance.\\nSecond,\\nAI-powered\\nLM\\ndevelopment\\ndemands vast computation power. Larger documents and higher\\nmodel parameters lead to better performance, whereas more\\nTABLE 2\\nSelected examples AI-based NLP applied in drug discovery.\\nNLP task\\nData set\\nAlgorithm\\nNotes\\nSource code\\nRefs\\nTarget identiﬁcation\\nBiomedical Named\\nEntity Recognition\\nBC2GM; BC5CD;\\nNCBI -Disease;\\nJNLPBA\\nBioBERT\\nA multi-ask (MT)-BioNER proposed for\\nbiomedical named entity recognition using\\nBioBERT as shared layers and different data\\nsets in task-speciﬁc layers\\nhttps://\\ngithub.com/cambridgeltl/MTL-\\nBioinformatics-2016\\n30\\nGene–disease\\nrelationship\\nextraction\\nDisGeNET:\\ndatabase of\\ngene–disease\\nassociations\\nConvolution\\nneural network\\n(CNN) and\\nattention-based\\nBiLSTM\\nProposed Deep-GDAE integrates\\nspeciﬁcities of CNN and an attention-based\\nBiLSTM to classify gene–disease\\nassociations\\nhttps://github.com/\\nEsmaeilNourani/Deep-GDAE/\\n33\\nBiomedical text\\nsummarization\\nPubMed\\nBERT and\\nhierarchical\\nclustering\\nalgorithm\\nBiomedical text summarizer (BERT-based-\\nSumm) proposed by interrogating BERT and\\nhierarchical clustering algorithm to extract\\nbiomedical content summarization\\nhttps://github.com/\\nBioTextSumm/BERT-based-\\nSumm\\n34\\nDrug properties\\nprediction\\n1 million SMILE\\ncodes of\\ncompounds in\\nChEMBL\\ndatabase\\nBiLSTM-based\\ntransfer learning\\nTransfer learning framework, MolPMoFiT, to\\npredict physical and biological endpoints,\\nsuch as lipophilicity and blood–brain barrier\\npenetration, for given compounds\\nhttps://github.com/\\nXinhaoLi74/MolPMoFiT\\n40\\nVirtual screening\\nSMILES\\nBERT\\nMOLBERT model proposed by applying\\nBERT model to SMILES for virtual screening\\nhttps://github.com/\\nBenevolentAI/MolBERT\\n41\\nReshape clinical trails\\nPatient–trial matching\\nPatient EHR data\\nClinicalBERT\\nProposed DeepEnroll based on ClinicalBERT\\njointly encodes enrollment criteria and\\npatient EHRs into shared latent space for\\npatient–trial matching\\nhttps://github.com/\\ndeepenroll/DeepEnroll\\n46\\nTrial eligibility criteria\\nPatient EHR data\\nCrOss-Modal\\nPseudO-SiamEse\\nnetwork\\n(COMPOSE)\\nCOMPOSE aims to address challenges for\\npatient–trial matching; one path of network\\nencodes EC using convolutional highway\\nnetwork\\nhttps://github.com/v1xerunt/\\nCOMPOSE\\n112\\nAssist the regulatory process\\nBiomedical entity\\nnormalization\\nClinical notes;\\nPubMed\\nabstract; drug\\nlabeling\\nBERT; BioBERT;\\nClinicalBERT\\nAuthors proposed entity normalization\\narchitecture by ﬁne-tuning pretrained BERT/\\nBioBERT/ClinicalBERT models and applying\\nthem to SNOMED-CT coding, MedDRA\\ncoding, and Medical Subject Headings\\n(MESH) coding\\n–\\n63\\nDisease coding\\nClinical notes\\nand ICD-10\\nBERT\\nAuthors proposed ML model, BERT-XML, for\\nlarge-scale automated ICD coding from EHR\\nnotes\\nhttps://\\ngithub.com/suamin/multilabel-\\nclassiﬁcation-bert-icd10\\n64\\nBiomedical mention\\ndisambiguation\\nCTDbase and\\ngene2pubmed\\nCNN with LSTM\\nAuthors developed biomedical corpus for\\ncurating biomedical terms ambiguous\\nbetween one or more concept types; model\\nis used by interrogating LSTM and CNN\\n–\\n22\\nAdvance postmarketing surveillance\\nADR detection\\nTwitter\\nBidirectional\\nBiLSTM\\nAuthors proposed RNN model using\\npretrained word embeddings created from\\na large, nondomain-speciﬁc Twitter data set\\nfor ADR extraction\\nhttps://github.com/chop-dbhi/\\ntwitter-adr-blstm\\n113\\nTwitter\\nEnsemble models\\nof BERT, BioBERT,\\nand ClinicalBERT\\nAuthors proposed ensemble model by\\nintegrating BERT, BioBERT, and ClinicalBERT\\nfor ADR detection from social media data\\n–\\n75\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2599\\n\\ncomputational power is a prerequisite to retrain these models\\nwith the custom corpus. For example, if the AI-powered LMs\\naim to identify potential adverse events from clinical notes, pre-\\ntrained BioBERT or ClinicalBERT are appropriate options to test\\nthe necessity of developing the model de novo. Third, an ensem-\\nble approach might allow for further performance improvement.\\nFor complicated drug discovery and development tasks, such as\\npatient recruitment, a single model might capture one aspect\\nof complexity, whereas a consensus approach might improve\\npatient matching.\\nAI-powered language models in drug discovery\\nAI offers great potential in drug discovery and development\\n(Table 2). Here, we highlight potential opportunities for AI-\\npowered LMs in different drug development stages and suggest\\npossible directions and solutions for further improvement.\\nOpportunity 1: AI-powered language models to accelerate\\ntarget identiﬁcation\\nTarget identiﬁcation is one of the most crucial steps in the drug\\ndiscovery pipeline to establish the biological origin of disease and\\ndesign appropriate interventions.28 Typically, target identiﬁca-\\ntion involves various considerations from both scientiﬁc and\\neconomic perspectives. The project team, with experts from\\ndiverse disciplines, need to deﬁne the disease area and the\\nexpected therapeutic effects. Then, they need to look for poten-\\ntial biochemical, cellular, or pathophysiological mechanisms tai-\\nlored to the disease. Next, a comprehensive investigation of the\\ntargets involving different tools might be conducted to further\\nprioritize targets for development. Importantly, the prioritized\\ntargets should be competitive in terms of therapeutic efﬁcacy,\\nsafety, and intellectual property perspectives. However, the vast\\narray of information might be widely distributed in the public\\ndomain literature, patent documents, and biomedical databases.\\nIt is often too great a challenge to curate the data manually using\\nconventional simple search-based approaches.\\nAI-powered LMs can advance ﬁndings and accelerate target\\nidentiﬁcation. Automatic biomedical named entity recognition\\n(BioNER) is a practical approach to uncover the hidden relation-\\nship among chemicals, genes, targets, and diseases embedded in\\nfree-text documents.29 Khan et al.30 proposed a multiple-tasking\\nlearning architecture for BioNER using the BioBERT. These\\napproaches outperformed state-of-the-art approaches, such as\\nbidirectional\\nLSTM\\n(BiLSTM),\\nconditional\\nrandom\\nﬁelds\\n(CRF),31 and multitask learning neural network with shared char-\\nacter and word layers (MTM-CW),32 for chemical, disease, and\\ngene entity recognition. Nourani et al.33 developed a hybrid\\ntransfer learning framework (Deep-GDAE) for biomedical associ-\\nation extraction from\\nPubMed literature,\\nwhich integrates\\nattrition-based BiLSTM and a convolutional neural network\\n(CNN) based on feature information extracted from BERT and\\nBioBERT base models. Deep-GDAE yielded a high performance\\n(79.8% of F-measure) for gene–disease relationship extraction.\\nAnother promising application of AI-powered LMs is to summa-\\nrize the essential information from biomedical literature for\\naccelerating target identiﬁcation. Moradi et al.34 applied the\\nBERT base and large models for biomedical text summarization\\nto create a synthetic abstract based on full PubMed articles. The\\napproaches achieved state-of-the-art results; performance could\\nbe further improved by using domain-speciﬁc contextual embed-\\nding from BioBERT.\\nThe concept of transformer-based LM has been leveraged into\\nchemoinformatics to advance drug–target relationship predic-\\ntion.35 The Simpliﬁed Molecular Input Line Entry System\\n(SMILES) is a comprehensive yet straightforward chemical lan-\\nguage in which molecules and reactions can be speciﬁed using\\nASCII characters representing atom and bond symbols. Similarly,\\nFASTA is useful in analyzing protein structure and function\\nbecause it ﬁnds regions of local or global similarity between pro-\\ntein or DNA sequences.36 Inspired by transformer-based pre-\\ntrained LMs, the large body of information in SMILES or FASTA\\nﬁles could be assimilated in the same way that humans do with\\nsentences to grasp the semantics of molecules and their relation-\\nship to downstream tasks. Unlike early attempts at chemical\\nrepresentation based on deep-learning frameworks, such as\\nWod2vec\\nand\\nVariational\\nAutoencoders\\n(VAEs),37,38\\ntransformer-based\\nchemical\\nrepresentation\\nincorporates\\nthe\\nattention mechanism (positional encoding) into the learning\\nprocess to maximize information extraction. One such example\\nis SMILES transformer, which trained 861 000 SMILES from the\\nChEMBL database, a chemical bioassay repository. The learned\\nchemical representations were ﬁne-tuned to different chemo-\\nphysical properties, therapeutic targets, and toxicity predictions.\\nThis\\napproach\\nsigniﬁcantly\\noutperformed\\nconventional\\nﬁngerprint-based strategies.39\\nOther AI frameworks have also been used for virtual screening\\nbased on SMILE sequences. Li et al.40 proposed a transfer learning\\nframework, named Molecular Prediction Model Fine-Tuning\\n(MolPMoFiT), to predict physical and biological endpoints, such\\nas lipophilicity and blood–brain barrier penetration, for the\\ngiven compounds. The proposed MolPMoFiT comprised two\\ncomponents. First, the authors developed a weight-dropped\\nLSTM model based on 1 million SMILE sequences in the\\nChEMBL database to predict the masked atoms in the SMILE\\nsequencing. Second, a trained weight-Dropped LSTM model\\nwas transferred and ﬁne-tuned for downstream tasks. Similarity,\\nFabian et al.41 also adopted the transfer learning framework by\\ntraining the SMILES sequences with BERT, which was then\\napplied for virtual screening of compound-binding afﬁnity of\\n69 individual protein targets.\\nAI-powered LMs have the potential to assess unmet medical\\nneeds and provide prioritized targets for high-throughput screen-\\ning (HTS). The opportunity to accelerate understanding of the\\ncurrent market and potential gaps could facilitate early drug\\ndevelopment planning. However, target identiﬁcation still relies\\non the generation of experimental data; AI-powered LMs have\\nthe potential to promote understanding of the data and to sup-\\nport target identiﬁcation and prioritization.\\nCurrently, AI-\\npowered LM models provide a more informative way to represent\\ntext-based input as an n-dimensional vector or high-level repre-\\nsentation. However, to further improve target identiﬁcation per-\\nformance, ﬁne-tuned models are vital for different downstream\\ntasks. Some more comparative studies and evaluations appeared\\nto set out the pros and cons of AI-powered models compared\\nwith conventional approaches, potentially guiding the ﬁt-for-\\npurpose selections of different AI models.\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2600\\nwww.drugdiscoverytoday.com\\n\\nOpportunity 2: AI-powered language models to reshape clinical\\ntrials\\nClinical trials are resource intensive, accounting for around half\\nof the cost and time in the drug development life cycle, yet there\\nis a high failure rate.42 Unsuccessful clinical trials are attributed\\nto various reasons, some of which are suboptimal patient cohort\\nselection, ineffective patient recruitment strategies, and unso-\\nphisticated patient monitoring systems.43 Diverse text-based\\ndata sets, including electronic health records (EHRs), clinical trial\\ndatabases, trial announcements, eligibility databases, social\\nmedia, and medical literature, provide a unique and immediate\\nentry point for AI-powered LMs to improve clinical trial\\noutcomes.44\\nApproximately 80% of clinical trials in the USA fail to meet\\ntheir timelines on patient recruitment.43 The complexity of\\ninclusion/exclusion criteria in terms of suitability, eligibility,\\nmotivation, and empowerment poses a challenge for patient\\nrecruitment. Poorly matched disease subtypes might make\\npatients\\nunsuitable,\\nand\\ninconsistencies\\nin\\nmedical-history\\nrecording could render suitable patients ineligible. Patient infor-\\nmation is often inconsistent and recorded in an unstructured for-\\nmat, hampering comprehensive patient screening for a given set\\nof inclusion/exclusion criteria.\\nAI-powered LMs enable automation of the patient recruit-\\nment process, alleviating the manual workload burden through\\nadvanced information retrieval and prioritization mechanisms.\\nFirst, AI-powered LMs can learn medical terms and their syn-\\nonyms to retrieve useful information from clinical documents\\nthat might be free-ﬂowing and unstructured. For example, dis-\\nease heterogeneity often hinders the determination of patient\\nsuitability; recurrent models based on the Bidirectional GRU\\narchitecture with contextual embedding could effectively boost\\nmultilabel disease extraction from EHRs.45 This approach has\\nthe potential to stratify patients precisely based on disease sub-\\ntypes for patient recruitment. Second, AI-powered LMs could\\nsynthesize the eligibility criteria into a standardized contextual\\nquery to improve the clinical trial-matching process. One such\\nexample described the use of BERT-based contextual embedding\\nto match eligibility criteria for patient selection.46 Powered by a\\ncross-model learning infrastructure, the proposed DeepEnroll\\ncould jointly encode enrollment criteria and patient EHRs into\\na shared latent space for matching inference. Eventually, the\\nmodel outperformed the rule-based matching strategies, with\\nup to a 12.4% improvement in the F-measure (a measure of the\\naccuracy of a test). Third, AI-powered LMs could be combined\\nwith other emerging technologies seamlessly to expedite patient\\nstratiﬁcation. A combination of EHR data, genomics data, or\\nimage data holds signiﬁcant promise to advance precision\\nmedicine.47,48 AI-powered LMs could be used to boost phenotyp-\\ning capabilities by deep mining from EHRs, registries, hospital\\nrecords, and health insurance data alongside biobank, genomic,\\nand digital phenotyping information. Finally, AI-powered LMs\\nenable higher patient enrollment rates and better site identiﬁca-\\ntion, leading to efﬁcient recruitment of patients. However, site\\nidentiﬁcation is a multifactorial decision. Factors, such as prior\\nexperience of the site for a therapeutic area, connection with\\nhealth nonproﬁts and patient advocates, historical data of\\npatient retention, and cost-effectiveness are signiﬁcant contribu-\\ntors. AI-powered LMs could be leveraged to support clinical\\ndecision-making by considering these diverse factors, allowing\\nfor a balanced decision and the best possibility of success.\\nSuccessful completion of clinical trials justiﬁes the massive\\ninvestment in patient recruitment, but the average patient drop-\\nout rate across all clinical trials is ~ 30%.44 Efforts to overcome\\nchallenges in clinical trial recruitment and retention continue.\\nThese efforts could safeguard the well-being of trial participants,\\nensure adherence to trial rules and procedures, enhance compli-\\nance and retention, collect reliable and high-quality trial data\\npoints, and improve real-world outcome monitoring.49 AI-\\npowered LMs, as a combination of ML and digital technologies,\\ncould have an essential role in enhancing patient monitoring\\ntoward a lower dropout rate and a more efﬁcient data uptake\\nframework.\\nDigital health technologies, such as wearables, voice tech-\\nnologies, and computer vision, make remote patient monitoring\\npossible.50 These emerging technologies also enable the collec-\\ntion of longitudinal and real-time biometric data sets to provide\\nunique insights into the long-term, real-world impact of pharma-\\ncotherapies and treatment protocols. Meanwhile, the implemen-\\ntation of such technologies could relieve patients of their more\\nburdensome tasks during the clinical trial and increase their\\nadherence. More importantly, AI and ML (particularly deep-\\nlearning models) could be used to carry out real-time patient\\nmonitoring for detecting and logging relevant information.51,52\\nFor example, powered by AI, voice assistants have been gradually\\nadopted in clinical trials for various tasks, including reminding\\npatients of appointments, recording patient diaries, fostering col-\\nlaboration between site investigators and sponsors, and increas-\\ning physician awareness.53\\nAlthough AI-powered LMs, along with digital technologies,\\nhave the potential to transform clinical trials, most of the inter-\\nventions to date have yet to deliver on that potential. Debate\\nover the adoption of AI and mobile platforms in clinical trials\\nis ongoing.54 Regulatory guidance is urgently needed to leverage\\nthese promising tools and technologies to advance clinical trials.\\nTo ﬁll the gap, the US Food and Drug Administration (FDA)\\nannounced a new strategic framework to promote the use of\\nreal-world evidence (RWE) to support the development of drugs\\nand biologicals.55 On the other side, to gain the trust of patients\\nin AI and digital technologies, the beneﬁt of RWE should be ver-\\niﬁed and communicated. One Community of Patients for\\nResearch (ComPaRe) study on patient uptake of wearable devices\\nand AI showed that only 20% of participants considered that the\\nbeneﬁts of the technology greatly outweighed the dangers.56 Fur-\\nthermore, the authors found that 35% of patients would refuse\\nto integrate at least one existing or soon-to-be-available interven-\\ntion using biometric monitoring devices (BMDs) and AI-based\\ntools into their care.\\nOpportunity 3: AI-powered language model to assist the\\nregulatory process\\nRegulatory submissions comprise a dossier of documents sent by\\npharmaceutical companies to health regulatory agencies as evi-\\ndence of compliance. The process includes many laws, regulatory\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2601\\n\\nrequirements, and regulatory guidance, which help deﬁne how\\ndrug companies manufacture their drugs, design clinical trials,\\nreport safety ﬁndings, and create promotional material. The\\nFDA promotes regulatory submissions of standardized study data\\nin electronic format for investigational new drugs (INDs), new\\ndrug applications (NDAs), and biologics license applications\\n(BLAs). For example, the FDA Data Standards Catalog indicates\\nthat these data sets should be formatted following the Clinical\\nData\\nInterchange\\nStandards\\nConsortium\\n(CDISC)\\nStandard\\nExchange for Nonclinical Data (SEND).57 These standards cur-\\nrently support single-dose general toxicology, repeat-dose gen-\\neral toxicology, and carcinogenicity studies.\\nMeanwhile, the FDA has an internal database to maintain and\\norganize submission in a data warehouse, such as its Document\\nArchiving,\\nReporting,\\nand\\nRegulatory\\nTracking\\nSystem\\n(DARRTS).58 Essential regulatory documents are required to pro-\\nceed through the drug development life cycle. These documents,\\nsuch as regulatory guidance, IND safety reports, NDA/BLA sub-\\nmissions, patient narratives, drug labeling, and FDA Adverse\\nEvent Reporting System (FAERS), are a rich source of informa-\\ntion. AI-powered LMs offer an unprecedented opportunity for\\nmedical ofﬁcers and others who ensure that drugs are safe and\\neffective in supporting RWE generation for regulatory decision\\nmaking and better patient outcomes. The FDA promotes devel-\\noping knowledge management systems to enable better leverag-\\ning of AI to advance NLP in the regulatory process.59 Although\\nNLP-derived clinical evidence has not yet been included in regu-\\nlatory submission documents, it is time to consider how this\\ncould be enabled without disrupting assumptions of data integ-\\nrity and acceptance in the future.\\nA standardized medical terminology could accurately repre-\\nsent medical knowledge stored in regulatory documents for efﬁ-\\ncient,\\nevidence-based\\ndecision\\nmaking,\\nand\\noptimal\\ncommunication among stakeholders.60 Standardized terminolo-\\ngies have been assigned and regulated by health agencies.61\\nThe suggested coding systems were recommended for different\\ndomains, such as the International Classiﬁcation of Diseases\\n(ICD) for disease, WHO Anatomical Therapeutic Chemical\\n(ATC) Classiﬁcation for medicine, the Systematized Nomencla-\\nture of Medicine–Clinical Terms (SNOMED-CT) for diagnoses,\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nTABLE 3\\nPublicly available FDA documents for promoting AI-powered LMs in regulatory applications.\\nData sets\\nDescriptions\\nPotential use in LMs\\nURL of data ﬁles\\nDrug labeling\\nDrug labeling comprises a summary of information\\nfor safe and effective use of the drug, which is\\nproposed by manufacturer and approved by FDA\\nDrug labeling could be a useful resource\\n(>120 000 product labelings) to develop\\nbiomedical named-entity recognition/\\nnormalization, and relation extraction between\\ndrug and AEs, drug–drug interaction, etc.\\nhttps://dailymed.nlm.\\nnih.gov/dailymed/spl-\\nresources-all-drug-\\nlabels.cfm\\nFAERS\\nFAERS is a database that contains information on\\nAE and medication error reports submitted to FDA\\nFAERS is designed to support the post-\\nmarketing safety surveillance program for\\ndrug and therapeutic biologic products of the\\nFDA. There are more than 19 million case\\nreports in FAERS; AI-powered LMs could be\\napplied to carry out AE detection, causal\\nrelationship extraction, etc.\\nhttps://ﬁs.fda.gov/\\nextensions/FPD-QDE-\\nFAERS/FPD-QDE-FAERS.\\nhtml\\nOrange Book\\nOrange Book identiﬁes drug products approved on\\nbasis of safety and effectiveness by FDA under the\\nFederal Food, Drug, and Cosmetic Act and related\\npatent and exclusivity information\\nOrange book provides crucial regulatory\\ninformation, such as biological equivalence,\\nreference listed drug (RLD), Reference\\nStandard (RS), and patent status. This\\ninformation could be included in AI-powered\\nLMs to compare drug product information\\nwith RLD and RS to facilitate abbreviated new\\ndrug application (ANDA) submissions\\nwww.\\nfda.gov/drugs/drug-\\napprovals-and-\\ndatabases/orange-book-\\ndata-ﬁles\\nDrugs@FDA\\nDrugs@FDA includes most drug products approved\\nsince 1939. Most patient information, labels,\\napproval letters, reviews, and other information are\\navailable for drug products approved since 1998\\nDrugs@FDA provides rich information on drug\\napproval history, which could be used as AI-\\npowered LMs to explore underlying reasons\\nfor labeling changes and increase business\\nsuccess\\nwww.\\nfda.gov/drugs/drug-\\napprovals-and-\\ndatabases/drugsfda-\\ndata-ﬁles\\nFDA Guidance\\nDocuments\\nGuidance documents describe FDA’s interpretation\\nof policy on a regulatory issue (21 CFR 10.115(b)).\\nThese documents usually discuss more speciﬁc\\nproducts or issues that relate to design, production,\\nlabeling, promotion, manufacturing, and testing of\\nregulated products\\nFDA Guidance Documents could be useful to\\nimplement AI-powered LMs for standardizing\\nand monitoring crucial steps in drug discovery\\nand development in terms of their consistency\\nand alignment with regulatory requirements\\nwww.fda.gov/\\nregulatory-information/\\nsearch-fda-guidance-\\ndocuments\\nFDA Acronyms and\\nAbbreviations\\nFDA Acronyms and Abbreviations database\\nprovides a quick reference to acronyms and\\nabbreviations related to FDA activities\\nEmphasis of FDA Acronyms and Abbreviations\\nis on scientiﬁc, regulatory, government\\nagency, and computer application terms. The\\ndatabase includes some FDA organizational\\nand program acronyms. It is a useful resource\\nto deﬁne vocabularies in AI-powered LMs and\\nincrease model generalization\\nwww.accessdata.\\nfda.gov/scripts/cder/\\nacronyms/index.cfm\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2602\\nwww.drugdiscoverytoday.com\\n\\nHealth Level 7 (HL7) for messaging, and Medical Dictionary for\\nRegulatory Activities (MedDRA) for adverse events.62 However,\\nthe uptake of standardized medical terminologies in regulatory-\\nrelated documents is a time-consuming and labor-intensive task.\\nAI-powered LMs could facilitate the coding of regulatory docu-\\nments for more efﬁcient and effective review, delivery, and infor-\\nmation recall.\\nBiomedical named-entity normalization aims to identify\\nbiomedical entities from documents and further link detected\\nentities to their corresponding concepts in a given knowledge\\nbase or ontology. Ji et al.63 proposed a BERT-based ranking model\\nfor biomedical entity normalization for SNOMED-CT coding,\\nMedDRA coding, and Medical Subject Headings (MESH) coding.\\nThe model adopted domain-specialized BERT architectures,\\nincluding BioBERT and ClinicalBERT, and yielded a superior per-\\nformance compared with state-of-the-art approaches without\\nany prior knowledge of medical terminology. BERT-XML was\\nalso developed for large-scale automated ICD coding from\\nEHRs.64 Notably, the authors trained the BERT model de novo\\non EHR notes with multilabel attention for better clinical vocab-\\nulary identiﬁcation. Thus, the proposed model outperformed\\nreported models and further demonstrated that the domain-\\nspeciﬁc BERT model could improve the performance of down-\\nstream tasks.\\nApparent ambiguity between different bioconcept types is a\\npotential obstacle for automated bio-NER method development.\\nThis ambiguity exists within the particular domains and across\\nother biological concepts. Abbreviation ambiguity means that\\none entity could map to multiple bioconcepts. For example,\\nthe abbreviation ‘BD’ could not only represent Binswanger’s dis-\\nease, but also Behçet’s disease; this example is relatively easy to\\nresolve.65 However, some ambiguities across bioconcepts are\\nchallenging for automated bio-NER methods. For example,\\nCO2 could mean carbon dioxide in chemicals and cytochrome\\nc oxidase subunit 2 gene. Bio-NER is developed based on the\\ndifferent standardized corpus in various domains. The uniﬁed\\nbiomedical corpus could be a potential solution for biomedical\\nmention disambiguation. For instance, Wei et al. developed a\\nbiomedical\\ncorpus for\\ncurating biomedical\\nterms\\nthat\\nare\\nambiguous between one or more concept types. Using the\\nensemble model by interrogating LSTM and CNN, the model\\ncould\\nachieve\\nF1-scores\\nof\\n91.94%\\n(micro-averaged)\\nand\\n85.42% (macro-averaged) for ambiguous entity identiﬁcation,\\noutperforming the transformer models, such as BioBERT.22 Thus,\\nwe recommend further efforts to standardize bioconcepts for\\nenhancing the performance of automated bio-NER methods.\\nThe decision-making process is tied to the regulatory frame-\\nwork, which yields consensual results by integrating different\\ndata sets. Medical ofﬁcers are required to not only review the sub-\\nmission documents, but also take account of historical data and\\nrelated documents to generate evidence and support decision-\\nmaking, which is a complicated and time-consuming process.\\nThe\\ncurrent\\nregulatory-related\\ndatabases\\nare\\nindependently\\nindexed and maintained, with no interconnection between each\\nother. More importantly, the indexing strategy is mainly identity\\nbased, and there exist no semantic relations of entity–entity and\\ndocument–document types. Reviewers must move from one\\ndatabase to another to collect the relevant information. Powered\\nby AI, a semantic search engine might be a potential solution to\\nimprove the effectiveness of information retrieval for extracting\\nthe most relevant documents for reviewer convenience.66\\nUnlike a lexical search, in which the search engine looks for\\nliteral matches of query words or variants of them, semantic\\nsearch can search and rank the relevance with meaning.67 Some\\nearly attempts at biomedical question & answering (Q&A) sys-\\ntems, such as BioBERT, have given rise to a new direction.14\\nBuilding on this, a publicly available evaluation infrastructure\\nfor biomedical semantic indexing and Q&A was developed to\\nevaluate the performance of the developed semantic search\\nengines.68 To take full advantage of the semantic search engine\\nin the regulatory arena, we strongly suggest that regulators\\nshould work with the community to develop a regulatory-\\nbased semantic search engine that would assist the review of reg-\\nulatory dossiers. To facilitate this, we list the regulatory data sets\\nthat are open to the public in support of these efforts (Table 3).\\nOpportunity 4: AI-powered model to advance postmarketing\\nsurveillance\\nPostmarket surveillance refers to the process of drug safety mon-\\nitoring once drugs have reached the market, and is an essential\\npart of the science of pharmacovigilance.69 The primary purpose\\nof postmarket surveillance is to further reﬁne, conﬁrm, or refute\\nthe safety of a drug or device after use in the general population\\nwith a variety of medical conditions. Postmarket surveillance\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today\\nFIGURE 3\\nArtificial intelligence (AI)-powered language models for accelerating\\nCoronavirus 2019 (COVID-19) treatment development. Potential opportuni-\\nties, data resources, and key questions are illustrated. Abbreviation: CDC,\\nCenters for Disease Control and Prevention.\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2603\\n\\ndata are primarily derived from: (i) spontaneous/voluntary\\nreporting of cases (e.g., FAERS, Local or Regional Joint Commis-\\nsion Requirement) and scientiﬁc literature publications; (ii)\\nobservational\\nstudies,\\nincluding\\nautomated\\nhealthcare\\ndatabases/social media, and randomized clinical trials; and (iii)\\nactive surveillance, such as the Drug-Induced Liver Injury Net-\\nwork (DILIN)70 and FDA Sentinel initiative.71 These real-world\\ndata (RWD) and RWE data sets are having an increasing role in\\nhealthcare decisions and are adopted by the FDA to monitor\\npostmarket safety and AEs and to make regulatory decisions.72\\nThe safety data accumulated in the postmarket stage provide\\nan excellent resource for AI to deeply mine safety signals and\\nadvance pharmacovigilance.\\nAI-powered LMs have also been shown to be useful in improv-\\ning the detection of drug–AE associations and in deciphering the\\ncausal relationship between the AE and clinical parameters.73,74\\nSocial media has gradually become one of the major resources\\nfor adverse drug reaction (ADR) monitoring. Breden et al.75 pro-\\nposed an ensemble model by integrating the BERT-large model,\\nBioBERT, and ClinicalBERT to generate an enhanced automatic\\nADR detection within Twitter tweets. Relation extraction from\\nclinical notes is a practical approach to detect the causality rela-\\ntionship between AE and relations. Guan et al.76 combined the\\nBERT model and Edge sampling to identify ADR and disease rela-\\ntionships from Electronic Health Records 1.0 (MADE) with\\nimproved performance. This developed model could be used to\\nextract causal relationships from unstructured documents.\\nUser cases: AI-powered model to combat emerging\\ninfections\\nEmerging infectious diseases have been an ever-present threat to\\npublic health, and COVID-19 is a recent example.77 At the time\\nof this review, COVID-19 had infected more than 31 million peo-\\nple, killed over 961 000, and resulted in catastrophic social and\\neconomic losses. Global efforts have been put into the develop-\\nment of effective treatments to combat this devastating deadly\\ndisease. Unfortunately, there are still no approved drugs or vac-\\ncines.78 Encouragingly, AI is proving invaluable in the battle\\nagainst the coronavirus pandemic. Here, we illustrate how AI-\\npowered LMs could aid the development of treatments for\\nCOVID-19 (Fig. 3).\\nCOVID-19 search engine\\nCOVID-19 may be the hottest topic in the scientiﬁc arena at this\\nmoment, resulting in more than 20,000 papers published in\\n2020 alone. The number is still increasing exponentially, and\\nan average of 300 articles are being published every day. The pub-\\nlished literature is a rich resource for promoting the development\\nof COVID-19 treatments. However, there are far too many publi-\\ncations for any researcher to read. Some ﬁrst efforts (CORD-19\\ndata\\nset)\\nhave\\nbeen\\nmade\\nto\\ncreate\\nthe\\nmost\\nextensive\\nmachine-readable coronavirus literature collection of COVID-\\n19 available for data mining to date.79 The AI-powered COVID-\\n19 search engine is a great solution to help researchers navigate\\nthe scientiﬁc literature for addressing different questions. More\\nthan 50 search and discovery tools have been developed and\\nused for various topics, such as drug repurposing, interaction\\nwith other diseases, infection, mortality by demographic, and\\nmanagement policies.80 These AI-powered search engines allow\\nresearchers to ask speciﬁc questions, such as ‘what approved drug\\ncould potentially treat COVID-19?’.\\nSafer repurposing candidates\\nDrug repositioning and repurposing is being promoted as a rapid\\ndrug development paradigm for COVID-19 therapy.81–84 Some\\nrepurposing\\ncandidates,\\nincluding\\nchloroquine\\nand\\nhydroxychloroquine, were initially authorized by the FDA for\\nhospitalized\\npatients\\nonly\\nunder\\ncareful\\nheart\\nmonitoring\\nbecause of the risk of heart rhythm problems.85,86 The use of\\nthese two drugs for COVID-19 has now been revoked because\\nthe evidence suggests that they are unlikely to be an effective\\ntreatment. The potential risk of experiencing QTc prolongation\\nwith chloroquine or hydroxychloroquine was included in the\\nFDA-approved drug label for their original indications.86 AI-\\npowered LMs could be applied to extract the relationship\\nbetween repurposed drugs and their potential AEs to prioritize\\nrepurposing candidates regarding their safety proﬁles.\\nFurthermore, COVID-19 has affected vulnerable populations\\nand patients with pre-existing conditions disproportionately\\nacross the world.87 Patients with COVID-19 and pre-existing\\nconditions and older patients have a high probability of encoun-\\ntering drug–drug interactions (DDIs) because they are more likely\\nto take multiple medications.88 Therefore, caution should be\\nconsidered before prescribing COVID-19 therapy to vulnerable\\npopulations. AI-powered models could be used to extract poten-\\ntial DDIs between COVID-19 repurposing candidates and other\\nmedicines for prevention.89\\nClinical trial optimization\\nTreating patients with COVID-19 is forcing doctors to make hard\\ndecisions between two equally unattractive options: (i) prescribe\\ndrugs off-label in the hope that there will be some beneﬁt; or (2)\\ntreat patients with standard supportive care for severe respiratory\\ndisease. This will continue to be the case until a conﬁrmed ran-\\ndomized controlled trial establishes an effective treatment. Based\\non statistics from clinicaltrial.gov, there are currently more than\\n2900 clinical trials related to COVID-19. The number of enrolled\\npatients, age groups, and demographic distributions within the\\nclinical trials are highly variable, with the potential for contro-\\nversies among trial sponsors. For example, disputes regarding\\ntreatment or preventative effects of hydroxychloroquine were\\nreported based on different clinical trials.79,90–92 A randomized\\ncontrolled trial is currently underway in dozens of hospitals\\naround the world proposed by REMAP-CAP using AI to guide\\nresearchers toward the most effective treatments for COVID-\\n19.93\\nConcluding remarks\\nAI-powered LMs have enormous potential to transform every\\nstep of the drug discovery and development pipeline. As such,\\nwe expect different stakeholders to implement more investiga-\\ntion and real-world applications. We have illustrated the poten-\\ntial opportunities of AI-powered models in drug discovery and\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2604\\nwww.drugdiscoverytoday.com\\n\\ndevelopment, focusing on the role of AI-powered LMs for accel-\\nerating target identiﬁcation, optimizing clinical trials, facilitating\\nregulatory decision-making, and enhancing pharmacovigilance.\\nMoreover, we highlighted how AI-powered LMs could promote\\ntreatment development in combating the COVID-19 pandemic.\\nHowever, the implementation of AI-powered LMs in drug discov-\\nery is still in its infancy. Furthermore, besides AI-powered LM,\\nother AI-based models been proposed and show promise in tack-\\nling different drug discovery and development questions. These\\nare out of the scope of the current review, but we recommended\\na closer look at other AI-based models, which might combine\\nwith AI-power language models to enhance drug discovery and\\ndevelopment.94–96\\nAI-powered LMs are a fast-evolving ﬁeld, and many model\\narchitectures have been proposed. However, most of the applica-\\ntions in drug discovery and development are based on BERT and\\nits derivates. Other newly developed LMs have claimed superior\\nperformance and strength based on evaluation data in the gen-\\neral domain. The utility of these transformer-based LMs in drug\\ndiscovery and development remains to be established via further\\ninvestigation and critical evaluation. To carry out a comprehen-\\nsive assessment of different transformer-based LMs for various\\ntasks in drug discovery and development, more standard bench-\\nmark data sets in the biomedical domain, such as BioASQ68 and\\nBiomedical Language Understanding Evaluation (BLUE),97 are\\nurgently needed.\\nThe beneﬁt of learning domain-speciﬁc corpus and knowledge\\nhas been demonstrated for LMs.14,19,98 However, these models\\nhave been retrained on top of the BERT-base model. Improved\\nmodel performance is expected by using the BERT large model.\\nFurthermore,\\nwe\\nstrongly\\nrecommend\\nthese\\nadvanced\\ntransformed-based LMs be retrained by using other regulatory-\\nrelated documents to enhance their application in the regulatory\\nprocess. Moreover, novel model architecture, such as GPT-3,\\nshowed potential in tackling downstream tasks without task-\\nrelated ﬁne-tuning data sets. A further investigation of biomedi-\\ncal applications should be conducted if a favorable performance\\nis obtained, which could expand this utility in drug discovery\\nand development.\\nBeing able to explain how AI-powered LMs could be used in\\ndrug discovery and development is vital to building trust. Large\\nLMs can produce powerful contextual representations that lead\\nto improvements across many NLP tasks. Our ability to explore\\nthe biological relevance of these contextual representations will\\nenhance adoption in the drug discovery and development pro-\\ncess. Initial efforts, such as ExBERT, have been proposed to pro-\\nvide\\ninsights\\ninto\\nthe\\nmeaning\\nof\\nthe\\ncontextual\\nrepresentations by matching a human-speciﬁed input to similar\\ncontexts in a large, annotated data set.99 More in-depth efforts\\nto develop explainable transformers would be beneﬁcial.\\nAI faces challenges in reproducibility because researchers have\\ndifﬁculty reproducing many vital results, hindering their real-\\nworld applications.100 Few efforts have been made to explore\\nthe reproducibility of AI-powered LMs.101 Some consortium\\nefforts, such as the Kaggle challenge, might be a suitable platform\\nto organize a comprehensive assessment of the reproducibility of\\nAI-powered LMs with biomedical data, such as EHRs or PubMed\\nliterature.\\nAI-powered LMs have been applied in many different areas of\\nbiomedical science. The impact of AI-powered models in all areas\\nof drug discovery and healthcare is already noticeable, especially\\nin transforming clinical trial design. Conventional NLP, along\\nwith rule-based matching strategies, has also been extensively\\napplied in drug discovery and development.7,102 We believe the\\nAI-based\\nLMs\\ncould\\nbe\\ncomplementary\\nto\\nconventional\\napproaches to promote drug discovery and development. Here,\\nwe have summarized the challenges and opportunities presented\\nby AI-powered LMs to stimulate community efforts for further\\nevaluation and to better position and promote AI-powered LMs\\nin drug discovery and development.\\nDeclaration of interests\\nR.A.R. is co-founder and co-director of ApconiX, an inte-\\ngrated toxicology and ion channel company that provides\\nexpert advice on nonclinical aspects of drug discovery and drug\\ndevelopment\\nto\\nacademia,\\nindustry,\\nand\\nnot-for-proﬁt\\norganizations.\\nAcknowledgments\\nX.C. and R.A.R. are grateful to the National Center for Toxico-\\nlogical Research (NCTR) for support through the Oak Ridge Insti-\\ntute for Science and Education (ORISE) via an interagency\\nagreement between the Department of Energy and the FDA. R.\\nH. is grateful for participation through part of the Tox21 project.\\nThe views presented in this article do not necessarily reﬂect the\\nopinions of the FDA or the NIH. Any mention of commercial\\nproducts is for clariﬁcation and is not intended as an endorse-\\nment. The work was funded and supported by the FDA Medical\\nCountermeasures Initiative (MCMi).\\nReferences\\n1 A.I.\\nbreakthroughs\\nin\\nnatural-language\\nprocessing\\nare\\nbig\\nfor\\nbusiness.\\nwww.fortune.com/2020/01/20/natural-language-processing-business/ [Accessed\\nJune 24, 2021].\\n2 Z. Liu, L. Zhu, R. Roberts, W. Tong, Toward clinical implementation of next-\\ngeneration sequencing-based genetic testing in rare diseases: where are we?,\\nTrends Genet 35 (2019) 852–867.\\n3 Y. Shi, H. Inoue, J.C. Wu, S. Yamanaka, Induced pluripotent stem cell\\ntechnology: a decade of progress, Nat Rev Drug Discov 16 (2017) 115–130.\\n4 P. Schneider, W.P. Walters, A.T. Plowright, N. Sieroka, J. Listgarten, R.A.\\nGoodnow Jr, et al., Rethinking drug design in the artiﬁcial intelligence era,\\nNat Rev Drug Discov 19 (2020) 353–364.\\n5 B. Chen, L. Garmire, D.F. Calvisi, M.-S. Chua, R.K. Kelley, X. Chen, Harnessing\\nbig ‘omics’ data and AI for drug discovery in hepatocellular carcinoma, Nat Rev\\nGastroenterol Hepatol 17 (2020) 238–251.\\n6 S. Shilo, H. Rossman, E. Segal, Axes of a revolution: challenges and promises of\\nbig data in healthcare, Nature Medicine 26 (1) (2020) 29–38.\\n7 P. Agarwal, D.B. Searls, Literature mining in support of drug discovery, Brieﬁngs\\nin Bioinformatics 9 (6) (2008) 479–492.\\n8 J. Vamathevan, D. Clark, P. Czodrowski, I. Dunham, E. Ferran, G. Lee, et al.,\\nApplications of machine learning in drug discovery and development, Nature\\nReviews Drug Discovery 18 (6) (2019) 463–477.\\n9 Vaswani A, Shazeer N, Parmar N, Uszkoreit J, Jones L, Gomez AN, et al. Attention\\nis all you need. arXiv preprint 2017; arXiv:1706.03762v5.\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2605\\n\\n10 Sutskever I, Vinyals O, Le QV. Sequence to sequence learning with neural\\nnetworks. arXiv preprint 2014: arXiv;1409.3215.\\n11 The\\nfall\\nof\\nRNN/LSTM.\\nwww.towardsdatascience.com/the-fall-of-rnn-lstm-\\n2d1594c74ce0. [Accessed June 24, 2021].\\n12 Salehinejad H, Sankar S, Barfett J, Colak E, Valaee S. Recent advances in recurrent\\nneural networks. arXiv preprint 2017; arXiv:1801.01078.\\n13 G.I. Parisi, R. Kemker, J.L. Part, C. Kanan, S. Wermter, Continual lifelong\\nlearning with neural networks: a review, Neural Networks 113 (2019) 54–71.\\n14 J. Lee, W. Yoon, S. Kim, D. Kim, S. Kim, C.H. So, et al., BioBERT: a pre-trained\\nbiomedical\\nlanguage\\nrepresentation\\nmodel\\nfor\\nbiomedical\\ntext\\nmining,\\nBioinformatics 36 (4) (2019) 1234–1240.\\n15 Alsentzer E, Murphy JR, Boag W, Weng, W-H, Jin D, Naumann T, et al. Publicly\\navailable clinical BERT embeddings. arXiv preprint 2019; arXiv:1904.03323.\\n16 Devlin J, Chang M-W, Lee K, Toutanova K. BERT: pre-training of deep\\nbidirectional transformers for language understanding. arXiv preprint 2018;\\narXiv:1810.04805.\\n17 Liu PJ, Saleh M, Pot E, Goodrich B, Sepassi R, Kaiser L et al. Generating Wikipedia\\nby summarizing long sequences. arXiv preprint 2018; arXiv:1801.10198.\\n18 Kitaev N, Kaiser Ł, Levskaya A. Reformer: the efﬁcient transformer. arXiv\\npreprint 2020; arXiv:2001.04451.\\n19 Huang K, Altosaar J, Ranganath R. Clinicalbert: modeling clinical notes and\\npredicting hospital readmission. arXiv preprint 2019; arXiv:1904.05342.\\n20 Sanz F, Pognan F, Steger-Hartmann T, Díaz C; eTOX, Cases M et al. Legacy data\\nsharing to improve drug safety assessment: the eTOX project. Nature Reviews\\nDrug Discovery 2017; 16(12): 811-812.\\n21 C. Harrison, GlaxoSmithKline opens the door on clinical data sharing, Nature\\nReviews Drug Discovery 11 (12) (2012) 891–892.\\n22 Wei C-H, Lee K, Leaman R, Lu Z. Biomedical mention disambiguation using a\\ndeep learning approach. arXiv preprint 2019; arXiv:1909.10416v1\\n23 M. Ziemann, Y. Eren, A. El-Osta, Gene name errors are widespread in the\\nscientiﬁc literature, Genome Biology 17 (1) (2016) 177.\\n24 Data labeling for natural language processing. www.topbots.com/data-labeling-\\nfor-natural-language-processing/. [Accessed June 24, 2021].\\n25 Sanh V, Debut L, Chaumond J, Wolf T. DistilBERT, a distilled version of BERT:\\nsmaller, faster, cheaper and lighter. arXiv preprint 2019; arXiv:1910.01108.\\n26 Clark K, Luong M-T, Le QV, Manning CD. Electra: pre-training text encoders\\nas discriminators rather than generators. arXiv preprint 2020; arXiv:2003.\\n10555.\\n27 Lan Z, Chen M, Goodman S, Gimpel K, Sharma P, Soricut R. Albert: A lite Bert for\\nself-supervised learning of language representations. arXiv preprint 2019;\\narXiv:1909.11942.\\n28 M. Schenone, V. Dan\\x01cík, B.K. Wagner, P.A. Clemons, Target identiﬁcation and\\nmechanism of action in chemical biology and drug discovery, Nature Chemical\\nBiology 9 (4) (2013) 232–240.\\n29 J.M. Giorgi, G.D. Bader, Towards reliable named entity recognition in the\\nbiomedical domain, Bioinformatics 36 (1) (2020) 280–286.\\n30 Khan MR, Ziyadi M, AbdelHady M. MT-BioNER: multi-task learning for\\nbiomedical named entity recognition using deep bidirectional transformers.\\narXiv preprint 2020; arXiv:2001.08904.\\n31 Z. Liu, H. Fang, J. Borlak, R. Roberts, W. Tong, In vitro to in vivo extrapolation for\\ndrug-induced liver injury using a pair ranking method, ALTEX 34 (3) (2017)\\n399–408.\\n32 X. Wang, Y. Zhang, X. Ren, Y. Zhang, M. Zitnik, J. Shang, et al., Cross-type\\nbiomedical\\nnamed\\nentity\\nrecognition\\nwith\\ndeep\\nmulti-task\\nlearning,\\nBioinformatics 35 (10) (2018) 1745–1752.\\n33 E. Nourani, V. Reshadat, Association extraction from biomedical literature based\\non representation and transfer learning, Journal of Theoretical Biology 488\\n(2020) 110112.\\n34 M. Moradi, G. Dorffner, M. Samwald, Deep contextualized embeddings for\\nquantifying\\nthe\\ninformative\\ncontent\\nin\\nbiomedical\\ntext\\nsummarization,\\nComputer Methods and Programs in Biomedicine 184 (2020) 105117.\\n35 Wang S, Guo Y, Wang Y, Sun H, Huang J. SMILES-BERT: large scale unsupervised\\npre-training for molecular property prediction. In Proceedings of the 10th ACM\\nInternational Conference on Bioinformatics, Computational Biology and Health\\nInformatics, New York: Association for Computing Machinery; 2019: 429–436\\n36 What\\nis\\nFASTA\\nformat?\\nhttps://zhanglab.dcmb.med.umich.edu/FASTA/\\n[Accessed June 24, 2021].\\n37 S. Jaeger, S. Fulle, S. Turk, Mol2vec: unsupervised machine learning approach\\nwith chemical intuition, Journal of Chemical Information and Modeling 58 (1)\\n(2018) 27–35.\\n38 R. Winter, F. Montanari, F. Noé, D.-A. Clevert, Learning continuous and data-\\ndriven molecular descriptors by translating equivalent chemical representations,\\nChemical Science 10 (6) (2019) 1692–1701.\\n39 Honda S, Shi S, Ueda HR. SMILES Transformer: pre-trained molecular ﬁngerprint\\nfor low data drug discovery. arXiv preprint 2019: arXiv:1911.04738.\\n40 X. Li, D. Fourches, Inductive transfer learning for molecular activity prediction:\\nnext-gen QSAR models with MolPMoFiT, Journal of Cheminformatics 12 (1)\\n(2020) 27.\\n41 Fabian B, Edlich T, Gaspar H, Segler M, Meyers J, Marco Fiscato M, et al.\\nMolecular representation learning with language models and domain-relevant\\nauxiliary tasks. arXiv preprint 2020: arXiv:2011.13230.\\n42 J.W. Scannell, A. Blanckley, H. Boldon, B. Warrington, Diagnosing the decline in\\npharmaceutical R&D efﬁciency, Nature Reviews Drug Discovery 11 (3) (2012)\\n191–200.\\n43 D.B. Fogel, Factors associated with clinical trials that fail and opportunities for\\nimproving the likelihood of success: a review, Contemp Clin Trials Commun 11\\n(2018) 156–164.\\n44 S. Harrer, P. Shah, B. Antony, J. Hu, Artiﬁcial intelligence for clinical trial design,\\nTrends in Pharmacological Sciences 40 (8) (2019) 577–591.\\n45 A. Blanco, O. Perez-de-Viñaspre, A. Pérez, A. Casillas, Boosting ICD multi-label\\nclassiﬁcation\\nof\\nhealth\\nrecords\\nwith\\ncontextual\\nembeddings\\nand\\nlabel-\\ngranularity, Computer Methods and Programs in Biomedicine 188 (2020)\\n105264.\\n46 Zhang X, Xiao C, Glass LM, Sun J. DeepEnroll: patient-trial matching with deep\\nembeddingand entailment prediction. arXiv preprint 2020; arXiv:2001.08179.\\n47 J.L. Hall, J.J. Ryan, B.E. Bray, C. Brown, D. Lanfear, L.K. Newby, et al., Merging\\nelectronic health record data and genomics for cardiovascular research: A\\nScience\\nAdvisory\\nfrom\\nthe\\nAmerican\\nHeart\\nAssociation.\\nCirculation:\\nCardiovascular, Genetics 9 (2) (2016) 193–202.\\n48 R. Dias, A. Torkamani, Artiﬁcial intelligence in clinical and genomic diagnostics,\\nGenome Medicine 11 (1) (2019) 70.\\n49 M. Woo, An AI boost for clinical trials, Nature 573 (7775) (2019) S100–S102.\\n50 S.R. Steinhubl, D.L. Wolff-Hughes, W. Nilsen, E. Iturriaga, R.M. Califf, Digital\\nclinical trials: creating a vision for the future, NPJ Digital Medicine 2 (1) (2019)\\n126.\\n51 Y. Liu, P.-H.C. Chen, J. Krause, L. Peng, How to read articles that use machine\\nlearning: users’ guides to the medical literature, JAMA 322 (18) (2019) 1806–\\n1816.\\n52 I. Sim, Mobile devices and health, New England Journal of Medicine 381 (10)\\n(2019) 956–968.\\n53 J.L. Wilder, D. Nadar, N. Gujral, B. Ortiz, R. Stevens, F. Holder-Niles, et al.,\\nPediatrician attitudes toward digital voice assistant technology use in clinical\\npractice, Appl Clin Inform 10 (2) (2019) 286–294.\\n54 Anon. Getting real with wearable data. Nature Biotechnology 2019; 37(4): 331-\\n331.\\n55 Statement from FDA Commissioner Scott Gottlieb, M.D., on FDA’s new strategic\\nframework to advance use of real-world evidence to support development of\\ndrugs\\nand\\nbiologics.\\nwww.fda.gov/news-events/press-announcements/\\nstatement-fda-commissioner-scott-gottlieb-md-fdas-new-strategic-framework-\\nadvance-use-real-world. [Accessed June 24, 2021].\\n56 V.-T. Tran, C. Riveros, P. Ravaud, Patients’ views of wearable devices and AI in\\nhealthcare: ﬁndings from the ComPaRe e-cohort, NPJ Digital Medicine 2 (1)\\n(2019) 53.\\n57 Study\\ndata\\nstandards:\\nwhat\\nyou\\nneed\\nto\\nknow.\\nwww.fda.gov/media/\\n98907/download. [Accessed June 24, 2021].\\n58 FDA’s\\nDocument\\nArchiving,\\nReporting,\\nand\\nRegulatory\\nTracking\\nSystem\\n(DARRTS). www.fda.gov/media/80214/download. [Accessed June 24, 2021].\\n59 The future of FDA’s electronic safety surveillance. www.fda.gov/news-events/\\nfda-voices/future-fdas-electronic-safety-surveillance. [Accessed June 24, 2021].\\n60 A. Awaysheh, J. Wilcke, F. Elvinger, L. Rees, W. Fan, K. Zimmerman, A review of\\nmedical terminology standards and structured reporting, J Vet Diagn Invest 30\\n(1) (2018) 17–25.\\n61 Centers for Medicare & Medicaid Services, HHS. Medicare and Medicaid\\nprograms; electronic health record incentive program. Final rule. Fed Regist\\n2010; 75(144): 44313-44588.\\n62 Anon., Standards for medical identiﬁers, codes, and messages needed to create an\\nefﬁcient computer-stored medical record, J. Am. Med. Informatics 1 (1994) 1–7.\\n63 Z. Ji, Q. Wei, H. Xu, Bert-based ranking for biomedical entity normalization,\\nAMIA Summits on Translational Science Proceedings 2020 (2020) 269.\\n64 Zhang Z, Liu J, Razavian N. BERT-XML: large scale automated ICD coding using\\nBERT pretraining. arXiv preprint 2020; arXiv:2006.03685.\\n65 S.\\nSohn,\\nD.C.\\nComeau,\\nW.\\nKim,\\nW.J.\\nWilbur,\\nAbbreviation\\ndeﬁnition\\nidentiﬁcation based on automatic precision estimates, BMC Bioinformatics 9\\n(1) (2008) 402.\\n66 N. Fiorini, R. Leaman, D.J. Lipman, Z. Lu, How user intelligence is improving\\nPubMed, Nature Biotechnology 36 (10) (2018) 937–945.\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\n2606\\nwww.drugdiscoverytoday.com\\n\\n67 Q. Chen, K. Lee, S. Yan, S. Kim, C.-H. Wei, Z. Lu, BioConceptVec: creating and\\nevaluating literature-based biomedical concept embeddings on a large scale,\\nPLOS Computational Biology 16 (4) (2020) e1007617.\\n68 G. Tsatsaronis, G. Balikas, P. Malakasiotis, I. Partalas, M. Zschunke, M.R. Alvers,\\net al., An overview of the BIOASQ large-scale biomedical semantic indexing and\\nquestion answering competition, BMC Bioinformatics 16 (1) (2015) 138.\\n69 P. Beninger, Pharmacovigilance: an overview, Clinical Therapeutics 40 (12)\\n(2018) 1991–2004.\\n70 R.J. Fontana, P.B. Watkins, H.L. Bonkovsky, N. Chalasani, T. Davern, J. Serrano,\\net al., Drug-Induced Liver Injury Network (DILIN) prospective study: rationale,\\ndesign and conduct, Drug Saf 32 (1) (2009) 55–68.\\n71 FDA’s Sentinel Initiative. www.fda.gov/safety/fdas-sentinel-initiative. [Accessed\\nJune 24, 2021].\\n72 Real-world\\nevidence.\\nwww.fda.gov/science-research/science-and-research-\\nspecial-topics/real-world-evidence. [Accessed June 24, 2021].\\n73 B. Fan, W. Fan, C. Smith, H.S. Garner, Adverse drug event detection and\\nextraction from open data: a deep learning approach, Information Processing &\\nManagement 57 (1) (2020) 102131.\\n74 Biseda B, Mo K. Enhancing pharmacovigilance with drug reviews and social\\nmedia. arXiv preprint 2020; arXiv:2004.08731.\\n75 Breden A, Moore L. Detecting adverse drug reactions from Twitter through\\ndomain-speciﬁc preprocessing and BERT ensembling. arXiv preprint 2020;\\narXiv:2005.06634.\\n76 H. Guan, M. Devarakonda, Leveraging contextual information in extracting\\nlong distance relations from clinical notes, AMIA Annu Symp Proc 2019 (2020)\\n1051–1060.\\n77 F. Wu, S. Zhao, B. Yu, Y.M. Chen, W. Wang, Z.G. Song, et al., A new coronavirus\\nassociated with human respiratory disease in China, Nature 579 (7798) (2020)\\n265–269.\\n78 D.E. Gordon, G.M. Jang, M. Bouhaddou, J. Xu, K. Obernier, K.M. White, et al., A\\nSARS-CoV-2 protein interaction map reveals targets for drug repurposing, Nature\\n583 (2020) 459–468.\\n79 W. Tang, Z. Cao, M. Han, Z. Wang, J. Chen, W. Sun, et al., Hydroxychloroquine\\nin patients with mainly mild to moderate coronavirus disease 2019: open label,\\nrandomised controlled trial, BMJ 369 (2020) m1849.\\n80 J. Brainard, New tools aim to tame pandemic paper tsunami, Science 368 (6494)\\n(2020) 924–925.\\n81 R.K. Guy, R.S. DiPaola, F. Romanelli, R.E. Dutch, Rapid repurposing of drugs for\\nCOVID-19, Science 368 (6493) (2020) 829–830.\\n82 Shaffer L. 15 drugs being tested to treat COVID-19 and how they would work.\\nNature Medicine. Published online May 15, 2020. http://dx.doi.org/10.1038/\\nd41591-020-00019-9.\\n83 B. Delavan, R. Roberts, R. Huang, W. Bao, W. Tong, Z. Liu, Computational drug\\nrepositioning for rare diseases in the era of precision medicine, Drug Discovery\\nToday 23 (2) (2018) 382–394.\\n84 C. Harrison, Coronavirus puts drug repurposing on the fast track, Nat\\nBiotechnol 38 (4) (2020) 379–381.\\n85 N.J. Mercuro, C.F. Yen, D.J. Shim, T.R. Maher, C.M. McCoy, P.J. Zimetbaum,\\net\\nal.,\\nRisk\\nof\\nQT\\ninterval\\nprolongation\\nassociated\\nwith\\nuse\\nof\\nhydroxychloroquine\\nwith\\nor\\nwithout\\nconcomitant\\nazithromycin\\namong\\nhospitalized patients testing positive for Coronavirus Disease 2019 (COVID-\\n19), JAMA Cardiology 5 (2020) 1036–1041.\\n86 E. Chorin, M. Dai, E. Shulman, L. Wadhwani, R. Bar-Cohen, C. Barbhaiya, et al.,\\nThe QT interval in patients with COVID-19 treated with hydroxychloroquine\\nand azithromycin, Nature Medicine 26 (2020) 808–809.\\n87 M.E.\\nSelvan,\\nRisk\\nfactors\\nfor\\ndeath\\nfrom\\nCOVID-19,\\nNature\\nReviews\\nImmunology 20 (2020) 407.\\n88 D.M. Roden, R.A. Harrington, A. Poppas, A.M. Russo, Considerations for drug\\ninteractions on QTc interval in exploratory COVID-19 treatment, Journal of the\\nAmerican College of Cardiology 75 (20) (2020) 2623–2624.\\n89 T. Zhang, J. Leng, Y. Liu, Deep learning for drug–drug interaction extraction\\nfrom the literature: a review, Brieﬁngs in Bioinformatics 21 (2019) 1609–1627.\\n90 D.R. Boulware, M.F. Pullen, A.S. Bangdiwala, K.A. Pastick, S.M. Lofgren, E.C.\\nOkafor, et al., A randomized trial of hydroxychloroquine as postexposure\\nprophylaxis for Covid-19, New England Journal of Medicine 383 (2020) 517–525.\\n91 Chen\\nZ,\\nHu\\nJ,\\nZhang\\nZ,\\nJiang\\nS,\\nHan\\nS,\\nYan\\nD,\\net\\nal.\\nEfﬁcacy\\nof\\nhydroxychloroquine in patients with COVID-19: results of a randomized\\nclinical trial. medRxiv 2020: 2020.2003.2022.20040758.\\n92 M. Mahévas, V.T. Tran, M. Roumier, A. Chabrol, R. Paule, C. Guillaud, et al.,\\nClinical efﬁcacy of hydroxychloroquine in patients with covid-19 pneumonia\\nwho require oxygen: observational comparative study using routine care data,\\nBMJ 369 (2020) m1844.\\n93 REMAP-CAP\\nresponse\\nto\\nthe\\nCOVID-19\\npandemic.\\nwww.remapcap.\\norg/coronavirus. [Accessed June 24, 20201].\\n94 D. Paul, G. Sanap, S. Shenoy, D. Kalyane, K. Kalia, R.K. Tekade, Artiﬁcial\\nintelligence in drug discovery and development, Drug Discovery Today 26 (1)\\n(2021) 80–93.\\n95 A. Bender, I. Cortés-Ciriano, Artiﬁcial intelligence in drug discovery: what is\\nrealistic, what are illusions? Part 1: ways to make an impact, and why we are not\\nthere yet, Drug Discovery Today 26 (2) (2021) 511–524.\\n96 K.-K. Mak, M.R. Pichika, Artiﬁcial intelligence in drug development: present\\nstatus and future prospects, Drug Discovery Today 24 (3) (2019) 773–780.\\n97 Peng Y, Yan S, Lu Z. Transfer learning in biomedical natural language\\nprocessing: an evaluation of BERT and ELMo on ten benchmarking datasets.\\narXiv preprint 2019; arXiv:1906.05474.\\n98 Beltagy I, Lo K, Cohan A. SciBERT: a pretrained language model for scientiﬁc\\ntext. arXiv preprint 2019: arXiv:1903.10676v3.\\n99 Hoover B, Strobelt H, Gehrmann S. exbert: A visual analysis tool to explore\\nlearned\\nrepresentations\\nin\\ntransformers\\nmodels.\\narXiv\\npreprint\\n2019;\\narXiv:1910.05276.\\n100 M. Hutson, Artiﬁcial intelligence faces reproducibility crisis, Science 359 (6377)\\n(2018) 725–726.\\n101 O.E. Gundersen, S. Kjensmo, State of the art: reproducibility in artiﬁcial\\nintelligence, AAAI 2018 (2018) 1644–1651.\\n102 S. Zhao, C. Su, Z. Lu, F. Wang, Recent advances in biomedical literature mining,\\nBrieﬁngs in Bioinformatics 22 (2020) bbaa057.\\n103 A.Z. Broder, S.C. Glassman, M.S. Manasse, G. Zweig, Syntactic clustering of the\\nWeb, Computer Networks and ISDN Systems 29 (8) (1997) 1157–1166.\\n104 H. Schwenk, Continuous space language models, Computer Speech & Language\\n21 (3) (2007) 492–518.\\n105 Y. Bengio, H. Schwenk, J.-S. Senécal, F. Morin, J.-L. Gauvain, Neural probabilistic\\nlanguage models, in: D.E. Holmes, L.C. Jain (Eds.), Innovations in machine\\nlearning: theory and applications, Springer, Berlin, 2006, pp. 137–186.\\n106 Mikolov T, Sutskever I, Chen K, Corrado GS, Dean J. Distributed representations\\nof words and phrases and their compositionality. arXiv preprint 2013;\\narXiv:1310.4546v1.\\n107 Pennington J, Socher R, Manning CD. GloVe: global vectors for word\\nrepresentation. In: Moschitti A, Pang B, Daelemans W, eds. Proceedings of the\\n2014 conference on empirical methods in natural language processing (EMNLP)\\n2014. Stroudsburg: Association for Computational Linguistics, 2014: 1532–\\n1543.\\n108 P. Bojanowski, E. Grave, A. Joulin, T. Mikolov, Enriching word vectors with\\nsubword information, Transactions of the Association for Computational\\nLinguistics 5 (2017) 135–146.\\n109 Liu Y, Ott M, Goyal N, et al. Roberta: A robustly optimized BERT pretraining\\napproach. arXiv preprint 2019: arXiv:1907.11692.\\n110 https: //commoncrawl.org/ [Accessed June 24, 2021].\\n111 Brown TB, Mann B, Ryder N, et al. Language models are few-shot learners. arXiv\\npreprints 2020; arXiv:2005.14165.\\n112 Gao J, Xiao C, Glass LM, Sun J. COMPOSE: cross-modal pseudo-Siamese network\\nfor patient trial matching. arXiv preprint 2020; arXiv:2006.08765v1.\\n113 A. Cocos, A.G. Fiks, A.J. Masino, Deep learning for pharmacovigilance: recurrent\\nneural network architectures for labeling adverse drug reactions in Twitter posts,\\nJ Am Med Inform Assoc 24 (4) (2017) 813–821.\\nKEYNOTE (GREEN)\\nDrug Discovery Today d Volume 26, Number 11 d November 2021\\nKEYNOTE (GREEN)\\nwww.drugdiscoverytoday.com\\n2607\\n',\n",
       " \"See discussions, stats, and author profiles for this publication at: https://www.researchgate.net/publication/380731266\\nA COMPREHENSIVE STUDY ON AI-POWERED DRUG DISCOVERY: RAPID\\nDEVELOPMENT OF PHARMACEUTICAL RESEARCH\\nArticle\\xa0\\xa0in\\xa0\\xa0Journal of Emerging Technologies and Innovative Research · February 2021\\nCITATIONS\\n0\\nREADS\\n111\\n1 author:\\nVenkateswaranaidu Kolluri\\nUniversity of Mary Hardin–Baylor\\n24 PUBLICATIONS\\xa0\\xa0\\xa023 CITATIONS\\xa0\\xa0\\xa0\\nSEE PROFILE\\nAll content following this page was uploaded by Venkateswaranaidu Kolluri on 21 May 2024.\\nThe user has requested enhancement of the downloaded file.\\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n191 \\n \\nA COMPREHENSIVE STUDY ON AI-POWERED DRUG \\nDISCOVERY: RAPID DEVELOPMENT OF \\nPHARMACEUTICAL RESEARCH \\n \\nVENKATESWARANAIDU KOLLURI \\nSr. Software Engineer, Department of Information Technology \\n \\nABSTRACT—The aim of this paper is to provide an overview of \\nAI-assisted drug discovery, which involves AI technologies \\nenabling the quick development of pharmaceutical research. The \\ndemand for unique therapeutics and increasing disease \\ncomplexity leads conventional drug discovery methods to \\nlamentable difficulties, which include time, cost, and inefficiency \\n[1]. To counter the problem, human assisted technologies show \\nproof-of-concept methods based on machine learning algorithms, \\ndata analytics, and computational modeling to vet drug candidates \\nfaster, and optimize for treatment [1]. The study analyzes the \\nproblem at the juncture of AI and drug discoveries, conducts a \\ncomprehensive literature review to review the current situation, \\nand emphasizes the significance and advantages of utilizing AI in \\ndrug designing for the nation. Moreover, the article illustrates the \\nfuture perspectives and predictions of AI for pharmaceutical \\nstudies and for healthcare innovation in the U.S.With AI \\nalgorithms getting more and more complex, sensitive, and \\nautonomous, the issues connected with data privacy, patient \\nconsent, and the algorithm bias are arising. Implementing ethical \\nprinciples and regulatory frameworks are of crucial importance \\nto ensure the responsible use of AI in pharmaceutical research, as \\nwell as to maintain protection of patient rights and safety \\nthroughout the drug discovery process. \\nKeywords— AI, artificial intelligence, drug discovery, \\npharmaceutical research, machine learning, \\ncomputational modeling, healthcare innovation \\nI. \\nINTRODUCTION \\nPharmaceutical research and development are redefining \\ntheir landscape by digitizing biomedical research with AI \\ntechnologies. Conventional drug-finding strategies, consisting \\nof high expenditure, sloping timelines and low effectiveness, \\nare being substituted by AI-powered techniques that utilize \\nmachine-learning algorithms, data-analytics and computational \\nmodeling to accelerate the identification of novel drugs. This \\npaper will provide a view of AI-based drug discovery which in \\naddition to revolutionizing the pharmaceutical industry will \\ndramatically hasten treatment development. Innovation, by its \\nnature, is altering demand for new therapies to meet the trade-\\noffs in decision-making regarding patients´ wellbeing as well \\nas the complexity of the disease mechanisms to make possible \\nthe work of experts in developing alternatives. \\nAI is a fast-emerging field which enables the scientists to \\nprocess large volumes of biomedical data and predicts drug \\ninteractions as well as detecting the drug candidates with more \\npower and sheer accuracy than at any time in history. \\nLeveraging the power of AI  [2], pharmaceutical companies \\nwill be able to speed up the drug discovery process, lower the \\ncosts and take the new medicines to the market faster by doing \\nthis, patients and the society as a whole will benefit. At the core \\nof AI pharmaceutical research is the potential to extract \\nmeaning from extensive biological artifacts and convert them \\ninto practical understanding. Machine learning models, trained \\non broad datasets combining genomics, proteomics and clinical \\ndata, have the potential to hide patterns, biomarkers and lead to \\ntherapeutic developments that not all traditional analytical \\nmethods can achieve. Such an experience-based approach does \\nnot only shorten the period of finding out promising therapeutic \\ncandidates but also allows the development of individualized \\ntreatment plans based on the profiles of the patients as a whole, \\nthus transforming the field of medicine, namely personalized \\nmedicine [3]. \\nWhile the use of AI in drug discovery presents numerous \\nopportunities, its adoption also comes with challenges. \\nProblems, like data variability, algorithm stability and model \\nunderstandability should be supported; otherwise, we cannot \\ntrust AI outputs and their replicability. Furthermore, the ethical \\nquestions of data privacy, patient consent, and algorithm bias \\nshould be dealt with with caution in order to prevent \\nunanticipated side effects and adhere to ethical standards in \\npharmaceutical research [4]. Despite these issues, there is no \\ndoubt that AI, through its transformation capability, is one of \\nthe biggest drivers of innovation, collaboration, and study \\namong researchers in the field of drug discovery and human \\nhealth. \\nIn this essay, we will explore the research problem that lies \\nat the point where AI and pharmaceutical research meet, \\nsummarize the current practices in this area by reviewing the \\nliterature, and make an argument for the advantages of machine \\nlearning in drug discovery in the United States. Furthermore, \\nwe will look into the prospects and implications that AI could \\nhold for the U.S. pharmaceutical industry development and \\ninnovation in healthcare that highlights that investments, \\npolicies, and collaboration come in handy to tap the real \\npotential of AI in shaping the future of medicine. \\nII. \\nRESEARCH PROBLEM \\nThe primary research problem addressed in this study ison \\nexploring  the importance of revolutionizing drug discovery by \\nintegrating \\nartificial \\nintelligence \\n(AI) \\ntechnology \\ncomprehensively Modern drug discovery methods are \\nhandicapped by different challenges like high costs, long \\ndevelopment timelines, and low success rates, which make it an \\nindescribably demanding and resource-consumption process of \\ndiscovering and producing new drugs. The AI application in \\ndrug discovery is built on the fact that this can speed up the \\nprocess of discovery of novel drug candidates and of treatment \\nstrategies optimisation and can also lead to better patient \\noutcomes. Earth, the only place in the universe where we have \\nextraterrestrial intelligence contact. Firstly, there is a necessity \\nto understand how AI-based techniques will fill in the gaps and \\nenhance the current drug discovery approaches. This process \\ninvolves exploration of the abilities and inadequacies of \\nartificial intelligence algorithms in deciphering complex \\nbiological data, predicting drug-target interactions, and sorting \\nout potential drug candidates that deserve further investigation  \\n[5]. Furthermore, the technical issues of data accuracy, \\nalgorithm reliability, and model comprehensibility must be \\naddressed to maintain the authenticity of the AI-driven results \\nresearch. This will help in understanding ethical and regulatory \\nmatters which arise from augmentation of AI in drug discovery \\nprocesses. \\n \\n \\n \\n \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n192 \\n \\nIII. \\nLITERATURE REVIEW \\nA. AI IN DRUG DISCOVERY \\nThe emergence of AI in drug discovery is an underlying \\nprocess full of revolutionary progresses which are enabled by \\ninnovative computational procedures and data-oriented \\ntechniques. At the beginning, AI-based solutions were \\nrestricted to the rule-based systems and semi-intuitive \\nalgorithms, which rendered only limited data but were unable \\nto solve the complexity of biological systems. Nonetheless, \\nadvances in the area of machine learning and deep learning \\ndriven by the availability of the vast biological databases and \\ncomputational resources, made AI to be revolutionary in the \\nprocess of drug discovery [6]. Machine learning (ML) \\nalgorithms have been widely used for analyzing various types \\nof biological data which are represented by genomics, \\nproteomics and chemical compounds. For example, support \\nvector machines (SVMs), random forests, and neural networks \\nare often applied to accomplish this purpose  [7]. These \\nalgorithms can discriminate highly sophisticated connections \\nand correlations all around large databases; therefore, drug-\\ntarget interactions prediction, leads optimization and the \\nsuccessful medical treatment can be achieved by using these \\nalgorithms. Besides this, the birth of the deep learning \\narchitectures including convolutional neural networks (CNNs) \\nand recurrent neural networks (RNNs) have changed the way \\ndata analysis is done by offering machine-learning algorithms \\nthat can be used to extract features from the raw data and \\nprovide more accurate predictions and insights  [7]. \\nThe past several years have observed a coming together of \\ncomputational as well as experimental approaches where AI \\nalgorithms \\ndetermine \\nthe \\nexperimental \\ndesign, \\ntarget \\nvalidation, and lead optimization. Artificial intelligence in the \\ndrug discovery process has led to accelerated identification of \\nnovel drug candidates, reduction in the cost of discovery and \\nenhanced the success rate of clinical trials. Another important \\ncapability of AI is its ability to analyze collective biological \\ninteractions and predict drug response in the populations where \\ntreatment entirely depends on individual genetic profiles and \\ndisease characteristics. \\n \\n \\n [5].\\n \\nFig. 1  A bar graph showing the distribution of various AI \\ntechniques in Drug Discovery \\nB. AI-POWERED DRUG TARGET IDENTIFICATION \\nAI-based drug target identification is leading-edge \\ntechnology for boosting drug discovery for different diseases. \\nThe use of these machine learning and data analytics tools \\nallows for the analysis in detail of massive volumes of \\nbiological information with the objective of identifying \\nmolecules, proteins, or pathways that could be affected by \\ndrugs in order to treat different diseases  [8]. One of the main \\nadvantages of AI in drug target identification is that it can sift \\nthrough large-scale -omics or genomic, transcriptomic, \\nproteomic, or metabolomic data - to discover useful \\nrelationships and patterns. Machine learning algorithms can be \\nemployed to analyze complex datasets in order to enhance the \\nsearch of genes, proteins or signaling pathways that are \\ndysregulated in disease states offering a clue for potential \\ntherapeutic \\ntargets \\n[9]. \\nFurther, \\nartificial \\nintelligence \\nalgorithms can ease the process of personnel drug targeting by \\nconsidering various multifaceted factors, such as their \\nbiological significance, the issue of druggability, and the \\nresulting unwanted effects. Through the integration of varied \\ndata sources and by applying advanced feature selection skills, \\nAI-based drug target identification platforms firstly seek out the \\nmost prominent candidates (according to the calculated for \\nsuccess$) with a consequent smoothing of the process of drug \\ndiscovery. Moreover, AI algorithms are not only capable of \\nhandling conventional biological data but also features such as \\nelectronic health records, patient demographics, and clinical \\ntrial data, to identify new pathways and predictive biomarkers \\nfor personalized medicine strategies. It provides for a unified \\nand holistic approach to therapy that assists in covering missed \\ntargets that can be obtained by traditional methods and deliver \\npersonalized treatments to the patients considering their distinct \\ngenetic makeup and other factors affecting their disease. \\n \\nFig. 2  AI in Drug Target Identification \\nC. AI-DRIVEN DRUG DESIGN AND OPTIMIZATION \\nArtificial intelligence (AI)-facilitated drug design and \\noptimization of new therapeutics is a game changer in the drug \\ndevelopment platform because of the higher efficacy and safety \\nthat they provide. This process takes advantage of the machine \\nlearning, in-silico modeling and structure-based screening \\napproaches to speed up the process of finding and optimizing \\ndrug candidates with an aim to decrease the size of the drug \\ndevelopment pipeline  [10,11]. The capability of AI to process \\nlarge amounts of chemical compounds and predict their \\nproperties such as activity, pharmacokinetics, and toxicity is \\none of importance. Machine learning algorithms are capable of \\nconforming to the existing drug data for finding patterns and \\nlinks among chemical structures and biological activities [12], \\nwhich in turn makes possible the emergence of new compound \\nstructures with desired properties. \\nAI-driven drug design utilizes model extensions such as \\nmolecular docking/ molecular dynamics simulations to predict \\nthe interactions between a drug and its target protein. Such \\nsimulations are useful for researchers to test the binding \\nstrength and the selectivity of the molecules that can be adapted \\nas drug candidates which then guide the researchers to select a \\nlead compound for candidates for further optimization [13]. \\nMoreover, drug candidates can be optimized by AI algorithms \\nwhich can \\niteratively generate and screen chemical \\nmodifications to improve their potency, selectivity, and \\npharmacokinetic properties. The most time-consuming and \\nlabor-demanding step in the lead optimization process is \\nexecuted with conventional methods; however, it can be \\nsignificantly improved using AI-driven ones. \\n \\n \\n \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n193 \\n \\nD. AI-ASSISTED DRUG REPURPOSING AND \\nCOMBINATION THERAPY \\nAI-driven repurposing and synergistic drug combination \\ntechniques represent innovative methods for drug development \\nthat aim to take advantage of existing drugs and to address \\nunmet medical needs more effectively  [14]. Drug repurposing, \\nalso referred to as drug repositioning, is the process of \\ndiscovering new therapeutic uses for pre-existing drugs with \\nindications other than those they were originally designed for. \\nAI algorithms have proved that they significantly contribute to \\nthe drug discovery process through analyzing large-scale \\nbiomedical data, such as electronic health records and drug \\ndatabases from where relationships between drugs, diseases, \\nand \\nbiological \\npathways \\ncan \\nbe \\nextracted \\n[15,16]. \\nImplementing machine learning methodologies, such as those \\nbased on networks and similarity, can enable AI to quickly \\ndetermine potential candidates for repurposing based on their \\ncurrent known pharmacological profiles and suitability for the \\ntreatment of different diseases. On the other hand, AI-powered \\ndrug repurposing tools can rank drug candidates in accordance \\nwith different factors like their safety profiles, the way they act \\nand their probability to be successful in clinical trials. The \\nrepurposing of existing drugs not only bypasses most of the \\ntime-consuming and expensive steps in the traditional drug \\ndevelopment process, it also speeds up the time to market a new \\nmedication. \\n \\n \\n \\nFig. 1 A heatmap on AI-Assisted Drug Repurposing and \\nCombination Therapy \\nIV. \\nSIGNIFICANCE AND BENEFITS TO THE \\nU.S \\nThe AI-driven drug invention and development are highly \\nvaluable to the United States and have wide ranging significant \\nbenefits. First of all AI in pharmaceutical research can bring a \\ncomplete rethinking of the drug discovery process that is going \\nto help develop new designer medicines addressing a broad \\nrange of diseases  [17]. AI technologies provide efficiency with \\nstreamlining of the drug development system, thus, cutting the \\ntime and cost that go to the market bringing new drugs, which \\nat the end give patients a wider reach to treatment options. \\nAdditionally, artificial intelligence (AI)-driven drug discovery \\nimplies a potential of improving the positions of the U.S. \\npharmaceutical companies in a global market. Through \\nsupporting innovation and nurturing of partnerships among \\nacademic institutions, industry, and regulatory bodies, the US \\nwill stay on a path of successful biomedical research and \\ndevelopment. This thereby boosts economic growth, creates \\nhigh-paying jobs, and draws the life sciences industry players \\ninto the investments  [18]. Moreover, AI in drug discovery not \\nonly poses solutions to the most crucial challenges in the United \\nStates healthcare such as drug-resistant pathogens, opioid \\nepidemic, but also tobacco-related diseases [19,20]. The U.S. \\nwill be able to enhance patient outcomes, decrease healthcare \\nexpenditures, and offset the burden on the healthcare system by \\nusing AI to discover new therapeutic targets, repurpose existing \\ndrugs, and optimize treatment regimens. \\nV. \\nFUTURE IN THE U.S \\nAI-driven drug development and discovery in the future of \\nUS healthcare will create a tremendous opportunity for medical \\nresearch advancement, health outcomes improvement, and \\neconomic growth. As AI technologies gradually evolve and \\nprogress, their impact on pharmaceutical innovation in the U.S. \\nis also expected to grow in importance. One of the major \\naspects of AI-powered drug discovery's future relates to the \\nprogressive assimilation of AI into pharmaceutical firms' R&D \\npipelines. AI is becoming more attractive to pharmaceutical \\ncompanies to reduce the burden of drug discovery, to accelerate \\ndrug discovery, and to improve clinical trial design [21]. This \\ntrend is expected to extend with AI becoming an irreplaceable \\ninstrument for improving productivity, decreasing costs, and \\nincreasing the rate of drug development programs' success. \\nAdditionally, AI in the future will not work in just one sector \\nof the life sciences but it will be used in the different sectors \\nwith more collaboration and convergence. AI technologies find \\ntheir application in academic institutions, biotech startups, \\npharmaceutical companies, and government agencies as they \\nare becoming partners to share data and resources. Such joint \\nefforts are likely to push for innovation, emphasize \\ninterdisciplinary research, and speed up the transition from \\nscientific discoveries to clinical application. \\nVI. \\nCONCLUSION \\nThis paper has offered a detailed analysis on how AI is \\ntransforming the pharmaceutical landscape and offering better \\nsolutions to patients' healthcare issues. By focusing on the study \\nof this issue, a literature review, and a discussion of the \\nrelevance of this topic to the US, we see that AI technology has \\nmany advantages as a means of speeding up drug development, \\nminimizing costs, and achieving personalized medicine. The \\nmost important role of AI-based drug discovery in the USA can \\nbe viewed as the contributor to the innovation, the process of \\neconomic growth, and the way to deal with impressive health \\nissues. AI technologies can serve as a game changer in terms of \\nmaintaining the U.S.'s front line position with respect to \\nbiomedical research and development thus allowing attainment \\nof such great ideas like precision medicine, drug repurposing \\nand combination therapy. In summary on the topic, the future \\nof AI- supported drug development in the United States is \\npromising with new AI algorithms together with both data and \\ncomputational resources being the future of the industry. Using \\nthese technologies and making collaboration between \\nacademia, industry and government possible, the U.S. can \\ncontinue to make great strides and improve the lives of a lot of \\npatients across the globe. \\n \\n \\nREFERENCES \\n[1] \\nN. Brown. Artificial Intelligence in Drug Discovery. Royal Society of \\nChemistry, 2020. \\n[2] \\nJ. Cassidy and B. Taylor, Artificial Intelligence in Oncology Drug \\nDiscovery and Development. BoD – Books on Demand, 2020. \\n[3] \\nM. Chang, Artificial Intelligence for Drug Development, Precision \\nMedicine, and Healthcare. CRC Press, 2020. \\n[4] \\nM. Chang, AI for Drug Development and Well-being. 2020. \\n[5] \\nF. Lake, “Artificial intelligence in drug discovery: what is new, and what \\nis next?,” Future Drug Discovery, vol. 1, no. 2, p. FDD19, Oct. 2019, \\ndoi: https://doi.org/10.4155/fdd-2019-0025 \\n[6] \\nA. Varun, “Artificial Intelligence (AI) in Drug Discovery and \\nMedicine,” Journal of Clinical Cases & Reports, vol. 2, no. 3, pp. 76–\\n80, Jul. 2019, doi: https://doi.org/10.46619/joccr.2019.2-1043 \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n194 \\n \\n[7] \\nR. Lawrence, “Now the future, we see our dreams: artificial intelligence \\nin drug discovery,” Future Drug Discovery, vol. 1, no. 2, p. FDD22, Oct. \\n2019, doi: https://doi.org/10.4155/fdd-2019-0027 \\n[8] \\nK.-K. Mak and M. R. Pichika, “Artificial intelligence in drug \\ndevelopment: present status and future prospects,” Drug Discovery \\nToday, \\nvol. \\n24, \\nno. \\n3, \\npp. \\n773–780, \\nMar. \\n2019, \\ndoi: \\nhttps://doi.org/10.1016/j.drudis.2018.11.014. \\nAvailable: \\nhttps://www.sciencedirect.com/science/article/abs/pii/S1359644618300\\n916 \\n[9] \\nP. Agrawal, “Artificial Intelligence in Drug Discovery and \\nDevelopment,” Journal of Pharmacovigilance, vol. 06, no. 02, 2018, doi: \\nhttps://doi.org/10.4172/2329-6887.1000e173 \\n[10] A. Laghaee, C. Malcolm, J. Hallam, and P. Ghazal, “Artificial \\nintelligence and robotics in high throughput post-genomics,” Drug \\nDiscovery Today, vol. 10, no. 18, pp. 1253–1259, Sep. 2005, doi: \\nhttps://doi.org/10.1016/s1359-6446(05)03581-6 \\n[11] A. Bohr and K. Memarzadeh, Artificial Intelligence in Healthcare. \\nAcademic Press, 2020. \\n[12] R. Jagannathan, “AI and drug discovery : Search for new psychoactive \\ncandidates \\nfrom \\ncannabis \\nsativa,” \\nSep. \\n2020, \\ndoi: \\nhttps://doi.org/10.1021/scimeetings.0c00478 \\n[13] J. T. Chang and R. B. Altman, “Promises of text processing: natural \\nlanguage processing meets AI,” Drug Discovery Today, vol. 7, no. 19, \\npp. \\n992–993, \\nOct. \\n2002, \\ndoi: \\nhttps://doi.org/10.1016/s1359-\\n6446(02)02457-1 \\n[14] J. Kirchmair, “Editorial for the Special Section ‘Artificial Intelligence in \\nDrug Discovery,’” Drug Discovery Today: Technologies, Dec. 2020, \\ndoi: https://doi.org/10.1016/j.ddtec.2020.12.001 \\n[15] E. Smalley, “AI-powered drug discovery captures pharma interest.” \\nNature Biotechnology, vol. 35, no. 7, pp. 604-605, 2017, doi: \\n10.1038/nbt0717-604. \\n[16] M. Hervey, “Harnessing AI in drug discovery without losing patent \\nprotection.” Drug Discovery Today, vol. 25, no. 6, pp. 949-950, 2020, \\ndoi: 10.1016/j.drudis.2020.03.007. \\n[17] A. Varun, “Artificial Intelligence (AI) in Drug Discovery and \\nMedicine.” Journal of Clinical Cases & Reports, vol. 2, no. 3, pp. 76-80, \\n2019, doi: 10.46619/joccr.2019.2-1043. \\n[18] K. Linton-Reid, “Introduction: An Overview of AI in Oncology Drug \\nDiscovery and Development.” Artificial Intelligence in Oncology Drug \\nDiscovery and Development, 2020, doi: 10.5772/intechopen.92799. \\n[19] R. Dousa, “Toward the Clinic: Understanding Patient Perspectives on \\nAI and Data-Sharing for AI-Driven Oncology Drug Development.” \\nArtificial Intelligence in Oncology Drug Discovery and Development, \\n2020, doi: 10.5772/intechopen.92787. \\n[20] M. Hervey, “Harnessing AI in drug discovery without losing patent \\nprotection.” Drug Discovery Today, vol. 25, no. 6, pp. 949-950, 2020, \\ndoi: 10.1016/j.drudis.2020.03.007. \\n[21] N. Ai, M. D. Krasowski, W. J. Welsh, and S. Ekins, “Understanding \\nnuclear receptors using computational methods.” Drug Discovery \\nToday, \\nvol. \\n14, \\nno. \\n9, \\npp. \\n486-494, \\n2009, \\ndoi: \\n10.1016/j.drudis.2009.03.003. \\n \\n \\n \\nView publication stats\\n\",\n",
       " 'Received January 21, 2019, accepted March 3, 2019, date of current version May 29, 2019.\\nDigital Object Identifier 10.1109/ACCESS.2019.2906402\\nTowards Sustainable Energy: A Systematic\\nReview of Renewable Energy Sources,\\nTechnologies, and Public Opinions\\nATIKA QAZI\\n1, FAYAZ HUSSAIN1, NASRUDIN ABD. RAHIM1,2, (Senior Member, IEEE),\\nGLENN HARDAKER3, DANIYAL ALGHAZZAWI4, KHALED SHABAN5,\\nAND KHALID HARUNA\\n6\\n1University Malaya Power Energy Dedicated Advanced Centre (UMPEDAC), University of Malaya, 59990 Kuala Lumpur, Malaysia\\n2Renewable Energy Research Group, King Abdulaziz University, Jeddah 21589, Saudi Arabia\\n3Centre for Lifelong Learning, University of Brunei Darussalam, Gadong BE1410, Brunei\\n4Information Systems Department, Faculty of Computing and Information Technology, King Abdulaziz University, Jeddah 21589, Saudi Arabia\\n5Computer Science and Engineering Department, College of Engineering, Qatar University, Doha, Qatar\\n6Department of Computer Science, Faculty of Computer Science and Information Technology, Bayero University, Kano, Kano, Nigeria\\nCorresponding author: Atika Qazi (atikaqazium@gmail.com)\\nThis work was supported in part by the UM Power Energy Dedicated Advanced Centre (UMPEDAC) and the Higher Institution Centre of\\nExcellence (HICoE) Program Research under Grant UMPEDAC-2016 (MOHE HICOE-UMPEDAC), and in part by the Deanship of\\nScientiﬁc Research (DSR) at King Abdulaziz University, Jeddah, under Grand no. RG-6-611-39.\\nABSTRACT The use of renewable energy resources, such as solar, wind, and biomass will not diminish their\\navailability. Sunlight being a constant source of energy is used to meet the ever-increasing energy need. This\\nreview discusses the world’s energy needs, renewable energy technologies for domestic use, and highlights\\npublic opinions on renewable energy. A systematic review of the literature was conducted from 2009 to 2018.\\nDuring this process, more than 300 articles were classiﬁed and 42 papers were ﬁltered for critical review.\\nThe literature analysis showed that despite serious efforts at all levels to reduce reliance on fossil fuels by\\npromoting renewable energy as its alternative, fossil fuels continue to contribute 73.5% to the worldwide\\nelectricity production in 2017. Conversely, renewable sources contributed only 26.5%. Furthermore, this\\nstudy highlights that the lack of public awareness is a major barrier to the acceptance of renewable energy\\ntechnologies. The results of this study show that worldwide energy crises can be managed by integrating\\nrenewable energy sources in the power generation. Moreover, in order to facilitate the development of\\nrenewable energy technologies, this systematic review has highlighted the importance of public opinion and\\nperformed a real-time analysis of public tweets. This example of tweet analysis is a relatively novel initiative\\nin a review study that will seek to direct the attention of future researchers and policymakers toward public\\nopinion and recommend the implications to both academia and industries.\\nINDEX TERMS\\nEnergy policies, public opinion, renewable energy sources (RES), renewable energy\\ntechnology (RET), solar energy, wind energy.\\nI. INTRODUCTION\\nThe subject of renewable energy (RE) concerns experts as\\nwell as the general public increasingly. Studies on renewable\\nenergy sources (RES) has increased in the last years in abso-\\nlute and relative terms [1]. RES can perform an important role\\nby addressing the issues of fossil fuel depletion and global\\nwarming [2]. Fossil fuels, nuclear resources, and renewable\\nresources are the three main sources of energy. RES such\\nThe associate editor coordinating the review of this manuscript and\\napproving it for publication was Liandong Zhu.\\nas solar, wind, biomass, geothermal, and hydro-power are\\nutilized to reproduce energy and are therefore extensively\\nuseful to combat energy crises [3]–[5]. A recent study [6] of\\ncommunities in Western Greece focused on the public attitude\\nand the willingness to pay for electricity from renewable\\nelectricity sources.\\nRenewable energy resources are considered clean energy\\nresources [7] and are critically important due to their\\nenvironmental-friendly nature [8], [9]. With the increase\\nin awareness of a clean environment, it is believed that\\ntraditional dependence on fossil fuels has led to carbon\\nVOLUME 7, 2019\\n2169-3536 \\n 2019 IEEE. Translations and content mining are permitted for academic research only.\\nPersonal use is also permitted, but republication/redistribution requires IEEE permission.\\nSee http://www.ieee.org/publications_standards/publications/rights/index.html for more information.\\n63837\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\ndioxide (CO2) emissions, greenhouse gas (GHG) problems\\nand environmental pollution [10], [11]. RES can cover\\ndomestic energy requirements with the potential to provide\\nenergy services with zero or almost zero emission of air\\npollutants and GHGs [12], [13]. Pivotal tasks such as sustain-\\nable development of remote regions in the desert and moun-\\ntain zones in addition to implementation of the obligations\\nto fulﬁl international agreements relating to environmental\\nprotection are expected to be resolved with the development\\nof RE [14]–[16]. To fulﬁl the excessive energy demand\\nthere is currently a global tendency to replace conventional\\nfuels with RES [17]. There are many challenges, such as\\nGHG emissions, CO2\\nemissions, climate change and\\nenergy security, that have resulted in RE seeking to ful-\\nﬁll a growing need in today’s environment [18]–[20].\\nUnlike fossil fuels, RES provides environmental protection,\\na pollution-free environment, energy security and economic\\nbeneﬁts [21], [86]. Hence, in order to meet the energy need,\\nit is indeed vital for present and future generations to rely\\non RES. To achieve this, policy-making and exploring public\\nopinion is required to promote the use of RE. Analyzing\\npublic opinion toward RE is crucial due to the inﬂuence it\\ncan have on policy creation. The research has suggested an\\nimportant link between public opinion and public policy [22].\\nPublic opinions that have become a hot ﬁeld of research\\nare growing intensively through computer-mediated environ-\\nments such as Web 2.0. These enable an increasing num-\\nber of people to communicate and interact with each other\\nusing social media such as Twitter, Amazon, blogosphere,\\netc. [23], [24]. This survey aims at drawing useful insights\\non the use of RE, the challenges faced therein, the awareness\\non using RES and the acceptance of renewable energy tech-\\nnology (RET) for domestic use by mining public opinion on\\nsocial media. The interaction through social media provides a\\nvariety of useful opinions that can be a guideline for making\\nbetter plans for the future [25]. The opinions capture certain\\nsentiment words which are useful in diverse areas, such as\\nmarketing, education, business, and health care. Further, they\\ncan attract more ﬁelds to utilize their signiﬁcance [26], [27].\\nOpinions can be recorded and analyzed using survey studies\\nand/or sentiment analysis techniques in order to determine\\nthe present and future intentions of users related to RE and\\nexplore new insights [28], [23]. Hence it is believed that opin-\\nions inﬂuence public decision to use any services and tech-\\nnologies such as RES [24], [29]. While most of the research\\nhas focused on technical, policy, and ﬁnancial challenges to\\nRE deployment [30], less attention has been paid on its social\\ndimensions, yet public opinion is highly relevant. In this age\\nof technology, opinions created by social networks provide an\\nexceptional opportunity to mine valuable insights from them\\nthat can be used for the development of RES [31].\\nThe aim of this survey paper is to study the worldwide\\nneed of RE, types of RES used at domestic scale and draw\\nuseful conclusions on use and acceptance of the public opin-\\nion on the use and acceptance of RET and RES. Section II\\nexplains the research method; Section III presents the results;\\nSection IV analyze the opinions regarding renewable energy;\\nsection V provides the discussion; Section VI discuss the\\nimplications of the study and section VII concludes the study.\\nII. RESEARCH METHOD\\nTo advance our understanding of utilizing RES effectively the\\npresent study consists of a systematic literature review (SLR)\\nwith a speciﬁc focus on RES and RET. Recently a SLR has\\nbeen carried out in the combination of computer science and\\nRE ﬁeld [32], [33] and a detailed SLR was performed about\\npublic opinions [23]. A SLR is a means of identifying, evalu-\\nating and interpreting available research relevant to a partic-\\nular research question or topic area [34]. The design of the\\nSLR reported in this paper started in December 2017. After\\nseveral reﬁnements and improvements, publication search\\nwas started in March 2018. The following subsections present\\nthe research questions of the review, search strategy, searched\\ndatabase, search terms, inclusion/exclusion criteria and the\\nquality criteria of the SLR.\\nA. PLANNING THE REVIEW\\nThe review is planned by proposing the research ques-\\ntions relevant to our research objective. Further, we deﬁned\\nthe search strategy, search strings and inclusion/exclusion\\ncriteria.\\n1) RESEARCH QUESTIONS\\nThe main objective of this work is to develop an under-\\nstanding of the necessity for RE, RES, and RET. We have\\nformulated the following research questions (RQs) to achieve\\nthe research objectives.\\nRQ1: WHY THERE IS WORLDWIDE NEED FOR\\nRENEWABLE ENERGY?\\nThe aim of this research question is to recognize the rea-\\nsons for the use of RE. The results obtained will be useful to\\nhighlight the challenges with the use of energy sources other\\nthan RE and bring awareness for the use of environmentally\\nfriendly energy sources such as RES.\\nRQ2: WHAT ARE THE TYPES OF REWNEWABLE\\nENERGY SOURCES USED FOR DOMESTIC USAGES?\\nThe aim of this research question is to identify the various\\ntypes of RES such as wind, solar and biomass, used to provide\\nenergy for domestic scale. The results will assist to develop\\nthe awareness for the use of these RES.\\nRQ3: WHAT ARE THE PUBLIC OPINIONS ON\\nRENEWABLE ENERGY TECHNOLOGIES?\\nThe aim of this research question is to identify the public\\nopinions addressed in the literature regarding the use of RET.\\nThe results obtained will be useful to identify emerging trends\\nand provide a possible solution to tackle the barriers for\\naccepting RE.\\n2) SEARCH STRATEGY\\nThe study has used the guidelines of Kitchenham [35] for\\ncarrying out this SLR. We began with the formal search\\nstrategy to explore the articles relevant to our research\\nobjective\\nand\\nthe\\nresearch\\nquestions.\\nTo\\ncollect\\nthe\\n63838\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nTABLE 1. Number of filtered publications according to the\\ninclusion/exclusion criteria.\\nappropriate articles, the search space is deﬁned to cover the\\nprominent databases listed in Table 1. At ﬁrst, the studies\\nwere retrieved and afterwards examined further for other\\nsigniﬁcant studies (i.e. snowballing) [35]. Then the inclusion\\nand exclusion criteria in two distinct rounds (Round 1 &\\nRound 2) were applied as explained in Section 4.\\n3) SEARCH STRING\\nThe search terms were derived from the key terms used in\\nthe topic area and the objective of the review. A number\\nof pilot searches were performed to reﬁne the keywords\\nin the search string using trial and error. The terms whose\\ninclusion did not yield additional papers in the automatic\\nsearch were removed. The following search string was used:\\n(‘‘renewable energy‘‘ OR ‘‘public opinions’’ OR ‘‘use of\\nrenewable energy’’ OR ‘‘solar systems for domestic use’’\\nOR ‘‘solar heaters’’ OR ‘‘solar cooker’’ OR ‘‘solar cooler’’\\nOR ‘‘biomass’’ OR ‘‘hydropower’’, OR ’’wind’’) AND\\n(‘‘energy sources’’ OR ‘‘energy demand’’ OR ‘‘public opin-\\nions’’ OR ‘‘domestic products’’ OR ‘‘home systems’’ OR\\n‘‘acceptance’’ OR ‘‘attitude’’ OR ‘‘effects‘‘ OR ‘‘motiva-\\ntion’’) AND (‘‘energy crises’’ OR ‘‘ energy demand’’ OR\\n‘‘clean energy’’ OR ‘‘electricity need’’ OR ‘‘greenhouse gas\\nemission’’).\\n4) INCLUSION AND EXCLUSION CRITERIA\\nTo select the appropriate studies for inclusion in the review,\\nthe meta data and the abstracts of the papers were reviewed,\\nand the following inclusion criteria were applied. Selected\\npapers were I1: peer-reviewed; I2: in the English language;\\nI3: empirical research papers; I4: published between the\\nyear 2009 to 2018; where I1-I4 denote the inclusion criteria.\\nPapers that did not meet the inclusion criteria were excluded\\nfrom the study. We applied the exclusion criteria E1-E4 to\\nﬁlter out irrelevant papers. E1: papers not focusing explicitly\\non RE; E2: papers that do not discuss the public opinions\\nabout renewable energy technologies; E3: studies that do\\nnot discuss RES; E4: Grey literature [35], e.g., working\\npapers, project deliverables, and PhD theses were the exclu-\\nsion criterias applied. Therefore, the ﬁnal selection consisted\\nof 42 articles.\\nB. CONDUCTING THE REVIEW\\nIn this section, we present the ﬁndings of our search\\nand extraction of information from relevant sources and\\ndatabases.\\n1) ARTICLE SEARCH AND SELECTION\\nBy following the search strategy explained in Section 2,\\nthe selected electronic databases were searched to retrieve\\nthe studies. The initial search returned 362 papers as shown\\nin Table 1. We followed the inclusion and exclusion cri-\\nteria (Round 1) explained above, and detailed examination\\nof the titles and abstracts of the articles was performed by\\na researcher and ended up with 135 studies. Afterwards,\\nin the second stage (Round 2) using the exclusion criteria\\n(E1, E2, E3 and E4), the preselected studies were assessed\\nby a second researcher (one of the co-authors) and a third\\none (experienced and independent researcher). In case of\\nany agreements and disagreements among the researchers\\nin their assessments, a face-to-face meeting was arranged to\\nreview and reach consensus. For papers on which consensus\\nwas not met, the three researchers read the paper and then\\nexcluded the studies based on the deﬁned exclusion criteria.\\nFinally, 42 papers were selected for critical review as shown\\nin Table 1.\\n2) DATA EXTRACTION AND ANALYSIS\\nBy following the guidelines in [35], we conducted a data\\nextraction process to identify the relevant information from\\nthe selected 42 studies after round 2. This data extraction\\nprocess included the following steps: First, using Mendeley\\nreference manager, we arranged a database of publications\\nand recorded the relevant information about ideas, contribu-\\ntions, and ﬁndings of each of the 42 studies in a spreadsheet.\\nFrom each publication the following data were extracted:\\nreview date, title, authors, references; database, relevance to\\nthe theme, i.e. need of RE, RES at domestic scale, public\\nopinion on RE, and year of publication.\\nAfter the extraction process, we used content analysis to\\ncharacterize the focus of each study [36]. Content analysis\\nhas been recently used in a systematic review [23], [32].\\nThe result of data extraction was assessed by using inter-\\nrater agreement between two researchers using the κ coef-\\nﬁcient [37]. The κ coefﬁcient is a measure of the agreement\\nbetween any two raters who classify the studies into various\\ncategories. The value of κ for this study is 0.5 which is consid-\\nered to be a good agreement [38]. Subsequently, an indepen-\\ndent quality assessment was recorded for 42 selected studies.\\n3) QUALITY OF STUDY\\nThe quality criteria for this systematic review is similarly\\nadopted in other SLRs [39]. The ﬁrst criterion was (C1):\\nif the aims and objectives of the conducted research were\\nclearly deﬁned. This question was positively answered for\\n92% of the studies. The second criterion was (C2): if the\\nresearch context was adequately addressed. This question was\\npositively answered by 86% of the studies. The last crite-\\nrion was (C3): if the outcome of the research was sufﬁcient\\nfor our research purpose. For the quality measures (C3) the\\nheuristic scores were established by a group of two experi-\\nenced researchers and validated by an independent reviewer.\\nVOLUME 7, 2019\\n63839\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nTABLE 2. Quality criteria for study selection.\\nThe data was normalized for the 42 papers, combining the\\npercentage obtained in all quality criteria. The results of\\nquality assessment including normalized scores are described\\nin Table 2.\\nIII. RESULTS\\nIn this section, results from the SLR are described in the light\\nof the proposed research questions.\\nA. OVERVIEW OF STUDIES\\nTo provide more focused results related to our research objec-\\ntive, the current review has focused on 42 papers that were\\nquality-rated as high. Figure 1 shows the stages of the study\\nselection process for the systematic review presented in this\\nwork. These studies focused on, speciﬁcally: the necessity\\nof RE (9 out of 42); RES at domestic scale (14 out of 42);\\nand public opinion on RE (19 of 42). The results are fur-\\nther discussed in light of the proposed research questions\\n(RQ1, RQ2 & RQ3).\\n1) RQ 1: WHY THERE IS WORLDWIDE NEED FOR\\nRENEWABLE ENERGY?\\nThe demand for energy is increasing rapidly and, hence,\\nit is crucial to introduce processes to meet the requirements\\nof the growing world population and to avoid energy crises.\\nAt the current rate of energy consumption, the demand is\\nexpected to increase by 65% by the year 2030, using 2004 as\\nthe base year [40]. Currently, most of the energy used globally\\nis produced from non-renewable sources such as coal-ﬁred\\npower plants. These are known to cause serious problems\\nincluding GHG (CO2, NOx, and SOx) emissions as well as\\ncontributing to global warming [41], [42].\\nWith the levels of GHGs in the atmosphere rising at a\\nhazardous rate, it is important to develop sources of non-fossil\\nfuel based energy in addition to determining ways to reduce\\nCO2 [43]. In contrast to fossil fuels, RE offers alternative\\nsources of clean energy. Moreover, it is also expected to\\nreduce energy crises by playing a key role in meeting future\\nelectricity demands. Solar and wind energy are the most\\npromising forms of the RE sources that encourage interest in\\nincreasing their use worldwide [44], [45]. There are a variety\\nof reasons identiﬁed and policies deﬁned in the literature\\nFIGURE 1. Study selection phase.\\npromoting the use of RE instead of fossil fuels to produce\\npower.\\nShaﬁullah [46] developed a hybrid RE integration system\\nwhich aimed to reduce the energy crisis while minimizing\\nglobal warming issues in order to facilitate RE integration in\\nthe Capricornia region of Central Queensland, Australia. The\\nproposed system comprised a techno-economic model that\\nanalyzed the techno-economic and environmental prospects\\nof RE, as well as a load management system by which\\nutilities could efﬁciently manage customer load demand.\\nIrandoust [47] applied a vector autoregression (VAR) model\\nto examine the relationship between RE consumption, tech-\\nnological innovation, economic growth, and CO2 emissions\\nin the four Nordic countries. In a study [48] to decar-\\nbonize electricity renewables, nuclear energy and fossil fuels\\nwere compared regarding cost, emissions reductions, and\\nenergy security for power systems of Great Britain (England,\\nWales and Scotland). To promote a clean environment by\\nreducing GHG emissions and dependence on fossil fuels,\\nPark and Kim [49] have identiﬁed different measures, for\\nexample: (1) consideration of subjecting ocean renewable\\nenergy development to strategic environmental assessment;\\n(2) new standards; (3) integration of the environmental impact\\nassessment with the procedures for consultation on utilization\\nof sea areas; and (4) reinforcement of post-development envi-\\nronmental monitoring processes.\\nCompared to developed countries, energy crises (partic-\\nularly electricity crises), are profound in underdeveloped\\ncountries. A study presented by Shakeel et al. [50] deﬁned\\na roadmap by which to overcome energy crises in Pakistan\\nby integrating RES in the power generation. To meet the\\nneed for energy, a review study by [51] recognized the heavy\\ndependency on fossil fuels in rural areas of Iran. This study\\nsuggested that, due to dependency on fossil fuels, there is\\nconsiderable pollution in the environment. According to the\\nreport [52], a large amount of global electricity produc-\\ntion in 2017 was obtained from fossil fuels. RE production\\n63840\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nFIGURE 2. Estimated share of global electricity production [53].\\nFIGURE 3. Total CO2 emissions from the consumption of energy [53].\\nFIGURE 4. Number of Countries with Renewable Energy Policies [53].\\ncomprises a total of 26 percent (26%) of electricity from\\nsources such as hydropower, biopower, solar PV and Geother-\\nmal. This is depicted in Figure 2 following. Figure 2 presents\\nthe contribution of electricity for the year 2017 at a global\\nlevel. Due to the large-scale use of fossil fuels, CO2 emis-\\nsions are rising exponentially. Figure 3 presents the CO2\\nemissions for the period 2000-2017 [53]. In order to control\\nCO2 emissions, a number of policies have been introduced\\nglobally promoting the use of clean energy sources, such\\nas RE. Figure 4 presents the number of countries which have\\ncontributed in carrying out these policies for the use of RES\\nduring the period 2014 to 2017. These policies include power\\npolicies, heating and cooling policies, and transport policies.\\nAs reported [54], at the end of 2017, global renewable\\ngeneration capacity amounted to 2,179 GW. Accordingly,\\nhydro energy is noted as having the largest share of the global\\ntotal, with an installed capacity of 1,152 GW. Wind and solar\\nenergy accounted for most of the remainder, with capaci-\\nties of 514 GW and 397 GW respectively. Other renewable\\nsources included 109 GW of bioenergy, 13 GW of geothermal\\nenergy and 500 MW of marine energy (tide, wave and ocean\\nenergy).\\nRenewable capacity growth continues to be driven mostly\\nby new installations of solar and wind energy, which together\\naccounted for 85% of all new capacity installed in 2017. The\\nrapid improvement in RE capacity is favorable for, namely:\\neconomic growth and job creation; limiting carbon emissions;\\nreduction of air pollution; expansion of energy access; and\\nimprovement of energy security.\\n2) RQ 2: WHAT ARE THE TYPES OF REWNEWABLE\\nENERGY SOURCES USED FOR DOMESTIC USAGES?\\nReliable energy sources such as RES can play a vital role\\nin improving standard of life [44]. Solar thermal energy is the\\nmost abundant type of RE and is available in both directions\\nas well as in indirect forms. Solar energy has a vast scope for\\nthermal applications such as: solar water heaters [55], [56];\\ncooling systems [57], solar home systems [58], [59]; cookers;\\nand refrigeration. These can meet the needs of a family and\\nachieve a sustainable future [60].\\nSolar thermal-based technologies are extensively used to\\nproduce hot water in several countries [61], [62]. They are\\ndesigned as both a closed loop and open loop. The closed\\nloop solar system is suitable for extremely cold areas [63].\\nThe literature presented different studies related to RET\\nfor speciﬁcally: domestic use; the performance of portable\\nheating systems [64]; experimental investigation of graphene\\nnanoplatelets nanoﬂuid-based volumetric solar collector for\\ndomestic hot water systems [65]; local market ﬂow of solar\\nwater heaters in Taiwan [66]; as well as hybrid renewable\\nenergy system for household applications. These applications\\ninclude: boxed hybrid solar cooker designed by photovoltaic\\nand thermal technologies [67]; parabolic-type portable solar\\ncooker [68]; boxed-type solar cooker using internal reﬂec-\\ntor [69]; design and installation of cooling and heating water\\nsystems [70]; in addition to a novel opaque roof solar chimney\\nconﬁguration at home [71].\\nIn order to minimize the electricity consumption of a\\nhousehold and fulﬁl the energy demand for a single-family\\ndetached house, RES applications (solar and biomass) are\\nused [72], [73]. A vast amount of literature is available on the\\nsubject of usage of the artiﬁcial neural network [74]–[76] for\\nsolar radiation prediction to produce effective solar products.\\nAll of these gadgets, including solar cookers, water heaters,\\nair conditioners, and dryers are potential means of energy\\nconservation in developing nations. A report [52] shows that\\nin the year 2015, Bangladesh stood in top position for the use\\nof solar home systems. An analysis of the energy consump-\\ntion of households [56], found that total household energy\\nconsumption could be reduced by around 13% with the\\npresence of a solar water heater. In order to improve the use\\nof solar energy on a domestic scale, the study [77] explored\\nthe potential of the hybrid solar Photovoltaic/Thermal (PV/T)\\ncollector integrated with a thermochemical sorption thermal\\nVOLUME 7, 2019\\n63841\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nFIGURE 5. Global capacity of solar heating collectors for 2007-2017 [54].\\nstorage\\nsystem\\nfor\\na\\nhot\\nwater\\nheater.\\nSolar\\nhome\\nsystems (SHS), e.g. LED lamps and energy efﬁcient multi-\\ncooker innovations, are used to provide a solution for domes-\\ntic energy poverty in developing countries [78]. Another\\nstudy [79] described that the solar photovoltaic system can be\\nused as a supplementary source by which to recharge batteries\\nfor home energy management systems (HEMS).\\nThese renewable resources are considered as being safe\\nfor health and also inexpensive, which is another motivation\\nfor the public to utilize them at home. These RES are used\\nin a variety of forms for domestic usage. For example, RES\\n(i.e. sun) is utilized for solar home systems, solar dryers, solar\\ncookers, solar refrigeration, solar heaters, cooling systems\\nand power generation. Similarly, wind power is utilized for\\nwindmills, water pumps and wind generators; while biomass\\nis utilized for heat and power generation. Figure 5 presents\\nthe global capacity of solar heating collectors during the\\nperiod 2007 -2017. Solar thermal heating and cooling systems\\nserved millions of residential as well as commercial clients\\nin 2017. Solar thermal technology was used for a wide range\\nof applications such as: hot water; space heating and cooling;\\nproduct drying; water desalination; and cooking services.\\nThe systems with glazed and unglazed collectors provided\\n388 TWh (1,397 PJ) of heat annually by the end of 2017.\\nThe most signiﬁcant application for solar thermal systems is\\nthat of domestic hot water heating. This type of application is\\nconsidered as being environmentally-friendly thereby help-\\ning to avoid CO2. Figure 6 presents the results for worldwide\\nuse of thermal energy in 2017 for heating of domestic hot\\nwater. Mainly, the hot water is used by small-scale systems\\nin a single-family house (67%) and large applications such as\\nmulti-family houses, hotels, schools, etc. (27%). The heating\\nshare of swimming pools was noted as 4% while 2 % was\\nused for solar combi-systems.\\nIn the following section, we will explain the opinions of\\nthe public regarding the acceptance of RES and RET at the\\ndomestic level.\\n3) RQ 3: WHAT ARE THE PUBLIC OPINIONS ON\\nRENEWABLE ENERGY TECHNOLOGIES?\\nOur literature review found that the use of RE is essential\\nin order to meet future energy-related challenges. To achieve\\nthis, awareness of RET and RES is crucial to increase\\nFIGURE 6. The share of the hot water system at a domestic and large\\nscale [80].\\npublic acceptance on these clean energy resources [16], [81].\\nTo boost the acceptance of RES and to avoid the critical\\nproblem of global warming (due to the energy provided by\\nfossil fuels), analysis of public opinions towards RES is\\nvital [82]. In this situation, RE is a good replacement for\\nkeeping the environment clean [83]. A survey was conducted\\nin Turkey among secondary school students to determine\\ntheir interest in RE-based systems. The vast majority of the\\nstudents were familiar with solar and wind plants as RES to\\nproduce electricity. These students were aware of the issue of\\nglobal warming and, in their opinion, the use of RE would\\nreduce this [84]. The opinion of the public on the use of\\nRET was obtained through a survey study in Portugal [85].\\nIt was observed that Portuguese are generally supportive of\\nthe development of more RE projects in their country, but they\\nare not familiar with all RES such as biomass. Among the\\ndeveloping countries, Malaysia is signiﬁcantly contributing\\nto achieving a sustainable environment and reducing climate\\nchange. However, based on quantitative analysis, it was per-\\nceived that the people in Peninsular Malaysia have a nega-\\ntive attitude towards the use of RET [86]. There are cases\\nwhere local residents are reluctant to allow the introduction of\\nRE projects such as wind farms. A case study in Japan\\nrevealed that silent local respondents are not willing to allow\\nwind farms in their backyards [87]. China is the world’s\\nlargest emitter of GHG, and Chinese are aware that global\\nefforts to stop climate change depends largely on China’s\\npolicies. A survey was conducted among 2086 Chinese\\ninternet users, and great support was received [88]. Public\\nopinion on installing a project for offshore wind power in\\nMassachusetts was supportive [89]. An upcoming trend in\\nCanada is the establishment of a community energy plan,\\nwhereby decisions are considered at the community level.\\nAn analysis of 10 of the ﬁrst community energy plans in\\nCanadian communities found that communities are selecting\\npolicies and programs centered on increasing energy efﬁ-\\nciency, while RE receives much less attention [90]. A survey\\nbased on a random sample of residents in the state of Maine,\\nUSA found that they preferred speciﬁc types of RES [91].\\nIn some cases, it was noted that demographic parameters\\ninﬂuence the acceptability or the decline of RES. Female\\nrespondents generally sounded more appreciative towards the\\n63842\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nacceptability of wind energy as compared to male respon-\\ndents [92]. A choice experiment was conducted in two Greek\\nAegean islands (Naxos and Skyros) to attempt to determine\\nthe resistance factors to installing RES. It was deduced that\\nthe majority of respondents perceived institutional factors as\\nmore important for acceptance than the physical attributes of\\nwind farms [93]. The results showed that social acceptance\\nchanged rather than diminished when entrepreneurs extend\\na project’s focus from biopower to smart biomass use [94].\\nA survey study on people of different age groups was con-\\nducted in Finland to obtain an opinion on renewable sources\\nand technologies [95]. The study found that less than 50%\\nof people were willing to install wind turbines in their back-\\nyards or solar panels on their roofs. Analysis of [96] study\\nshows that users do not understand the energy consumption\\nof their water heaters and they do not know how to control\\ntheir efﬁciency. The study [97] reveals considerable differ-\\nences in perception depending on familiarity and involvement\\nwith energy sources, environmental friendliness, and speciﬁc\\nenvironmental impacts respectively.\\nAnalyses of public opinion by [98] regarding climate\\nchange and RE in Peninsular Malaysia have proved that the\\nmajority of Malaysians are adequately learned about RE and\\nare genuinely concerned about climate change. However,\\nthe cost of RE is a signiﬁcant barrier to the acceptance of\\nRET technologies. The study [99] results revealed that the\\nKorean public’s level of acceptability is high regarding RES\\nsource expansion policy. However, the education level of\\nrespondents had a signiﬁcant effect on the preference for\\nan increase of RES. In the analysis of public perceptions of\\nRET acceptance, the study [100] deduced that education is\\nparticularly relevant for justifying economic, environmental\\nand social perceptions and these are also signiﬁcant vari-\\nables for the acceptance of the technology. In particular,\\nthe respondent’s attitude towards risk can play a major role for\\neach respondent’s willingness to accept new RET. Another\\nstudy [60] has analyzed public opinion by using both qual-\\nitative and quantitative methods and found that there is a\\nrelationship between RE usage and environmental education.\\nThe respondents, however, generally did not agree with the\\ncomplete transfer towards RE. The ﬁndings of a study [101]\\nshowed that public acceptance is highly dependent on the\\ntechnology in question, the dimension of social acceptance\\n(community versus socio-political dimension) and previous\\nexperiences with RE.\\nFigure 7 explains the sources for public opinion and the\\nsources of RE on which public opinion in this review study\\nis based. Table 3 summarizes the results obtained from\\nreviewed studies, which discussed the opinions of the public\\non the acceptance of RES and RET. Table 4 highlights fre-\\nquently found barriers to the acceptance of RE, for example:\\nresources of interest; policies of interest; noise; lack of knowl-\\nedge and awareness; economic conditions; demographics;\\nand climatic conditions.\\nIn this review, public’s opinions on the acceptance of RES\\nin the last 9 years, from 2009 to 2018, are discussed. It is\\nFIGURE 7. Opinions & renewable energy sources.\\nnoted that the lack of knowledge is the most inﬂuential factor\\nin preventing the public from accepting RE.\\nIV. ANALYSIS OF OPINIONS REGARDING RENEWABLE\\nENERGY SHARED ON SOCIAL MEDIA\\nWe have performed an analysis of opinions related to\\nRE received through Twitter to provide an overview of social\\nmedia towards current practices of RE.\\nA. EXAMPLE FROM TWITTER\\nTwitter provides useful real-time information on a variety\\nof topics ranging from social issues to serious topics and\\nalso entertaining ones. Tweet feeds are handled by different\\napproaches so as to obtain useful information e.g. handle\\nfuture outbreaks in real time [103]. These tweets are encapsu-\\nlated with multiple types of opinions inside. Opinion-mining\\nstudies have focused on and explored three types of opin-\\nions including: 1. Regular; 2. Comparative; and 3 Suggestive\\nopinions [29]. Based on the opinion data results available on\\nsocial media (Twitter) related to RE, it was observed that\\nusers’ perceptions about RE are conveyed with signiﬁcant\\nopinion types. We have found that there are large numbers of\\ntweets available related to sustainable and RE sources. This\\nprovides an overview that social media is productive towards\\ngenerating awareness on various events [104] that may help in\\npromoting RE. As discussed in the above sections, awareness\\nof public opinion is a crucial attribute for a successful out-\\ncome of RE acceptance. Results of social media data analysis\\nshows that it is a very useful tool when it comes to pub-\\nlic relations issues. In addition to the advantages discussed\\nabove, social media is signiﬁcantly cheaper than traditional\\nadvertising. In addition, it is a mode of communication which\\nallows getting in touch with people worldwide anytime and\\nsharing their opinions about RE.\\n1) ETHICS\\nNo ethical matters were perceived as being essential for\\nthis study, as there was no involvement with any human data\\nbeyond measuring internet activity among Twitter users.\\nVOLUME 7, 2019\\n63843\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nTABLE 3. Public opinions for attitudes & promotion clues of res & RET’s.\\n1) DATA ACQUISITION TOOLS AND ANALYSIS OF\\nTWEETS\\nTweet extractions took place using the NodeXL the Twitter\\nImporter module [105]. The extracted opinions or tweets\\nwere parsed to remove the HTML formatting from the text\\nand then transformed into an XML ﬁle that separated the data\\ninto records (the opinion) and ﬁelds (the data in each tweet).\\nWe have labeled the tweets into multiple classes (A, B and C)\\nbased on a linguistic construct using a similar method to card\\nsorting [106]. The regular reviews are labeled A, comparative\\nas B, and suggestive reviews as C [26]. The aim of this\\nexample is to explain the qualitative measures associated with\\ntweets related to RE and monitor the current impressions of\\npublic perceptions during the time of this review study.\\n63844\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nTABLE 4. Identified barriers and challenges for acceptance of renewable energy sources.\\nThere was a time restriction which meant that the obser-\\nvational study of Twitter [107] was performed during the\\nlast month of the study, e.g. from 1st December 2018 to\\n31st December 2018. It was carried out in different time\\nslots using one of the authors’ Twitter accounts. There were\\nthree popular RE information sources, namely, (1) The Inter-\\nnational renewable energy agency (IRENA); (2) Renewable\\nenergy policy network for the 21st century (REN21); and (3)\\nInternational energy agency (IEA). The selection of three dif-\\nferent information sources e.g. IRENA, REN21 and IEA also\\nsatisﬁed the requirement of triangulation that is commonly\\nused in qualitative research [108].\\nWe picked up on the top few tweets that were used e.g.,\\n#Renewable energy hashtag and performed manual annota-\\ntion until we received a variety of review types related to RE.\\nFurther, for detailed analysis total, nine tweets were selected;\\nthree tweets which were against each source and that were\\nlarge in size and had gained more ‘‘likes’’. As likes and\\nhelpfulness\\nindication\\nshow\\nthe\\nimportance\\nof\\nan\\nopinion [25], we therefore excluded the tweets that had an\\ninsufﬁcient number or no likes on it. The detailed analysis\\nof these tweets provides useful information that divides them\\ninto different classes (see Table 5).\\nThe tweets were collected with time and date stamp e.g.\\nfrom IRENA (2:10 PM -24 Dec 2018, 4:44 AM-28 Dec\\n2018,10:10 AM-31 Dec 2018), REN21 (10:59 PM -18 Nov\\n2018, 10:15 22 PM-7 Dec2018, 2:20 AM-10 Dec 2018)\\nand IEA (2:04 AM -26 Dec 2018 3:10 AM 30-Dec 2018,\\n10:01 PM- 24 Dec-2018). This analysis showed that people\\nposted tweets frequently in different time spans and, overall,\\nthere is a supportive trend towards RE and future adoption\\nof RES. This information will open ways to bring up innova-\\ntive ideas of research in the ﬁeld.\\nV. DISCUSSION\\nDriven by the requirement for world energy demand and clean\\nenergy, research on RE has uncovered a high potential to\\ncreate a sustainable environment and meet today’s need of\\nVOLUME 7, 2019\\n63845\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nTABLE 5. The analysis of three #renewable energy tweets.\\nenergy [109]. In this review study, 42 papers were critically\\nexamined in an attempt to explain the importance of RE and\\nRET practices at a domestic level in addition to public opinion\\non the adaptation of RE. With reference to a hybrid renewable\\nenergy integration system [46], results revealed that the large-\\nscale renewable energy integration system (wind/PV/grid-\\nconnected) is more economically viable in contrast to a\\ngrid-connected-only system because of high upfront costs.\\nThe study [47] using the Granger non-causality test for\\nfour Nordic countries reveals unidirectional causality running\\nfrom RE to CO2 emissions for Denmark and Finland and\\nbidirectional causality between these variables for Sweden\\nand Norway. Interestingly, the results could not conﬁrm any\\ncausality from RE to growth. In terms of costs, emissions\\nand energy security, the study [48] revealed that the cost for\\nrenewables, nuclear energy and fossil fuels are essentially\\nthe same; whereas energy security of up to 60% of variable\\nrenewable capacity is possible with little cost increase. Large\\namounts of emissions are prominent with the use of fossil\\nfuels. In relation to a clean environment, a study by [49]\\nhas identiﬁed several issues which it addressed in the current\\nenvironmental assessment systems. In the case of developing\\ncountries being able to meet the challenges of energy crises\\n(particularly electricity shortages), a study [50] indicated\\nseveral issues such as lack of foresight, ﬂawed policies,\\npoor decision-making and lack of seriousness at a higher\\nlevel. It is stated that currently, a mix of energy is skewed\\ntowards the thermal, where hydrocarbons constitute 87% of\\ntotal primary energy supply that ultimately increases GHG\\nemissions. To confront such issues, Pakistan needs to switch\\ntowards RES as it has rich land reserves of 800,000km2 with\\nplenty of opportunities for solar, wind, bio and hydro energy.\\nAnother study [51] explains the need for RES to combat the\\nenergy crises in rural areas of Iran. Estimated solar radiation\\nin Iran is between 1800–2200kWh/m2 per year, which is\\nhigher than the global average. Despite having productive\\nland, some infrastructural, managerial, socio-cultural and\\neconomic challenges are seen for the instalment of RES.\\nOne of the proposed solutions to meet these challenges in\\ndeveloping countries is the development of aggressive and\\ninnovative policymaking.\\nWith reference to RET for domestic use, hot water pro-\\nduction was one of the considering factors for public need at\\nthe domestic level [64]. For domestic hot water production,\\ndirect solar with thermal store integration (DSTSI) conﬁg-\\nuration showed high efﬁciency with a demand of 277 kWh\\nthermal energy in order to supply domestic hot water at the\\nrate of 80 L/min [65]. The main diffusion barriers to solar\\nwater heaters (SWH) for households are economy, household\\ncomposition, and the structure of the house for available\\ninstallation location of SWH [66]. For household items pow-\\nered by RES, the solar cooker is perceived as a most important\\nand useful appliance. There are three different types of solar\\ncookers, speciﬁcally: solar panel cookers; solar box cookers;\\nand parabolic solar cookers. A box-type hybrid solar cooker\\nhas been speciﬁcally designed for a small family, by keeping\\nit small in size and light in weight, weighing 4.8 kg [67].\\nIt was deduced that the combination of thermal and photo-\\nvoltaic energy reveals 38% higher efﬁciency as compared to\\nthe baseline model. It comprises attractive features includ-\\ning being: user-friendly; convenient (unattended cooking is\\nallowed at any time); fast-cooking; affordable; able to cook\\n4-5 times meals/day. A portable parabolic-type solar cooker\\nhas been designed with reused metallic cardboard [68]. The\\nparabolic design gives the best result compared to other tested\\ndesigns with an average efﬁciency of about 14-18% as com-\\npared to rough ﬁre. Such portable solar cookers could be eas-\\nily usable in the case of emergencies in humanitarian camps.\\nIn a study [69], a box-type solar cooker was designed and\\nexperimented upon by using internal reﬂectors; higher per-\\nformance was observed compared to the same cooker without\\na reﬂector. A tri-generation system has been developed at\\nthe University of Technology, Sydney (UTS), which delivers:\\n(1) hot water; (2) cold water; and (electricity) to the Faculty\\nof Engineering [70]. This unique tri-generation system uses\\nphotovoltaic panels, ﬂat plate collectors, and wind power as\\ngenerators with thermal and chemical energy storage. For\\nventilation and cooling of a building, a roof solar chimney\\n63846\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\n(RSC) was installed and tested against four different climatic\\nconditions [71]. Results found that a ventilated roof with RSC\\nprovides free cooling and natural ventilation in all four cli-\\nmates and seasons. A numerical investigation was conducted\\non a single family detached house in order to reduce the\\ncost of household electricity in the Philippines [72]. As a\\nresult, it was observed that installing solar photovoltaic panels\\non the rooftop is helpful for electricity consumption. In the\\nstudy [73], it was revealed that by using RES, such as solar\\nenergy and biomass energy, a household could minimize its\\ndependency on both fossil fuels and grid line electricity.\\nMoreover, progress has been made to investigate public\\nopinion on the adoption of RES and RET. In relation to the use\\nof RES and power stations, a supportive survey response was\\nreceived from secondary school students in Turkey [84]. The\\nsuccess behind the supportive response was due to the aware-\\nness of RES and the beneﬁts of RET. Similarly, Portuguese\\nopinions towards the use of RET showed as being positive\\ntowards the acceptance of RES because of awareness of\\nit [85]. According to this survey, the public’s attitude towards\\naccepting hydropower was most welcome as compared to\\nbiomass. It is evident that people do not know much about\\nthe concept of biomass. We argue that public awareness about\\nRES is a signiﬁcant factor in the acceptance of RES and\\nunawareness is observed as a major barrier in acceptance\\nof RES. Although there are RE policies, RES has not yet\\nbeen established in Malaysia [86]. The main reason can be\\nattributed to unawareness and the high subsidy for fossil fuels\\nwhich made Malaysia uncompetitive economically in term of\\nRET technologies. There was a case in Japan on the accep-\\ntance of a presently installed wind farm; however, the accep-\\ntance of a new wind farm was at ﬁrst very low among\\nsilent residents [87]. Possible solutions for the acceptance of\\nsustainable wind farms are: (1) providing opportunities for\\nthe locals to express their opinions; and (2) the developers\\nproviding a response to the locals on their requests during the\\nproject planning phase. These may enhance the possibility of\\nacceptance in future wind farm projects. In terms of current\\nglobal warming, China is the world’s largest emitter of GHS.\\nA signiﬁcant response in the acceptance of RET was reported\\nfrom a survey conducted in China [88]. Interestingly, people\\nwere willing to accept RET because of the cost and security\\nof these technologies, as opposed to reasons of pollution\\nand climate change. It shows that energy security is one of\\nthe prominent aspects in the acceptance of RET. In another\\ncase [90], regarding the usage of RES among Canadian com-\\nmunities, biofuels received greater acceptance compared to\\nwind, passive solar design, solar photovoltaics and solar ther-\\nmal. In the latest survey study in Massachusetts, an offshore\\nwind project received maximum support [89]. The observed\\nminimal rejection can be overcome by providing education\\nabout offshore wind technology. This shows that with grow-\\ning knowledge and awareness of RET, people become more\\ninclined to accept it. Preference and selection of RES make\\na signiﬁcant difference in the acceptance of RE policies\\nfor the development of RET. There is case reporting that,\\nwithout giving preferences to any RES, investments may face\\nrejection by the public who may have a preference on energy\\ntype [91]. The public demographic and their attitude towards\\nusing the beach is a contributing factor to accepting offshore\\nwind farms [92].\\nPeople who visit the beach less frequently sounded more\\nwilling to use offshore wind farms. Despite other factors\\ndiscussed above, welfare is considered as another promis-\\ning aspect in promoting RES. A study [93] among Greek\\nresidents showed that the governance characteristic of the\\nplanning procedure is a signiﬁcant aspect in local community\\nwelfare. The welfare aspect as opposed to the physical aspect\\nplays a vital role in the acceptance of wind farms. A case\\nstudy revealed that the improvement of bioenergy towards\\nsmart biomass is not enough for its acceptance [94]. Peo-\\nple may accept it for the purpose of enhancing energy and\\neconomic efﬁciency levels. Finland received support for RES\\nfrom people with more knowledge on RES as well as age and\\nexperience on home energy issues.\\nMore precisely, based on literature analysis, several\\ninsights concerning the acceptance of RES are provided.\\nFirstly, RES is the ultimate solution to combat energy crises\\nand environmental pollution. Secondly, the public need for\\nhome systems can be resolved by using RE-based tech-\\nnologies instead of fossil fuels. Thirdly, the perceived pub-\\nlic opinions on RES and RET disclose various important\\nfactors that signiﬁcantly inﬂuence the acceptance of RES.\\nThese inﬂuencing factors are well-versed with, speciﬁcally:\\nRE knowledge on RET and RES; demographics; usage of\\ncertain environments where RETs are installed; climatic con-\\nditions; economy; global warming; energy security; and poli-\\ncies embedded with public welfare and involvement in issues\\ninvolving energy. The geographical area also impacts the\\nacceptance of diverse RET; for example, people who live near\\nan island receive a higher amount of wind and solar power\\nthan hydropower, and therefore, they are acknowledged more\\nthan other types of RES.\\nVI. IMPLICATIONS OF THE STUDY\\nThrough the comprehensive analysis of previous and related\\nliterature, this study offers several implications for both\\nresearchers and policymakers regarding RE development.\\nResearchers need to focus more on factors inﬂuencing the\\nintention of people in using RE. Moreover, there is a def-\\ninite need to conduct more research that incorporates RES\\nusing public opinion, which can assist to reduce reservations\\namong the public and promote future use of RES and RET.\\nThe government can make plans to expand RE concepts and\\nintroduce study of this topic in secondary schools and higher\\nlevels of education. A variety of RET should be available\\nthrough government policies and incentives. It is evident\\nthat for the development of RET, government and research\\nand development (R&D) programs play a vital role [110].\\nIn the case of industry, the policymakers can develop\\nattractive policies embedded with the preferred choice of RES\\nby utilizing public opinion. Various public awareness policies\\nVOLUME 7, 2019\\n63847\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\ncan be proposed for the identiﬁed challenges e.g. awareness\\nof RES and RET use. Another important practical implication\\nfor policymakers and government sector is the introduction\\nof welfare programs related to RE projects so as to gain\\na positive response from the public. The implications for\\nacademia are considerable. They can include, for example,\\nthe university sector, where R&D could be working on RE\\nprojects. In this case, it would indeed be important to conduct\\nworkshops for students from diverse disciplines to deliver\\nknowledge on RES and RET. Furthermore, the study high-\\nlights the need to obtain useful information through public\\nopinion that can be used by future researchers for successful\\nacceptance of RE.\\nVII. CONCLUSIONS\\nThe objective of this article was to highlight the importance\\nof RES and the RET. RES such as solar, wind and biomass\\nare mostly used in the manufacturing of domestic products;\\nnamely, windmills to produce electricity, water pumps and\\nheat and power generation, etc. Among all types of renewable\\nenergies, solar energy is available in the majority of the world.\\nThis is the reason for solar energy being the most suitable sub-\\nstitute for fossil fuels and most household products leverage\\nsolar energy in RET. In order to ensure sustainable develop-\\nment for the future generations, it is indeed important to raise\\nawareness about RES. RE can be utilized in making many\\nproducts employed in daily use, such as solar cookers, solar\\ncoolers and heaters, as well as solar dryers. The application\\nof RES such as solar is worldwide. Biomass assists to absorb\\ndangerous gasses such as CO2 and can also be used for\\nelectricity and as a fuel. Wind is another useful RES resource\\nthat can be used to produce electricity and power generation.\\nOur ﬁndings endorse the fact that the success of RE may be\\ngained by providing greater awareness of RES and RET to\\nthe public. This can be achieved by introducing educational\\nprograms related to RE through domestic and international\\nplatforms.\\nThis SLR has been carried out in attempt to focus on\\nRES & RET in light of public opinions, that are indeed impor-\\ntant to promote the development of RES & RET. Even though\\nthe results evidently contribute to the RE literature, the study\\nhas certain some limitations. First, included articles discussed\\nthe need of RE, RE at domestic level and public opinions\\nabout the use of RET through the use of well-established key-\\nwords. However, there may be other relevant keywords that\\nhave not been considered in the present SLR. Second, to cer-\\ntify the study remained focused and quality conscious articles\\nwritten in a language other than English have been excluded.\\nOverall, our analysis is a qualitative interpretation of RES &\\nRET practices among public providing aggregated overview\\nof the research and thus allowing us to systematically identi-\\nfying future research avenues. We hope this systematic review\\ncan assist both academia and industry to promote renewable\\nenergy and encourage the analysis of public opinions that\\nare present in Twitter or other social network platforms such\\nas LinkedIn.\\nREFERENCES\\n[1] F. Rizzi, N. J. van Eck, and M. Frey, ‘‘The production of scientiﬁc knowl-\\nedge on renewable energies: Worldwide trends, dynamics and challenges\\nand implications for management,’’ Renew. Energy, vol. 62, pp. 657–671,\\nFeb. 2014.\\n[2] D. C. Momete, ‘‘Analysis of the potential of clean energy deployment in\\nthe European Union,’’ IEEE Access, vol. 6, pp. 54811–54822, 2018.\\n[3] A. Raheem et al., ‘‘Renewable energy deployment to combat energy crisis\\nin Pakistan,’’ Energy, Sustainab. Soc., vol. 6, no. 1, p. 16, 2016.\\n[4] A. Ashfaq and A. Ianakiev, ‘‘Features of fully integrated renewable\\nenergy atlas for Pakistan; wind, solar and cooling,’’ Renew. Sustain.\\nEnergy Rev., vol. 97, pp. 14–27, Dec. 2018.\\n[5] N. A. Ludin et al., ‘‘Prospects of life cycle assessment of renewable\\nenergy from solar photovoltaic technologies: A review,’’ Renew. Sustain.\\nEnergy Rev., vol. 96, pp. 11–28, Nov. 2018.\\n[6] J. A. Paravantis, E. Stigka, G. Mihalakakou, E. Michalena, J. M. Hills, and\\nV. Dourmas, ‘‘Social acceptance of renewable energy projects: A contin-\\ngent valuation investigation in Western Greece,’’ Renew. Energy, vol. 123,\\npp. 639–651, Aug. 2018.\\n[7] Q. Xu, P. Lan, B. Zhang, Z. Ren, and Y. Yan, ‘‘Energy sources, part\\nA: Recovery, utilization, and environmental effects,’’ Energy Sources,\\nvol. 35, pp. 848–858, Mar. 2013.\\n[8] A. M. Saleh, A. Bin Haris, and N. B. Ahmad, ‘‘Towards a UTAUT-based\\nmodel for the intention to use solar water heaters by Libyan households,’’\\nInt. J. Energy Econ. Policy, vol. 4, no. 1, pp. 26–31, Dec. 2014.\\n[9] M. Qu, P. Ahponen, L. Tahvanainen, D. Gritten, B. Mola-Yudego,\\nand P. Pelkonen, ‘‘Chinese university students’ knowledge and atti-\\ntudes regarding forest bio-energy,’’ Renew. Sustain. Energy Rev., vol. 15,\\npp. 3649–3657, Oct. 2011.\\n[10] H. Lucas, S. Pinnington, and L. F. Cabeza, ‘‘Education and training gaps\\nin the renewable energy sector,’’ Sol. Energy, vol. 173, pp. 449–455,\\nOct. 2018.\\n[11] P. Trop and D. Goricanec, ‘‘Comparisons between energy carriers’ pro-\\nductions for exploiting renewable energy sources,’’ Energy, vol. 108,\\npp. 155–161, Aug. 2015.\\n[12] F. Fornara, P. Pattitoni, M. Mura, and E. Strazzera, ‘‘Predicting intention\\nto improve household energy efﬁciency: The role of value-belief-norm\\ntheory, normative and informational inﬂuence, and speciﬁc attitude,’’\\nJ. Environ. Psychol., vol. 45, pp. 1–10, Mar. 2016.\\n[13] A. Mardani et al., ‘‘Sustainable and renewable energy: An overview\\nof the application of multiple criteria decision making techniques and\\napproaches,’’ Sustainability, vol. 7, no. 10, pp. 13947–13984, Oct. 2015.\\n[14] T. Kousksou, P. Bruel, A. Jamil, T. El Rhaﬁki, and Y. Zeraouli, ‘‘Energy\\nstorage: Applications and challenges,’’ Sol. Energy Mater. Sol. Cells,\\nvol. 120, pp. 59–80, Jan. 2014.\\n[15] E. K. Stigka, J. A. Paravantis, and G. K. Mihalakakou, ‘‘Social acceptance\\nof renewable energy sources: A review of contingent valuation applica-\\ntions,’’ Renew. Sustain. Energy Rev., vol. 32, pp. 100–106, Apr. 2014.\\n[16] K. P. Tsagarakis et al., ‘‘Clean vs. Green: Redeﬁning renewable\\nenergy. Evidence from Latvia, Lithuania, and Romania,’’ Renew. Energy,\\nvol. 121, pp. 412–419, Jun. 2018.\\n[17] M. R. Borovik and J. D. Albers, ‘‘Participation in the Illinois solar\\nrenewable energy market,’’ Electr. J., vol. 31, pp. 33–39, Mar. 2018.\\n[18] N. Zografakis, E. Sifaki, M. Pagalou, G. Nikitaki, V. Psarakis, and\\nK. P. Tsagarakis, ‘‘Assessment of public acceptance and willingness to\\npay for renewable energy sources in Crete,’’ Renew. Sustain. Energy Rev.,\\nvol. 14, no. 3, pp. 1088–1095, Apr. 2010.\\n[19] O. Bayulgen and S. Benegal, ‘‘Green priorities: How economic frames\\naffect perceptions of renewable energy in the United States,’’ Energy Res.\\nSoc. Sci., vol. 47, pp. 28–36, Jan. 2018.\\n[20] K.\\nM.\\nKeramitsoglou,\\nR.\\nC.\\nMellon,\\nM.\\nI.\\nTsagkaraki,\\nand\\nK. P. Tsagarakis, ‘‘Clean, not green: The effective representation of\\nrenewable energy,’’ Renew. Sustain. Energy Rev., vol. 59, pp. 1332–1337,\\nJun. 2016.\\n[21] C. Bhowmik, S. Bhowmik, A. Ray, and K. M. Pandey, ‘‘Optimal green\\nenergy planning for sustainable development: A review,’’ Renew. Sustain.\\nEnergy Rev., vol. 71, pp. 796–813, May 2017.\\n[22] P. Burstein, ‘‘Bringing the public back in: Should sociologists consider the\\nimpact of public opinion on public policy?’’ Soc. Forces, vol. 77, no. 1,\\npp. 27–62, Sep. 1998.\\n[23] A. Qazi, R. Raj, G. Hardaker, and C. Standing, ‘‘A systematic literature\\nreview on opinion types and sentiment analysis techniques: Tasks and\\nchallenges,’’ Internet Res., vol. 27, no. 3, pp. 608–630, 2017.\\n63848\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\n[24] A. Qazi, A. Tamjidyamcholo, R. G. Raj, G. Hardaker, and C. Standing,\\n‘‘Assessing consumers’ satisfaction and expectations through online opin-\\nions: Expectation and disconﬁrmation approach,’’ Comput. Hum. Behav.,\\nvol. 75, pp. 450–460, Oct. 2017.\\n[25] A. Qazi, K. B. S. Syed, R. G. Raj, E. Cambria, M. Tahir, and\\nD. Alghazzawi, ‘‘A concept-level approach to the analysis of online\\nreview helpfulness,’’ Comput. Hum. Behav., vol. 58, pp. 75–81,\\nMay 2016.\\n[26] A. Qazi, R. G. Raj, M. Tahir, E. Cambria, and K. B. S. Syed, ‘‘Enhancing\\nbusiness intelligence by means of suggestive reviews,’’ Sci. World J.,\\nvol. 2014, no. 1, Jan. 2014, Art. no. 879323.\\n[27] W. You, Y. Guo, and C. Peng, ‘‘Twitter’s daily happiness sentiment and\\nthe predictability of stock returns,’’ Finance Res. Lett., vol. 23, pp. 58–64,\\nNov. 2017.\\n[28] B. Liu and L. Zhang, ‘‘A survey of opinion mining and sentiment\\nanalysis,’’ in Mining Text Data. Boston, MA, USA: Springer, 2012,\\npp. 415–463.\\n[29] A. Qazi, R. G. Raj, M. Tahir, S. U. R. Khan, A. Abraham, and M. Waheed,\\n‘‘A preliminary investigation of user perception and behavioral intention\\nfor different review types: Customers and designers perspective,’’ Sci.\\nWorld, vol. 2014, Feb. 2014, Art. no. 872929.\\n[30] B. K. Sovacool, ‘‘What are we doing here? Analyzing ﬁfteen years of\\nenergy scholarship and proposing a social science research agenda,’’\\nEnergy Res. Soc. Sci., vol. 1, pp. 1–29, Mar. 2014.\\n[31] J. R. Ragini, P. M. R. Anand, and V. Bhaskar, ‘‘Big data analytics for\\ndisaster response and recovery through sentiment analysis,’’ Int. J. Inf.\\nManage., vol. 42, pp. 13–24, Oct. 2018.\\n[32] A. Qazi, H. Fayaz, A. Wadi, R. G. Raj, N. A. Rahim, and W. A. Khan,\\n‘‘The artiﬁcial neural network for solar radiation prediction and designing\\nsolar systems: A systematic literature review,’’ J. Cleaner Prod., vol. 104,\\npp. 1–12, Oct. 2015.\\n[33] A. McCabe, D. Pojani, and A. B. van Groenou, ‘‘The application of\\nrenewable energy to social housing: A systematic review,’’ Energy Policy,\\nvol. 114, pp. 549–557, Mar. 2018.\\n[34] B. Kitchenham, O. P. Brereton, D. Budgen, M. Turner, J. Bailey, and\\nS. Linkman, ‘‘Systematic literature reviews in software engineering—\\nA systematic literature review,’’ Inf. Softw. Technol., vol. 51, pp. 7–15,\\nJan. 2008.\\n[35] B. Kitchenham, ‘‘Procedures for performing systematic reviews,’’ Dept.\\nSoftw. Eng., Keele Univ., Keele, U.K., vol. 33, 2004, pp. 1–26.\\n[36] S. Elo and H. Kyngäs, ‘‘The qualitative content analysis process,’’ J. Adv.\\nNursing, vol. 62, no. 1, pp. 107–115, Apr. 2008.\\n[37] M. C. Paik, J. L. Fleiss, and B. Levin, ‘‘The measurement of interrater\\nagreement,’’ in Statistical Methods for Rates and Proportions, 3rd ed.\\nHoboken, NJ, USA: Wiley, 2004.\\n[38] J. R. Landis and G. G. Koch, ‘‘The measurement of observer agreement\\nfor categorical data,’’ Biometrics, vol. 33, no. 1, pp. 159–174, 1977.\\n[39] I. Inayat, S. Salim, S. Marczak, and M. Daneva, ‘‘A systematic literature\\nreview on agile requirements engineering practices and challenges,’’\\nComput. Hum., vol. 51, pp. 915–929, Oct. 2015.\\n[40] B. Metz, O. R. Davidson, P. R. Bosch, R. Dave, and L. A. Meyer,\\nContribution of Working Group III to the Fourth Assessment Report\\nof the Intergovernmental Panel on Climate Change. Cambridge, U.K.:\\nCambridge Univ. Press, 2007.\\n[41] F. Zabihian and A. Fung, ‘‘Fuel and GHG emission reduction potentials\\nby fuel switching and technology improvement in the Iranian electricity\\ngeneration sector,’’ Int. J. Eng., vol. 3, no. 2, p. 159, 2009.\\n[42] S. R. Sharvini, Z. Z. Noor, C. S. Chong, L. C. Stringer, and R. O. Yusuf,\\n‘‘Energy consumption trends and their linkages with renewable energy\\npolicies in East and Southeast Asian countries: Challenges and opportu-\\nnities,’’ Sustain. Environ. Res., vol. 28, pp. 257–266, Nov. 2018.\\n[43] F. Behrouzi, M. Nakisa, A. Maimun, and Y. M. Ahmed, ‘‘Global\\nrenewable energy and its potential in Malaysia: A review of Hydroki-\\nnetic turbine technology,’’ Renew. Sustain. Energy Rev., vol. 62,\\npp. 1270–1281, Sep. 2016.\\n[44] G. M. Shaﬁullah, M. T. O. Amanullah, A. B. M. S. Ali, D. Jarvis, and\\nP. Wolfs, ‘‘Prospects of renewable energy—A feasibility study in the\\nAustralian context,’’ Renew. Energy, vol. 39, no. 1, pp. 183–197, 2012.\\n[45] J.-J. Andreas, C. Burns, and J. Touza, ‘‘Renewable energy as a luxury?\\nA qualitative comparative analysis of the role of the economy in the EU’s\\nrenewable energy transitions during the ‘double crisis,’’’ Ecol. Econ.,\\nvol. 142, pp. 81–90, Dec. 2017.\\n[46] G. M. Shaﬁullah, ‘‘Hybrid renewable energy integration (HREI) system\\nfor subtropical climate in Central Queensland, Australia,’’ Renew. Energy,\\nvol. 96, pp. 1034–1053, Oct. 2016.\\n[47] M. Irandoust, ‘‘The renewable energy-growth nexus with carbon emis-\\nsions and technological innovation: Evidence from the Nordic countries,’’\\nEcol. Indicators, vol. 69, pp. 118–125, Oct. 2016.\\n[48] S. Pfenninger and J. Keirstead, ‘‘Renewables, nuclear, or fossil fuels?\\nScenarios for Great Britain’s power system considering costs, emissions\\nand energy security,’’ Appl. Energy, vol. 152, pp. 83–93, Aug. 2015.\\n[49] J.-I. Park and T. Kim, ‘‘Institutional improvement measures for environ-\\nmental assessment in the pursuit of eco-friendly ocean renewable energy\\ndevelopment in South Korea,’’ Renew. Sustain. Energy Rev., vol. 58,\\npp. 526–536, May 2016.\\n[50] S. R. Shakeel, J. Takala, and W. Shakeel, ‘‘Renewable energy sources\\nin power generation in Pakistan,’’ Renew. Sustain. Energy Rev., vol. 64,\\npp. 421–434, Oct. 2016.\\n[51] N. Afsharzade, A. Papzan, M. Ashjaee, S. Delangizan, S. Van Passel, and\\nH. Azadi, ‘‘Renewable energy development in rural areas of Iran,’’ Renew.\\nSustain. Energy Rev., vol. 65, pp. 743–755, Nov. 2016.\\n[52] Renewables 2015: Global Status Report, Secretariat Renew. Energy Pol-\\nicy Netw. 21st Century (REN21), Paris, France, 2015\\n[53] International Energy Statistics, Int. Energy Agency, Paris, France, 2018.\\n[54] Renewable Capacity Highlights, Int. Renew. Energy Agency, Abu Dhabi,\\nUnited Arab Emirates, 2018.\\n[55] H. S. Xue, ‘‘Experimental investigation of a domestic solar water heater\\nwith solar collector coupled phase-change energy storage,’’ Renew.\\nEnergy, vol. 86, pp. 257–261, Feb. 2016.\\n[56] E. Aydin, P. Eichholtz, and E. Yönder, ‘‘The economics of residential solar\\nwater heaters in emerging economies: The case of Turkey,’’ Energy Econ.,\\nvol. 75, pp. 285–299, Sep. 2018.\\n[57] I. B. Tijani, A. A. A. Al Hamadi, K. A. S. S. Al Naqbi,\\nR. I. M. Almarzooqi, and N. K. S. R. Al Rahbi, ‘‘Development of an\\nautomatic solar-powered domestic water cooling system with multi-stage\\nPeltier devices,’’ Renew. Energy, vol. 128, pp. 416–431, Dec. 2018.\\n[58] E.\\nAl-Hassan,\\nH.\\nShareef,\\nM.\\nM.\\nIslam,\\nA.\\nWahyudie,\\nand\\nA. A. Abdrabou, ‘‘Improved smart power socket for monitoring\\nand controlling electrical home appliances,’’ IEEE Access, vol. 6,\\npp. 49292–49305, 2018.\\n[59] M. Kurata, N. Matsui, Y. Ikemoto, and H. Tsuboi, ‘‘Do determinants\\nof adopting solar home systems differ between households and micro-\\nenterprises? Evidence from rural Bangladesh,’’ Renew. Energy, vol. 129,\\npp. 309–316, Dec. 2018.\\n[60] W. Al-Marri, A. Al-Habaibeh, and M. Watkins, ‘‘An investigation into\\ndomestic energy consumption behaviour and public awareness of renew-\\nable energy in Qatar,’’ Sustain. Cities Soc., vol. 41, pp. 639–646,\\nAug. 2018.\\n[61] J. G. Rogers, M. C. McManus, and S. J. G. Cooper, ‘‘Potential for\\nreliance on solar water heating throughout the summer in northern cloudy\\nclimates,’’ Energy Buildings, vol. 66, pp. 128–135, Nov. 2013.\\n[62] G.\\nTévar,\\nA.\\nGómez-Expósito,\\nA.\\nArcos-Vargas,\\nand\\nM. Rodríguez-Montañés, ‘‘Inﬂuence of rooftop PV generation on\\nnet demand, losses and network congestions: A case study,’’ Int. J. Elect.\\nPower Energy Syst., vol. 106, pp. 68–86, Mar. 2019.\\n[63] P. Veeraboina and G. Y. Ratnam, ‘‘Analysis of the opportunities and\\nchallenges of solar water heating system (SWHS) in India: Estimates from\\nthe energy audit surveys & review,’’ Renew. Sustain. Energy Rev., vol. 16,\\nno. 1, pp. 668–676, 2012.\\n[64] N. T. U. Kumar, G. Mohan, and A. Martin, ‘‘Performance analysis of solar\\ncogeneration system with different integration strategies for potable water\\nand domestic hot water production,’’ Appl. Energy, vol. 170, pp. 466–475,\\nMay 2016.\\n[65] M. Vakili, S. M. Hosseinalipour, S. Delfani, S. Khosrojerdi, and\\nM. Karami, ‘‘Experimental investigation of graphene nanoplatelets\\nnanoﬂuid-based volumetric solar collector for domestic hot water sys-\\ntems,’’ Sol. Energy, vol. 131, pp. 119–130, Jun. 2016.\\n[66] K. C. Chang, W. M. Lin, T. S. Lee, and K. M. Chung, ‘‘Local market of\\nsolar water heaters in Taiwan: Review and perspectives,’’ Renew. Sustain.\\nEnergy Rev., vol. 13, no. 9, pp. 2605–2612, Dec. 2009.\\n[67] S. B. Joshi and A. R. Jani, ‘‘Design, development and testing of a small\\nscale hybrid solar cooker,’’ Sol. Energy, vol. 122, pp. 148–155, Dec. 2015.\\n[68] A. Regattieri, F. Piana, M. Bortolini, M. Gamberi, and E. Ferrari,\\n‘‘Innovative portable solar cooker using the packaging waste of human-\\nitarian supplies,’’ Renew. Sustain. Energy Rev., vol. 57, pp. 319–326,\\nMay 2016.\\nVOLUME 7, 2019\\n63849\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\n[69] M. B. Kahsay, J. Paintin, A. Mustefa, A. Haileselassie, M. Tesfay,\\nand B. Gebray, ‘‘Theoretical and experimental comparison of box solar\\ncookers with and without internal reﬂector,’’ Energy Procedia, vol. 57,\\npp. 1613–1622, Dec. 2014.\\n[70] J. P. Osborne, P. Kohlenbach, U. Jakob, J. Dreyer, and J. Kim, ‘‘The design\\nand installation of a combined concentrating power station, solar cool-\\ning system and domestic hot water system,’’ Energy Procedia, vol. 70,\\npp. 486–494, May 2015.\\n[71] J. DeBlois, M. Bilec, and L. Schaefer, ‘‘Simulating home cooling load\\nreductions for a novel opaque roof solar chimney conﬁguration,’’ Appl.\\nEnergy, vol. 112, pp. 142–151, Dec. 2013.\\n[72] N. Enteria, H. Awbi, and H. Yoshino, ‘‘Application of renewable energy\\nsources and new building technologies for the Philippine single family\\ndetached house,’’ Int. J. Energy Environ. Eng., vol. 6, no. 3, pp. 267–294,\\n2015.\\n[73] N. Enteria, H. Yoshino, A. Satake, R. Takaki, H. Ishihara, and S. Baba,\\n‘‘Beneﬁts of utilizing on-site and off-site renewable energy sources for\\nthe single family detached house,’’ Int. J. Energy Environ. Eng., vol. 7,\\nno. 2, pp. 145–166, 2016.\\n[74] S. Alam, S. C. Kaushik, and S. N. Garg, ‘‘Assessment of diffuse solar\\nenergy under general sky condition using artiﬁcial neural network,’’ Appl.\\nEnergy, vol. 86, no. 4, pp. 554–564, 2009.\\n[75] Y. Jiang, ‘‘Computation of monthly mean daily global solar radiation\\nin China using artiﬁcial neural networks and comparison with other\\nempirical models,’’ Energy, vol. 34, no. 9, pp. 1276–1283, 2009.\\n[76] J. Qin, Z. Chen, K. Yang, S. Liang, and W. Tang, ‘‘Estimation of monthly-\\nmean daily global solar radiation based on MODIS and TRMM prod-\\nucts,’’ Appl. Energy, vol. 88, no. 7, pp. 2480–2489, 2011.\\n[77] K. Thinsurat, H. Bao, Z. Ma, and A. P. Roskilly, ‘‘Performance study\\nof solar photovoltaic-thermal collector for domestic hot water use and\\nthermochemical sorption seasonal storage,’’ Energy Convers. Manage.,\\nvol. 180, pp. 1068–1084, Jan. 2019.\\n[78] G. Zubi, G. V. Fracastoro, J. M. Lujano-Rojas, K. El Bakari, and\\nD. Andrews, ‘‘The unlocked potential of solar home systems; an effective\\nway to overcome domestic energy poverty in developing regions,’’ Renew.\\nEnergy, vol. 132, pp. 1425–1435, Mar. 2019.\\n[79] M. Shakeri et al., ‘‘Implementation of a novel home energy management\\nsystem (HEMS) architecture with solar photovoltaic system as supple-\\nmentary source,’’ Renew. Energy, vol. 125, pp. 108–120, Sep. 2018.\\n[80] W. Weiss, M. Spörk-Dür, and F. Mautnier, Solar Heat Worldwide. Global\\nMarket Development and Trends in 2016. Detailed Market Figures 2015,\\n2017 ed. AEE-Institute for Sustainable Technologies, 2017, pp. 1–83.\\n[81] B. Sütterlin and M. Siegrist, ‘‘Public acceptance of renewable energy\\ntechnologies from an abstract versus concrete perspective and the pos-\\nitive imagery of solar power,’’ Energy Policy, vol. 106, pp. 356–366,\\nJul. 2017.\\n[82] J. A. Van Biljon, ‘‘A model for representing the motivational and cultural\\nfactors that inﬂuence mobile phone usage variety,’’ Ph.D. dissertation,\\nUniv. South Africa, Pretoria, South Africa, 2006.\\n[83] E. Park and S. J. Kwon, ‘‘What motivations drive sustainable energy-\\nsaving behavior?: An examination in South Korea,’’ Renew. Sustain.\\nEnergy Rev., vol. 79, pp. 494–502, Nov. 2017.\\n[84] D. Çelikler and Z. Aksan, ‘‘The opinions of secondary school stu-\\ndents in Turkey regarding renewable energy,’’ Renew. Energy, vol. 75,\\npp. 649–653, Mar. 2015.\\n[85] F. Ribeiro, P. Ferreira, M. Araújo, and A. C. Braga, ‘‘Public opinion on\\nrenewable energy technologies in Portugal,’’ Energy, vol. 69, pp. 39–50,\\nMay 2014.\\n[86] R. Kardooni, S. B. Yusoff, and F. B. Kari, ‘‘Renewable energy technology\\nacceptance in Peninsular Malaysia,’’ Energy Policy, vol. 88, pp. 1–10,\\nJan. 2016.\\n[87] M. Motosu and Y. Maruyama, ‘‘Local acceptance by people with\\nunvoiced opinions living close to a wind farm: A case study from Japan,’’\\nEnergy Policy, vol. 91, pp. 362–370, Apr. 2016.\\n[88] D. Chen, C.-Y. Cheng, and J. Urpelainen, ‘‘Support for renewable energy\\nin China: A survey experiment with Internet users,’’ J. Cleaner Prod.,\\nvol. 112, pp. 3750–3758, Jan. 2016.\\n[89] D. Bush and P. Hoagland, ‘‘Public opinion and the environmental, eco-\\nnomic and aesthetic impacts of offshore wind,’’ Ocean Coast. Manag.,\\nvol. 120, pp. 70–79, Feb. 2016.\\n[90] G. S. Denis and P. Parker, ‘‘Community energy planning in Canada:\\nThe role of renewable energy,’’ Renew. Sustain. Energy Rev., vol. 13, no. 8,\\npp. 2088–2095, 2009.\\n[91] C. L. Noblet, M. F. Teisl, K. Evans, M. W. Anderson, S. McCoy, and\\nE. Cervone, ‘‘Public preferences for investments in renewable energy\\nproduction and energy efﬁciency,’’ Energy Policy, vol. 87, pp. 177–186,\\nDec. 2015.\\n[92] J. Ladenburg, ‘‘Attitudes towards offshore wind farms—The role of beach\\nvisits on attitude and demographic and attitude relations,’’ Energy Policy,\\nvol. 38, no. 3, pp. 1297–1304, 2010.\\n[93] A. Dimitropoulos and A. Kontoleon, ‘‘Assessing the determinants of local\\nacceptability of wind-farm investment: A choice experiment in the Greek\\nAegean Islands,’’ Energy Policy, vol. 37, no. 5, pp. 1842–1854, 2009.\\n[94] J. Ganzevles, L. Asveld, and P. Osseweijer, ‘‘Extending bioenergy\\ntowards smart biomass use Issues of social acceptance at Park Cuijk,\\nThe Netherlands,’’ Energy. Sustain. Soc., vol. 5, no. 1, pp. 1–12, 2015.\\n[95] M. M. E. Moula, J. Maula, M. Hamdy, T. Fang, N. Jung, and R. Lahdelma,\\n‘‘Researching social acceptability of renewable energy technologies in\\nFinland,’’ Int. J. Sustain. Built Environ., vol. 2, pp. 89–98, Jun. 2013.\\n[96] P. J. C. Nel, M. J. Booysen, and B. van der Merwe, ‘‘Energy perceptions\\nin South Africa: An analysis of behaviour and understanding of electric\\nwater heaters,’’ Energy Sustain. Develop., vol. 32, pp. 62–70, Jun. 2016.\\n[97] A. Botelho, L. M. C. Pinto, L. Lourenço-Gomes, M. Valente, and\\nS. Sousa, ‘‘Public perceptions of environmental friendliness of renew-\\nable energy power plants,’’ in Proc. 1st Energy Econ. Iberian, 2016,\\npp. 73–86.\\n[98] R. Kardooni, S. B. Yusoff, F. B. Kari, and L. Moeenizadeh, ‘‘Public opin-\\nion on renewable energy technologies and climate change in Peninsular\\nMalaysia,’’ Renew. Energy, vol. 116, pp. 659–668, Feb. 2018.\\n[99] J. Kim, S. Y. Park, and J. Lee, ‘‘Do people really want renewable\\nenergy? Who wants renewable energy?: Discrete choice model of\\nreference-dependent preference in South Korea,’’ Energy Policy, vol. 120,\\npp. 761–770, Sep. 2018.\\n[100] F. Ribeiro, P. Ferreira, M. Araújo, and A. C. Braga, ‘‘Modelling per-\\nception and attitudes towards renewable energy technologies,’’ Renew.\\nEnergy, vol. 122, pp. 688–697, Jul. 2018.\\n[101] K. Schumacher, F. Krones, R. McKenna, and F. Schultmann, ‘‘Public\\nacceptance of renewable energies and energy autonomy: A comparative\\nstudy in the French, German and Swiss Upper Rhine region,’’ Energy\\nPolicy, vol. 126, pp. 315–332, Mar. 2019.\\n[102] C. Bhowmik, S. Bhowmik, and A. Ray, ‘‘Social acceptance of green\\nenergy determinants using principal component analysis,’’ Energy,\\nvol. 160, pp. 1030–1046, Oct. 2018.\\n[103] A. Khatua, A. Khatua, and E. Cambria, ‘‘A tale of two epidemics: Con-\\ntextual Word2Vec for classifying Twitter streams during outbreaks,’’ Inf.\\nProcess. Manag., vol. 56, no. 1, pp. 247–257, Jan. 2019.\\n[104] T. M. Nisar, G. Prabhakar, and L. Strakova, ‘‘Social media information\\nbeneﬁts, knowledge management and smart organizations,’’ J. Bus. Res.,\\nvol. 94, pp. 264–272, Jan. 2018.\\n[105] M. A. Smith et al., ‘‘Analyzing (social media) networks with NodeXL,’’\\nin Proc. 4th Int. Conf. Communities Technol. (C&T), 2009, p. 255.\\n[106] S. C. Bowden et al., ‘‘The reliability and internal validity of the wisconsin\\ncard sorting test,’’ Neuropsychol. Rehabil., vol. 8, no. 3, pp. 243–254,\\nMay 1998.\\n[107] K. Søreide, G. Mackenzie, K. Polom, L. Lorenzon, H. Mohan, and\\nJ. Mayol, ‘‘Tweeting the meeting: Quantitative and qualitative Twitter\\nactivity during the 38th ESSO conference,’’ Eur. J. Surg. Oncol., vol. 45,\\nno. 2, pp. 284–289, Nov. 2018.\\n[108] B. J. Breitmayer, L. Ayres, and K. A. Knaﬂ, ‘‘Triangulation in qualitative\\nresearch: Evaluation of completeness and conﬁrmation purposes,’’ Image\\nJ. Nursing Scholarship, vol. 25, no. 3, pp. 237–243, Sep. 1993.\\n[109] N. Kannan and D. Vakeesan, ‘‘Solar energy for future world:—A review,’’\\nRenew. Sustain. Energy Rev., vol. 62, pp. 1092–1105, Sep. 2016.\\n[110] S. Jacobsson and A. Johnson, ‘‘The diffusion of renewable energy tech-\\nnology: An analytical framework and key issues for research,’’ Energy\\nPolicy, vol. 28, no. 9, pp. 625–640, 2000.\\nATIKA QAZI received the Ph.D. degree from the\\nUniversity of Malaya, Kuala Lumpur, Malaysia,\\nwhere she is currently a Postdoctoral Researcher.\\nShe was an Assistant Professor with COMSATS\\nUniversity Islamabad, Pakistan. Her research\\ninterests include opinion mining, sentiment anal-\\nysis, big data analytics, information systems, e-\\nlearning, ICT, and renewable energy. She received\\nthe Bright Spark Scholarship for her Ph.D. She is\\nactively involved as a Reviewer of ISI journals.\\n63850\\nVOLUME 7, 2019\\n\\nA. Qazi et al.: Toward Sustainable Energy: Systematic Review of RES, Technologies, and Public Opinions\\nFAYAZ HUSSAIN received the Master of Engi-\\nneering Science (M.E.Sc.) and Ph.D. degrees\\nin mechanical engineering from the University\\nof Malaya, Kuala Lumpur, Malaysia. He was a\\nResearch Assistant with the University of Malaya,\\nto contribute his skills in renewable energy\\nprojects.\\nNASRUDIN\\nABD.\\nRAHIM\\n(M’89–SM’08)\\nreceived the B.Sc. (Hons.) and M.Sc. degrees from\\nthe University of Strathclyde, Glasgow, U.K., and\\nthe Ph.D. degree from Heriot–Watt University,\\nEdinburgh, U.K., in 1995. He is currently a Profes-\\nsor with the University of Malaya, Kuala Lumpur,\\nMalaysia, where he is also the Director of the\\nUM Power Energy Dedicated Advanced Centre\\n(UMPEDAC). He is also a Distinguish Adjunct\\nProfessor with the Renewable Energy Research\\nGroup, King Abdulaziz University, Jeddah, Saudi Arabia. His research\\ninterests include power electronics, solar PV and wind technologies, real-\\ntime control systems, and electrical drives. He is a Fellow of the Institution of\\nEngineering and Technology, U.K. and the Academy of Sciences Malaysia.\\nHe is also a Chartered Engineer, U.K.\\nGLENN HARDAKER is currently the Director of\\nthe Centre for Lifelong Learning and a Professor\\nof education with the Sultan Hassanal Bolkiah\\nInstitute of Education, University of Brunei\\nDarussalam. He is also a National Teaching Fel-\\nlow (Higher Education Academy, U.K.). He has\\nworked in Universities in the U.K., for approxi-\\nmately 25 years in the areas of technological inno-\\nvation and learning. His research is focused on a\\nblend of Islam, innovation, and learning. He is an\\nEditor of two international publications: the Journal for Multicultural Educa-\\ntion and the International Journal of Information and Learning Technology.\\nDANIYAL ALGHAZZAWI received the B.Sc.\\ndegree (Hons.) in computer science from King\\nAbdulaziz University, in 1999, and the M.Sc.\\ndegree in teaching and leadership, which helped\\nhim to develop his teaching and leadership skills,\\nfrom the University of Kansas, USA, in 2004, and\\nthe M.Sc. and Ph.D. degrees in computer science\\nfrom the University of Kansas, in 2007. Since\\n2007, he has been the Head of Department for ﬁve\\nyears, and then he served as a Vice Dean of the\\nDevelopment of the Deanship of Information Technology for two years.\\nIn 2010, he became an Honorary Lecturer with the School of Computer Sci-\\nence and Electronic Engineering, University of Essex, U.K. He is currently\\na Professor with the Computing Information Systems Department and the\\nHead of the Information Security Group at King Abdulaziz University.\\nKHALED SHABAN received the Ph.D. degree\\nfrom the Department of Electrical and Computer\\nEngineering, University of Waterloo, Waterloo,\\nCanada, in 2006. He is currently an Associate Pro-\\nfessor with the Computer Science and Engineering\\nDepartment, College of Engineering, Qatar Uni-\\nversity, Doha, Qatar. His research experience in\\nacademic and industrial institutions covers a vari-\\nety of domains in intelligent systems application\\nand design.\\nKHALID HARUNA received the B.Sc. and M.Sc.\\ndegrees in computer science from Bayero Univer-\\nsity, Kano, Nigeria, in 2011 and 2015, respectively,\\nand the Ph.D. degree in information systems from\\nthe University of Malaya, Malaysia, in 2018. He is\\ncurrently a Lecturer with the Department of Com-\\nputer Science, Bayero University, Kano, Nigeria.\\nHis research interests include recommender sys-\\ntems, semantic web, big data analytics, sentiment\\nanalysis, and soft set and e-learning systems.\\nVOLUME 7, 2019\\n63851\\n',\n",
       " '1US Department of Energy, 1000 Independence Avenue SW, Washington DC 20585, USA. \\nT\\nhe industrial revolution began in the mid-eighteenth century, \\nand provided humans with capabilities well beyond animal and \\nhuman power. Steam-powered trains and ships, and then inter\\xad\\nnal combustion engines transformed how people moved and produced \\ngoods around the world. Electrification and related technologies contin\\xad\\nued the revolution in the nineteenth and twentieth centuries. Today, a \\ngrowing number of people keep their homes warm in the winter, cool in \\nthe summer and lit at night. They go to the local market in cars with the \\npower of over a hundred horses and fly across continents in wide-body \\naeroplanes with the power of a hundred thousand horses. This power is \\nderived largely from our ability to exploit fossil sources of energy. How\\xad\\never, in the transition from human and horse power to horsepower, the \\ncarbon emissions that result from the equivalent of over a billion horses \\nworking continuously have created significant climate-change risks.\\nAt the beginning of the industrial revolution, the population of the \\nworld was 700\\xa0million. Today, the population is 7\\xa0billion and is estimated \\nto grow to 9\\xa0billion by 2050, and about 10 billion by 2100 (ref.\\xa01). Most of \\nthis population growth will be in Asia and Africa, where rapidly rising \\neconomic growth will place additional demands on energy supply. The \\nInternational Energy Agency (IEA) based in Paris has projected that \\nthe world’s energy demand will increase from about 12\\xa0billion tonne \\noil equivalents (t.o.e.) in 2009 to either 18\\xa0billion t.o.e. or 17\\xa0billion \\nt.o.e. by 2035 under their ‘current policies’ or ‘new policies’ scenarios, \\nrespectively2. Carbon-dioxide emissions are expected to increase from \\n29 gigatonnes per year to 43\\u2009Gt\\u2009yr-1 or 36\\u2009Gt\\u2009yr-1 under the current and \\nnew policies, respectively. The actual path we follow will depend on how \\nefficiently and effectively we use existing and new sources of energy. \\nThe world needs another industrial revolution in which our sources \\nof energy are affordable, accessible and sustainable. Energy efficiency \\nand conservation, as well as decarbonizing our energy sources, are \\nessential to this revolution. Reducing carbon emissions on the timescale \\nneeded to mitigate the worst risks of climate change will not be driven \\nby our inability to find cost-effective sources of fossil fuels. Despite the \\nsignificant growth in the use of renewable energy, the fractional sum of \\nnon-carbon- emitting sources of energy that remained constant during \\nthe past two decades is sobering (Fig.\\xa01). \\nEnergy systems can be divided into transportation and stationary. \\nThe supply, demand and distribution infrastructures within each system \\nare highly coupled, but are currently largely independent of each other. \\nIn this Insight, the Reviews focus on specific subareas of biofuels, solar \\nelectricity generation, and electricity generation from salinity gradients \\nand waste water. The systems discussed are largely at the research and \\ndevelopment stage, and are not yet ready to displace more mature \\nsources of energy. Nevertheless, we should be constantly looking for \\ninnovations that make a marked improvement on today’s approaches \\nor provide an entirely new approach. To place the topics discussed in \\nthe Reviews in context, this Perspective discusses transportation and \\nstationary power systems, and indicates how the topics discussed in the \\nReviews fit into the energy landscape. \\nMany important and challenging research areas have the potential to \\nsignificantly affect our future energy needs. For example, energy effi\\xad\\nciency and the integration of energy sources with electricity transmis\\xad\\nsion, distribution and storage are vitally important, but these are only \\nbriefly mentioned in this Perspective. Excellent reviews of these topics \\ninclude the National Academies’ report America’s Energy Future3 and \\nthe US Department of Energy’s (DOE’s) Report on the First Quadrennial \\nTechnology Review4.\\nTransportation\\nPetroleum-derived liquid fuels are the overwhelming source of energy \\nin the current transportation infrastructure. The geographical distri\\xad\\nbution of petroleum resources is changing as reserves are found and \\naccessed with improved technologies for discovery and production. \\nHowever, this distribution of oil supply generally does not coincide \\nwith where the demand is located. For example, many countries import \\noil at an unprecedented scale, which can lead to significant balance-\\nof-trade and national-security challenges. In 2011, about 2.690\\xa0billion \\ntonnes of oil were consumed; of this, 1.895\\xa0billion tonnes of crude oil \\nand 0.791\\xa0billion tonnes of refined products crossed national borders5 \\nand significant discoveries of oil, natural-gas liquids and natural gas \\ncould potentially alter the global-energy landscape6. \\nWe will review some of the opportunities and challenges related to \\ntransportation technologies. For example, most of the future infrastruc\\xad\\nture of the world will be built in locations where we have the greatest \\nopportunity to transition to sustainable mobility. Desirable and afford\\xad\\nable public transportation that is fully integrated into urban planning7, \\nand the use of information technologies to assist and displace transpor\\xad\\ntation can significantly reduce fuel consumption, but are not discussed \\nin this Perspective. \\nThe Quadrennial Technology Review (QTR) of the US DOE provides \\nAccess to clean, affordable and reliable energy has been a cornerstone of the world’s increasing prosperity and economic \\ngrowth since the beginning of the industrial revolution. Our use of energy in the twenty-first century must also be sustain\\xad\\nable. Solar and water-based energy generation, and engineering of microbes to produce biofuels are a few examples of the \\nalternatives. This Perspective puts these opportunities into a larger context by relating them to a number of aspects in the \\ntransportation and electricity generation sectors. It also provides a snapshot of the current energy landscape and discusses \\nseveral research and development opportunities and pathways that could lead to a prosperous, sustainable and secure energy \\nfuture for the world.\\nOpportunities and challenges for \\na sustainable energy future\\nSteven Chu1 & Arun Majumdar1 \\n \\n2 9 4  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\ndoi:10.1038/nature11475\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\na broad overview of state-of-the-art technologies and opportunities \\nfor future research8. Improvements in energy efficiency of vehicles \\ncan greatly reduce oil dependency. These improvements include \\nincreased use of light-weight materials, such as advanced ultra-high \\ntensile strength steels, aluminium and magnesium alloys, polymers, \\nand carbon-fibre reinforced composite materials9. The integration of \\nlighter weight materials is especially important if more complex parts \\ncan be manufactured as a single unit. The potential for reducing the \\nweight of vehicles has already been shown, and in the next 10–20\\xa0years, \\nan additional 20–40% reduction in overall weight, without sacrificing \\nsafety, seems to be possible10. For every 10% weight reduction of the \\nvehicle, an improvement in fuel consumption of 6–8% is expected11. \\nReducing energy losses as a result of friction is also possible12 (Fig.\\xa02). \\nAdvances in cost-effective technologies such as tribology, tyres, braking \\nand waste-heat energy recovery, and aerodynamics could potentially \\nlead to efficiency improvements of 20% in the short term and more than \\n60% over a longer term (15–25\\xa0years).\\nInternal combustion engines\\nThe internal combustion engine using liquid-transportation fuel (or \\nliquid ICE) will probably continue to have a major role over the next \\nfew decades. However, improvements to the efficiency of the liquid ICE \\nare possible because the efficiency of most spark-ignition engines is \\ntypically 25–35%, whereas that for compression-ignition diesel engines \\nis about 40–50%. With direct injection, lean burn and turbocharger \\ntechnologies, the spark-ignition ICEs operating on premium octane rat\\xad\\nings can approach diesel efficiencies. The combination of in situ meas\\xad\\nurements of prototype ICE designs with detailed simulations, which \\nare made possible with high-performance computers, are increasing \\nthe energy efficiency and have lowered emissions13,14. Finally, low-cost \\nwaste-heat recovery can increase efficiency, especially in heavy-duty \\nvehicles. Approaches include use of the Rankine cycle to convert waste \\nheat to work, and the development of low-cost and high-efficiency \\nsolid-state thermoelectric systems15. \\nBattery-based electrification\\nPlug-in hybrid and all-electric light- and medium-duty vehicles have the \\nopportunity to displace a significant amount of liquid fuel use in trans\\xad\\nportation. The main challenges are performance and cost of the battery \\nsystems. The performance of battery systems is quantified by usable \\nenergy density, power density (including fast charging), cycle lifetime \\nand robustness. In the past 5–6\\xa0years, a remarkable amount of progress \\nin research has been made in battery cathodes, anodes and electro\\xad\\nlytes. Large volumetric changes in the electrodes have led to designs \\nin which micro- or nanostructures are embedded in a conducting and \\nflexible matrix to allow for the relief of mechanical stresses. State-of-the-\\nart batteries based on graphite anodes and lithium-manganese-oxide \\ncomposite cathodes for lithium batteries are being commercialized16. \\nWithin the next few years, battery system packs of 200\\xa0Watt-hour kg-1 \\nat a charging rate of full charge in 3 hours, which is double the cur\\xad\\nrent cell-energy density, are expected to become available. The current \\nproduction cost of a vehicle battery is estimated to be US$650\\u2009kWh-1 of \\nusable energy storage, but this is expected to drop to below $150\\u2009kWh-1 \\nby 2030 (ref. 17). \\nThe US DOE is supporting a research and development effort that \\nwould allow a four or five passenger plug-in hybrid electric vehicle \\n(PHEV) or an electric vehicle with about a 160-km range to be competi\\xad\\ntive as a mass-market ICE car within a decade. The US DOE EV-Eve\\xad\\nrywhere challenge for PHEV, will require the cost of the battery system \\nto be reduced by an estimated $190–300\\u2009kWh-1, depending on whether \\nthe car is a PHEV or an electric vehicle. The development of anode-pro\\xad\\ntecting materials and non-flammable electrolytes that are stable at high \\nvoltage and tolerate 55\\u2009°C are desired characteristics of third-generation \\nlithium-ion batteries. Developing batteries beyond lithium-ion batter\\xad\\nies, such as lithium-sulphur and metal-air batteries, could achieve up \\nto ten times the energy density of the current lithium-ion batteries, but \\nmaterials research is needed to develop anode and cathode protection, \\nas well as non-flammable electrolytes with electrochemical stability over \\na large potential range.\\nBattery packs typically use about 50% of the total battery capacity, \\nand the charging rates are limited to increase the lifetime18. If sensor \\ntechnologies are developed that can continuously monitor the proper\\xad\\nties of individual cells, such as internal impedance, temperature and \\nFigure 1 | Statistical review of world energy.\\u2002 a, Fossil energy comprises \\nroughly 86% of the world’s main energy consumption. Although the \\nconsumption of oil has increased by 31% between 1980 and 2008, the known \\nreserves have increased comparably owing to improvements in exploration and \\nextraction technologies. Much of the world’s shale-gas reserves are not ‘proven’ \\nand are not included. The fractional sum of non-carbon emitting sources of \\nenergy remained constant during the same time period. b, Growth of renewable \\nenergy was offset by the decline in nuclear power generation. Renewable \\nenergy sources in power generation grew by 17.7%. Wind generation (25.8%) \\naccounted for more than half of renewable power generation for the first time. \\nRenewables accounted for 3.8% of global power generation, with the highest \\nshare in Europe and Eurasia (7.1%). Adapted with permission from ref. 72.\\nTrillion bbls\\nTcm\\nOil and gas reserves\\n1981\\n1991\\n2001\\n2011 1981\\n1991\\n2001\\n2011\\n0\\n0.8\\n0.4\\n1.2\\n1.6\\n2.0\\n0\\n50\\n100\\n150\\n200\\n250\\nOther\\nNorth America\\nFSU\\nOPEC\\n0\\n10\\n20\\n30\\n40\\n50\\n1966\\nPercentage\\n1981\\nShares of world primary energy\\n1996\\n2011\\nOil\\nCoal\\nGas\\nHydro\\nNuclear\\nRenewable\\na \\nMillion tonne oil equivalent\\n0\\n91\\n95\\n93\\n99\\n97\\n01\\n03\\n05\\n07\\n09\\n11\\n50\\n100\\n150\\n200\\nAsia Paciﬁc\\nAfrica\\nMiddle East\\nEurope and Eurasia\\nSouth and Central America\\nNorth America\\nOther renewables consumption by region\\nb \\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 5\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nstate of charge, the lifetime and useful capacity could be improved. \\nStandardized battery cells that are designed to be integrated with the \\noriginal equipment  manufacturer’s thermal management systems could \\nalso reduce cost19.\\nFuel-cell-based electrification \\nThe high efficiency of fuel-cell-powered electric vehicles makes this \\nform of electrification a potentially viable option for the future. Invest\\xad\\nment in this technology is driven by the potential of extended range \\nand faster fuelling times of moderately low-priced cars. Fuel-cell cost \\nhas been lowered and their lifetime increased20, but further gains are \\nneeded. Platinum-group catalyst loading has been reduced fivefold since \\n2005; however, further reductions are needed4,21, or these catalysts need \\nto be replaced with less costly alternatives. Costs can also be lowered \\nand performance improved through robust higher-conductivity and \\nhigher-temperature membranes, improvements in balance-of-plant \\ncomponents, such as humidifiers and compressors, as well as thermo\\xad\\nfluid design and control. \\nThere are inherent volumetric energy density issues for hydrogen-gas \\nstorage. To achieve a range of 480 km, fuel-cell electric vehicles need \\nto store about 4–7 kg of hydrogen4,22. A carbon-fibre-composite tank \\npressurized to 700 bar is the best current option for personal vehicles, \\nbut this costs23 about $3,000. Research is under way to develop materials \\nand manufacturing processes to reduce the cost of composite tanks. In \\nparallel, researchers are searching for lower-pressure storage assisted by \\nhigh-surface area materials that could physisorb or weakly chemisorb \\nhydrogen and still maintain fast-fuelling times24,25. \\nThe supply infrastructure and a low-carbon source of hydrogen are \\nalso a challenge. The technology advances in shale-gas production, and \\nthe possibility of large reserves in Europe and Asia, in addition to the \\nconsiderable reserves in North America, could have a significant affect \\non the transportation sector. In addition to the direct use of natural gas \\nas a fuel (see later), low-cost natural gas could spur the deployment of \\nlocal reforming or hydrogen filling stations for near-term hydrogen \\nproduction. Alternatively, commercial reforming plants, such as hybrid \\npower plants that produce hydrogen as well as CO2, for enhanced oil \\nrecovery located near oil-field and refinery sites can serve as an eco\\xad\\nnomical source of hydrogen4,26. In regions that are close to a large com\\xad\\nmercial production plant, delivery to local filling stations can be made \\nthrough high-pressure gaseous tube trailers, but in the long run, a cost-\\ncompetitive method to produce hydrogen with considerably lower net \\ncarbon emissions is needed. \\nNatural gas for transportation\\nThe projected low cost of natural gas in the United States in the next \\nfew decades compared with that of petrol is expected to lead to wider \\nadoption of natural-gas vehicles. Displacing diesel fuel with liquefied \\nnatural gas (LNG) for class 8 tractor-trailer trucks commonly used on \\nlong routes in the United States is already economically viable because \\na typical long-haul truck uses about 90,000 litres of fuel per year (about \\n$80,000 per year in fuel costs, today). The incremental purchase price \\nof LNG trucks can be up to $100,000 per truck for cryogenic tanks \\nand related upgrades, as a result of low-volume market conditions. \\nEven so, the payback period is currently 3–4 years on a net-present-\\nvalue basis using a 7% discount rate, and would drop considerably with \\neven modest increases in production volumes27. Developing a fuelling \\ninfrastructure for LNG long-haul trucks would require fuelling stations \\nabout every 240–320\\u2009km for a truck range of 800–960\\u2009km. Commer\\xad\\ncial viability is reinforced by plans in the private sector to make the \\nrequired investments in infrastructure. LNG-powered freight trains \\nare also being considered.\\nCompressed natural gas (CNG) has been used for buses, delivery \\ntrucks and light-duty vehicles. To make vehicles using CNG economi\\xad\\ncally viable without subsidies, low-cost CNG storage technologies are \\nneeded. In the United States, light-duty vehicles account for 75% of \\non-road fuel consumption. There are roughly 160,000 gasoline (petrol) \\nservice stations in the United States28; creating a similar nationwide \\ninfrastructure for CNG vehicles would be prohibitively expensive (more \\nthan $100 billion). However, about 60 million homes in the United \\nStates have natural-gas delivery. Economic viability for CNG cars and \\nrefuelling systems can be achieved if the payback period for the addi\\xad\\ntional system-level cost is typically 5 years or less; at present it is about \\n10–15 years for a vehicle with average mileage. For vehicles with high \\nannual mileage and for vehicles with low kilometres per litre, natural-gas \\nvehicles can have less than a 5-year payback, even today. Research into \\nfibre-matrix composites for high-pressure light-weight tank materials \\nis needed, as well as into natural-gas sorbents for low-pressure storage. \\nAlthough seldom discussed, multifuel ICEs can be designed to operate \\non CNG for 30–60\\u2009km (the CNG–ICE hybrid equivalent of a PHEV) \\nand then switch to petrol. Similar to a PHEV, a CNG–petrol–ICE vehicle \\ncould compensate for the partial coverage of CNG fuelling stations.\\nNatural gas can also be converted into liquid fuels using either the \\nFischer–Tropsch or the methanol process. The capital cost per barrel \\nof liquid fuel reduces with increasing capacity of a Fischer–Tropsch \\nplant according to scaling laws. However, with increasing capacity, \\ncapital costs are suggested to deviate and are higher than scaling law \\npredictions, which increases the financial risk for gas-to-liquid plants. \\nResearch is needed to find alternative approaches for exciting the car\\xad\\nbon–hydrogen bond and synthesizing carbon–carbon bonds. Biologi\\xad\\ncal approaches that use organisms, such as methanotrophs, that can \\nmetabolize natural gas and produce long-chain hydrocarbons seem \\nworth exploring. Even if this approach is successful at laboratory scales, \\nit will need to be scalable to large-volume production. Large quanti\\xad\\nties of methanol are already produced from natural gas for industrial \\nFuel\\nenergy\\n100%\\nExhaust\\n33%\\nCooling\\n29%\\nMechanical\\npower\\n38%\\nFriction\\nlosses\\n33%\\nAir drag 5%\\nAir drag 5%\\nEngine\\n11.5%\\nTransmission 5%\\nBrakes 5%\\nRolling\\nresist 11.5%\\nEnergy \\nused to \\nmove the \\ncar 21.5% \\nThermodynamic\\nlosses\\nTotal\\nenergy\\nlosses\\nFigure 2 | Vehicle energy losses.\\u2002 Of the \\nenergy that fuel provides to vehicles a \\nsubstantial proportion is lost. A breakdown \\nof the average losses of internal-combustion-\\nengine cars (fleet make up 70% petrol and 30% \\ndiesel) is shown. Heat lost constitutes 30–37% \\nof the energy as a result of exhaust gases with \\nlower energy content and convection. The \\nother losses come from heat dissipation (25–\\n33%), mechanical losses (33–40%), air drag \\n(3–12%), rolling friction (12–45%) and brake \\nlosses (about 5%). These losses mean only \\nabout 21.5% of the energy is used to move the \\ncar. Adapted with permission from ref. 12.\\n2 9 6  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\npurposes at costs that are roughly equivalent to petrol. Methanol could \\nbe used in a petrol–alcohol blend, much like ethanol in the United \\nStates. However, pure or high-percentage methanol-based transporta\\xad\\ntion could face distribution-infrastructure challenges.\\nAlternative liquid-transportation fuels\\nLiquid fuels derived from oil became the main form of energy for \\ntransportation largely because of their high energy densities. Associ\\xad\\nated with their high energy content, the energy transfer rate during \\nvehicle refuelling is about 6 MW; in contrast, electrical charging will \\nbe tens of kilowatts. Apart from inherent limitations in battery chemis\\xad\\ntries, there are practical limits to the size of the electrical connector that \\ncould accommodate megawatt-scale power transfer. However, a search \\nfor alternatives to oil for transportation energy is required to deal with \\nthe growing concerns over the rising and volatile price of oil, the vul\\xad\\nnerability to supply disruptions, and balance-of-trade issues. Biofuels, \\nparticularly those produced domestically at competitive prices, would \\nstrengthen a nation’s energy and economic security. \\nThere are a number of approaches to alternative transportation fuels \\nbeing actively explored (Fig.\\xa03). The estimated future (10–15 year) costs \\nfor classes of alternative fuels can vary29 (Fig.\\xa04). The US Environmental \\nProtection Agency (EPA) estimates that Brazilian sugarcane ethanol \\n— already price competitive with oil-based fuels in Brazil — reduces \\ntotal life-cycle greenhouse gas emissions, including direct and indi\\xad\\nrect land-use change emissions, by 61% (ref.\\xa030). There are numerous, \\nand sometimes contentious, studies of carbon life-cycle emissions30–32. \\nMinimization of indirect land-use concerns and the sequestration of \\nprocess CO2 could result in net-negative carbon emissions, so that fuel \\nproduction and use becomes a net carbon sink. On the other hand, if \\nbiofuels based on energy-intensive crops were coupled with poor-land \\nmanagement this could result in environmental costs that are far higher \\nthan those associated with oil-based fuels29.\\nThe production of ethanol by domesticated yeast for fermentation \\nis perhaps 4,000 years old. In this Insight, Paralta-Yahya et al.33  review \\nthe application of metabolic engineering and synthetic biology to alter \\nmicrobes for the production of advanced biofuels, and of precursors to \\ndrop-in substitutes for petrol, diesel and jet fuels. Static adjustment of \\ntranscription, translation and post-translational modifications does not \\nallow engineered organisms to respond to changing bioreactor condi\\xad\\ntions and cellular changes. Remarkably, dynamic sensing and regulation \\nof fatty-acid ethyl ester (FAEE) intermediates in Escherichia coli has \\nbeen shown to increase FAEE yield by threefold, to reduce the cellular \\nconcentration of toxic intermediates and to significantly improve the \\ngenetic stability of producing strains.\\nSuccessful commercial-scale deployment of this class of technologies \\nwill depend on microbial productivity and robustness. The cost of \\nmicrobial feedstock is also a major factor: production of advanced fuels \\nfrom simple sugars and starches is further along in the development of \\nthese fuels, but it is held back by high feedstock prices. Lignocelluloses, \\nincluding agricultural and wood-waste streams, are roughly an order of \\nmagnitude less costly, and are generally viewed as the end-goal feedstock. \\nMuch attention is now focussed on reducing the cost of converting ligno\\xad\\ncelluloses into forms that are more readily used by microbial organisms.\\nThe net energy yield per hectare per year is a major factor in \\ndetermining feedstock cost. The overall yield for biofuels varies widely: \\nBrazilian sugarcane can produce as much as 800 gallons per acre-year, \\nwhich is about twice as high as productivities based on North American \\nmaize. High-yield grasses or fast-growing trees that could be feedstocks \\nfor advanced biofuels have the potential to surpass sugarcane produc\\xad\\ntivities by 1.3 times or more34. Most lignocellulosic feedstocks have low-\\nvolume energy density: relative to fossil fuels they are ‘light and fluffy’. To \\nimprove the economies of scale of biorefineries, especially \\nthermochemical refineries, increasing the feedstock col\\xad\\nlection radius is desirable. Biofuels could become more \\ncompetitive if a means of efficiently concentrating the bio\\xad\\nmass during the harvesting process (such as combining the \\nreaping and pelleting the woody materials) could be devel\\xad\\noped. The same densification of the feedstock could also \\nlead to the biofuel equivalent of grain elevators. Reducing \\n‘first touch’ costs are an integral part of efficient agricul\\xad\\ntural practice, and progress in this area would significantly \\nlower the overall feedstock cost delivered to biorefineries. \\nIn this Insight, Georgianna and Mayfield35 discuss the \\nuse of algae to produce next generation biofuels. Algae \\ncan provide high fuel yields, especially if areal sunlight \\ncollection can be used to drive volumetric growth of \\nalgae. However, algal growth in closed systems requires \\nvery high capital investments relative to energy crops \\nsuch as grasses or trees. In open-pond systems, water use \\nis a major issue but may be partially ameliorated through \\nthe use of species that can grow in brackish or saltwater.\\nOther approaches to bio-based fuel production include \\nPlatforms/pathways \\nFeedstock \\nproduction \\nand logistics \\n• Energy crops \\n• Agricultural \\nby-products \\n• Waste \\nstreams \\n• Algae \\n• Coal  \\n• Natural gas \\n• Ethanol \\n• Methanol \\nReﬁning\\n• Butanol \\n• Alkenes\\n• Aromatics \\n• Petrol\\n• Diesel \\n• Jet fuel \\n• Dimethyl \\nether  \\n• Heat and \\npower \\nBy-products  \\nPower\\n \\nPyrolysis oil platform\\nSyngas platform \\nLiquid\\nbio-oil  \\nEnzymatic\\nhydrolysis\\nSugars\\nFermentation\\nCellulosic sugar platform \\nProducts \\n \\nFast pyrolysis  \\nGasiﬁcation \\nRaw\\nsyngas  \\nFiltration and\\nclean-up  \\nUpgrading \\n  \\nAlgal and other \\nbio-oils \\nTransesteriﬁcation\\nCatalytic upgrading \\nLipid (oil) platform \\nOther enzymatic or biochemical methods \\nFigure 3 | Methods of producing alternative fuels from various feedstocks to \\nproducts.\\u2002 Various feedstocks are being explored, and the pathways for producing energy or fuel \\ninvestigated. Adapted with permission from ref. 8.\\nCO2 cost (ignoring indirect CO2 consequences)\\nCarbon storage cost\\n280\\n240\\n200\\n160\\nCost US$ per barrel petrol-equivalent product\\nCFT\\nCFT-CCS\\nCMTG\\nCMTG-CCS\\nCorn ethanol\\nCellulosic\\nethanol\\nBTL\\nBTL-CCS\\nCBFT\\nCBFT-CCS\\nCBMTG\\nCBMTG-CCS\\nCrude oil\\n$60, bbl\\nCrude oil\\n$100, bbl\\n120\\n80\\n40\\n0\\n–40\\nAdditional transportation\\nNon-feedstock operating cost\\nCapital cost\\nFeedstock cost\\nTotal cost (without carbon price)\\nFigure 4 | Alternative fuel costs.\\u2002 Estimated costs, achievable within 10–15 \\nyears, of alternative liquid fuels produced from coal, biomass, or coal and \\nbiomass with a CO2 price of $50 per tonne and capital costs are 20% lower \\nthan the America’s Energy Future panel’s estimates. BTL, biomass-to-liquid \\nfuel; CBFT, coal-and-biomass-to-liquid fuel, Fischer–Tropsch; CBMTG, coal-\\nand-biomass-to-liquid fuel, methanol-to-gasoline; CCS, carbon capture and \\nstorage; CFT, coal-to-liquid fuel, Fischer–Tropsch; CMTG, coal-to-liquid fuel, \\nmethanol-to-gasoline. Adapted with permission from ref. 3.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 7\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nthe manipulation of photosynthetic bacteria to produce biofuels, diverting \\nhigh-energy Calvin-cycle intermediates upstream of glucose — for exam\\xad\\nple energy-dense terpene (a biofuel precursor) production in trees or \\nmicrobes — or developing alternatives to C3 and C4 carbon fixation. \\nBiological enzymes can synthesize carbon–carbon bonds with an \\nunparalleled high specificity, but photosynthesis may not be the only \\napproach to converting sunlight into hydrocarbon fuels36. Several non-\\nphotosynthetic alternatives are under investigation that could poten\\xad\\ntially overcome many of the limitations of photosynthesis. To appreciate \\nthe potential of such approaches, the energy conversion process can be \\ndivided into three steps. First is to identify what reducing equivalents, \\nother than solar photons captured through photosystems I and II, can \\nbe accepted by various microorganisms. Several organisms are known \\nto be capable of growth on hydrogen sulphide, hydrogen, electrons, \\nammonia and reduced ions such as iron (II). Second is to investigate \\nopportunities to fix CO2 using pathways other than those used in C3 or \\nC4 plants. Potential systems may include the reverse tricarboxylic-acid \\ncycle (often called the reverse Krebs cycle), the Woods–Ljungdahl cycle \\nused by acetogens, the hydroxypropionate–hydroxybutyrate cycle or \\nnewly designed biochemical pathways. The final step is to determine \\nwhether we can metabolically engineer direct carbon products into a \\nmolecule such as acetyl-CoA, which is a precursor for many energy-\\ndense fuels. These three steps can be engineered into autotrophic organ\\xad\\nisms, an approach that is now being supported by a US DOE Advanced \\nResearch Projects Agency-Energy programme called Electrofuels37. \\nFinally, researchers are investigating highly efficient non-biological \\nenergy-conversion approaches that generate fuel from sunlight by the \\noxidation of water into hydrogen and oxygen and reduction of CO2 to \\nfuel. The Joint Center for Artificial Photosynthesis, a US DOE funded \\nEnergy Innovation Research Center, was established to identify Earth-\\nabundant, robust light absorbers with optimal bandgaps to harvest sun\\xad\\nlight most effectively and efficiently, to accelerate the rate of catalyst \\ndiscovery for solar energy-to-fuel conversion reactions and to provide \\nsystem integration and scale-up so that laboratory experiments can \\nquickly transition into prototypes for commercial development.\\nClean and affordable electricity generation\\nThe IEA has projected, to 2035, how electricity is expected to be gen\\xad\\nerated38 (Fig. 5). In the agency’s new policies scenario, the growth \\nof carbon emissions is curtailed, but does not decline. The scenario \\nassumes that government policies such as carbon pricing are adopted \\nin several Organisation for Economic Co-operation and Development \\ncountries and in China, but the projections of cost reduction in renew\\xad\\nable energy are overly conservative. As an example, the agency esti\\xad\\nmates the full levellized cost of electricity (LCOE) of onshore wind to \\nbe about $90\\u2009MWh-1 in 2010 real dollars by 2020 (ref.\\xa039). Similarly, the \\nUS Energy Information Administration estimates40 the LCOE of wind \\nfor 2016 will be about $80–120\\u2009MWh-1 (Fig.\\xa06). \\nIn contrast to these estimates, an analysis of the LCOE of various \\nforms of renewable energy, based on the available data from projects in \\nwhich investment or purchase contracts have been completed41, shows \\nthat wind at level (class) 4 sites in the United States is currently about \\n$73\\u2009MWh-1 (Fig.\\xa07). These costs do not include transmission line costs, \\nwhich average42 at about $300 kW-1. Some experts estimate that by 2020, \\nthe LCOE of wind energy at level 4 sites will fall below $60\\u2009MWh-1. \\nIn the United States, utility-scale solar photovoltaic projects are being \\ninstalled in 2011–2012 at an unsubsidized cost of about $150\\u2009MWh-1 \\n(the US investment tax credit has been taken out43,44), which is con\\xad\\nsistent with the Bloomberg New Energy Finance estimates41. Utility-\\nscale solar photovoltaics are projected to be between $60–120\\u2009MWh-1 \\nin good insolation areas. The lowest LCOE for combined-cycle natural \\ngas is about $50–60\\u2009MWh-1 in the United States, assuming gas costs of \\n$3–4\\xa0per million British thermal units. Although LCOE is an important \\nfactor, time of delivery, LCOE at various size scales and the potential for \\nenergy storage are also important. \\nAffordable and environmentally sustainable electricity generation is \\nalready at cost parity in areas in which energy costs exceed $200\\u2009MWh-1. \\nOn the basis of the current and projected learning curves for wind and \\nsolar energy, renewable energy will become increasingly affordable on \\na global scale. In addition, energy storage with durable and inexpensive \\nbatteries will allow electricity accessibility in micro- or meso-scale grids \\nto leapfrog the need to bring centrally generated electricity to distant \\nrural areas. \\nEven though the costs of renewable energies such as wind and solar \\ncontinue to decline, the full economic benefit of these variable sources \\nof energy will not be realized until the more established forms of energy \\ngeneration are integrated with renewable sources into transmission and \\ndistribution, load response and storage of electricity. Eighty per cent \\n24\\n0\\n20\\n40\\nPercentage\\n60\\n80\\n100\\n20,043 TWh\\n27,881 TWh\\n36,250 TWh\\n2009\\n2020\\n2035\\n22\\n20\\n18\\n16\\n14\\n12\\n10\\n2010\\n2015\\nGT\\n2020\\n2025\\n2030\\n2035\\nMore eﬃcient plants\\nCoal\\nGas\\nOil\\nNuclear\\nBiomass\\nHydro\\nWind\\nOther renewables\\nNuclear\\nHydro\\nBiomass\\nWind\\nOther renewables\\nCarbon capture and storage\\n2009 fuel mix\\nNew policies scenario\\nFigure 5 | Projections from the \\nInternational Energy Agency for the \\ngeneration of electricty until 2035.\\u2002 In \\nthe top panel, the relative energy \\nmix in 2009 and that projected in the \\nInternational Energy Agency’s ‘new \\npolicies’ scenario. In the bottom panel, \\nin the new policies scenario, carbon \\nemissions continue to grow, but slower \\nthan the 2009 fuel-mix curve. The 2009 \\nfuel-mix curve is based on emissions \\nthat would have been generated for the \\nprojected level of electricity generation \\nwere there no change in the mix of fuels \\nand technologies, and no change in the \\nefficiency of thermal generation plants \\nafter 2009. The cost of renewable energy is \\ndeclining at a faster rate than assumed in \\nthe International Energy Agency’s report. \\nIn addition, the relative mix of natural gas \\nto coal may be larger than that projected. \\nAdapted with permission from ref. 38.\\n2 9 8  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\npenetration of renewable energy in the United States is possible by 2050, \\nbut to achieve even half this goal, a transformation of the electricity sys\\xad\\ntem on the basis of technology advances and new operating procedures, \\nbusiness models and regulatory approaches is required45. \\nSolar\\nIn 2010, the US DOE created the SunShot initiative to reduce the LCOE \\nfrom solar sources (photovoltaic and thermal) to $50–60\\u2009MWh-1, with\\xad\\nout subsidies, by 2020. For solar photovoltaics at utility scale, the fully \\ninstalled cost target must be about $1 per Watt of electricity generated. \\nThe costs are broken down into $0.50\\u2009W-1 for the solar module, $0.10\\u2009W-1 \\nfor the  power electronics for grid integration and $0.40\\u2009W-1 for the bal\\xad\\nance of the system (including installation, permits and all other costs)46. \\nIn the four years between the second quarter of 2008 and the second \\nquarter of 2012, the actual spot market price of silicon solar modules \\nfor immediate delivery declined from $4\\u2009W-1 to about $1\\u2009W-1 (ref.\\xa047). \\nAlthough part of this drop was due to a temporary oversupply coupled \\nto the worldwide recession, continual engineering and manufacturing \\ninnovations contributed to this decline and are likely to continue doing \\nso in the coming years: future delivery contracts are about $0.80 W-1, \\nand there is a growing confidence that module costs will achieve the \\ngoal of $0.50\\u2009W-1 by 2020 (ref.\\xa048). In contrast, the balance-of-system \\ncosts, specifically the soft costs such as permits, interconnection and \\ninspection, financing and customer acquisition have declined but at a \\nsignificantly slower rate. Because of this trend, a focus of the US DOE \\nSunShot programme is lowering all balance-of-system costs commen\\xad\\nsurate with module prices49.\\nAll costs are affected by increasing the efficiency of the solar cells while \\nreducing their production cost. The Shockley–Queisser limit is a well-\\nknown efficiency limit of single-junction solar cells, which suffer from \\nspectrum losses, recombination and black-body radiation. The Shock\\xad\\nley–Queisser limit of silicon is 32.7%, but a practical limit is considered \\nto be 29% (ref.\\xa050). The ultimate efficiency limit in the conversion of sun\\xad\\nlight energy to electrical energy is determined by a Carnot heat engine. \\nThis limit is about 94%, based on the high temperature of the sun’s \\nblack body at 5,800\\u2009K, and the cell temperature of about 350\\u2009K. There \\nis, therefore, significant room to surpass the Shockley–Queisser limit.\\nMost of the solar cells manufactured today are made of crystalline \\nsilicon, and have shown efficiencies as high as 23% at a production-\\nscale, although most cells are within the 17–19% range. In this Insight, \\nGraetzel\\xa0et al.51 review how solution-processed solar cells could be com\\xad\\npetitive if the cost could be reduced and efficiency increased, but this is \\njust one approach, competing with other rapidly advancing techniques. \\nPolycrystalline-silicon photovoltaics typically have efficiencies in \\nthe range of 14–16%, although recent progress indicates that refine\\xad\\nments in bulk polysilicon production could yield efficiencies similar \\nto that of single-crystal silicon for half of the ingot material52. The cost \\nof photovoltaic-grade silicon forms half the cost of a solar module. In \\nconventional silicon-cell production, about half the silicon is lost in saw\\xad\\ning the bulk silicon into 150–200-μm-thick wafers. With modest light-\\ntrapping techniques53, the wafers can be less than 50\\u2009μm thick. Other \\napproaches to lowering costs include single-crystal epitaxial growth \\nand lift-off, pursued by Solexel of Milpitas, California54, and kerf-less \\nmanufacturing of wafers with large and aligned grains in which the thin \\nwafers are directly made55,56.\\nThe other competitive approaches are based on direct-gap \\nsemiconductors such as cadmium telluride and copper-indium-gal\\xad\\nlium-selenide (CIGS), in which only a thin film (less than 1\\u2009μm) is \\nneeded to absorb solar radiation. Thin-film solar cells can be deposited \\non light and low-cost substrates, reducing their overall cost and weight, \\nwhich is an advantage for transportation and installation. However, \\nthin-film deposition on heterosubstrates introduces defects in these \\nfilms, which currently limit production-scale efficiency of cadmium-\\ntelluride and CIGS thin-film solar cells to 12–14% — their practical \\nupper limit is 29%. If the defects in cadmium-telluride thin films could \\nbe reduced in production, this would increase the carrier lifetime and \\nincrease the efficiency57. This reduction requires an understanding of \\nthe origins of defects, as well as ways to reduce and control them. The \\nrole of interfaces is a crucial aspect of solar cells and requires significant \\ninvestment in research and development. \\nIn addition to scientific understanding, engineered online monitoring \\nand control systems may be needed to improve manufacturing quality \\ncontrol and as a means of continual improvement. Currently, panels made \\nof cadmium-telluride thin films have the lowest cost (about $0.70\\u2009W-1). \\nOwing to the financial discount that is applied to any new solar technol\\xad\\nogy that is expected to perform for at least 20–25 years, any photovoltaic \\ntechnology developed has to have the technological ‘head-room’ to be \\ncompetitive with the projected progress of more established technologies. \\nAn increase in efficiency from 17% to 20% would significantly reduce the \\npanel cost and some of the balance-of-system costs, which could have a \\ntransformative effect on the market similar to that expected for polycrys\\xad\\ntalline silicon cells with an efficiency of over 18% (ref.\\xa052). \\nThin-film solar cells made of direct-gap III–V semiconductors are \\nattractive because of their higher carrier mobilities, and their superior \\nmaterial quality that can make use of the materials research that focused \\non optoelectronics. This allows III–V semiconductor cells to achieve \\na much higher efficiency that is close to the practical upper limit. For \\nexample, solar-cell efficiencies of about 29% have been shown for III–V \\ndevices, which approaches the Shockley–Queisser limit of 33.5% for any \\nsingle bandgap material58. Although high-quality films have generally \\nbeen achieved on crystalline substrates, use of bulk materials would be \\ncost-prohibitive for solar applications. The main challenge is to achieve \\nthe crystalline quality of III–V thin films, but on low-cost substrates \\nsuch as plastics or sheet metals. As for techniques in solar cells that \\nuse silicon epitaxial growth followed by lift-off, good progress is being \\nmade towards manufacturing, on a larger scale, lift-off of thin crystalline \\ngallium-arsenide layers grown epitaxially from bulk crystals. \\nImproving solar-module efficiency may also come from \\nmultijunction cells that capture a larger fraction of the solar spec\\xad\\ntrum, multi-exciton generation, multiphoton absorption or photon \\nup-conversion, and light concentration. The use of quantum dots and \\nmolecules to utilize some of these phenomena are being pursued, and, \\n0 \\n50 \\n100 \\n150 \\n200 \\n250 \\n300 \\n350 \\n400 \\nConventional coal\\nAdvanced coal with CCS\\nNG ACC\\nNG ACC with CCS\\nNG CCT\\nAdvanced nuclear\\nWind\\nWind-oﬀshore\\nSolar PV\\nSolar thermal\\nGeothermal\\nBiomass\\nHydro\\nLevellized cost ($ MWhr–1) \\nFigure 6 | Projections by the US Energy Information Administration made \\nin 2010 of the 2016 average cost and range of the levellized cost of electricity \\nfrom various sources.\\u2002 Note the cost of onshore wind at level 4 sites where \\nthe wind speed is 7.0–7.5 ms-1 and there are no financial incentives. Wind, \\nutility-scale solar photovoltaics and solar thermal energy in 2012 in the United \\nStates (green) is already at or below the lower end of the 2016 estimates. ACC, \\nadvanced combined cycle; CCS, carbon capture and storage; CCT, conventional \\ncombustion turbine; NG, natural gas; PV, photovoltaics.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 9\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nas described by Graetzel et al.51, are leading to a deeper understanding \\nof light–matter interactions. The competition in solar technologies is \\nfierce, and to have an effect on solar-electricity generation, the science \\nmust be translated into engineered systems that can meet the techno\\xad\\neconomic target of about $0.50\\u2009W-1 for a solar panel at commercial scale. \\nAlthough the discussion so far has focused on solar flat panels, there are \\nopportunities to generate electricity at $50–60\\u2009MWh-1 using concentrated \\nsolar photovoltaics. Most of the cost is in the balance of systems includ\\xad\\ning the mirrors, lenses and their tracking system; however, the cost of the \\nsolar cell at the focal point of the optical system is not as significant. This \\nis when multijunction solar cells with efficiencies that exceed 40% would \\nbe ideal. Innovations in low-cost and high-quality mirrors59, inexpensive \\nFresnel optics, as well as creative engineering to track them throughout \\nthe day at low cost are being developed.\\nComplementary to electricity generation using photovoltaics, solar \\nthermal approaches that concentrate sunlight in parabolic troughs and \\npower towers have also been used to convert absorbed solar radiation \\ninto heat (typically below 450\\u2009°C). Electricity is then generated at about \\n30–35% efficiency using a heat engine46. Heat can also be stored to allow \\nbetter demand matching, especially the offset peak electricity demand, \\nby several hours in climates and economies in which air-conditioning \\ndemand is rising. \\nTo reduce the LCOE to $50–60\\u2009MWh-1, the efficiency of the heat \\nengines must be increased. This requires more than 600\\u2009°C, suggest\\xad\\ning that the higher-light concentration, as achieved in power towers or \\nparabolic dish mirrors, is needed. There is significant opportunity for \\nresearch to identify ways to store heat through the enthalpy of phase \\ntransitions or chemical reactions at more than 600\\u2009°C, as well as systems \\nengineering to efficiently store and access this enthalpy on demand to \\nrun the heat engine for power generation60. Furthermore, research into \\nlow-cost and high-temperature materials, and their use in engineered \\nsystems, is needed to design and operate solar thermal reactors at these \\nunprecedented high temperatures. \\nAlthough most of the work on concentrated solar power has involved \\neither photovoltaic or thermal approaches, there are opportunities to \\ncombine them. For example, the visible part can be spectrally filtered \\nout of the concentrated solar radiation for photovoltaics, and the rest \\nof the radiation used for solar thermal approaches. Furthermore, solar \\nthermal techniques rarely use all of the concentrated solar radiation, \\nbecause there is generally some spillover from the solar absorber. Inte\\xad\\ngrating photovoltaics in a solar thermal absorber may be possible, allow\\xad\\ning both approaches to be integrated. Achieving more than 600\\u2009°C in a \\nsolar thermal absorber would also make it compatible with gas turbines \\nused in natural-gas combined cycle technology, which would allow inte\\xad\\ngration of solar and natural-gas power plants to address baseload and \\npeak demands.\\nTo convert direct current to alternating current electricity and to \\nintegrate solar power into the grid, power electronics is crucial and \\nmust be reduced to $0.10\\u2009W-1 or less — roughly half the cost it is today46. \\nWe are currently witnessing a revival of power-conversion electron\\xad\\nics in which the introduction of wide bandgap semiconductors such \\nas silicon-carbide and gallium-nitride, low-loss soft magnetics, solid-\\nstate capacitors and new-circuit topologies is having multiple positive \\neffects61. These allow a significant increase in the operating frequency, \\nwhich reduces the sizes of inductors and capacitors without altering \\ntheir electrical impedance. The size reduction can allow integration of \\nvarious components onto the same platform, reducing cost and weight. \\nThe use of solid-state capacitors as opposed to electrolytic ones can sig\\xad\\nnificantly increase reliability, which reduces the LCOE of solar energy. \\nThe need to address the variable nature of intermittent technolo\\xad\\ngies such as solar will become crucial at high levels of penetration. \\nApproaches to forecast the time-dependent solar insolation are needed to \\nplan electricity dispatch. In parallel, storage technologies that will offset \\nthe variations in solar electricity output need to be developed (see later). \\nWind\\nThe cost decline of wind energy is a result of significant improvements \\nto turbines, blades and gearboxes, and an increase in the height of \\nthe wind towers. Although the largest wind turbine has a capacity of \\n7.5\\u2009MW, most turbines being installed are 1.5–2\\u2009MW (ref.\\xa062). \\nAs wind turbines increase in height and size, there is a marked \\nincrease in the mechanical stresses on the gearbox, blades and tower. \\nDirect-drive wind turbines that can convert the slow rotary motion \\ndirectly into alternating current electrical power using electromagnetic \\ngenerators are receiving increased attention. However, these generators \\ncurrently use permanent magnets based on rare-earth materials such as \\nneodymium and dysprosium for their high remnant fields, field anisot\\xad\\nropy and high-temperature stability. Although rare-earth deposits are \\nfound in many countries around the world, more than 90% of current \\nproduction is in China63,64.\\nResearch in magnetic materials that can produce large magnetic \\nenergy density without the need for rare-earth metals is needed. \\nFor example, there are certain phases of iron-born compounds (for \\nexample, Fe16N2) that produce extremely high magnetocrystalline \\nanisotropy, but that are difficult to access and stabilize under ambi\\xad\\nent conditions65,66. Today’s computing power could be used to predict \\nmagnetic behaviour in a combinatorial search that could guide the \\ndiscovery of magnetic materials with high crystalline anisotropy and \\nthose with accessible stable phases. Often, accessing these metastable \\nphases requires nanostructuring, as well as manufacturing composites \\nwith materials such as soft magnets67. \\nOffshore wind farms are an attractive source of energy because \\nthey can be placed in near-proximity to population centres and yet far \\nenough away to mitigate local opposition. However, the cost remains \\nhigh because of the need to build stable marine structures that can \\nwithstand the harsh marine environment for many years without the \\n0\\n100\\n200\\n300\\n400\\n500\\nMarine: wave\\nMarine: tidal\\nSTEG: LFR\\nSTEG: tower and heliostat\\nSTEG: parabolic trough + storage\\nWind: oﬀshore\\nSTEG: parabolic trough\\nSTEG: tower and heliostat storage\\nPV: thin ﬁlm\\nPV: c-Si\\nPV: c-Si tracking\\nBiomass: anaerobic digestion\\nBiomass: gasiﬁcation\\nGeothermal: binary plant\\nBiomass: incineration\\nWind: onshore\\nLandﬁll gas\\nMunicipal solid waste\\nGeothermal: ﬂash plant\\nSmall hydro\\nLarge hydro\\nNuclear\\nCoal-ﬁred\\nNatural gas CCGT\\n789 +9%\\n1,049 +9%\\n+11%\\n–2%\\n+5%\\n–4%\\n–5%\\n–18%\\n–9%\\n+1%\\n–8%\\n–1%\\n–14%\\n–\\n–1%\\n+2%\\n–1%\\n–\\n–\\n–\\n–\\n+1%\\n+1%\\n+2%\\nFigure 7 | Levellized cost of \\nenergy for quarter three, \\n2012.\\u2002 The analysis of cost in \\ndollar per megawatt-hour is \\nbased on available data from \\nprojects in which investment \\nor purchase contracts have \\nbeen completed. To produce \\ncomparisons of levellized cost \\nof energy at a global level, \\nthe analysis assumed a single \\ncorporate tax rate of 35%, an \\nannual inflation of 2% and the \\nequity investor internal rate of \\nreturn of 10%. Quarter three \\ncentral scenarios, filled triangles; \\nquarter two central scenarios, \\nopen triangles. Adapted with \\npermisssion from ref. 41.\\n3 0 0  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nneed for replacement. Furthermore, because heavy-duty maintenance \\ncurrently needs calm seas, offshore wind turbines have to be highly reli\\xad\\nable. One approach to servicing the highest wind turbines is to design \\ncranes that can ‘inch worm’ themselves up the towers, analogous to the \\nself-lifting cranes used to build tall buildings.\\nDecarbonizing fossil-fuel emissions\\nIf the world is to continue to generate electricity from fossil fuels, the \\ncarbon emissions from the major stationary ‘point’ sources will have \\nto be significantly reduced in the coming decades. Achieving higher \\nprimary energy use through cost-effective improvements in thermo\\xad\\ndynamic efficiency, co-generation of power and heat, and use of waste \\nheat can reduce carbon emissions. However, maximizing efficiency is \\nnot sufficient. Carbon capture and storage (CCS) from coal and natural-\\ngas power plants and other major emitters, such as cement and steel \\nplants, will be necessary. \\nA detailed analysis of post-combustion carbon capture science, \\nengineering and economics can be found in a recent American Physical \\nSociety (APS) report68. The cost of CO2 capture can be largely divided \\ninto two components: the capital cost, mostly of a sorbent-based CO2 \\nseparation unit, the sorbent regenerator and the CO2 compressor; and \\nthe operating cost, mainly a result of the parasitic energy needed to \\ncapture and compress CO2. The capital cost depends on the size of the \\nseparation unit, which is inversely correlated with the rate constant of \\nthe CO2 binding reaction with a sorbent. The energy cost, however, \\ndepends on the enthalpy of the sorbent-CO2 dissociation reaction. The \\nkinetics and the thermodynamics of the binding and unbinding reac\\xad\\ntions of today’s sorbents are correlated (Fig.\\xa08).\\nResearch is needed to identify sorbents that have rate constants of more \\nthan 10,000\\u2009M-1\\u2009s-1, but of enthalpies 40–80\\u2009kJ\\u2009mol-1. There are various sec\\xad\\nondary and tertiary amines, metal-organic frameworks and ionic liquids \\nwhose reactions could perhaps be tailored to approach desirable kinetic \\nand thermodynamic performance levels68 (Fig.\\xa08). In biological organ\\xad\\nisms, enzymes such as carbonic anhydrase greatly reduce the free-energy \\nbarrier in the capture and release of CO2. Researchers are making progress \\nin genetically altering the properties of carbonic anhydrase69 so that it can \\nbe used to improve the kinetics of low-enthalpy sorbents under the harsh \\nconditions of the exhaust of a coal-fired power plant. \\nThe cost of retrofitting existing pulverized coal plants to achieve 90% \\nCO2 capture with the technology available is estimated to require capital \\nexpenditures that approach those of the original plant3. In addition, \\n20–40% of the plant’s energy would be diverted for separation, compres\\xad\\nsion and transmission of the CO2, so only the most thermodynamically \\nefficient supercritical and ultra-supercritical plants could be considered \\nfor retrofits.\\nCO2 capture can be achieved relatively easily if a highly concentrated \\nstream of CO2 exhaust is produced, which is possible by oxycombus\\xad\\ntion (in pulverized coal or natural-gas power plants) or integrated \\ngasification combined-cycle (IGCC) technology. Cryogenic separa\\xad\\ntion of oxygen from air is the most common industrial-scale method \\nof production. \\nThere is no clear choice between oxycombustion and post-\\ncombustion capture3, but it is clear that innovative chemical and physi\\xad\\ncal separation methods need to be developed, piloted and demonstrated \\nat near-commercial scale before wide-scale deployment can occur. \\nOther economical solutions to CCS are needed, but without the pros\\xad\\npect of a meaningful price on CO2, the private sector is less motivated to \\ninvest in technology development. For this reason, research and devel\\xad\\nopment of CCUS (carbon capture, utilization and storage of carbon) \\nin enhanced oil recovery and possibly enhanced geothermal energy70 \\nprovides a financial incentive to invest in carbon-capture technologies. \\nAdvancing CCS and CCUS could come from high-performance \\ncomputing for in silico optimization of system performance, rather \\nthan expensive pre-commercial projects. High-fidelity simulations \\ncan also be used to reduce the risk of scaling small demonstration \\nprojects to larger systems, and thereby accelerate these projects \\nadoption by industry. \\nNuclear\\nNuclear power can have an important role in efforts to decarbonize \\nthe production of electricity. Worldwide, nuclear energy constituted \\nabout 14% of the total electrical power generated in 2009 (ref.\\xa071), but \\nin 2011 this percentage dropped to about 12%, with a loss of 180\\u2009TWh \\nof generation from Germany and Japan5,72. The reaction to the 2011 \\nFukushima nuclear disaster has been varied: some countries that have \\nnuclear reactors are proceeding with caution, others continue to show \\ninterest in starting a nuclear-power programme, some countries have \\nput their plans on hold and some have opted to reverse their decision \\non nuclear energy or accelerate planned phase-outs. \\nAn increasing number of older nuclear plants will need to have their \\noperating licenses extended beyond their original time limit if they are \\nto continue operating. The cost of operating fully amortized plants in the \\nUnited States is estimated to be about $23\\u2009MWh-1, including the contri\\xad\\nbution of $1\\u2009MWh-1 to the nuclear-waste fund for the cost of spent fuel \\ndisposal73. The estimated cost of decommissioning each plant, including \\nused fuel and site restoration costs is about $500 million per plant. The \\nlow cost of operating the current fleet of reactors has to be balanced with \\nsafety concerns, disposition of spent fuel and risks of nuclear weapons \\nproliferation. Unfortunately, space does not allow a full discussion of \\nthese important concerns. Although there are significant challenges, the \\ndevelopment of carbon-free sources of baseload power (nuclear and fossil \\nplants with CCS) are needed to mitigate the worst risks of climate change. \\nThird-generation nuclear-power plants are engineered to be signifi\\xad\\ncantly safer than previous generation ones. In the event of an emergency \\nshutdown of a nuclear reactor, the first few days — when the heat from \\ndaughter nuclear decay is largest and has to be dissipated — are the \\nmost critical. New reactors being constructed are designed to safely \\ncool down for three days without access to grid or emergency electrical \\npower or to water outside the reactor facility. The ultimate goal is to \\nachieve passive safety, so there is very little chance of thermal damage to \\nthe reactor without requiring any active control after sudden shutdown.\\nThe LCOE of new nuclear plants is about $100–120\\u2009MWh-1; although \\nthis cost is not competitive with natural gas, it is less than estimates \\nfor fossil-fuel plants with CCS4. Another challenge is that the size of \\na single reactor is 1.0–1.5 GW with associated cooling and electri\\xad\\ncal distribution requirements. The cost of nuclear-reactor engineer\\xad\\ning, procurement and construction, operation and maintenance, and \\ndecommissioning is about $6,000–6,600 per installed kilowatt, which \\nis roughly five times higher than comparable natural-gas generation74 \\ncapital costs, and is quantized in increments of about $6 billion to \\n$10\\xa0billion. The large financial exposure and the risk of construction \\nand licensing delays have driven up the cost of financing plants75. \\nThe US DOE intends to support the engineering designs required \\nby the Nuclear Regulatory Commission for licensing small-modular \\nIdeal absorbents\\nor adsorbents\\n0\\n20\\n20,000\\nRate constant, kf (M–1s–1) \\nReaction enthalpy, ΔH (kJ Mol–1) \\n15,000\\n10,000\\n5,000\\n40\\n60\\n80\\n100\\n120\\nCurrent absorbents or adsorbents\\nFigure 8 | Rate constant of current absorbents or adsorbents is about \\nlinearly proportional to the reaction enthalpy.\\u2002 New sorbents are needed to \\nenhance the rate constant but maintain the energy required to release CO2, to \\ndrive down the capital and operating costs of carbon capture. Adapted with \\npermission from ref. 68.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  3 0 1\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nreactors between 80\\u2009MW and 300\\u2009MW (refs.\\xa076, 77). It is possible that \\nsafe nuclear power can be made more accessible through the economy \\nof constructing dozens of reactors in a factory rather than one at a time \\nat each site. Also, with the risk of licensing and construction delays \\nreduced, small-modular reactors may represent a new paradigm in \\nnuclear construction. The US DOE has also established an Energy \\nInnovation Hub to develop multiphysics computational simulation \\ntools to reduce the time needed to design and certify many aspects of \\nboth conventional reactors and small-modular reactors78.\\nGrid energy storage\\nThe reliability and efficiency of the transmission and distribution \\nsystem is improved by grid energy storage. Balancing and optimizing \\npower flow and generation are challenges that need improved tech\\xad\\nnology, and business and regulatory vision. Significant improvements \\nwould help to realize more efficient energy markets, a lower cost of \\nenergy, increased reliability and security, and penetration of renewable \\nenergy beyond 20%. \\nThe mismatch between an intermittent electricity supply and demand \\noccurs over multiple time and energy scales. For seconds-to-minutes \\nscales, voltage and frequency stabilization is currently achieved by grid-\\nancillary services that mostly use large banks of capacitors and hot-\\nspinning reserves from fossil-fuel plants, but also banks of batteries and \\nmechanical fly wheels. Hot-spinning reserves are also used to shore-up \\ngeneration from minutes to hours. Modern natural-gas carbon-cycle \\nplants are designed to ramp up rapidly at about 50\\u2009MW min-1, which, \\nwhen combined with improved sensing and control of the grid, will \\nbe sufficient to accommodate most of the intermittency problems of \\nrenewable energy. For hours-to-overnight scales, power and energy \\ncapacities of hundreds to thousands of megawatt and megawatt hour \\nwill be needed, alternatively many smaller (hundreds to thousands of \\nkilowatts) of distributed energy storage sites can be used.\\nThe most inexpensive method to store electricity for the medium- \\nand long-term is to use pumped hydroelectric and compressed air in \\nair-tight caverns, for which the capital cost is about $100\\u2009kWh-1. How\\xad\\never, geographical constraints warrant alternative approaches. New \\nelectrochemical and modular and low-cost fabricated compressed-air \\nsystems are promising79.\\nPerspective\\nGiven the anticipated technological advances in finding and extracting \\nfossil fuels, the cost of these forms of energy could remain competitive \\nwith carbon-free sources for decades. In addition, the mix of energy \\nsources in the United States (Fig.\\xa09) indicates that the timescale of change \\nwill be roughly half a century80. If we are to mitigate climate risks in a \\ntimely manner, the speed of the adoption of clean-energy technologies \\nat a widely-deployed commercial scale has to be accelerated. Although \\nthe deployment of any innovation will depend on its benefits relative to \\nestablished technologies, inertia as well as real and perceived financial \\nrisks always favour the status quo. To accelerate the use of economically \\nviable clean-energy solutions, government policies are needed to stimu\\xad\\nlate invention, innovation, and align the market forces. \\nWith increases over the past 30 years in the number of extreme \\nweather events, such as temperature extremes, floods, wildfires, \\ndroughts and storms, the overall loss trend is beginning to exceed $150 \\nbillion per year81. There is increasing evidence that these changes are \\nlinked with climate change82. Although the full costs of mitigating or \\nadapting to these changes have significant uncertainties, prudent risk \\nmanagement does not equate uncertainty with inaction. Policies are \\nneeded that begin to include the full cost of the various sources of energy \\nmore directly into their market prices82. \\nThis Perspective highlights a few of the challenges and opportunities \\nfor science, engineering and policy. Our ability to find and extract fossil \\nfuels continues to improve, and economically recoverable reservoirs \\naround the world are likely to keep pace with the rising demand for \\ndecades. The Stone Age did not end because we ran out of stones; we \\ntransitioned to better solutions. The same opportunity lies before us \\nwith energy efficiency and clean energy.\\nThe cost of renewable energy is rapidly becoming competitive with \\nother sources of energy. With additional engagement of the scientific, \\nfinancial and public-policy communities, as well as the general public, \\nwe can accelerate the transition to affordable, accessible and sustainable \\nenergy that will power economic growth, increase energy security and \\nmitigate the risks of climate change. On the other hand, to quote from \\nthe IEA’s World Energy Outlook, “If we don’t change direction soon, \\nwe’ll end up where we’re heading”. ■\\n1.\\t\\nLee, R. The outlook for population growth. Science 333, 569–573 (2011). \\n2.\\t\\nInternational Energy Agency in World Energy Outlook 2011, 546–547 \\nhttp://www.worldenergyoutlook.org (International Energy Agency, 2011). \\n3.\\t\\nCommittee on America’s Energy Future, National Academy of Sciences, National \\nAcademy of Engineering and National Research Council. America’s Energy \\nFuture: Technology and Transformation (National Academies Press, 2009).\\n4.\\t\\nUS Department of Energy. Report on the First Quadrennial Technology Review. \\nhttp://energy.gov/sites/prod/files/QTR_report.pdf (US DOE, 2011).\\n5.\\t\\nBP. BP Statistical Review of World Energy. http://bp.com/statisticalreview2012. \\n(BP, 2012).\\n6.\\t\\nMaugeri, L. Oil: The Next Revolution. Discussion Paper 2012–10. \\nhttp://belfercenter.ksg.harvard.edu/files/Oil-%20The%20Next%20\\nRevolution.pdf (Belfer Center for Science and International Affairs, Harvard \\nKennedy School, 2012).\\n7.\\t\\nInderwildi, O. & King, D. (eds). Energy, Transport, & the Environment, Addressing \\nthe Sustainable Mobility Paradigm. (Springer, 2012).\\n8.\\t\\nUS Department of Energy. Quadrennial Technology Review report, vol. II Technology \\nAssessments http://energy.gov/articles/department-energy-releases-inaugural-\\nquadrennial-technology-review-report (US Department of Energy, 2011).\\n9.\\t\\nGibbs, J., Pesaran, A. A., Sklad, P. S. & Marlino, L. D. in Fundamentals of Materials \\nfor Energy and Environmental Sustainability (eds Ginley, D. S. & Cahen D.) Ch. 31, \\n426–444 (Cambridge Univ. Press, 2012).\\n10.\\t Powers, W. F., Automotive materials in the 21st century. Adv. Mater. Process. \\n157, 38–41 (2000).\\n11.\\t Cuddy, M. R. & Wipke, K. B. Analysis of Fuel Economy Benefit of Drivetrain \\nHybridization. http://www.nrel.gov/vehiclesandfuels/vsa/pdfs/22309.pdf \\n(National Renewable Energy Laboratory, 1997).\\n12.\\t Holmberg, K., Anderssona, P. & Erdemirb, A. Global energy consumption due to \\nfriction in passenger cars. Tribol. Int. 47, 221–234 (2012). \\n13.\\t Schafer, A., Heywood, J. B. & Weiss, M. A. Future fuel cell and internal \\ncombustion engine automobile technologies: a 25-year life cycle and fleet \\nimpact assessment. Energy 31, 2064–2087 (2006). \\n14.\\t Manley, D. K., McIlroy, A. & Taatjes, C. A. Research needs for future internal \\ncombustion engines. Physics Today 61, 47–52 (2008).\\n15.\\t Yang, J. & Caillat, T. Thermoelectric materials for space and automotive power \\ngeneration. MRS Bull. 31, 224–229 (2006).\\n16.\\t Scrosati, B. & Grache, J. Lithium batteries: status, prospects and future. J. Power \\nSources 195, 2419–2430 (2010).\\n17.\\t McCone, A. Electric Vehicle Battery Prices Down 14% Year on Year http://www.\\nbnef.com/PressReleases/view/210 (Bloomberg New Energy Finance, 2012).\\n18.\\t Chan, C. C. The state of the art of electric, hybrid, and fuel cell vehicles. \\nProc. IEEE 95, 704–718 (2007).\\n19.\\t Lukic, S. M., Cao, J., Bansal, R. C., Rodriguez, F. & Emadi, A. Energy storage \\nsystems for automotive applications. IEEE Trans. Ind. Electron. 55, 2258–2267 \\n(2008). \\n20.\\t Spendelow, J. & Papageorgopoulos, D. Progress in PEMFC MEA component \\n100\\n90\\n80\\n70\\n80\\n60\\n50\\n40\\n30\\n10\\n0\\n1850\\n1970\\n1940\\n1910\\n1880\\nPercentage\\n2000\\nRenewables\\nNuclear\\nGas\\nOil\\nHydro\\nCoal\\nWood\\nFigure 9 | History of the relative mix of the main source of energy used in \\nthe United States. The timescale for a shift in the energy mix is measured in \\ndecades. The growth in use of oil was increasingly driven by the transportation \\nsector, for which it makes up 94% of transportation energy. Electricity \\nis generated from coal (48%), natural gas (19%), nuclear power (21%), \\nrenewables 10%, and oil (1%). Adapted with permission from ref. 80.\\n3 0 2  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nR&D at the DOE fuel cell technologies program. Fuel Cells 11, 775–786 (2011).\\n21.\\t Debe, M. K. Electrocatalyst approaches and challenges for automotive fuel cells. \\nNature 486, 43–51 (2012).\\n22.\\t Yang, J., Sudik, A., Wolverton, C. & Siegel, D. J. High capacity hydrogen storage \\nmaterials: attributes for automotive applications and techniques for materials \\ndiscovery. Chem. Soc. Rev. 39, 656–675 (2010).\\n23.\\t Paster, M. D. et al. Hydrogen storage technology options for fuel cell vehicles: \\nwell-to-wheel costs, energy efficiencies, and greenhouse gas emissions. Int. J. \\nHydrogen Energy 36, 14534–14551 (2011).\\n24 .\\tDavid, E. An overview of advanced materials for hydrogen storage. J. Mater. \\nProcess. Technol. 162–163, 169–177 (2005).\\n25.\\t Murray, L. J., Dincă, M. & Long, J. R. Hydrogen storage in metal–organic \\nframeworks. Chem. Soc. Rev. 38, 1294–1314 (2009).\\n26.\\t Abbas, H. F. & Wan Daud, W. M. A. Hydrogen production by methane \\ndecomposition: A review. Int. J. Hydrogen Energy 35, 1160–1190 (2010).\\n27.\\t Krupnick, A. J. Will Natural Gas Vehicles be in Our Future. http://www.rff.org/RFF/\\nDocuments/RFF-IB-11-06.pdf (Resources for the Future, 2011).\\n28.\\t NACS. Key Facts and Figures. http://www.nacsonline.com/NACS/Resources/\\ncampaigns/GasPrices_2011/Pages/default.aspx (NACS, 2011).\\n29.\\t America’s Energy Future Panel. Liquid Transportation Fuels. \\n(National Academies Press, 2009).\\n30.\\t Environmental Protection Agency. Renewable Fuel Standard Program (RFS2): \\nFinal Rulemaking http://www.epa.gov/otaq/fuels/renewablefuels/regulations.\\nhtm (US Environmental Protection Agency, 2010). \\n31.\\t United Nations Environment Programme. Towards Sustainable Production of \\nand use of resources: Assessing Biofuels http://www.unep.org/publications/\\ncontents/pub_details_search.asp?ID=4082 (United Nations Environment \\nProgramme, 2009).\\n32.\\t Stratton, R. W., Wong, H. M. & Hileman, J. I. Quantifying variability in life cycle \\ngreenhouse gas inventories of alternative middle distillate transportation fuels. \\nEnviron. Sci. Technol. 45, 4637–4644 (2011).\\n33.\\t Peralta-Yahya, P. P., Zhang, F., del Cardayre, S. B. & Keasling, J. D. Progress \\nin the microbial production of advanced biofuels: from feedstocks to fuels. \\nNature 488, 320–328 (2012).\\n34.\\t Worldwatch Institute. Biofuels for Transport (Routledge, 2007).\\n35.\\t Georgianna, D. R. & Mayfield, S. P. Exploiting diversity and synthetic biology for \\nthe production algae biofuels, Nature 488, 329–335 (2012).\\n36.\\t Zhu, X-, G., Long, S. P. & Ort, D. R. What is the maximum efficiency with \\nwhich photosynthesis can convert solar energy into biomass? Curr. Opin. \\nBiotechnol. 19, 153–159 (2008).\\n37.\\t Conrado, R. J. et al. in Electrofuels: A new paradigm for renewable fuels. Advanced \\nBiofuels and Bioproducts (Lee, J. ed.) (Springer, 2012).\\n38.\\t International Energy Agency. World Energy Outlook 2011 179,186 \\nhttp://www.worldenergyoutlook.org/ (International Energy Agency, 2011).\\n39.\\t International Energy Agency. World Energy Outlook 2011, 180 \\nhttp://www.worldenergyoutlook.org/ (International Energy Agency, 2011).\\n40.\\t US Energy Information Administration. Annual Energy Outlook 2011 Levelized \\nCost Supplement http://www.eia.gov/forecasts/archive/aeo11/electricity_\\ngeneration.cfm  (US Energy Information Administration, 2011)\\n41.\\t Bloomberg New Energy Finance. Levelised Cost of Energy Update, Q3 2012 \\nhttp://www.bnef.com/WhitePapers/download/114 (Bloomberg New Energy \\nFinance, 2012).\\n42.\\t Wiser, R., Lantz E., Bolinger, M. & Hand, M. Recent Developments in the Levelized \\nCost of Energy from U.S. Wind Power Projects. http://eetd.lbl.gov/ea/ems/\\nreports/wind-energy-costs-2-2012.pdf (US Department of Energy, 2012).\\n43.\\t Galen Barbose, G. Darghouth, N., Wiser, R. & Seel, J. Tracking the Sun: An \\nHistorical Summary of the Installed Cost of Photovoltaics in the United States from \\n1998 to 2010 http://eetd.lbl.gov/ea/ems/reports/lbnl-5047e.pdf (Lawrence \\nBerkeley National Laboratory report, 2011). \\n44.\\t Solar Energy Industries Association. US Solar Market Insight Report Q1 2012 \\nhttp://www.seia.org/research-resources/us-solar-market-insight (Solar Energy \\nIndustries Association, 2012).\\n45.\\t Hand, M. M. et al. Renewable Electricity Futures Study http://www.nrel.gov/\\nanalysis/re_futures/ (National Renewable Energy Laboratory, 2012).\\n46.\\t US Department of Energy. SunShot Vision Study http://www1.eere.energy.gov/\\nsolar/pdfs/47927.pdf (US Department of Energy, 2012).\\n47.\\t Baziliana, M. et al. Re-considering the Economics of Photovoltiac Power www.\\nbnef.com/WhitePapers/download/82 (Bloomberg New Energy Finance, 2012).\\n48.\\t Swanson, R. M. Plenary talk at the DOE SunShot Grand Challenge: Summit and \\nTechnology Forum on June 14, 2012 https://www.eeremultimedia.energy.\\ngov/solar/videos/sunshot_summit_richard_swanson_plenary_session (US \\nDepartment of Energy, 2012).\\n49.\\t US Department of Energy. SunShot Initiative http://www1.eere.energy.gov/solar/\\nsunshot/accelerating_deployment.html (US Department of Energy, 2012).\\n50.\\t Swanson, R. M. Proc. 31st IEEE Photovoltaic Specialists Conf. 889–894 (2005).\\n51.\\t Graetzel M. Jannsen, R. A., Mitzi, D. B. & Sargent, E. H. Materials interface \\nengineering for solution-processed photovoltaics, Nature 488, 304–312 (2012).\\n52.\\t Stoddard, N., Wu B., Maisano, L., Russell, Clark, R. & Fernandez, J. M. The \\nLeading Edge of Silicon Casting Technology and BP Solar’s Mono2 Wafers, in 18th \\nWorkshop on Crystalline Silicon Solar Cells & Modules: Materials and Processes. \\npp. 7–14. http://www.nrel.gov/docs/fy08osti/45745.pdf (National Renewable \\nEnergy Laboratory, 2008).\\n53.\\t Mavrokefalos, A., Han, S. E., Yerci, S., Branham, M. S. & Chen, G. Efficient light \\ntrapping in inverted nanopyramid thin crystalline silicon membranes for solar \\ncell applications. Nano Lett. 12, 2792–2796 (2012).\\n54.\\t Solexel. Solexel unveils ultra-thin, high-performance silicon solar cell at Intersolar. \\nhttp://www.solexel.com/IntersolarNA2012.php (Solexel, 2012).\\n55.\\t Rao, R. A. et al. Proc. 37th IEEE Photovoltaic Specialists Conf. 1504–1507 (2011). \\n56.\\t Sachs, E. M., Wallace R. L., Hantsoo, E. T., Lorenz, A. M., Hudelson S.G.D. \\n& Jonczyk, R. Methods for efficiently making thin semiconductor bodies \\nfrom molten material for solar cells and the like, US Patent Application \\n20120067273 (2012).\\n57.\\t Garland J. W., Biegala, T., Carmody, M., Gilmore, C. & Sivananthan, S. Next-\\ngeneration multijunction solar cells: The promise of II-VI materials. J. Appl. \\nPhys. 109, 102423 (2011). \\n58.\\t Miller, O. D., Yablonovitch, E. & Kurtz, S. R. Strong internal and external \\nluminescence as solar cells approach the Shockley–Queisser limit. IEEE \\nJ. Photovolt. 2, 303–311 (2012).\\n59.\\t Angel, R. & Olbert, B. H. Method of manufacturing large dish reflectors for a \\nsolar concentrator apparatus. US Patent Application 20120125400 (2011).\\n60.\\t Gur, I., Sawyer, K. & Prasher, R. Searching for a better thermal battery. \\nScience 335, 1454–1455 (2012).\\n61.\\t Advanced Research Projects Agency-Energy. ARPA-E Efficient Power Conversion \\nFactsheet. http://arpa-e.energy.gov/Portals/0/Documents/FundedProjects/\\nADEPT%20Slicksheets/ADEPT_Program_Final_One%20Pager.pdf (Advanced \\nResearch Projects Agency-Energy, 2010).\\n62.\\t Bolinger, M. & Wiser, R. Understanding wind turbine price trends in the U.S. \\nover the past decade. Energy Pol. 42, 628–641 (2012). \\n63.\\t US Department of Energy. Critical Materials Strategy. http://energy.gov/sites/\\nprod/files/piprod/documents/cms_dec_17_full_web.pdf (US Department of \\nEnergy, 2010)\\n64.\\t National Research Council. Minerals, Critical Minerals, and the U.S. Economy \\n(National Academies Press, 2008).\\n65.\\t Komuro, M., Kozono Y., Hanazono, M. & Yutaka, S. Epitaxial growth and \\nmagnetic properties of Fe16N2 films with high saturation magnetic flux density. \\nJ. Appl. Phys. 67, 5126–5130 (1990).\\n66.\\t Nian, J., Lawrence F. A., Edgar L. C., &Wang, J.-P. N-site ordering effect on \\npartially ordered Fe16N2. Appl. Phys. Lett. 98, 092506 (2011).\\n67.\\t Bader, S. J. Opportunities in nanomagnetism. Rev. Mod. Phys. 78, 1–15 (2006).\\n68.\\t American Physical Society. Direct air capture of CO2 with chemicals: a Technology \\nAssessment for the APS Panel on Public Affairs. www.aps.org/policy/reports/\\nassessments/upload/dac2011.pdf (American Physical Society, 2011).\\n69.\\t Turner, N. J. Directed evolution drives the next generation of biocatalysts. Nature \\nChem. Biol. 5, 567–573 (2009).\\n70.\\t Pruess, K. Enhanced geothermal systems (EGS) using CO2 as working fluid \\n— a novel approach for generating renewable energy with simultaneous \\nsequestration of carbon. Geothermics 35, 351–367 (2006).\\n71.\\t International Energy Agency. Technology Roadmap — Nuclear Energy http://\\nwww.iea.org/papers/2010/nuclear_roadmap.pdf (International Energy Agency, \\n2010).\\n72.\\t Ruhl, C. Energy in 2011 — disruption and continuity BP Statistical Review of World \\nEnergy http://www.bp.com/sectiongenericarticle800.do?categoryId=9037130\\n&contentId=7068669 (BP, 2012).\\n73.\\t Nuclear Energy Institute. Costs: Fuel, Operation and Waste Disposal http://www.\\nnei.org/resourcesandstats/nuclear_statistics/costs/ (National Energy Institute, \\n2012). \\n74.\\t Nuclear Energy Institute. The Cost of New Generating Capacity in Perspective \\nhttp://www.nei.org/resourcesandstats/documentlibrary/newplants/\\ngraphicsandcharts/the-cost-of-new-generating-capacity-in-perspective \\n(Nuclear Energy Institute, 2012).\\n75.\\t Massachusetts Institute of Technology. Update of the MIT 2003 Future of Nuclear \\nPower Study http://web.mit.edu/nuclearpower/pdf/nuclearpower-update2009.\\npdf (Massachusetts Institute of Technology, 2009).\\n76.\\t Chu, S. America’s new nuclear option. Wall Street J. (March 23, 2010).\\n77.\\t US Department of Energy. Small Modular Reactor Fact Sheet http://www.ne.doe.\\ngov/pdfFiles/factSheets/2012_SMR_Factsheet_final.pdf (US Department of \\nEnergy, 2012).\\n78.\\t US Department of Energy. Consortium for Advanced Simulation of LWRs Project \\nSummary. http://www.ornl.gov/sci/nsed/docs/CASL_Project_Summary.pdf (US \\nDepartment of Energy).\\n79.\\t Advanced Research Projects Agency-Energy. Grid-scale renewable energy \\nstorage factsheet. http://arpa-e.energy.gov/Portals/0/Documents/\\nFundedProjects/GRIDS%20Slicks/_GRIDS_ProgramSheet_Final.pdf (Advanced \\nResearch Projects Agency-Energy, 2010).\\n80.\\t Energy Information Administration. Annual Energy Review 2010. Report No. \\nDOE/EIA-0384(2010) www.eia.gov/totalenergy/data/annual/pdf/aer.pdf \\n(Energy Information Administration, 2010).\\n81.\\t Munich Re. Topics Geo. http://www.munichre.com/publications/302-07225_\\nen.pdf (Munich Re, 2012).\\n82.\\t Intergovernmental Panel on Climate Change. Managing the Risks of \\nExtreme Events and Disasters to Advance Climate Change Adaptation. \\nhttp://www.ipcc-wg2.gov/SREX/images/uploads/SREX-All_FINAL.pdf \\n(Cambridge Univ. Press, 2012).\\nAcknowledgements The authors wish to acknowledge T. J. Augustine, P. Davis, \\nH. Gruenspecht, M. Le, P. Lyons, R. Ramamoorthy, D. Sandalow, S. Satyapal and \\nE. Toone for suggestions.\\nAuthor Information Reprints and permissions information is available at \\nwww.nature.com/reprints. The authors declare no competing financial \\ninterests. Readers are welcome to comment on the online version of this \\narticle at go.nature.com/7pb7ff. Correspondence should be addressed to S.C. \\n(the.secretary@hq.doe.gov) and A.M. (4majumdar.pubs@gmail.com).\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  3 0 3\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n',\n",
       " '1US Department of Energy, 1000 Independence Avenue SW, Washington DC 20585, USA. \\nT\\nhe industrial revolution began in the mid-eighteenth century, \\nand provided humans with capabilities well beyond animal and \\nhuman power. Steam-powered trains and ships, and then inter\\xad\\nnal combustion engines transformed how people moved and produced \\ngoods around the world. Electrification and related technologies contin\\xad\\nued the revolution in the nineteenth and twentieth centuries. Today, a \\ngrowing number of p',\n",
       " 'eople keep their homes warm in the winter, cool in \\nthe summer and lit at night. They go to the local market in cars with the \\npower of over a hundred horses and fly across continents in wide-body \\naeroplanes with the power of a hundred thousand horses. This power is \\nderived largely from our ability to exploit fossil sources of energy. How\\xad\\never, in the transition from human and horse power to horsepower, the \\ncarbon emissions that result from the equivalent of over a billion horses \\nworking co',\n",
       " 'ntinuously have created significant climate-change risks.\\nAt the beginning of the industrial revolution, the population of the \\nworld was 700\\xa0million. Today, the population is 7\\xa0billion and is estimated \\nto grow to 9\\xa0billion by 2050, and about 10 billion by 2100 (ref.\\xa01). Most of \\nthis population growth will be in Asia and Africa, where rapidly rising \\neconomic growth will place additional demands on energy supply. The \\nInternational Energy Agency (IEA) based in Paris has projected that \\nthe wor',\n",
       " 'ld’s energy demand will increase from about 12\\xa0billion tonne \\noil equivalents (t.o.e.) in 2009 to either 18\\xa0billion t.o.e. or 17\\xa0billion \\nt.o.e. by 2035 under their ‘current policies’ or ‘new policies’ scenarios, \\nrespectively2. Carbon-dioxide emissions are expected to increase from \\n29 gigatonnes per year to 43\\u2009Gt\\u2009yr-1 or 36\\u2009Gt\\u2009yr-1 under the current and \\nnew policies, respectively. The actual path we follow will depend on how \\nefficiently and effectively we use existing and new sources of ener',\n",
       " 'gy. \\nThe world needs another industrial revolution in which our sources \\nof energy are affordable, accessible and sustainable. Energy efficiency \\nand conservation, as well as decarbonizing our energy sources, are \\nessential to this revolution. Reducing carbon emissions on the timescale \\nneeded to mitigate the worst risks of climate change will not be driven \\nby our inability to find cost-effective sources of fossil fuels. Despite the \\nsignificant growth in the use of renewable energy, the fracti',\n",
       " 'onal sum of \\nnon-carbon- emitting sources of energy that remained constant during \\nthe past two decades is sobering (Fig.\\xa01). \\nEnergy systems can be divided into transportation and stationary. \\nThe supply, demand and distribution infrastructures within each system \\nare highly coupled, but are currently largely independent of each other. \\nIn this Insight, the Reviews focus on specific subareas of biofuels, solar \\nelectricity generation, and electricity generation from salinity gradients \\nand wast',\n",
       " 'e water. The systems discussed are largely at the research and \\ndevelopment stage, and are not yet ready to displace more mature \\nsources of energy. Nevertheless, we should be constantly looking for \\ninnovations that make a marked improvement on today’s approaches \\nor provide an entirely new approach. To place the topics discussed in \\nthe Reviews in context, this Perspective discusses transportation and \\nstationary power systems, and indicates how the topics discussed in the \\nReviews fit into th',\n",
       " 'e energy landscape. \\nMany important and challenging research areas have the potential to \\nsignificantly affect our future energy needs. For example, energy effi\\xad\\nciency and the integration of energy sources with electricity transmis\\xad\\nsion, distribution and storage are vitally important, but these are only \\nbriefly mentioned in this Perspective. Excellent reviews of these topics \\ninclude the National Academies’ report America’s Energy Future3 and \\nthe US Department of Energy’s (DOE’s) Report on t',\n",
       " 'he First Quadrennial \\nTechnology Review4.\\nTransportation\\nPetroleum-derived liquid fuels are the overwhelming source of energy \\nin the current transportation infrastructure. The geographical distri\\xad\\nbution of petroleum resources is changing as reserves are found and \\naccessed with improved technologies for discovery and production. \\nHowever, this distribution of oil supply generally does not coincide \\nwith where the demand is located. For example, many countries import \\noil at an unprecedented sc',\n",
       " 'ale, which can lead to significant balance-\\nof-trade and national-security challenges. In 2011, about 2.690\\xa0billion \\ntonnes of oil were consumed; of this, 1.895\\xa0billion tonnes of crude oil \\nand 0.791\\xa0billion tonnes of refined products crossed national borders5 \\nand significant discoveries of oil, natural-gas liquids and natural gas \\ncould potentially alter the global-energy landscape6. \\nWe will review some of the opportunities and challenges related to \\ntransportation technologies. For example, ',\n",
       " 'most of the future infrastruc\\xad\\nture of the world will be built in locations where we have the greatest \\nopportunity to transition to sustainable mobility. Desirable and afford\\xad\\nable public transportation that is fully integrated into urban planning7, \\nand the use of information technologies to assist and displace transpor\\xad\\ntation can significantly reduce fuel consumption, but are not discussed \\nin this Perspective. \\nThe Quadrennial Technology Review (QTR) of the US DOE provides \\nAccess to clean,',\n",
       " ' affordable and reliable energy has been a cornerstone of the world’s increasing prosperity and economic \\ngrowth since the beginning of the industrial revolution. Our use of energy in the twenty-first century must also be sustain\\xad\\nable. Solar and water-based energy generation, and engineering of microbes to produce biofuels are a few examples of the \\nalternatives. This Perspective puts these opportunities into a larger context by relating them to a number of aspects in the \\ntransportation and el',\n",
       " 'ectricity generation sectors. It also provides a snapshot of the current energy landscape and discusses \\nseveral research and development opportunities and pathways that could lead to a prosperous, sustainable and secure energy \\nfuture for the world.\\nOpportunities and challenges for \\na sustainable energy future\\nSteven Chu1 & Arun Majumdar1 \\n \\n2 9 4  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\ndoi:10.1038/nature11475\\n© 2012 Macmillan Publishers Limited. All rights re',\n",
       " 'served\\n\\na broad overview of state-of-the-art technologies and opportunities \\nfor future research8. Improvements in energy efficiency of vehicles \\ncan greatly reduce oil dependency. These improvements include \\nincreased use of light-weight materials, such as advanced ultra-high \\ntensile strength steels, aluminium and magnesium alloys, polymers, \\nand carbon-fibre reinforced composite materials9. The integration of \\nlighter weight materials is especially important if more complex parts \\ncan be manu',\n",
       " 'factured as a single unit. The potential for reducing the \\nweight of vehicles has already been shown, and in the next 10–20\\xa0years, \\nan additional 20–40% reduction in overall weight, without sacrificing \\nsafety, seems to be possible10. For every 10% weight reduction of the \\nvehicle, an improvement in fuel consumption of 6–8% is expected11. \\nReducing energy losses as a result of friction is also possible12 (Fig.\\xa02). \\nAdvances in cost-effective technologies such as tribology, tyres, braking \\nand wa',\n",
       " 'ste-heat energy recovery, and aerodynamics could potentially \\nlead to efficiency improvements of 20% in the short term and more than \\n60% over a longer term (15–25\\xa0years).\\nInternal combustion engines\\nThe internal combustion engine using liquid-transportation fuel (or \\nliquid ICE) will probably continue to have a major role over the next \\nfew decades. However, improvements to the efficiency of the liquid ICE \\nare possible because the efficiency of most spark-ignition engines is \\ntypically 25–35%,',\n",
       " ' whereas that for compression-ignition diesel engines \\nis about 40–50%. With direct injection, lean burn and turbocharger \\ntechnologies, the spark-ignition ICEs operating on premium octane rat\\xad\\nings can approach diesel efficiencies. The combination of in situ meas\\xad\\nurements of prototype ICE designs with detailed simulations, which \\nare made possible with high-performance computers, are increasing \\nthe energy efficiency and have lowered emissions13,14. Finally, low-cost \\nwaste-heat recovery can i',\n",
       " 'ncrease efficiency, especially in heavy-duty \\nvehicles. Approaches include use of the Rankine cycle to convert waste \\nheat to work, and the development of low-cost and high-efficiency \\nsolid-state thermoelectric systems15. \\nBattery-based electrification\\nPlug-in hybrid and all-electric light- and medium-duty vehicles have the \\nopportunity to displace a significant amount of liquid fuel use in trans\\xad\\nportation. The main challenges are performance and cost of the battery \\nsystems. The performance o',\n",
       " 'f battery systems is quantified by usable \\nenergy density, power density (including fast charging), cycle lifetime \\nand robustness. In the past 5–6\\xa0years, a remarkable amount of progress \\nin research has been made in battery cathodes, anodes and electro\\xad\\nlytes. Large volumetric changes in the electrodes have led to designs \\nin which micro- or nanostructures are embedded in a conducting and \\nflexible matrix to allow for the relief of mechanical stresses. State-of-the-\\nart batteries based on graph',\n",
       " 'ite anodes and lithium-manganese-oxide \\ncomposite cathodes for lithium batteries are being commercialized16. \\nWithin the next few years, battery system packs of 200\\xa0Watt-hour kg-1 \\nat a charging rate of full charge in 3 hours, which is double the cur\\xad\\nrent cell-energy density, are expected to become available. The current \\nproduction cost of a vehicle battery is estimated to be US$650\\u2009kWh-1 of \\nusable energy storage, but this is expected to drop to below $150\\u2009kWh-1 \\nby 2030 (ref. 17). \\nThe US DO',\n",
       " 'E is supporting a research and development effort that \\nwould allow a four or five passenger plug-in hybrid electric vehicle \\n(PHEV) or an electric vehicle with about a 160-km range to be competi\\xad\\ntive as a mass-market ICE car within a decade. The US DOE EV-Eve\\xad\\nrywhere challenge for PHEV, will require the cost of the battery system \\nto be reduced by an estimated $190–300\\u2009kWh-1, depending on whether \\nthe car is a PHEV or an electric vehicle. The development of anode-pro\\xad\\ntecting materials and no',\n",
       " 'n-flammable electrolytes that are stable at high \\nvoltage and tolerate 55\\u2009°C are desired characteristics of third-generation \\nlithium-ion batteries. Developing batteries beyond lithium-ion batter\\xad\\nies, such as lithium-sulphur and metal-air batteries, could achieve up \\nto ten times the energy density of the current lithium-ion batteries, but \\nmaterials research is needed to develop anode and cathode protection, \\nas well as non-flammable electrolytes with electrochemical stability over \\na large po',\n",
       " 'tential range.\\nBattery packs typically use about 50% of the total battery capacity, \\nand the charging rates are limited to increase the lifetime18. If sensor \\ntechnologies are developed that can continuously monitor the proper\\xad\\nties of individual cells, such as internal impedance, temperature and \\nFigure 1 | Statistical review of world energy.\\u2002 a, Fossil energy comprises \\nroughly 86% of the world’s main energy consumption. Although the \\nconsumption of oil has increased by 31% between 1980 and 20',\n",
       " '08, the known \\nreserves have increased comparably owing to improvements in exploration and \\nextraction technologies. Much of the world’s shale-gas reserves are not ‘proven’ \\nand are not included. The fractional sum of non-carbon emitting sources of \\nenergy remained constant during the same time period. b, Growth of renewable \\nenergy was offset by the decline in nuclear power generation. Renewable \\nenergy sources in power generation grew by 17.7%. Wind generation (25.8%) \\naccounted for more than ',\n",
       " 'half of renewable power generation for the first time. \\nRenewables accounted for 3.8% of global power generation, with the highest \\nshare in Europe and Eurasia (7.1%). Adapted with permission from ref. 72.\\nTrillion bbls\\nTcm\\nOil and gas reserves\\n1981\\n1991\\n2001\\n2011 1981\\n1991\\n2001\\n2011\\n0\\n0.8\\n0.4\\n1.2\\n1.6\\n2.0\\n0\\n50\\n100\\n150\\n200\\n250\\nOther\\nNorth America\\nFSU\\nOPEC\\n0\\n10\\n20\\n30\\n40\\n50\\n1966\\nPercentage\\n1981\\nShares of world primary energy\\n1996\\n2011\\nOil\\nCoal\\nGas\\nHydro\\nNuclear\\nRenewable\\na \\nMillion tonne oil equiva',\n",
       " 'lent\\n0\\n91\\n95\\n93\\n99\\n97\\n01\\n03\\n05\\n07\\n09\\n11\\n50\\n100\\n150\\n200\\nAsia Paciﬁc\\nAfrica\\nMiddle East\\nEurope and Eurasia\\nSouth and Central America\\nNorth America\\nOther renewables consumption by region\\nb \\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 5\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nstate of charge, the lifetime and useful capacity could be improved. \\nStandardized battery cells that are designed to be integrated with the \\noriginal equipment  manu',\n",
       " 'facturer’s thermal management systems could \\nalso reduce cost19.\\nFuel-cell-based electrification \\nThe high efficiency of fuel-cell-powered electric vehicles makes this \\nform of electrification a potentially viable option for the future. Invest\\xad\\nment in this technology is driven by the potential of extended range \\nand faster fuelling times of moderately low-priced cars. Fuel-cell cost \\nhas been lowered and their lifetime increased20, but further gains are \\nneeded. Platinum-group catalyst loading ',\n",
       " 'has been reduced fivefold since \\n2005; however, further reductions are needed4,21, or these catalysts need \\nto be replaced with less costly alternatives. Costs can also be lowered \\nand performance improved through robust higher-conductivity and \\nhigher-temperature membranes, improvements in balance-of-plant \\ncomponents, such as humidifiers and compressors, as well as thermo\\xad\\nfluid design and control. \\nThere are inherent volumetric energy density issues for hydrogen-gas \\nstorage. To achieve a ran',\n",
       " 'ge of 480 km, fuel-cell electric vehicles need \\nto store about 4–7 kg of hydrogen4,22. A carbon-fibre-composite tank \\npressurized to 700 bar is the best current option for personal vehicles, \\nbut this costs23 about $3,000. Research is under way to develop materials \\nand manufacturing processes to reduce the cost of composite tanks. In \\nparallel, researchers are searching for lower-pressure storage assisted by \\nhigh-surface area materials that could physisorb or weakly chemisorb \\nhydrogen and sti',\n",
       " 'll maintain fast-fuelling times24,25. \\nThe supply infrastructure and a low-carbon source of hydrogen are \\nalso a challenge. The technology advances in shale-gas production, and \\nthe possibility of large reserves in Europe and Asia, in addition to the \\nconsiderable reserves in North America, could have a significant affect \\non the transportation sector. In addition to the direct use of natural gas \\nas a fuel (see later), low-cost natural gas could spur the deployment of \\nlocal reforming or hydrog',\n",
       " 'en filling stations for near-term hydrogen \\nproduction. Alternatively, commercial reforming plants, such as hybrid \\npower plants that produce hydrogen as well as CO2, for enhanced oil \\nrecovery located near oil-field and refinery sites can serve as an eco\\xad\\nnomical source of hydrogen4,26. In regions that are close to a large com\\xad\\nmercial production plant, delivery to local filling stations can be made \\nthrough high-pressure gaseous tube trailers, but in the long run, a cost-\\ncompetitive method to',\n",
       " ' produce hydrogen with considerably lower net \\ncarbon emissions is needed. \\nNatural gas for transportation\\nThe projected low cost of natural gas in the United States in the next \\nfew decades compared with that of petrol is expected to lead to wider \\nadoption of natural-gas vehicles. Displacing diesel fuel with liquefied \\nnatural gas (LNG) for class 8 tractor-trailer trucks commonly used on \\nlong routes in the United States is already economically viable because \\na typical long-haul truck uses ab',\n",
       " 'out 90,000 litres of fuel per year (about \\n$80,000 per year in fuel costs, today). The incremental purchase price \\nof LNG trucks can be up to $100,000 per truck for cryogenic tanks \\nand related upgrades, as a result of low-volume market conditions. \\nEven so, the payback period is currently 3–4 years on a net-present-\\nvalue basis using a 7% discount rate, and would drop considerably with \\neven modest increases in production volumes27. Developing a fuelling \\ninfrastructure for LNG long-haul trucks',\n",
       " ' would require fuelling stations \\nabout every 240–320\\u2009km for a truck range of 800–960\\u2009km. Commer\\xad\\ncial viability is reinforced by plans in the private sector to make the \\nrequired investments in infrastructure. LNG-powered freight trains \\nare also being considered.\\nCompressed natural gas (CNG) has been used for buses, delivery \\ntrucks and light-duty vehicles. To make vehicles using CNG economi\\xad\\ncally viable without subsidies, low-cost CNG storage technologies are \\nneeded. In the United States, l',\n",
       " 'ight-duty vehicles account for 75% of \\non-road fuel consumption. There are roughly 160,000 gasoline (petrol) \\nservice stations in the United States28; creating a similar nationwide \\ninfrastructure for CNG vehicles would be prohibitively expensive (more \\nthan $100 billion). However, about 60 million homes in the United \\nStates have natural-gas delivery. Economic viability for CNG cars and \\nrefuelling systems can be achieved if the payback period for the addi\\xad\\ntional system-level cost is typically',\n",
       " ' 5 years or less; at present it is about \\n10–15 years for a vehicle with average mileage. For vehicles with high \\nannual mileage and for vehicles with low kilometres per litre, natural-gas \\nvehicles can have less than a 5-year payback, even today. Research into \\nfibre-matrix composites for high-pressure light-weight tank materials \\nis needed, as well as into natural-gas sorbents for low-pressure storage. \\nAlthough seldom discussed, multifuel ICEs can be designed to operate \\non CNG for 30–60\\u2009km (',\n",
       " 'the CNG–ICE hybrid equivalent of a PHEV) \\nand then switch to petrol. Similar to a PHEV, a CNG–petrol–ICE vehicle \\ncould compensate for the partial coverage of CNG fuelling stations.\\nNatural gas can also be converted into liquid fuels using either the \\nFischer–Tropsch or the methanol process. The capital cost per barrel \\nof liquid fuel reduces with increasing capacity of a Fischer–Tropsch \\nplant according to scaling laws. However, with increasing capacity, \\ncapital costs are suggested to deviate ',\n",
       " 'and are higher than scaling law \\npredictions, which increases the financial risk for gas-to-liquid plants. \\nResearch is needed to find alternative approaches for exciting the car\\xad\\nbon–hydrogen bond and synthesizing carbon–carbon bonds. Biologi\\xad\\ncal approaches that use organisms, such as methanotrophs, that can \\nmetabolize natural gas and produce long-chain hydrocarbons seem \\nworth exploring. Even if this approach is successful at laboratory scales, \\nit will need to be scalable to large-volume pr',\n",
       " 'oduction. Large quanti\\xad\\nties of methanol are already produced from natural gas for industrial \\nFuel\\nenergy\\n100%\\nExhaust\\n33%\\nCooling\\n29%\\nMechanical\\npower\\n38%\\nFriction\\nlosses\\n33%\\nAir drag 5%\\nAir drag 5%\\nEngine\\n11.5%\\nTransmission 5%\\nBrakes 5%\\nRolling\\nresist 11.5%\\nEnergy \\nused to \\nmove the \\ncar 21.5% \\nThermodynamic\\nlosses\\nTotal\\nenergy\\nlosses\\nFigure 2 | Vehicle energy losses.\\u2002 Of the \\nenergy that fuel provides to vehicles a \\nsubstantial proportion is lost. A breakdown \\nof the average losses of intern',\n",
       " 'al-combustion-\\nengine cars (fleet make up 70% petrol and 30% \\ndiesel) is shown. Heat lost constitutes 30–37% \\nof the energy as a result of exhaust gases with \\nlower energy content and convection. The \\nother losses come from heat dissipation (25–\\n33%), mechanical losses (33–40%), air drag \\n(3–12%), rolling friction (12–45%) and brake \\nlosses (about 5%). These losses mean only \\nabout 21.5% of the energy is used to move the \\ncar. Adapted with permission from ref. 12.\\n2 9 6  |  N A T U R E  |  V O L',\n",
       " '  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\npurposes at costs that are roughly equivalent to petrol. Methanol could \\nbe used in a petrol–alcohol blend, much like ethanol in the United \\nStates. However, pure or high-percentage methanol-based transporta\\xad\\ntion could face distribution-infrastructure challenges.\\nAlternative liquid-transportation fuels\\nLiquid fuels derived from oil became the main form of energy for \\ntransportatio',\n",
       " 'n largely because of their high energy densities. Associ\\xad\\nated with their high energy content, the energy transfer rate during \\nvehicle refuelling is about 6 MW; in contrast, electrical charging will \\nbe tens of kilowatts. Apart from inherent limitations in battery chemis\\xad\\ntries, there are practical limits to the size of the electrical connector that \\ncould accommodate megawatt-scale power transfer. However, a search \\nfor alternatives to oil for transportation energy is required to deal with \\nth',\n",
       " 'e growing concerns over the rising and volatile price of oil, the vul\\xad\\nnerability to supply disruptions, and balance-of-trade issues. Biofuels, \\nparticularly those produced domestically at competitive prices, would \\nstrengthen a nation’s energy and economic security. \\nThere are a number of approaches to alternative transportation fuels \\nbeing actively explored (Fig.\\xa03). The estimated future (10–15 year) costs \\nfor classes of alternative fuels can vary29 (Fig.\\xa04). The US Environmental \\nProtection',\n",
       " ' Agency (EPA) estimates that Brazilian sugarcane ethanol \\n— already price competitive with oil-based fuels in Brazil — reduces \\ntotal life-cycle greenhouse gas emissions, including direct and indi\\xad\\nrect land-use change emissions, by 61% (ref.\\xa030). There are numerous, \\nand sometimes contentious, studies of carbon life-cycle emissions30–32. \\nMinimization of indirect land-use concerns and the sequestration of \\nprocess CO2 could result in net-negative carbon emissions, so that fuel \\nproduction and u',\n",
       " 'se becomes a net carbon sink. On the other hand, if \\nbiofuels based on energy-intensive crops were coupled with poor-land \\nmanagement this could result in environmental costs that are far higher \\nthan those associated with oil-based fuels29.\\nThe production of ethanol by domesticated yeast for fermentation \\nis perhaps 4,000 years old. In this Insight, Paralta-Yahya et al.33  review \\nthe application of metabolic engineering and synthetic biology to alter \\nmicrobes for the production of advanced bi',\n",
       " 'ofuels, and of precursors to \\ndrop-in substitutes for petrol, diesel and jet fuels. Static adjustment of \\ntranscription, translation and post-translational modifications does not \\nallow engineered organisms to respond to changing bioreactor condi\\xad\\ntions and cellular changes. Remarkably, dynamic sensing and regulation \\nof fatty-acid ethyl ester (FAEE) intermediates in Escherichia coli has \\nbeen shown to increase FAEE yield by threefold, to reduce the cellular \\nconcentration of toxic intermediates',\n",
       " ' and to significantly improve the \\ngenetic stability of producing strains.\\nSuccessful commercial-scale deployment of this class of technologies \\nwill depend on microbial productivity and robustness. The cost of \\nmicrobial feedstock is also a major factor: production of advanced fuels \\nfrom simple sugars and starches is further along in the development of \\nthese fuels, but it is held back by high feedstock prices. Lignocelluloses, \\nincluding agricultural and wood-waste streams, are roughly an ord',\n",
       " 'er of \\nmagnitude less costly, and are generally viewed as the end-goal feedstock. \\nMuch attention is now focussed on reducing the cost of converting ligno\\xad\\ncelluloses into forms that are more readily used by microbial organisms.\\nThe net energy yield per hectare per year is a major factor in \\ndetermining feedstock cost. The overall yield for biofuels varies widely: \\nBrazilian sugarcane can produce as much as 800 gallons per acre-year, \\nwhich is about twice as high as productivities based on North',\n",
       " ' American \\nmaize. High-yield grasses or fast-growing trees that could be feedstocks \\nfor advanced biofuels have the potential to surpass sugarcane produc\\xad\\ntivities by 1.3 times or more34. Most lignocellulosic feedstocks have low-\\nvolume energy density: relative to fossil fuels they are ‘light and fluffy’. To \\nimprove the economies of scale of biorefineries, especially \\nthermochemical refineries, increasing the feedstock col\\xad\\nlection radius is desirable. Biofuels could become more \\ncompetitive if',\n",
       " ' a means of efficiently concentrating the bio\\xad\\nmass during the harvesting process (such as combining the \\nreaping and pelleting the woody materials) could be devel\\xad\\noped. The same densification of the feedstock could also \\nlead to the biofuel equivalent of grain elevators. Reducing \\n‘first touch’ costs are an integral part of efficient agricul\\xad\\ntural practice, and progress in this area would significantly \\nlower the overall feedstock cost delivered to biorefineries. \\nIn this Insight, Georgianna ',\n",
       " 'and Mayfield35 discuss the \\nuse of algae to produce next generation biofuels. Algae \\ncan provide high fuel yields, especially if areal sunlight \\ncollection can be used to drive volumetric growth of \\nalgae. However, algal growth in closed systems requires \\nvery high capital investments relative to energy crops \\nsuch as grasses or trees. In open-pond systems, water use \\nis a major issue but may be partially ameliorated through \\nthe use of species that can grow in brackish or saltwater.\\nOther appro',\n",
       " 'aches to bio-based fuel production include \\nPlatforms/pathways \\nFeedstock \\nproduction \\nand logistics \\n• Energy crops \\n• Agricultural \\nby-products \\n• Waste \\nstreams \\n• Algae \\n• Coal  \\n• Natural gas \\n• Ethanol \\n• Methanol \\nReﬁning\\n• Butanol \\n• Alkenes\\n• Aromatics \\n• Petrol\\n• Diesel \\n• Jet fuel \\n• Dimethyl \\nether  \\n• Heat and \\npower \\nBy-products  \\nPower\\n \\nPyrolysis oil platform\\nSyngas platform \\nLiquid\\nbio-oil  \\nEnzymatic\\nhydrolysis\\nSugars\\nFermentation\\nCellulosic sugar platform \\nProducts \\n \\nFast pyr',\n",
       " 'olysis  \\nGasiﬁcation \\nRaw\\nsyngas  \\nFiltration and\\nclean-up  \\nUpgrading \\n  \\nAlgal and other \\nbio-oils \\nTransesteriﬁcation\\nCatalytic upgrading \\nLipid (oil) platform \\nOther enzymatic or biochemical methods \\nFigure 3 | Methods of producing alternative fuels from various feedstocks to \\nproducts.\\u2002 Various feedstocks are being explored, and the pathways for producing energy or fuel \\ninvestigated. Adapted with permission from ref. 8.\\nCO2 cost (ignoring indirect CO2 consequences)\\nCarbon storage cost\\n280\\n',\n",
       " '240\\n200\\n160\\nCost US$ per barrel petrol-equivalent product\\nCFT\\nCFT-CCS\\nCMTG\\nCMTG-CCS\\nCorn ethanol\\nCellulosic\\nethanol\\nBTL\\nBTL-CCS\\nCBFT\\nCBFT-CCS\\nCBMTG\\nCBMTG-CCS\\nCrude oil\\n$60, bbl\\nCrude oil\\n$100, bbl\\n120\\n80\\n40\\n0\\n–40\\nAdditional transportation\\nNon-feedstock operating cost\\nCapital cost\\nFeedstock cost\\nTotal cost (without carbon price)\\nFigure 4 | Alternative fuel costs.\\u2002 Estimated costs, achievable within 10–15 \\nyears, of alternative liquid fuels produced from coal, biomass, or coal and \\nbiomass with a ',\n",
       " 'CO2 price of $50 per tonne and capital costs are 20% lower \\nthan the America’s Energy Future panel’s estimates. BTL, biomass-to-liquid \\nfuel; CBFT, coal-and-biomass-to-liquid fuel, Fischer–Tropsch; CBMTG, coal-\\nand-biomass-to-liquid fuel, methanol-to-gasoline; CCS, carbon capture and \\nstorage; CFT, coal-to-liquid fuel, Fischer–Tropsch; CMTG, coal-to-liquid fuel, \\nmethanol-to-gasoline. Adapted with permission from ref. 3.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 7\\nPERSPE',\n",
       " 'CTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nthe manipulation of photosynthetic bacteria to produce biofuels, diverting \\nhigh-energy Calvin-cycle intermediates upstream of glucose — for exam\\xad\\nple energy-dense terpene (a biofuel precursor) production in trees or \\nmicrobes — or developing alternatives to C3 and C4 carbon fixation. \\nBiological enzymes can synthesize carbon–carbon bonds with an \\nunparalleled high specificity, but photosynthesis may not be the only \\napproac',\n",
       " 'h to converting sunlight into hydrocarbon fuels36. Several non-\\nphotosynthetic alternatives are under investigation that could poten\\xad\\ntially overcome many of the limitations of photosynthesis. To appreciate \\nthe potential of such approaches, the energy conversion process can be \\ndivided into three steps. First is to identify what reducing equivalents, \\nother than solar photons captured through photosystems I and II, can \\nbe accepted by various microorganisms. Several organisms are known \\nto be c',\n",
       " 'apable of growth on hydrogen sulphide, hydrogen, electrons, \\nammonia and reduced ions such as iron (II). Second is to investigate \\nopportunities to fix CO2 using pathways other than those used in C3 or \\nC4 plants. Potential systems may include the reverse tricarboxylic-acid \\ncycle (often called the reverse Krebs cycle), the Woods–Ljungdahl cycle \\nused by acetogens, the hydroxypropionate–hydroxybutyrate cycle or \\nnewly designed biochemical pathways. The final step is to determine \\nwhether we can ',\n",
       " 'metabolically engineer direct carbon products into a \\nmolecule such as acetyl-CoA, which is a precursor for many energy-\\ndense fuels. These three steps can be engineered into autotrophic organ\\xad\\nisms, an approach that is now being supported by a US DOE Advanced \\nResearch Projects Agency-Energy programme called Electrofuels37. \\nFinally, researchers are investigating highly efficient non-biological \\nenergy-conversion approaches that generate fuel from sunlight by the \\noxidation of water into hydrog',\n",
       " 'en and oxygen and reduction of CO2 to \\nfuel. The Joint Center for Artificial Photosynthesis, a US DOE funded \\nEnergy Innovation Research Center, was established to identify Earth-\\nabundant, robust light absorbers with optimal bandgaps to harvest sun\\xad\\nlight most effectively and efficiently, to accelerate the rate of catalyst \\ndiscovery for solar energy-to-fuel conversion reactions and to provide \\nsystem integration and scale-up so that laboratory experiments can \\nquickly transition into prototype',\n",
       " 's for commercial development.\\nClean and affordable electricity generation\\nThe IEA has projected, to 2035, how electricity is expected to be gen\\xad\\nerated38 (Fig. 5). In the agency’s new policies scenario, the growth \\nof carbon emissions is curtailed, but does not decline. The scenario \\nassumes that government policies such as carbon pricing are adopted \\nin several Organisation for Economic Co-operation and Development \\ncountries and in China, but the projections of cost reduction in renew\\xad\\nable en',\n",
       " 'ergy are overly conservative. As an example, the agency esti\\xad\\nmates the full levellized cost of electricity (LCOE) of onshore wind to \\nbe about $90\\u2009MWh-1 in 2010 real dollars by 2020 (ref.\\xa039). Similarly, the \\nUS Energy Information Administration estimates40 the LCOE of wind \\nfor 2016 will be about $80–120\\u2009MWh-1 (Fig.\\xa06). \\nIn contrast to these estimates, an analysis of the LCOE of various \\nforms of renewable energy, based on the available data from projects in \\nwhich investment or purchase contr',\n",
       " 'acts have been completed41, shows \\nthat wind at level (class) 4 sites in the United States is currently about \\n$73\\u2009MWh-1 (Fig.\\xa07). These costs do not include transmission line costs, \\nwhich average42 at about $300 kW-1. Some experts estimate that by 2020, \\nthe LCOE of wind energy at level 4 sites will fall below $60\\u2009MWh-1. \\nIn the United States, utility-scale solar photovoltaic projects are being \\ninstalled in 2011–2012 at an unsubsidized cost of about $150\\u2009MWh-1 \\n(the US investment tax credit h',\n",
       " 'as been taken out43,44), which is con\\xad\\nsistent with the Bloomberg New Energy Finance estimates41. Utility-\\nscale solar photovoltaics are projected to be between $60–120\\u2009MWh-1 \\nin good insolation areas. The lowest LCOE for combined-cycle natural \\ngas is about $50–60\\u2009MWh-1 in the United States, assuming gas costs of \\n$3–4\\xa0per million British thermal units. Although LCOE is an important \\nfactor, time of delivery, LCOE at various size scales and the potential for \\nenergy storage are also important. ',\n",
       " '\\nAffordable and environmentally sustainable electricity generation is \\nalready at cost parity in areas in which energy costs exceed $200\\u2009MWh-1. \\nOn the basis of the current and projected learning curves for wind and \\nsolar energy, renewable energy will become increasingly affordable on \\na global scale. In addition, energy storage with durable and inexpensive \\nbatteries will allow electricity accessibility in micro- or meso-scale grids \\nto leapfrog the need to bring centrally generated electricit',\n",
       " 'y to distant \\nrural areas. \\nEven though the costs of renewable energies such as wind and solar \\ncontinue to decline, the full economic benefit of these variable sources \\nof energy will not be realized until the more established forms of energy \\ngeneration are integrated with renewable sources into transmission and \\ndistribution, load response and storage of electricity. Eighty per cent \\n24\\n0\\n20\\n40\\nPercentage\\n60\\n80\\n100\\n20,043 TWh\\n27,881 TWh\\n36,250 TWh\\n2009\\n2020\\n2035\\n22\\n20\\n18\\n16\\n14\\n12\\n10\\n2010\\n2015',\n",
       " '\\nGT\\n2020\\n2025\\n2030\\n2035\\nMore eﬃcient plants\\nCoal\\nGas\\nOil\\nNuclear\\nBiomass\\nHydro\\nWind\\nOther renewables\\nNuclear\\nHydro\\nBiomass\\nWind\\nOther renewables\\nCarbon capture and storage\\n2009 fuel mix\\nNew policies scenario\\nFigure 5 | Projections from the \\nInternational Energy Agency for the \\ngeneration of electricty until 2035.\\u2002 In \\nthe top panel, the relative energy \\nmix in 2009 and that projected in the \\nInternational Energy Agency’s ‘new \\npolicies’ scenario. In the bottom panel, \\nin the new policies scenari',\n",
       " 'o, carbon \\nemissions continue to grow, but slower \\nthan the 2009 fuel-mix curve. The 2009 \\nfuel-mix curve is based on emissions \\nthat would have been generated for the \\nprojected level of electricity generation \\nwere there no change in the mix of fuels \\nand technologies, and no change in the \\nefficiency of thermal generation plants \\nafter 2009. The cost of renewable energy is \\ndeclining at a faster rate than assumed in \\nthe International Energy Agency’s report. \\nIn addition, the relative mix of ',\n",
       " 'natural gas \\nto coal may be larger than that projected. \\nAdapted with permission from ref. 38.\\n2 9 8  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\npenetration of renewable energy in the United States is possible by 2050, \\nbut to achieve even half this goal, a transformation of the electricity sys\\xad\\ntem on the basis of technology advances and new operating procedures, \\nbusiness models and regulatory appr',\n",
       " 'oaches is required45. \\nSolar\\nIn 2010, the US DOE created the SunShot initiative to reduce the LCOE \\nfrom solar sources (photovoltaic and thermal) to $50–60\\u2009MWh-1, with\\xad\\nout subsidies, by 2020. For solar photovoltaics at utility scale, the fully \\ninstalled cost target must be about $1 per Watt of electricity generated. \\nThe costs are broken down into $0.50\\u2009W-1 for the solar module, $0.10\\u2009W-1 \\nfor the  power electronics for grid integration and $0.40\\u2009W-1 for the bal\\xad\\nance of the system (including ',\n",
       " 'installation, permits and all other costs)46. \\nIn the four years between the second quarter of 2008 and the second \\nquarter of 2012, the actual spot market price of silicon solar modules \\nfor immediate delivery declined from $4\\u2009W-1 to about $1\\u2009W-1 (ref.\\xa047). \\nAlthough part of this drop was due to a temporary oversupply coupled \\nto the worldwide recession, continual engineering and manufacturing \\ninnovations contributed to this decline and are likely to continue doing \\nso in the coming years: fut',\n",
       " 'ure delivery contracts are about $0.80 W-1, \\nand there is a growing confidence that module costs will achieve the \\ngoal of $0.50\\u2009W-1 by 2020 (ref.\\xa048). In contrast, the balance-of-system \\ncosts, specifically the soft costs such as permits, interconnection and \\ninspection, financing and customer acquisition have declined but at a \\nsignificantly slower rate. Because of this trend, a focus of the US DOE \\nSunShot programme is lowering all balance-of-system costs commen\\xad\\nsurate with module prices49.\\n',\n",
       " 'All costs are affected by increasing the efficiency of the solar cells while \\nreducing their production cost. The Shockley–Queisser limit is a well-\\nknown efficiency limit of single-junction solar cells, which suffer from \\nspectrum losses, recombination and black-body radiation. The Shock\\xad\\nley–Queisser limit of silicon is 32.7%, but a practical limit is considered \\nto be 29% (ref.\\xa050). The ultimate efficiency limit in the conversion of sun\\xad\\nlight energy to electrical energy is determined by a Ca',\n",
       " 'rnot heat engine. \\nThis limit is about 94%, based on the high temperature of the sun’s \\nblack body at 5,800\\u2009K, and the cell temperature of about 350\\u2009K. There \\nis, therefore, significant room to surpass the Shockley–Queisser limit.\\nMost of the solar cells manufactured today are made of crystalline \\nsilicon, and have shown efficiencies as high as 23% at a production-\\nscale, although most cells are within the 17–19% range. In this Insight, \\nGraetzel\\xa0et al.51 review how solution-processed solar cell',\n",
       " 's could be com\\xad\\npetitive if the cost could be reduced and efficiency increased, but this is \\njust one approach, competing with other rapidly advancing techniques. \\nPolycrystalline-silicon photovoltaics typically have efficiencies in \\nthe range of 14–16%, although recent progress indicates that refine\\xad\\nments in bulk polysilicon production could yield efficiencies similar \\nto that of single-crystal silicon for half of the ingot material52. The cost \\nof photovoltaic-grade silicon forms half the cos',\n",
       " 't of a solar module. In \\nconventional silicon-cell production, about half the silicon is lost in saw\\xad\\ning the bulk silicon into 150–200-μm-thick wafers. With modest light-\\ntrapping techniques53, the wafers can be less than 50\\u2009μm thick. Other \\napproaches to lowering costs include single-crystal epitaxial growth \\nand lift-off, pursued by Solexel of Milpitas, California54, and kerf-less \\nmanufacturing of wafers with large and aligned grains in which the thin \\nwafers are directly made55,56.\\nThe othe',\n",
       " 'r competitive approaches are based on direct-gap \\nsemiconductors such as cadmium telluride and copper-indium-gal\\xad\\nlium-selenide (CIGS), in which only a thin film (less than 1\\u2009μm) is \\nneeded to absorb solar radiation. Thin-film solar cells can be deposited \\non light and low-cost substrates, reducing their overall cost and weight, \\nwhich is an advantage for transportation and installation. However, \\nthin-film deposition on heterosubstrates introduces defects in these \\nfilms, which currently limit ',\n",
       " 'production-scale efficiency of cadmium-\\ntelluride and CIGS thin-film solar cells to 12–14% — their practical \\nupper limit is 29%. If the defects in cadmium-telluride thin films could \\nbe reduced in production, this would increase the carrier lifetime and \\nincrease the efficiency57. This reduction requires an understanding of \\nthe origins of defects, as well as ways to reduce and control them. The \\nrole of interfaces is a crucial aspect of solar cells and requires significant \\ninvestment in resea',\n",
       " 'rch and development. \\nIn addition to scientific understanding, engineered online monitoring \\nand control systems may be needed to improve manufacturing quality \\ncontrol and as a means of continual improvement. Currently, panels made \\nof cadmium-telluride thin films have the lowest cost (about $0.70\\u2009W-1). \\nOwing to the financial discount that is applied to any new solar technol\\xad\\nogy that is expected to perform for at least 20–25 years, any photovoltaic \\ntechnology developed has to have the techno',\n",
       " 'logical ‘head-room’ to be \\ncompetitive with the projected progress of more established technologies. \\nAn increase in efficiency from 17% to 20% would significantly reduce the \\npanel cost and some of the balance-of-system costs, which could have a \\ntransformative effect on the market similar to that expected for polycrys\\xad\\ntalline silicon cells with an efficiency of over 18% (ref.\\xa052). \\nThin-film solar cells made of direct-gap III–V semiconductors are \\nattractive because of their higher carrier mo',\n",
       " 'bilities, and their superior \\nmaterial quality that can make use of the materials research that focused \\non optoelectronics. This allows III–V semiconductor cells to achieve \\na much higher efficiency that is close to the practical upper limit. For \\nexample, solar-cell efficiencies of about 29% have been shown for III–V \\ndevices, which approaches the Shockley–Queisser limit of 33.5% for any \\nsingle bandgap material58. Although high-quality films have generally \\nbeen achieved on crystalline substr',\n",
       " 'ates, use of bulk materials would be \\ncost-prohibitive for solar applications. The main challenge is to achieve \\nthe crystalline quality of III–V thin films, but on low-cost substrates \\nsuch as plastics or sheet metals. As for techniques in solar cells that \\nuse silicon epitaxial growth followed by lift-off, good progress is being \\nmade towards manufacturing, on a larger scale, lift-off of thin crystalline \\ngallium-arsenide layers grown epitaxially from bulk crystals. \\nImproving solar-module eff',\n",
       " 'iciency may also come from \\nmultijunction cells that capture a larger fraction of the solar spec\\xad\\ntrum, multi-exciton generation, multiphoton absorption or photon \\nup-conversion, and light concentration. The use of quantum dots and \\nmolecules to utilize some of these phenomena are being pursued, and, \\n0 \\n50 \\n100 \\n150 \\n200 \\n250 \\n300 \\n350 \\n400 \\nConventional coal\\nAdvanced coal with CCS\\nNG ACC\\nNG ACC with CCS\\nNG CCT\\nAdvanced nuclear\\nWind\\nWind-oﬀshore\\nSolar PV\\nSolar thermal\\nGeothermal\\nBiomass\\nHydro\\nL',\n",
       " 'evellized cost ($ MWhr–1) \\nFigure 6 | Projections by the US Energy Information Administration made \\nin 2010 of the 2016 average cost and range of the levellized cost of electricity \\nfrom various sources.\\u2002 Note the cost of onshore wind at level 4 sites where \\nthe wind speed is 7.0–7.5 ms-1 and there are no financial incentives. Wind, \\nutility-scale solar photovoltaics and solar thermal energy in 2012 in the United \\nStates (green) is already at or below the lower end of the 2016 estimates. ACC, \\na',\n",
       " 'dvanced combined cycle; CCS, carbon capture and storage; CCT, conventional \\ncombustion turbine; NG, natural gas; PV, photovoltaics.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  2 9 9\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nas described by Graetzel et al.51, are leading to a deeper understanding \\nof light–matter interactions. The competition in solar technologies is \\nfierce, and to have an effect on solar-electricity generation, the science ',\n",
       " '\\nmust be translated into engineered systems that can meet the techno\\xad\\neconomic target of about $0.50\\u2009W-1 for a solar panel at commercial scale. \\nAlthough the discussion so far has focused on solar flat panels, there are \\nopportunities to generate electricity at $50–60\\u2009MWh-1 using concentrated \\nsolar photovoltaics. Most of the cost is in the balance of systems includ\\xad\\ning the mirrors, lenses and their tracking system; however, the cost of the \\nsolar cell at the focal point of the optical system i',\n",
       " 's not as significant. This \\nis when multijunction solar cells with efficiencies that exceed 40% would \\nbe ideal. Innovations in low-cost and high-quality mirrors59, inexpensive \\nFresnel optics, as well as creative engineering to track them throughout \\nthe day at low cost are being developed.\\nComplementary to electricity generation using photovoltaics, solar \\nthermal approaches that concentrate sunlight in parabolic troughs and \\npower towers have also been used to convert absorbed solar radiation',\n",
       " ' \\ninto heat (typically below 450\\u2009°C). Electricity is then generated at about \\n30–35% efficiency using a heat engine46. Heat can also be stored to allow \\nbetter demand matching, especially the offset peak electricity demand, \\nby several hours in climates and economies in which air-conditioning \\ndemand is rising. \\nTo reduce the LCOE to $50–60\\u2009MWh-1, the efficiency of the heat \\nengines must be increased. This requires more than 600\\u2009°C, suggest\\xad\\ning that the higher-light concentration, as achieved i',\n",
       " 'n power towers or \\nparabolic dish mirrors, is needed. There is significant opportunity for \\nresearch to identify ways to store heat through the enthalpy of phase \\ntransitions or chemical reactions at more than 600\\u2009°C, as well as systems \\nengineering to efficiently store and access this enthalpy on demand to \\nrun the heat engine for power generation60. Furthermore, research into \\nlow-cost and high-temperature materials, and their use in engineered \\nsystems, is needed to design and operate solar t',\n",
       " 'hermal reactors at these \\nunprecedented high temperatures. \\nAlthough most of the work on concentrated solar power has involved \\neither photovoltaic or thermal approaches, there are opportunities to \\ncombine them. For example, the visible part can be spectrally filtered \\nout of the concentrated solar radiation for photovoltaics, and the rest \\nof the radiation used for solar thermal approaches. Furthermore, solar \\nthermal techniques rarely use all of the concentrated solar radiation, \\nbecause ther',\n",
       " 'e is generally some spillover from the solar absorber. Inte\\xad\\ngrating photovoltaics in a solar thermal absorber may be possible, allow\\xad\\ning both approaches to be integrated. Achieving more than 600\\u2009°C in a \\nsolar thermal absorber would also make it compatible with gas turbines \\nused in natural-gas combined cycle technology, which would allow inte\\xad\\ngration of solar and natural-gas power plants to address baseload and \\npeak demands.\\nTo convert direct current to alternating current electricity and t',\n",
       " 'o \\nintegrate solar power into the grid, power electronics is crucial and \\nmust be reduced to $0.10\\u2009W-1 or less — roughly half the cost it is today46. \\nWe are currently witnessing a revival of power-conversion electron\\xad\\nics in which the introduction of wide bandgap semiconductors such \\nas silicon-carbide and gallium-nitride, low-loss soft magnetics, solid-\\nstate capacitors and new-circuit topologies is having multiple positive \\neffects61. These allow a significant increase in the operating freque',\n",
       " 'ncy, \\nwhich reduces the sizes of inductors and capacitors without altering \\ntheir electrical impedance. The size reduction can allow integration of \\nvarious components onto the same platform, reducing cost and weight. \\nThe use of solid-state capacitors as opposed to electrolytic ones can sig\\xad\\nnificantly increase reliability, which reduces the LCOE of solar energy. \\nThe need to address the variable nature of intermittent technolo\\xad\\ngies such as solar will become crucial at high levels of penetrati',\n",
       " 'on. \\nApproaches to forecast the time-dependent solar insolation are needed to \\nplan electricity dispatch. In parallel, storage technologies that will offset \\nthe variations in solar electricity output need to be developed (see later). \\nWind\\nThe cost decline of wind energy is a result of significant improvements \\nto turbines, blades and gearboxes, and an increase in the height of \\nthe wind towers. Although the largest wind turbine has a capacity of \\n7.5\\u2009MW, most turbines being installed are 1.5–2',\n",
       " '\\u2009MW (ref.\\xa062). \\nAs wind turbines increase in height and size, there is a marked \\nincrease in the mechanical stresses on the gearbox, blades and tower. \\nDirect-drive wind turbines that can convert the slow rotary motion \\ndirectly into alternating current electrical power using electromagnetic \\ngenerators are receiving increased attention. However, these generators \\ncurrently use permanent magnets based on rare-earth materials such as \\nneodymium and dysprosium for their high remnant fields, field ',\n",
       " 'anisot\\xad\\nropy and high-temperature stability. Although rare-earth deposits are \\nfound in many countries around the world, more than 90% of current \\nproduction is in China63,64.\\nResearch in magnetic materials that can produce large magnetic \\nenergy density without the need for rare-earth metals is needed. \\nFor example, there are certain phases of iron-born compounds (for \\nexample, Fe16N2) that produce extremely high magnetocrystalline \\nanisotropy, but that are difficult to access and stabilize und',\n",
       " 'er ambi\\xad\\nent conditions65,66. Today’s computing power could be used to predict \\nmagnetic behaviour in a combinatorial search that could guide the \\ndiscovery of magnetic materials with high crystalline anisotropy and \\nthose with accessible stable phases. Often, accessing these metastable \\nphases requires nanostructuring, as well as manufacturing composites \\nwith materials such as soft magnets67. \\nOffshore wind farms are an attractive source of energy because \\nthey can be placed in near-proximity ',\n",
       " 'to population centres and yet far \\nenough away to mitigate local opposition. However, the cost remains \\nhigh because of the need to build stable marine structures that can \\nwithstand the harsh marine environment for many years without the \\n0\\n100\\n200\\n300\\n400\\n500\\nMarine: wave\\nMarine: tidal\\nSTEG: LFR\\nSTEG: tower and heliostat\\nSTEG: parabolic trough + storage\\nWind: oﬀshore\\nSTEG: parabolic trough\\nSTEG: tower and heliostat storage\\nPV: thin ﬁlm\\nPV: c-Si\\nPV: c-Si tracking\\nBiomass: anaerobic digestion\\nBi',\n",
       " 'omass: gasiﬁcation\\nGeothermal: binary plant\\nBiomass: incineration\\nWind: onshore\\nLandﬁll gas\\nMunicipal solid waste\\nGeothermal: ﬂash plant\\nSmall hydro\\nLarge hydro\\nNuclear\\nCoal-ﬁred\\nNatural gas CCGT\\n789 +9%\\n1,049 +9%\\n+11%\\n–2%\\n+5%\\n–4%\\n–5%\\n–18%\\n–9%\\n+1%\\n–8%\\n–1%\\n–14%\\n–\\n–1%\\n+2%\\n–1%\\n–\\n–\\n–\\n–\\n+1%\\n+1%\\n+2%\\nFigure 7 | Levellized cost of \\nenergy for quarter three, \\n2012.\\u2002 The analysis of cost in \\ndollar per megawatt-hour is \\nbased on available data from \\nprojects in which investment \\nor purchase contracts have',\n",
       " ' \\nbeen completed. To produce \\ncomparisons of levellized cost \\nof energy at a global level, \\nthe analysis assumed a single \\ncorporate tax rate of 35%, an \\nannual inflation of 2% and the \\nequity investor internal rate of \\nreturn of 10%. Quarter three \\ncentral scenarios, filled triangles; \\nquarter two central scenarios, \\nopen triangles. Adapted with \\npermisssion from ref. 41.\\n3 0 0  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited',\n",
       " '. All rights reserved\\n\\nneed for replacement. Furthermore, because heavy-duty maintenance \\ncurrently needs calm seas, offshore wind turbines have to be highly reli\\xad\\nable. One approach to servicing the highest wind turbines is to design \\ncranes that can ‘inch worm’ themselves up the towers, analogous to the \\nself-lifting cranes used to build tall buildings.\\nDecarbonizing fossil-fuel emissions\\nIf the world is to continue to generate electricity from fossil fuels, the \\ncarbon emissions from the majo',\n",
       " 'r stationary ‘point’ sources will have \\nto be significantly reduced in the coming decades. Achieving higher \\nprimary energy use through cost-effective improvements in thermo\\xad\\ndynamic efficiency, co-generation of power and heat, and use of waste \\nheat can reduce carbon emissions. However, maximizing efficiency is \\nnot sufficient. Carbon capture and storage (CCS) from coal and natural-\\ngas power plants and other major emitters, such as cement and steel \\nplants, will be necessary. \\nA detailed analy',\n",
       " 'sis of post-combustion carbon capture science, \\nengineering and economics can be found in a recent American Physical \\nSociety (APS) report68. The cost of CO2 capture can be largely divided \\ninto two components: the capital cost, mostly of a sorbent-based CO2 \\nseparation unit, the sorbent regenerator and the CO2 compressor; and \\nthe operating cost, mainly a result of the parasitic energy needed to \\ncapture and compress CO2. The capital cost depends on the size of the \\nseparation unit, which is in',\n",
       " 'versely correlated with the rate constant of \\nthe CO2 binding reaction with a sorbent. The energy cost, however, \\ndepends on the enthalpy of the sorbent-CO2 dissociation reaction. The \\nkinetics and the thermodynamics of the binding and unbinding reac\\xad\\ntions of today’s sorbents are correlated (Fig.\\xa08).\\nResearch is needed to identify sorbents that have rate constants of more \\nthan 10,000\\u2009M-1\\u2009s-1, but of enthalpies 40–80\\u2009kJ\\u2009mol-1. There are various sec\\xad\\nondary and tertiary amines, metal-organic fra',\n",
       " 'meworks and ionic liquids \\nwhose reactions could perhaps be tailored to approach desirable kinetic \\nand thermodynamic performance levels68 (Fig.\\xa08). In biological organ\\xad\\nisms, enzymes such as carbonic anhydrase greatly reduce the free-energy \\nbarrier in the capture and release of CO2. Researchers are making progress \\nin genetically altering the properties of carbonic anhydrase69 so that it can \\nbe used to improve the kinetics of low-enthalpy sorbents under the harsh \\nconditions of the exhaust of',\n",
       " ' a coal-fired power plant. \\nThe cost of retrofitting existing pulverized coal plants to achieve 90% \\nCO2 capture with the technology available is estimated to require capital \\nexpenditures that approach those of the original plant3. In addition, \\n20–40% of the plant’s energy would be diverted for separation, compres\\xad\\nsion and transmission of the CO2, so only the most thermodynamically \\nefficient supercritical and ultra-supercritical plants could be considered \\nfor retrofits.\\nCO2 capture can be a',\n",
       " 'chieved relatively easily if a highly concentrated \\nstream of CO2 exhaust is produced, which is possible by oxycombus\\xad\\ntion (in pulverized coal or natural-gas power plants) or integrated \\ngasification combined-cycle (IGCC) technology. Cryogenic separa\\xad\\ntion of oxygen from air is the most common industrial-scale method \\nof production. \\nThere is no clear choice between oxycombustion and post-\\ncombustion capture3, but it is clear that innovative chemical and physi\\xad\\ncal separation methods need to be',\n",
       " ' developed, piloted and demonstrated \\nat near-commercial scale before wide-scale deployment can occur. \\nOther economical solutions to CCS are needed, but without the pros\\xad\\npect of a meaningful price on CO2, the private sector is less motivated to \\ninvest in technology development. For this reason, research and devel\\xad\\nopment of CCUS (carbon capture, utilization and storage of carbon) \\nin enhanced oil recovery and possibly enhanced geothermal energy70 \\nprovides a financial incentive to invest in c',\n",
       " 'arbon-capture technologies. \\nAdvancing CCS and CCUS could come from high-performance \\ncomputing for in silico optimization of system performance, rather \\nthan expensive pre-commercial projects. High-fidelity simulations \\ncan also be used to reduce the risk of scaling small demonstration \\nprojects to larger systems, and thereby accelerate these projects \\nadoption by industry. \\nNuclear\\nNuclear power can have an important role in efforts to decarbonize \\nthe production of electricity. Worldwide, nuc',\n",
       " 'lear energy constituted \\nabout 14% of the total electrical power generated in 2009 (ref.\\xa071), but \\nin 2011 this percentage dropped to about 12%, with a loss of 180\\u2009TWh \\nof generation from Germany and Japan5,72. The reaction to the 2011 \\nFukushima nuclear disaster has been varied: some countries that have \\nnuclear reactors are proceeding with caution, others continue to show \\ninterest in starting a nuclear-power programme, some countries have \\nput their plans on hold and some have opted to revers',\n",
       " 'e their decision \\non nuclear energy or accelerate planned phase-outs. \\nAn increasing number of older nuclear plants will need to have their \\noperating licenses extended beyond their original time limit if they are \\nto continue operating. The cost of operating fully amortized plants in the \\nUnited States is estimated to be about $23\\u2009MWh-1, including the contri\\xad\\nbution of $1\\u2009MWh-1 to the nuclear-waste fund for the cost of spent fuel \\ndisposal73. The estimated cost of decommissioning each plant, in',\n",
       " 'cluding \\nused fuel and site restoration costs is about $500 million per plant. The \\nlow cost of operating the current fleet of reactors has to be balanced with \\nsafety concerns, disposition of spent fuel and risks of nuclear weapons \\nproliferation. Unfortunately, space does not allow a full discussion of \\nthese important concerns. Although there are significant challenges, the \\ndevelopment of carbon-free sources of baseload power (nuclear and fossil \\nplants with CCS) are needed to mitigate the w',\n",
       " 'orst risks of climate change. \\nThird-generation nuclear-power plants are engineered to be signifi\\xad\\ncantly safer than previous generation ones. In the event of an emergency \\nshutdown of a nuclear reactor, the first few days — when the heat from \\ndaughter nuclear decay is largest and has to be dissipated — are the \\nmost critical. New reactors being constructed are designed to safely \\ncool down for three days without access to grid or emergency electrical \\npower or to water outside the reactor faci',\n",
       " 'lity. The ultimate goal is to \\nachieve passive safety, so there is very little chance of thermal damage to \\nthe reactor without requiring any active control after sudden shutdown.\\nThe LCOE of new nuclear plants is about $100–120\\u2009MWh-1; although \\nthis cost is not competitive with natural gas, it is less than estimates \\nfor fossil-fuel plants with CCS4. Another challenge is that the size of \\na single reactor is 1.0–1.5 GW with associated cooling and electri\\xad\\ncal distribution requirements. The cost',\n",
       " ' of nuclear-reactor engineer\\xad\\ning, procurement and construction, operation and maintenance, and \\ndecommissioning is about $6,000–6,600 per installed kilowatt, which \\nis roughly five times higher than comparable natural-gas generation74 \\ncapital costs, and is quantized in increments of about $6 billion to \\n$10\\xa0billion. The large financial exposure and the risk of construction \\nand licensing delays have driven up the cost of financing plants75. \\nThe US DOE intends to support the engineering design',\n",
       " 's required \\nby the Nuclear Regulatory Commission for licensing small-modular \\nIdeal absorbents\\nor adsorbents\\n0\\n20\\n20,000\\nRate constant, kf (M–1s–1) \\nReaction enthalpy, ΔH (kJ Mol–1) \\n15,000\\n10,000\\n5,000\\n40\\n60\\n80\\n100\\n120\\nCurrent absorbents or adsorbents\\nFigure 8 | Rate constant of current absorbents or adsorbents is about \\nlinearly proportional to the reaction enthalpy.\\u2002 New sorbents are needed to \\nenhance the rate constant but maintain the energy required to release CO2, to \\ndrive down the capit',\n",
       " 'al and operating costs of carbon capture. Adapted with \\npermission from ref. 68.\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  3 0 1\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nreactors between 80\\u2009MW and 300\\u2009MW (refs.\\xa076, 77). It is possible that \\nsafe nuclear power can be made more accessible through the economy \\nof constructing dozens of reactors in a factory rather than one at a time \\nat each site. Also, with the risk of licensing and constru',\n",
       " 'ction delays \\nreduced, small-modular reactors may represent a new paradigm in \\nnuclear construction. The US DOE has also established an Energy \\nInnovation Hub to develop multiphysics computational simulation \\ntools to reduce the time needed to design and certify many aspects of \\nboth conventional reactors and small-modular reactors78.\\nGrid energy storage\\nThe reliability and efficiency of the transmission and distribution \\nsystem is improved by grid energy storage. Balancing and optimizing \\npower',\n",
       " ' flow and generation are challenges that need improved tech\\xad\\nnology, and business and regulatory vision. Significant improvements \\nwould help to realize more efficient energy markets, a lower cost of \\nenergy, increased reliability and security, and penetration of renewable \\nenergy beyond 20%. \\nThe mismatch between an intermittent electricity supply and demand \\noccurs over multiple time and energy scales. For seconds-to-minutes \\nscales, voltage and frequency stabilization is currently achieved by',\n",
       " ' grid-\\nancillary services that mostly use large banks of capacitors and hot-\\nspinning reserves from fossil-fuel plants, but also banks of batteries and \\nmechanical fly wheels. Hot-spinning reserves are also used to shore-up \\ngeneration from minutes to hours. Modern natural-gas carbon-cycle \\nplants are designed to ramp up rapidly at about 50\\u2009MW min-1, which, \\nwhen combined with improved sensing and control of the grid, will \\nbe sufficient to accommodate most of the intermittency problems of \\nrene',\n",
       " 'wable energy. For hours-to-overnight scales, power and energy \\ncapacities of hundreds to thousands of megawatt and megawatt hour \\nwill be needed, alternatively many smaller (hundreds to thousands of \\nkilowatts) of distributed energy storage sites can be used.\\nThe most inexpensive method to store electricity for the medium- \\nand long-term is to use pumped hydroelectric and compressed air in \\nair-tight caverns, for which the capital cost is about $100\\u2009kWh-1. How\\xad\\never, geographical constraints war',\n",
       " 'rant alternative approaches. New \\nelectrochemical and modular and low-cost fabricated compressed-air \\nsystems are promising79.\\nPerspective\\nGiven the anticipated technological advances in finding and extracting \\nfossil fuels, the cost of these forms of energy could remain competitive \\nwith carbon-free sources for decades. In addition, the mix of energy \\nsources in the United States (Fig.\\xa09) indicates that the timescale of change \\nwill be roughly half a century80. If we are to mitigate climate ris',\n",
       " 'ks in a \\ntimely manner, the speed of the adoption of clean-energy technologies \\nat a widely-deployed commercial scale has to be accelerated. Although \\nthe deployment of any innovation will depend on its benefits relative to \\nestablished technologies, inertia as well as real and perceived financial \\nrisks always favour the status quo. To accelerate the use of economically \\nviable clean-energy solutions, government policies are needed to stimu\\xad\\nlate invention, innovation, and align the market forc',\n",
       " 'es. \\nWith increases over the past 30 years in the number of extreme \\nweather events, such as temperature extremes, floods, wildfires, \\ndroughts and storms, the overall loss trend is beginning to exceed $150 \\nbillion per year81. There is increasing evidence that these changes are \\nlinked with climate change82. Although the full costs of mitigating or \\nadapting to these changes have significant uncertainties, prudent risk \\nmanagement does not equate uncertainty with inaction. Policies are \\nneeded ',\n",
       " 'that begin to include the full cost of the various sources of energy \\nmore directly into their market prices82. \\nThis Perspective highlights a few of the challenges and opportunities \\nfor science, engineering and policy. Our ability to find and extract fossil \\nfuels continues to improve, and economically recoverable reservoirs \\naround the world are likely to keep pace with the rising demand for \\ndecades. The Stone Age did not end because we ran out of stones; we \\ntransitioned to better solutions',\n",
       " '. The same opportunity lies before us \\nwith energy efficiency and clean energy.\\nThe cost of renewable energy is rapidly becoming competitive with \\nother sources of energy. With additional engagement of the scientific, \\nfinancial and public-policy communities, as well as the general public, \\nwe can accelerate the transition to affordable, accessible and sustainable \\nenergy that will power economic growth, increase energy security and \\nmitigate the risks of climate change. On the other hand, to qu',\n",
       " 'ote from \\nthe IEA’s World Energy Outlook, “If we don’t change direction soon, \\nwe’ll end up where we’re heading”. ■\\n1.\\t\\nLee, R. The outlook for population growth. Science 333, 569–573 (2011). \\n2.\\t\\nInternational Energy Agency in World Energy Outlook 2011, 546–547 \\nhttp://www.worldenergyoutlook.org (International Energy Agency, 2011). \\n3.\\t\\nCommittee on America’s Energy Future, National Academy of Sciences, National \\nAcademy of Engineering and National Research Council. America’s Energy \\nFuture: Te',\n",
       " 'chnology and Transformation (National Academies Press, 2009).\\n4.\\t\\nUS Department of Energy. Report on the First Quadrennial Technology Review. \\nhttp://energy.gov/sites/prod/files/QTR_report.pdf (US DOE, 2011).\\n5.\\t\\nBP. BP Statistical Review of World Energy. http://bp.com/statisticalreview2012. \\n(BP, 2012).\\n6.\\t\\nMaugeri, L. Oil: The Next Revolution. Discussion Paper 2012–10. \\nhttp://belfercenter.ksg.harvard.edu/files/Oil-%20The%20Next%20\\nRevolution.pdf (Belfer Center for Science and International Af',\n",
       " 'fairs, Harvard \\nKennedy School, 2012).\\n7.\\t\\nInderwildi, O. & King, D. (eds). Energy, Transport, & the Environment, Addressing \\nthe Sustainable Mobility Paradigm. (Springer, 2012).\\n8.\\t\\nUS Department of Energy. Quadrennial Technology Review report, vol. II Technology \\nAssessments http://energy.gov/articles/department-energy-releases-inaugural-\\nquadrennial-technology-review-report (US Department of Energy, 2011).\\n9.\\t\\nGibbs, J., Pesaran, A. A., Sklad, P. S. & Marlino, L. D. in Fundamentals of Materia',\n",
       " 'ls \\nfor Energy and Environmental Sustainability (eds Ginley, D. S. & Cahen D.) Ch. 31, \\n426–444 (Cambridge Univ. Press, 2012).\\n10.\\t Powers, W. F., Automotive materials in the 21st century. Adv. Mater. Process. \\n157, 38–41 (2000).\\n11.\\t Cuddy, M. R. & Wipke, K. B. Analysis of Fuel Economy Benefit of Drivetrain \\nHybridization. http://www.nrel.gov/vehiclesandfuels/vsa/pdfs/22309.pdf \\n(National Renewable Energy Laboratory, 1997).\\n12.\\t Holmberg, K., Anderssona, P. & Erdemirb, A. Global energy consumpt',\n",
       " 'ion due to \\nfriction in passenger cars. Tribol. Int. 47, 221–234 (2012). \\n13.\\t Schafer, A., Heywood, J. B. & Weiss, M. A. Future fuel cell and internal \\ncombustion engine automobile technologies: a 25-year life cycle and fleet \\nimpact assessment. Energy 31, 2064–2087 (2006). \\n14.\\t Manley, D. K., McIlroy, A. & Taatjes, C. A. Research needs for future internal \\ncombustion engines. Physics Today 61, 47–52 (2008).\\n15.\\t Yang, J. & Caillat, T. Thermoelectric materials for space and automotive power \\ng',\n",
       " 'eneration. MRS Bull. 31, 224–229 (2006).\\n16.\\t Scrosati, B. & Grache, J. Lithium batteries: status, prospects and future. J. Power \\nSources 195, 2419–2430 (2010).\\n17.\\t McCone, A. Electric Vehicle Battery Prices Down 14% Year on Year http://www.\\nbnef.com/PressReleases/view/210 (Bloomberg New Energy Finance, 2012).\\n18.\\t Chan, C. C. The state of the art of electric, hybrid, and fuel cell vehicles. \\nProc. IEEE 95, 704–718 (2007).\\n19.\\t Lukic, S. M., Cao, J., Bansal, R. C., Rodriguez, F. & Emadi, A. En',\n",
       " 'ergy storage \\nsystems for automotive applications. IEEE Trans. Ind. Electron. 55, 2258–2267 \\n(2008). \\n20.\\t Spendelow, J. & Papageorgopoulos, D. Progress in PEMFC MEA component \\n100\\n90\\n80\\n70\\n80\\n60\\n50\\n40\\n30\\n10\\n0\\n1850\\n1970\\n1940\\n1910\\n1880\\nPercentage\\n2000\\nRenewables\\nNuclear\\nGas\\nOil\\nHydro\\nCoal\\nWood\\nFigure 9 | History of the relative mix of the main source of energy used in \\nthe United States. The timescale for a shift in the energy mix is measured in \\ndecades. The growth in use of oil was increasingly',\n",
       " ' driven by the transportation \\nsector, for which it makes up 94% of transportation energy. Electricity \\nis generated from coal (48%), natural gas (19%), nuclear power (21%), \\nrenewables 10%, and oil (1%). Adapted with permission from ref. 80.\\n3 0 2  |  N A T U R E  |  V O L  4 8 8  |  1 6  A U G U S T  2 0 1 2\\nPERSPECTIVE\\nINSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n\\nR&D at the DOE fuel cell technologies program. Fuel Cells 11, 775–786 (2011).\\n21.\\t Debe, M. K. Electrocatalyst',\n",
       " ' approaches and challenges for automotive fuel cells. \\nNature 486, 43–51 (2012).\\n22.\\t Yang, J., Sudik, A., Wolverton, C. & Siegel, D. J. High capacity hydrogen storage \\nmaterials: attributes for automotive applications and techniques for materials \\ndiscovery. Chem. Soc. Rev. 39, 656–675 (2010).\\n23.\\t Paster, M. D. et al. Hydrogen storage technology options for fuel cell vehicles: \\nwell-to-wheel costs, energy efficiencies, and greenhouse gas emissions. Int. J. \\nHydrogen Energy 36, 14534–14551 (201',\n",
       " '1).\\n24 .\\tDavid, E. An overview of advanced materials for hydrogen storage. J. Mater. \\nProcess. Technol. 162–163, 169–177 (2005).\\n25.\\t Murray, L. J., Dincă, M. & Long, J. R. Hydrogen storage in metal–organic \\nframeworks. Chem. Soc. Rev. 38, 1294–1314 (2009).\\n26.\\t Abbas, H. F. & Wan Daud, W. M. A. Hydrogen production by methane \\ndecomposition: A review. Int. J. Hydrogen Energy 35, 1160–1190 (2010).\\n27.\\t Krupnick, A. J. Will Natural Gas Vehicles be in Our Future. http://www.rff.org/RFF/\\nDocuments/R',\n",
       " 'FF-IB-11-06.pdf (Resources for the Future, 2011).\\n28.\\t NACS. Key Facts and Figures. http://www.nacsonline.com/NACS/Resources/\\ncampaigns/GasPrices_2011/Pages/default.aspx (NACS, 2011).\\n29.\\t America’s Energy Future Panel. Liquid Transportation Fuels. \\n(National Academies Press, 2009).\\n30.\\t Environmental Protection Agency. Renewable Fuel Standard Program (RFS2): \\nFinal Rulemaking http://www.epa.gov/otaq/fuels/renewablefuels/regulations.\\nhtm (US Environmental Protection Agency, 2010). \\n31.\\t United N',\n",
       " 'ations Environment Programme. Towards Sustainable Production of \\nand use of resources: Assessing Biofuels http://www.unep.org/publications/\\ncontents/pub_details_search.asp?ID=4082 (United Nations Environment \\nProgramme, 2009).\\n32.\\t Stratton, R. W., Wong, H. M. & Hileman, J. I. Quantifying variability in life cycle \\ngreenhouse gas inventories of alternative middle distillate transportation fuels. \\nEnviron. Sci. Technol. 45, 4637–4644 (2011).\\n33.\\t Peralta-Yahya, P. P., Zhang, F., del Cardayre, S. ',\n",
       " 'B. & Keasling, J. D. Progress \\nin the microbial production of advanced biofuels: from feedstocks to fuels. \\nNature 488, 320–328 (2012).\\n34.\\t Worldwatch Institute. Biofuels for Transport (Routledge, 2007).\\n35.\\t Georgianna, D. R. & Mayfield, S. P. Exploiting diversity and synthetic biology for \\nthe production algae biofuels, Nature 488, 329–335 (2012).\\n36.\\t Zhu, X-, G., Long, S. P. & Ort, D. R. What is the maximum efficiency with \\nwhich photosynthesis can convert solar energy into biomass? Curr. O',\n",
       " 'pin. \\nBiotechnol. 19, 153–159 (2008).\\n37.\\t Conrado, R. J. et al. in Electrofuels: A new paradigm for renewable fuels. Advanced \\nBiofuels and Bioproducts (Lee, J. ed.) (Springer, 2012).\\n38.\\t International Energy Agency. World Energy Outlook 2011 179,186 \\nhttp://www.worldenergyoutlook.org/ (International Energy Agency, 2011).\\n39.\\t International Energy Agency. World Energy Outlook 2011, 180 \\nhttp://www.worldenergyoutlook.org/ (International Energy Agency, 2011).\\n40.\\t US Energy Information Administr',\n",
       " 'ation. Annual Energy Outlook 2011 Levelized \\nCost Supplement http://www.eia.gov/forecasts/archive/aeo11/electricity_\\ngeneration.cfm  (US Energy Information Administration, 2011)\\n41.\\t Bloomberg New Energy Finance. Levelised Cost of Energy Update, Q3 2012 \\nhttp://www.bnef.com/WhitePapers/download/114 (Bloomberg New Energy \\nFinance, 2012).\\n42.\\t Wiser, R., Lantz E., Bolinger, M. & Hand, M. Recent Developments in the Levelized \\nCost of Energy from U.S. Wind Power Projects. http://eetd.lbl.gov/ea/ems/',\n",
       " '\\nreports/wind-energy-costs-2-2012.pdf (US Department of Energy, 2012).\\n43.\\t Galen Barbose, G. Darghouth, N., Wiser, R. & Seel, J. Tracking the Sun: An \\nHistorical Summary of the Installed Cost of Photovoltaics in the United States from \\n1998 to 2010 http://eetd.lbl.gov/ea/ems/reports/lbnl-5047e.pdf (Lawrence \\nBerkeley National Laboratory report, 2011). \\n44.\\t Solar Energy Industries Association. US Solar Market Insight Report Q1 2012 \\nhttp://www.seia.org/research-resources/us-solar-market-insight',\n",
       " ' (Solar Energy \\nIndustries Association, 2012).\\n45.\\t Hand, M. M. et al. Renewable Electricity Futures Study http://www.nrel.gov/\\nanalysis/re_futures/ (National Renewable Energy Laboratory, 2012).\\n46.\\t US Department of Energy. SunShot Vision Study http://www1.eere.energy.gov/\\nsolar/pdfs/47927.pdf (US Department of Energy, 2012).\\n47.\\t Baziliana, M. et al. Re-considering the Economics of Photovoltiac Power www.\\nbnef.com/WhitePapers/download/82 (Bloomberg New Energy Finance, 2012).\\n48.\\t Swanson, R. M',\n",
       " '. Plenary talk at the DOE SunShot Grand Challenge: Summit and \\nTechnology Forum on June 14, 2012 https://www.eeremultimedia.energy.\\ngov/solar/videos/sunshot_summit_richard_swanson_plenary_session (US \\nDepartment of Energy, 2012).\\n49.\\t US Department of Energy. SunShot Initiative http://www1.eere.energy.gov/solar/\\nsunshot/accelerating_deployment.html (US Department of Energy, 2012).\\n50.\\t Swanson, R. M. Proc. 31st IEEE Photovoltaic Specialists Conf. 889–894 (2005).\\n51.\\t Graetzel M. Jannsen, R. A., ',\n",
       " 'Mitzi, D. B. & Sargent, E. H. Materials interface \\nengineering for solution-processed photovoltaics, Nature 488, 304–312 (2012).\\n52.\\t Stoddard, N., Wu B., Maisano, L., Russell, Clark, R. & Fernandez, J. M. The \\nLeading Edge of Silicon Casting Technology and BP Solar’s Mono2 Wafers, in 18th \\nWorkshop on Crystalline Silicon Solar Cells & Modules: Materials and Processes. \\npp. 7–14. http://www.nrel.gov/docs/fy08osti/45745.pdf (National Renewable \\nEnergy Laboratory, 2008).\\n53.\\t Mavrokefalos, A., Han',\n",
       " ', S. E., Yerci, S., Branham, M. S. & Chen, G. Efficient light \\ntrapping in inverted nanopyramid thin crystalline silicon membranes for solar \\ncell applications. Nano Lett. 12, 2792–2796 (2012).\\n54.\\t Solexel. Solexel unveils ultra-thin, high-performance silicon solar cell at Intersolar. \\nhttp://www.solexel.com/IntersolarNA2012.php (Solexel, 2012).\\n55.\\t Rao, R. A. et al. Proc. 37th IEEE Photovoltaic Specialists Conf. 1504–1507 (2011). \\n56.\\t Sachs, E. M., Wallace R. L., Hantsoo, E. T., Lorenz, A. M',\n",
       " '., Hudelson S.G.D. \\n& Jonczyk, R. Methods for efficiently making thin semiconductor bodies \\nfrom molten material for solar cells and the like, US Patent Application \\n20120067273 (2012).\\n57.\\t Garland J. W., Biegala, T., Carmody, M., Gilmore, C. & Sivananthan, S. Next-\\ngeneration multijunction solar cells: The promise of II-VI materials. J. Appl. \\nPhys. 109, 102423 (2011). \\n58.\\t Miller, O. D., Yablonovitch, E. & Kurtz, S. R. Strong internal and external \\nluminescence as solar cells approach the Sh',\n",
       " 'ockley–Queisser limit. IEEE \\nJ. Photovolt. 2, 303–311 (2012).\\n59.\\t Angel, R. & Olbert, B. H. Method of manufacturing large dish reflectors for a \\nsolar concentrator apparatus. US Patent Application 20120125400 (2011).\\n60.\\t Gur, I., Sawyer, K. & Prasher, R. Searching for a better thermal battery. \\nScience 335, 1454–1455 (2012).\\n61.\\t Advanced Research Projects Agency-Energy. ARPA-E Efficient Power Conversion \\nFactsheet. http://arpa-e.energy.gov/Portals/0/Documents/FundedProjects/\\nADEPT%20Slickshee',\n",
       " 'ts/ADEPT_Program_Final_One%20Pager.pdf (Advanced \\nResearch Projects Agency-Energy, 2010).\\n62.\\t Bolinger, M. & Wiser, R. Understanding wind turbine price trends in the U.S. \\nover the past decade. Energy Pol. 42, 628–641 (2012). \\n63.\\t US Department of Energy. Critical Materials Strategy. http://energy.gov/sites/\\nprod/files/piprod/documents/cms_dec_17_full_web.pdf (US Department of \\nEnergy, 2010)\\n64.\\t National Research Council. Minerals, Critical Minerals, and the U.S. Economy \\n(National Academies ',\n",
       " 'Press, 2008).\\n65.\\t Komuro, M., Kozono Y., Hanazono, M. & Yutaka, S. Epitaxial growth and \\nmagnetic properties of Fe16N2 films with high saturation magnetic flux density. \\nJ. Appl. Phys. 67, 5126–5130 (1990).\\n66.\\t Nian, J., Lawrence F. A., Edgar L. C., &Wang, J.-P. N-site ordering effect on \\npartially ordered Fe16N2. Appl. Phys. Lett. 98, 092506 (2011).\\n67.\\t Bader, S. J. Opportunities in nanomagnetism. Rev. Mod. Phys. 78, 1–15 (2006).\\n68.\\t American Physical Society. Direct air capture of CO2 with',\n",
       " ' chemicals: a Technology \\nAssessment for the APS Panel on Public Affairs. www.aps.org/policy/reports/\\nassessments/upload/dac2011.pdf (American Physical Society, 2011).\\n69.\\t Turner, N. J. Directed evolution drives the next generation of biocatalysts. Nature \\nChem. Biol. 5, 567–573 (2009).\\n70.\\t Pruess, K. Enhanced geothermal systems (EGS) using CO2 as working fluid \\n— a novel approach for generating renewable energy with simultaneous \\nsequestration of carbon. Geothermics 35, 351–367 (2006).\\n71.\\t I',\n",
       " 'nternational Energy Agency. Technology Roadmap — Nuclear Energy http://\\nwww.iea.org/papers/2010/nuclear_roadmap.pdf (International Energy Agency, \\n2010).\\n72.\\t Ruhl, C. Energy in 2011 — disruption and continuity BP Statistical Review of World \\nEnergy http://www.bp.com/sectiongenericarticle800.do?categoryId=9037130\\n&contentId=7068669 (BP, 2012).\\n73.\\t Nuclear Energy Institute. Costs: Fuel, Operation and Waste Disposal http://www.\\nnei.org/resourcesandstats/nuclear_statistics/costs/ (National Energy ',\n",
       " 'Institute, \\n2012). \\n74.\\t Nuclear Energy Institute. The Cost of New Generating Capacity in Perspective \\nhttp://www.nei.org/resourcesandstats/documentlibrary/newplants/\\ngraphicsandcharts/the-cost-of-new-generating-capacity-in-perspective \\n(Nuclear Energy Institute, 2012).\\n75.\\t Massachusetts Institute of Technology. Update of the MIT 2003 Future of Nuclear \\nPower Study http://web.mit.edu/nuclearpower/pdf/nuclearpower-update2009.\\npdf (Massachusetts Institute of Technology, 2009).\\n76.\\t Chu, S. Americ',\n",
       " 'a’s new nuclear option. Wall Street J. (March 23, 2010).\\n77.\\t US Department of Energy. Small Modular Reactor Fact Sheet http://www.ne.doe.\\ngov/pdfFiles/factSheets/2012_SMR_Factsheet_final.pdf (US Department of \\nEnergy, 2012).\\n78.\\t US Department of Energy. Consortium for Advanced Simulation of LWRs Project \\nSummary. http://www.ornl.gov/sci/nsed/docs/CASL_Project_Summary.pdf (US \\nDepartment of Energy).\\n79.\\t Advanced Research Projects Agency-Energy. Grid-scale renewable energy \\nstorage factsheet. h',\n",
       " 'ttp://arpa-e.energy.gov/Portals/0/Documents/\\nFundedProjects/GRIDS%20Slicks/_GRIDS_ProgramSheet_Final.pdf (Advanced \\nResearch Projects Agency-Energy, 2010).\\n80.\\t Energy Information Administration. Annual Energy Review 2010. Report No. \\nDOE/EIA-0384(2010) www.eia.gov/totalenergy/data/annual/pdf/aer.pdf \\n(Energy Information Administration, 2010).\\n81.\\t Munich Re. Topics Geo. http://www.munichre.com/publications/302-07225_\\nen.pdf (Munich Re, 2012).\\n82.\\t Intergovernmental Panel on Climate Change. Mana',\n",
       " 'ging the Risks of \\nExtreme Events and Disasters to Advance Climate Change Adaptation. \\nhttp://www.ipcc-wg2.gov/SREX/images/uploads/SREX-All_FINAL.pdf \\n(Cambridge Univ. Press, 2012).\\nAcknowledgements The authors wish to acknowledge T. J. Augustine, P. Davis, \\nH. Gruenspecht, M. Le, P. Lyons, R. Ramamoorthy, D. Sandalow, S. Satyapal and \\nE. Toone for suggestions.\\nAuthor Information Reprints and permissions information is available at \\nwww.nature.com/reprints. The authors declare no competing finan',\n",
       " 'cial \\ninterests. Readers are welcome to comment on the online version of this \\narticle at go.nature.com/7pb7ff. Correspondence should be addressed to S.C. \\n(the.secretary@hq.doe.gov) and A.M. (4majumdar.pubs@gmail.com).\\n1 6  A U G U S T  2 0 1 2  |  V O L  4 8 8  |  N A T U R E  |  3 0 3\\nPERSPECTIVE INSIGHT\\n© 2012 Macmillan Publishers Limited. All rights reserved\\n']"
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Initialize HuggingFace embeddings\n",
    "embedding_model = HuggingFaceEmbeddings(model_name=\"sentence-transformers/all-MiniLM-L6-v2\")\n",
    "embeddings = np.array([embedding_model.embed_query(doc) for doc in documents])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Store in FAISS index\n",
    "d = embeddings.shape[1]\n",
    "index = faiss.IndexFlatL2(d)\n",
    "index.add(embeddings)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "384"
      ]
     },
     "execution_count": 65,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<faiss.swigfaiss_avx2.IndexFlatL2; proxy of <Swig Object of type 'faiss::IndexFlatL2 *' at 0x000001D5235F0FF0> >"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "test_query = \"What are the recent advancements in AI-powered drug discovery?\"\n",
    "\n",
    "\n",
    "query_embedding = embedding_model.embed_query(test_query)\n",
    "D, I = index.search(np.array([query_embedding]), k=3)  # Retrieve top 3 similar docs\n",
    "similar_docs = [documents[i] for i in I[0]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Expert Opinion on Drug Discovery\\nISSN: (Print) (Online) Journal homepage: www.tandfonline.com/journals/iedc20\\nInduﬆrializing AI-powered drug discovery: lessons\\nlearned from the Patrimony computing platform\\nMickaël Guedj, Jack Swindle, Antoine Hamon, Sandra Hubert, Emiko\\nDesvaux, Jessica Laplume, Laura Xuereb, Céline Lefebvre, Yannick Haudry,\\nChristine Gabarroca, Audrey Aussy, Laurence Laigle, Isabelle Dupin-Roger &\\nPhilippe Moingeon\\nTo cite this article: Mickaël Guedj, Jack Swindle, Antoine Hamon, Sandra Hubert, Emiko\\nDesvaux, Jessica Laplume, Laura Xuereb, Céline Lefebvre, Yannick Haudry, Christine Gabarroca,\\nAudrey Aussy, Laurence Laigle, Isabelle Dupin-Roger & Philippe Moingeon (2022) Industrializing\\nAI-powered drug discovery: lessons learned from the Patrimony computing platform, Expert\\nOpinion on Drug Discovery, 17:8, 815-824, DOI: 10.1080/17460441.2022.2095368\\nTo link to this article:  https://doi.org/10.1080/17460441.2022.2095368\\n© 2022 The Author(s). Published by Informa\\nUK Limited, trading as Taylor & Francis\\nGroup.\\nView supplementary material \\nPublished online: 10 Jul 2022.\\nSubmit your article to this journal \\nArticle views: 3907\\nView related articles \\nView Crossmark data\\nCiting articles: 11 View citing articles \\nFull Terms & Conditions of access and use can be found at\\nhttps://www.tandfonline.com/action/journalInformation?journalCode=iedc20\\n\\nTECHNOLOGY EVALUATION\\nIndustrializing AI-powered drug discovery: lessons learned from the Patrimony \\ncomputing platform\\nMickaël Guedj\\na, Jack Swindleb, Antoine Hamonb, Sandra Huberta, Emiko Desvauxa, Jessica Laplumea, \\nLaura Xuereba, Céline Lefebvrea, Yannick Haudrya, Christine Gabarrocaa, Audrey Aussya, Laurence Laiglea, \\nIsabelle Dupin-Rogera and Philippe Moingeon\\na\\naServier, Research & Development, Suresnes, France; bLincoln, Research & Development, Boulogne-Billancourt, France\\nABSTRACT\\nIntroduction: As a mid-size international pharmaceutical company, we initiated 4 years ago the launch \\nof a dedicated high-throughput computing platform supporting drug discovery. The platform named \\n‘Patrimony’ was built up on the initial predicate to capitalize on our proprietary data while leveraging \\npublic data sources in order to foster a Computational Precision Medicine approach with the power of \\nartificial intelligence.\\nAreas covered: Specifically, Patrimony is designed to identify novel therapeutic target candidates. With \\nseveral successful use cases in immuno-inflammatory diseases, and current ongoing extension to \\napplications to oncology and neurology, we document how this industrial computational platform \\nhas had a transformational impact on our R&D, making it more competitive, as well time and cost \\neffective through a model-based educated selection of therapeutic targets and drug candidates.\\nExpert opinion: We report our achievements, but also our challenges in implementing data access and \\ngovernance processes, building up hardware and user interfaces, and acculturing scientists to use \\npredictive models to inform decisions.\\nARTICLE HISTORY\\nReceived 19 April 2022  \\nAccepted 24 June 2022  \\nKEYWORDS\\nDrug discovery; target \\nidentification; data \\nintegration; artificial \\nintelligence; multi-omics; \\ncomputing platform; \\nComputational Precision \\nMedicine\\n1. Introduction\\nIn the last decades, the pharmaceutical industry has faced \\na continuous decrease in productivity. R&D efficiency, mea\\xad\\nsured by the number of new drugs brought to patients per \\ndollar spent, has halved approximately every 10 years since \\n1950. This trend is often referred to as the Eroom’s Law, i.e. \\na reverse of the well-known Moore’s Law reflecting the expo\\xad\\nnential growth observed over time for numbers of transistors \\non a microchip [1]. As of today, it takes on average 12 years \\nand 2.6 billion U.S. dollars to bring a new drug to patients, \\nwith a probability of success of around 5–10%[2]. Root causes \\nencompass an increase in regulatory requirements, the lack of \\nsufficient validation in the selection of both therapeutic tar\\xad\\ngets and drug candidates prior to setting up costly and time- \\nconsuming clinical trials, and a need to optimize organiza\\xad\\ntional processes to better integrate scientific knowledge \\nwithin R&D [3–5]. Arguably, the most actionable lever for the \\npharmaceutical industry in order to increase the probability of \\nsuccess during drug development is to strengthen the ratio\\xad\\nnale behind decision-making, most particularly as it relates to \\nthe choice of the therapeutic target and the selection of the \\ndrug candidate.\\nTo this aim, recent breakthroughs in both biomedical and \\ncomputational sciences create new opportunities to inform \\ndrug development through computer-based approaches [6– \\n13]. Rapid advances in omics, imaging, and electronic capture \\ntechnologies make it now possible to characterize individuals \\nat both molecular, cellular, and clinical levels in a cost and \\ntime effective way [14]. Those advances occur in parallel with \\nthe exponential accumulation of information and knowledge \\naccessible through hundreds of structured biomedical data\\xad\\nbases, such as those managed by the European Bioinformatics \\nInstitute (EBI) or the US National Center for Biotechnology \\nInformation (NCBI). An effective use of these massive amounts \\nof data is facilitated by new computational approaches includ\\xad\\ning artificial intelligence (AI) and machine-learning (ML), thus \\ncreating an unprecedented opportunity to better inform deci\\xad\\nsion-making and decrease both costs and attrition rates at all \\nstages of drug discovery and development [5]. This ongoing \\nrevolution toward AI-powered drug discovery already trans\\xad\\nlates into concrete successes, with machine-designed anti- \\ncancer molecules reported to reach Phase 1 in less than \\n2 years in contrast to the 5–7 years commonly needed for \\nthe discovery phase [15].\\nAn emerging strategy for the pharmaceutical industry to \\ncapitalize on those new approaches is to build-an internal \\ncomputing platform to support the identification of disease \\ntargets as well as the repurposing of existing drugs in \\na systematic and efficient way. To this aim, a technical chal\\xad\\nlenge is to integrate the ever-increasing number of data \\nsources compiling massive and multidimensional information \\ncoming from genetics, multi-omics, molecular interactions, \\nCONTACT Mickaël Guedj \\nmickael.guedj@gmail.com \\nServier, Research & Development, Suresnes Cedex, France\\nSupplemental data for this article can be accessed online at https://doi.org/10.1080/17460441.2022.2095368\\nEXPERT OPINION ON DRUG DISCOVERY                                                                                                                        \\n2022, VOL. 17, NO. 8, 815–824\\nhttps://doi.org/10.1080/17460441.2022.2095368\\n© 2022 The Author(s). Published by Informa UK Limited, trading as Taylor & Francis Group. \\nThis is an Open Access article distributed under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives License (http://creativecommons.org/licenses/by-nc-nd/4.0/), \\nwhich permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited, and is not altered, transformed, or built upon in any way.\\n\\npreclinical experiments, clinical as well as real-life evidence \\ndata. A dedicated and specifically designed computational \\nframework is necessary to capture the value of all these infor\\xad\\nmation in order to guide decision-making and increase the \\nprobability of success during drug development.\\nIn light of the rapidly evolving environment driven by AI \\nand digital technologies, a decision was made in early 2018 \\nby the pharmaceutical group Servier to implement within the \\nR&D new data processes and computational methods \\nthrough a dedicated high-throughput computing platform. \\nGiven that a primary objective of this initiative was to valorize \\nexisting data, knowledge, and assets produced by the com\\xad\\npany during previous or ongoing R&D projects, the platform \\nwas named ‘Patrimony.’ The latter was built up with the intent \\nto capitalize on both proprietary and public data to drive \\ninnovation. After 4 years of implementation, the Patrimony \\nplatform has transformed very significantly Servier’s approach \\nto drug discovery and development. Herein, we share the \\nmain lessons learned from this initiative during its conception \\nand implementation, as well as the challenges that were \\nfaced to make it impactful on our drug development \\nprocesses.\\n2. Overview of the market\\nSeveral public or public–private initiatives have emerged with \\nthis orientation such as Open Targets [16,17]. In parallel, the \\nperspective of substantial cost savings offered to pharmaceuti\\xad\\ncal industries by substituting computerized modeling systems to \\ntraditional wet-bench biology, prompted the emergence of \\nnumerous start-ups aiming to reinvent drug discovery (Supp. \\nTable 1). In this context, in parallel to initiating this project, we \\nperformed a benchmark analysis of external solutions. We con\\xad\\ncluded that given its foreseen strategic relevance for the com\\xad\\npany, there was a strong added value to implement the project \\ninternally for gaining reactivity, flexibility, as well as for a better \\nintegration of internal and external data sources.\\n3. How the Patrimony computing platform works\\n3.1. General framework\\nA computing platform comprises a set of hardware infrastruc\\xad\\ntures, software, and user interface components that allow stor\\xad\\ning data and running algorithms in order to achieve a set of \\nwell-defined tasks within a specific field of application. The \\nmethodology we applied to build up such a computing platform \\nfor drug discovery is summarized in Figure 1, combining a range \\nof computational approaches in workflows to integrate, analyze, \\nand interpret data. A first step was to identify all relevant exist\\xad\\ning biomedical databases and knowledge sources, both struc\\xad\\ntured and unstructured, public or internal. A second step was to \\ncurate and integrate data sources into a knowledge graph (or \\nnetwork) as defined below. Lastly, algorithms were developed to \\nmine this graph for a specific application (related to a given \\ndisease or a therapeutic area) in order to generate and prioritize \\nhypotheses regarding new therapeutic targets or opportunities \\nfor drug repurposing. Analyzing such a volume and diversity of \\ndata turned out to be challenging, thus requiring an adapted \\ncomputational framework to ensure both a good integration, \\nappropriate use, and traceability. This methodology was con\\xad\\ncretized into a high-throughput scalable process encompassing \\nall steps from data acquisition, hypothesis generation, prioritiza\\xad\\ntion of outcomes, and experimental validation.\\nWe separated data sources based upon whether they were \\neither in-house or public, as well as application-agnostic versus \\napplication-related. From a core set of in-house/public sources \\nshared for all applications, we subsequently added a data pack\\xad\\nage specific to each single application of interest, making the \\nPatrimony framework both robust and highly flexible. In-house \\nsources broadly used to support Servier’s R&D projects com\\xad\\nprised data related to both therapeutic targets being investi\\xad\\ngated, potentially relevant proprietary drug(s), the phase within \\nresearch or development as well as the therapeutic area. Core \\npublic sources used such as DrugBank or UniProt listed in Supp. \\nTable 2 encompass the existing public knowledge on multiple \\ndomains of interest (e.g. biomolecular, pharmacological, clinical) \\n[16–52]. The application-related data package further assembled \\nfocused on the disease or set of diseases of interest, which for \\nServier relates to immuno-inflammatory, oncological, and neu\\xad\\nrological disorders. The level of implication of various genes and \\nproteins in pathophysiological processes was obtained from \\nmulti-omics patient profiling data, by comparing cases and \\ncontrols in different relevant conditions; either from aggregated \\nstatistics or derived from sample-level molecular data (subse\\xad\\nquently turned internally into aggregated statistics) retrieved \\nfrom both public repositories such as GEO or UK Biobank listed \\nSupp. Table 2, partnerships such as public–private IMI projects \\nas well as proprietary experimental data [53–57].\\n3.2. Building up a proprietary and adaptable knowledge \\ngraph\\nKnowledge graphs are network-like digital structures repre\\xad\\nsenting knowledge as a set of concepts and their relationships. \\nAs such, they facilitate the interface between humans and \\nmachines to analyze their content and support complex \\nArticle highlights\\n●Patrimony is a computing platform implemented by the pharmaceu\\xad\\ntical company Servier to capture the value of massive biomedical \\ndata (proprietary and public) with machine-learning techniques to \\nsupport drug discovery.\\n●The platform is based on a knowledge-graph connecting biomolecu\\xad\\nlar, pharmacological, and clinical domains, which can be mined to \\ngenerate new therapeutic hypotheses.\\n●Hypotheses are assessed and prioritized by aggregating information \\naround five summary criteria (Biological Relevance, Causality, \\nTractability, Safety, and Innovativeness).\\n●The platform was implemented following three iterations (proof-of- \\nconcept, structuration, and industrialization), while relying upon an \\nAgile operating model and FAIR guiding principles for data \\nmanagement.\\n●Reported applications encompass immuno-inflammatory diseases \\nand COVID-19; from these successes, the use of the platform is \\nbeing extended to oncology and neurology.\\n●Patrimony fosters an open innovation mindset, requiring both agility \\nand transversality across a broad range of existing and emerging \\nexpertises associated with AI.\\n816\\nM. GUEDJ ET AL.\\n\\ndecision-making. They are well adapted to model the inter\\xad\\nconnectivity of biomedical systems. Numerous detailed and \\nformal introductions to their application in Medicine have \\nbeen published elsewhere [58–60].\\nFrom the data sources described above, we built \\na knowledge graph connecting an unbiased mapping of all \\nknown molecular interactions (i.e. the Human Interactome) to \\npharmacological and clinical domains (Figure 2a). The result\\xad\\ning graph was made of overall 50k+ nodes and 200k+ relation\\xad\\nships, combined with all information we could gather or \\ngenerate as attributes on each node or interaction, and further \\nenriched with semantics or ontologies to help navigating \\nthrough the concepts (e.g. GO, ChEBI, EFO, MedDRA) [29– \\n31,36,48]. For each application, specific related data sources \\nwere used to complement the core knowledge graph with \\na set of additional nodes, interactions and attributes. \\nAggregate statistics resulting from multi-omics analyses (e.g. \\nfold-changes, p-values, etc.) were mapped to the knowledge \\ngraph as attributes of gene or protein nodes. Interactions \\ninferred from patient-level data such as gene–gene co- \\nexpression values were used to weight or complete the \\nknown interactions between them.\\nAt the heart of Patrimony is the relationship between the \\nbiomolecular, pharmacological, and clinical spaces in order to \\nidentify the most relevant therapeutic targets with regard to \\nthe measured pathophysiological manifestations of a disease \\nor a set of diseases (Figure 2a). The resulting knowledge graph \\nis unique to the owner and proprietary in the sense that it \\nintegrates public data with internal inputs. Consequently, it \\nrepresents a very strategic entry point to capture the value of \\nthe scientific knowledge accumulated over time by the \\ncompany.\\nIn order to mine this knowledge graph and extract the \\nmost relevant information, some specific methodological \\nexpertise has been developed (Figure 2b). As main examples, \\nthe identification of nodes exhibiting more frequent interac\\xad\\ntions with other nodes (i.e. hubs), or nodes having frequent \\ninteractions with each other (i.e. clusters) were deemed of \\nFigure 1. Patrimony general framework. Data sources are curated and integrated into a knowledge graph. For each application starting from one disease or a set of \\ndiseases, the knowledge graph is mined in order to evaluate the putative targets, generate hypotheses, and assist the consolidation.\\na\\nb\\nFigure 2. Patrimony knowledge graph. (a) Data sources are integrated through a relational diagram to form the Patrimony knowledge graph covering and \\ninterconnecting biomolecular, pharmacological, disease, and clinical domains. (b) Different metrics and approaches are used to mine the knowledge graph.\\nEXPERT OPINION ON DRUG DISCOVERY\\n817\\n\\nparticular interest as they tend to correspond to molecules \\npredicted to play a key role in biological or pathological \\nprocesses. The concept of distance within networks was also \\nfound to be critical, albeit not easy to quantify. Whereas the \\ndistance between two nodes can appear trivial to determine, \\nassessing the distance between two sets of nodes raises many \\nquestions depending upon the use case. We generally use \\na topological distance corresponding to the shortest path \\nlength in the graph between the nodes of interest. As the \\nsignificance of the distance depends on the density of the \\ngraph, we generate a distribution from bootstrapping similar \\nnodes defined by same degree in the graph; from this given \\ndistribution, \\nwe \\nderive \\na \\nstrandardized \\nz-score \\nand \\na corresponding p-value. Diffusion/propagation algorithms \\nwere selected as particularly useful because – as their name \\ninfer – they diffuse or propagate the information along the \\nlinks of the graph. It assesses the impact of a perturbation \\nstarting from a given node on the network by use of radom- \\nwalk probabilities. As such, they allow to capture the informa\\xad\\ntion on both the nodes and their neighborhood [61]. \\nFurthermore, the graph structure turned out to be very \\nadapted to facilitate the direct application of graph-based \\ndeep learning approaches such as Graph Convolutional \\nNetworks (GCNs) that aggregate features of the different \\ntypes of nodes and their relationships in order to predict \\nnew associations between drugs, targets, and diseases \\n[60,62]. The general principle is to learn how to represent \\nthe graph and map its nodes into a compact embedding \\nspace. As a result, it embeds diseases associated with similar \\ngenes or drugs whose target proteins have similar local neigh\\xad\\nborhoods close together in the embedding space. Altogether, \\none advantage of the knowledge graph supporting the \\nPatrimony computing platform turned out to allow initiating \\ninvestigations from any entry point among genes, diseases, or \\ndrugs depending on the data sources already integrated \\nwithin the platform. Potential queries related for instance to \\nthe most relevant targets for a given disease, or to potential \\nindications in which a drug could be repurposed. As such, the \\nPatrimony computing platform has now been well established \\nwithin Servier as a versatile tool to create and maintain \\na competitive advantage when developing drugs against dis\\xad\\neases of interest.\\n3.3. Target hypothesis assessment and prioritization\\nA central application of Patrimony is to identify novel thera\\xad\\npeutic target candidates from the modeling of a disease of \\ninterest. To capitalize on the high quantity of information \\ncontained in its knowledge graph, some metrics have been \\nestablished in order to rationalize the assessment and prior\\xad\\nitization of actionable therapeutic targets. Inspired by princi\\xad\\nples originally found in Open Targets, we eventually selected \\nstrategic summary criteria along five distinct dimensions \\n(Figure 3a).\\nThe first and most important criterion relates to Biological \\nRelevance. For a given use, it summarizes all the activities \\ncontributing to the understanding of the pathophysiology of \\na disease from multi-omics data. A score is generated to \\nquantify the level of cumulated evidence predicting a gene \\nor a protein to be a highly relevant target because of biomo\\xad\\nlecular associations or dysregulations, by itself and/or when \\nconsidering its neighborhood within the graph. Genes with \\na high Biological Relevance are referred to as disease-related \\ngenes, in that they are likely to contribute to the pathophy\\xad\\nsiology as a cause or a consequence. Also, they tend to cluster \\nand form identifiable disease modules within the knowledge \\na\\nc\\nb\\nFigure 3. Target assessment and prioritization. (a) Targets are assessed based on strategic summary criteria along five distinct dimensions. (b)The five criteria are \\nindividually quantified and a global scoring is then computed in order to prioritize the top targets for which the overall rationale is the highest. (c) Individual target \\nID cards are generated to represent the global assessment in a visual and easily interpretable format to end users.\\n818\\nM. GUEDJ ET AL.\\n\\ngraph. Computing techniques described previously including \\nAI/ML support here the prediction of new disease–gene asso\\xad\\nciations as well as disease modules. A second criterion, related \\nto biological relevance but nevertheless assessed indepen\\xad\\ndently, is Causality, which we consider as a highly critical \\ndimension. The latter is being assessed to discriminate \\nbetween causes and consequences in the observed pathophy\\xad\\nsiology. As such and depending on the information available, \\nit can be derived from (i) the force of genetic associations \\naround the target (taking into account that genetic signals are \\nrather causal compared to transcriptomic profiles which often \\nresult from downstream regulations), (ii) the expression of the \\ntarget in cells or tissues relevant to the disease of interest \\n(documented for instance from GTeX), and (iii) AI/ML predic\\xad\\ntions of what is likely to be a real target based on graph \\nfeatures and known approved targets as training. On \\na concrete basis, Causality may for instance refer to master \\nregulators or driver mutations [32,33]. A third criterion relates \\nto Tractability (also referred to as Druggability), which repre\\xad\\nsents the likelihood to modulate the function of a target with \\neither small synthetic or biological drugs. To date, we have \\nbeen using the measure proposed by Open Targets based on \\nboth precedence of the target in clinical trials, discovery \\nexperiments and computational predictions. In the future, \\nthis assessment will be supplemented with our own measures \\nof Tractability to include additional modalities of interest, e.g. \\nRNA-based antisense oligonucleotides. The fourth criterion is \\nlinked to potential Safety implications when interfering with \\na given target. It is assessed by the number of safety events \\nassociated with drugs known to bind the target reported in \\ndatabases such as SIDER [44]. In subsequent developments, we \\nwill consider not only the number but also the severity of \\nadverse events. The last criterion considers Innovativeness in \\nrelationship with the application on a disease of interest, \\ndocumented from either clinical trials, as well as patent or \\nliterature-mining by using Natural Language Processing (NLP) \\n[63]. Given the flexibility of the Patrimony computational fra\\xad\\nmework, any other relevant summary criterion could be easily \\nincluded in the future. For instance, assessing the Feasibility of \\ndrug development could be added to support target ranking, \\nas suggested by others [64].\\nThe above mentioned five criteria are individually quanti\\xad\\nfied and subsequently computed in a global scoring in order \\nto prioritize the top targets for which the overall rationale is \\nthe highest (Figure 3b). For any given target, individual ‘target \\nID cards’ are generated to represent the overall assessment in \\na visual and easily interpretable manner to end users \\n(Figure 3c).\\n3.4. From targets to drugs\\nFollowing the prioritization of top hypotheses on candidate \\ntargets with the Patrimony scoring system, the rationale is \\nsubsequently consolidated through a deep-dive investigation \\nby biologists and pharmacologists. During this consolidation \\nphase, both an extensive literature review and in house trans\\xad\\nlational data analyses are performed to confirm that identified \\ntarget candidates are involved in specific disease pathways \\nand are druggable with a specific compound modality.\\nResearchers can then validate target hypotheses, through \\nan experimental confirmation that disease activity is impacted \\nfollowing perturbation of the target of interest with a drug or \\na tool compound. Conducting wet-lab gene inhibition (e.g. via \\nCRISPR-Cas9 deletion or RNA silencing) or preclinical experi\\xad\\nments by using cellular assays or animal models are com\\xad\\nmonly implemented to corroborate the hypothesis that \\ndrugs interacting with the target exhibit the anticipated phar\\xad\\nmacological activity. Once a therapeutic target has been \\nselected, multiple processes streamlined by the pharmaceuti\\xad\\ncal industry can be used to identify small molecules or biolo\\xad\\ngicals interacting with it. For instance, High-Throughput \\nScreening (HTS) can be implemented to test the company’s \\nproprietary compound library in various molecular or cell- \\nbased assay systems in order to identify drug candidates \\n[65]. As of today, another strategy relies upon dedicated com\\xad\\nputational methods to select in silico drugs predicted to \\nengage the target of interest [15].\\nOne specific use of the Patrimony platform consists in \\nidentifying existing drugs modulating the target, e.g. within \\nthe R&D pipeline of the company or among marketed com\\xad\\npounds. Referred to as drug repurposing, drug rescuing or \\nindication extension, this approach has generated a growing \\ninterest during the last decade, with evidence that mining \\navailable biomedical data with proper algorithms can gener\\xad\\nate fast and valid innovative hypotheses [41,66–68]. The \\nknowledge graph used in Patrimony has thus also been \\napplied to such drug repurposing approaches, by using both \\ndistances between drug targets and disease-related proteins \\n[69], connectivity maps [23,24], and deep-learning methodol\\xad\\nogy to identify in silico new targets for known drugs [70].\\n3.5. Implementation\\nWe defined three iterations to implement the Patrimony com\\xad\\nputing platform: a proof-of-concept, a structuration, and an \\nindustrialization step. The proof-of-concept was pilot initiative \\nperformed within 3 months aiming for quick-wins. Based on \\na minimal set of datasets and algorithms, the aim was to \\nposition a given set of targets of interest into one disease \\n(namely Sjögren syndrome). Then, the structuration step \\naimed to list, retrieve, and implement all the necessary data\\xad\\nsets and algorithms along with conceiving a first dedicated \\nand adapted computational infrastructure. This step was per\\xad\\nformed around a well-defined application focused on \\nimmuno-inflammatory diseases. Finally, the industrialization \\nstep aimed to transfer the existing Patrimony platform into \\na more scalable architecture in order to pave the ground for \\na subsequent application to all therapeutic areas of interest for \\nthe company. Within each iteration, we adopted an Agile \\noperating model for software development with alternated \\nsequences of brainstorming, implementation (sometimes in \\nprecisely defined sprints), generation of results, consolidation, \\nand feedback [71]. Technical choices for IT infrastructure, soft\\xad\\nware and algorithms have proven to be challenging to ensure \\na robust and flexible solution in a fast-evolving field. Codes \\nwere developed in Python and R. For the two first iterations, \\nwe put in place a Microsoft Azure cloud-based sandbox with \\nMongoDB for database management. We subsequently moved \\nEXPERT OPINION ON DRUG DISCOVERY\\n819\\n\\nto Google Cloud Platform (GCP) and BigQuery for scalability. We \\nalso used a mixture of Neo4j, Cytoscape, Python graph-tool, and \\nR igraph to support graph storage, mining, and visualization \\n[72]. We followed FAIR guiding principles to enable findability, \\naccessibility, interoperability, and reusability of the data when \\nbuilding up our new data governance. Integrating large-scale \\nand multidimensional data generated from multiple technol\\xad\\nogies with proper quality attributes in terms of consistency \\nand reliability remained a significant difficulty throughout data \\nlife-cycle management. Assessing the right to use data from \\npublic sources also turned out to be complex. Whereas some \\ndata sources apply a clear ‘no restriction’ policy for any use of \\nthe data, e.g. under a Creative Common license for sharing, \\nothers make a distinction between the type of requesting \\ninstitution (profit versus nonprofit) or intended use (research \\nversus commercial purposes). Of note, it has been decided not \\nto include any patient-level data in order to overcome the \\nregulatory complexity linked to their use. Also, the possibility \\nto include data from partnerships such as IMI projects needs \\nto be evaluated case by case.\\n4. Applications\\nThe two first iterations (proof-of-concept and structuration) of \\nthe Patrimony initiative were built while focusing on immuno- \\ninflammatory diseases as selected indications. They were spe\\xad\\ncifically designed to evaluate the capacity of the Patrimony \\nplatform to support two direct applications, i.e. identifying \\ntherapeutic targets and generating hypotheses for drug \\nrepurposing.\\nAs a first run, we mapped multi-omics profiling data from \\nthe PRECISESADS cohort of patients with various autoim\\xad\\nmune diseases into the knowledge graph to support drug \\ndevelopment against primary Sjögren’s syndrome [73]. \\nA particular challenge in this indication was to rationally \\ndesign immunotherapeutic approaches acting at a systemic \\nlevel and/or target organs (i.e. salivary and lachrymal glands). \\nAs concrete outputs, Patrimony helped to identify and prior\\xad\\nitize several innovative therapeutic targets, supported by \\na robust and multidimensional set of evidence. Most of the \\ntargets identified as of interest in primary Sjogren Syndrome \\nwere confirmed to be valid as well in several autoimmune \\ndiseases sharing common pathophysiological mechanisms, \\ne.g. Systemic Lupus Erythematosus. Furthermore, Patrimony \\nwas very useful not only to identify new therapeutic targets \\nbut also to validate other ones for which our company had \\nalready initiated the development of drug candidates. \\nSpecifically, two monoclonal antibodies at an early clinical \\ndevelopment stage for autoimmune diseases, including anti- \\ntype 1 interferon and anti-IL7R antibodies (ClinicalTrial.gov \\nNCT04605978) were confirmed as valid therapeutic options \\nin Sjogren Syndrome and Systemic Lupus Erythematosus. \\nLastly, the availability of disease models providing emphasis \\non specific therapeutic targets as being relevant in various \\nautoimmune diseases has been a powerful tool to support \\nServier’s assessment of external licensing opportunities for \\ndrug candidates.\\nAs an effort to contribute to the global fight against the \\nCOVID-19 pandemics, a second application of the Patrimony \\nplatform aimed at identifying existing drugs that could be \\nrepurposed to treat those patients infected by the SARS-CoV \\n-2 virus who develop severe forms of the disease requiring \\nhospitalization. Specifically, we modeled the severe lung \\ninflammation associated with the life-threatening acute \\nrespiratory distress syndrome, which affects up to 75% of \\nCOVID-19 patients transferred to intensive care units [74]. \\nFrom data available in the scientific literature documenting \\ndifferences at a molecular level in both immune responses and \\ntissue inflammation between patients with either mild or very \\nsevere forms of the disease, an interactome of proteins pre\\xad\\ndicted to contribute to lung inflammation in severe COVID-19 \\nwas produced. The latter was used to confirm the interest of \\nseveral drugs already used in this indication such as dexa\\xad\\nmethasone, anti-IL6R antibodies or JAK2 inhibitors. It further \\nidentified additional drugs, either available in other indications \\nor in development, as being relevant for repurposing in severe \\nCOVID-19, such as inhibitors of alarmins and their recep\\xad\\ntors [75].\\nBased on these promising pilot applications to support our \\ndrug development in immuno-inflammation, we have now \\ninitiated an industrialization phase to extend its application \\nto oncology and neurology. Whereas the methodology and \\nprocesses behind were broadly applicable in those additional \\nindications, new challenges emerged reflecting the specifici\\xad\\nties in terms of categories of data predominantly used in \\ndistinct therapeutic areas. For instance, a complexity in neu\\xad\\nrology is to get access to data related to biological processes \\noccurring in the brain beyond solely postmortem samples. \\nThis hurdle induces as a consequence an emphasis in this \\nfield in genetics and animal data when compared with other \\ntherapeutic areas. In contrast, disease modeling in oncology \\ncan benefit from an abundance of molecular data from a great \\nvariety of sources, encompassing both constitutional and \\ntumor-specific molecular alterations. In the latter field, docu\\xad\\nmenting gene essentiality through CRISPR-Cas9 deletion \\nexperiments, e.g. as the ones centralized in the Depmap pro\\xad\\nject, are critical to prioritize among a wealth of hypoth\\xad\\neses [54].\\nVarious additional applications could be considered in the \\nfuture, as an extension to further support the identification \\nand optimization of drug candidates, beyond the repurposing \\nof existing drugs. The current models produced by Patrimony \\nleading to therapeutic target identification could be nicely \\nextended by machine-learning approaches to perform multi\\xad\\ntask parallel prediction of drug candidate characteristics. The \\nlatter include training artificial neural networks to select sui\\xad\\ntable therapeutic modalities for engaging a given target, pre\\xad\\ndicting both binding characteristics as well as pharmacological \\nand ADMET properties of virtual compounds, and even creat\\xad\\ning new molecules by using generative adversarial networks \\n[6]. Other applications requiring specific computational meth\\xad\\nods include generating hypotheses on potential combination \\ntherapies to address complex diseases [76]. Whereas the inter\\xad\\nest of the latter approach is well identified in oncology, we \\nhave as well recently documented how it could be \\n820\\nM. GUEDJ ET AL.\\n\\nimplemented in autoimmune diseases, and arguably in many \\nother therapeutic areas [77].\\n5. Challenges\\nImplementing ex nihilo in a pharmaceutical environment at an \\nindustrial scale, an initiative such as Patrimony has been dis\\xad\\nruptive. A major challenge was that it required a high level of \\ntransversality between multidisciplinary teams assembling \\nnumerous expertises encompassing computational hardware, \\ncloud computing, network computing, machine-learning and \\nAI, statistics, bioinformatics, large-scale biology, pharmacol\\xad\\nogy, clinical knowledge, but also legal skills to assess data \\naccessibility and use. Aligning those very diverse human \\nexpertises necessitated continuous training and internal com\\xad\\nmunication to foster acculturation to computational modeling \\nas it applies to drug discovery and development. Another \\nimportant challenge was the consolidation step needed to \\nvalidate \\nthe \\nscientific \\nrationale \\nof \\npredicted \\ntargets. \\nAlgorithms can generate many hypotheses in a short time \\nframe. The extensive literature search by human experts \\nremained time consuming in order to corroborate or refute \\nthe hypotheses generated. In this exercise, we found it critical \\nfor each given application to a disease of interest to evaluate \\nthe outputs with known targets and disease-modifying drugs. \\nTo that end, clinically approved drugs and their respective \\ntargets with established proof of efficacy have been assessed \\non the platform and their relevance verified. Altogether, dur\\xad\\ning the consolidation step, we consistently observed that the \\nrelevance of the hypotheses generated depends on the qual\\xad\\nity, completeness and continuous updates of data sources. \\nThe interpretation of model outputs by human experts was \\nalso difficult, owing to the inherent complexity of the analysis \\nof biomedical data, but also in light of the numerous existing \\ngaps in the thesaurus of human knowledge. The current \\nHuman Interactome is estimated to cover only circa 25% of \\nall possible molecular interactions [78]. As a consequence, \\nmany genes related to a given disease may appear to be \\ndispersed in the interactomes making the identification of \\ncoherent disease modules challenging. In our experience, we \\nfound that protein expression, documented for example by \\nproteomics or flow cytometry, was more translatable to dis\\xad\\nease pathological mechanisms than gene expression solely \\nassessed by transcriptomics.\\n6. Conclusions\\nThe knowledge-graph represented within Patrimony is \\na sophisticated and evolving way to represent a disease within \\na computer system. Disease modeling has recently emerged \\nas a powerful mean to educate the design and development \\nof drug candidates, in the context of Precision Medicine \\napproaches aiming to offer therapies better tailored to the \\npatients’ specificities. Such strategies are thus being estab\\xad\\nlished to capitalize on a comprehensive knowledge of the \\ndisease and of patient population heterogeneity to select \\ntherapeutic targets and drug candidates predicted to be \\nmost suitable in this indication. This trend that we termed \\nComputational Precision Medicine has become \\nhighly \\nstrategic for pharmaceutical industries to differentiate from \\nthe competition [6]. We thus emphasize the tremendous \\nvalue of developing a proprietary computational platform to \\ncreate a major competitive advantage for drug developers in \\ntheir disease areas of interest.\\nWithin Servier, we designed the Patrimony platform by \\ncombining innovative concepts, methodologies and suppor\\xad\\ntive infrastructure to grow significantly our ability to integrate \\nlarge-scale biomedical data. As of today, Patrimony allows to \\ngenerate multidimensional models relating to diseases, ther\\xad\\napeutic targets, and to some level drug candidates. Based on \\nour positive experience in exploring pilot applications in \\nimmuno-inflammatory diseases, the use of this computational \\nplatform is now being deployed throughout all therapeutic \\nareas of interest for the company.\\n7. Expert opinion\\nAs AI has the potential to transform drug discovery and sev\\xad\\neral AI-driven molecules have progressed into clinical trials \\n(most often with accelerated timelines and reduced costs), its \\nimpact and remaining challenges need to be globally \\naddressed [79,80]. As explainabilty is often invoked, we recom\\xad\\nmend paying attention to generating results that are both \\nrobust (e.g. to small changes in parameters and data), inter\\xad\\npretable (e.g. avoiding applying transformation to variables \\nthat would disconnect them to their biological or clinical \\nmeaning), and reproducible (e.g. by applying good coding \\nand data practices). Another important hurdle to implement \\ndisease modeling in support of drug discovery is the difficulty \\nto distinguish causal from incidental genes or proteins in the \\npathophysiology. In future analyses, computational inferences \\nof causality based for example on Bayesian networks represent \\nan interesting option to shed light on disease-related master \\nregulators or driver mutations [81,82]. The converging \\nadvances in high-throughput technologies such as single-cell \\nRNA sequencing or deep immunophenotyping along with \\nprotein structure prediction with the AlphaFold algorithms \\nwill contribute to expand continuously our knowledge space \\n[83,84]. Integrating this flow of new data implies to regularly \\nupdate the knowledge graph, thus raising concerns in terms \\nof trustworthiness of data sources and quality control. The \\nanalysis of disease-related processes requires complementing \\ntopology with dynamic properties at large scale, which \\nremains a major challenge [85,86]. Also, as computational \\nmodels are accumulating, there is an increasing need for \\nrapid validation and consolidation with data generated on \\npurpose or well-established external reference datasets such \\nas those proposed in the frame of data challenges (e.g. Kaggle, \\nDream, or GeneDisco). Eventually, computational innovative \\ndevelopments should also encompass other therapeutic mod\\xad\\nalities beyond small molecules, such as biologics, antisense \\noligonucleotides, protein degradation targeting and nanopar\\xad\\nticles as well as potential combination therapies.\\nImportantly, the Patrimony platform has become a highly \\ntransforming asset within our company’s R&D. It was seminal \\nto initiate a transition from classical drug development relying \\nupon life sciences, chemical expertise, and biotechnologies to \\nan educated computer-based approach powered by AI. As \\nEXPERT OPINION ON DRUG DISCOVERY\\n821\\n\\nsuch, it still challenges traditional organizational structures for \\ndrug discovery and imposes a close cooperation between \\nmultidisciplinary teams. It progressively fosters an open inno\\xad\\nvation mindset within the company, requiring both agility and \\ntransversality across a broad range of existing and emerging \\nexpertises. In light of this very significant cultural change, \\nacculturation to help acceptance by human experts of com\\xad\\nputational approaches to drug R&D is presently one of the \\nmost critical issues to solve in order to capture their full \\nbenefit.\\nAcknowledgments\\nThe authors would like to thank all members of Servier R&D who sup\\xad\\nported the initiation and implementation of Patrimony. In particular \\nM Coste, S Ollivier, V Robert, C Hébert, A Saugeot, A Bril, R Jeggo, \\nF Schmidlin, and C Bertrand. The authors also thank B Theofilopoulou \\nand T Bolba for legal assistance.\\nFunding\\nThis work was funded by Servier.\\nDeclaration of Interest\\nAll authors are employees at Servier, an international pharmaceutical \\ncompany governed by a non-profit foundation. The authors have no \\nother relevant affiliations or financial involvement with any organization \\nor entity with a financial interest in or financial conflict with the subject \\nmatter or materials discussed in the manuscript apart from those \\ndisclosed.\\nAuthor contributions\\nConceptualization and supervision: M Guedj, S Hubert, J Laplume, \\nI Dupin-Roger, and P Moingeon\\nImplementation: M Guedj, J Swindle, A Hamon, J Laplume, L Xuereb, \\nC Lefebvre, Y Haudry, and C Gabarroca\\nData access, curation, and integration: J Swindle, A Hamon, and \\nJ Laplume\\nDesign of applications: S Hubert, E Desvaux, A Aussy, L Laigle, I Dupin- \\nRoger, and P Moingeon\\nWriting of the manuscript: M Guedj and P Moingeon\\nReview and editing: all the authors\\nURLs\\nPublic sources\\nAlphaFold https://alphafold.ebi.ac.uk\\nBindingDB https://www.bindingdb.org\\nBlueprint https://www.blueprint-epigenome.eu\\nChEBI https://www.ebi.ac.uk/chebi\\nClinicalTrials.gov https://clinicaltrials.gov\\nCMAP https://clue.io/cmap\\nDepmap https://depmap.org\\nDisGenet https://www.disgenet.org\\nDrugBank https://www.drugbank.ca\\nEBI https://www.ebi.ac.uk\\nEFO http://www.ebi.ac.uk/efo\\nEnsembl https://www.ensembl.org\\nGEO https://www.ncbi.nlm.nih.gov/geo\\nGO http://geneontology.org\\nGTeX https://gtexportal.org\\nGWAS Catalog https://www.ebi.ac.uk/gwas\\nIMI https://www.imi.europa.eu\\nMedDRA https://www.meddra.org\\nMSigDB https://www.gsea-msigdb.org\\nNCBI https://www.ncbi.nlm.nih.gov\\nNCBI Genes (ex Entrez) https://www.ncbi.nlm.nih.gov/gene\\nOmicSoft DiseaseLand https://digitalinsights.qiagen.com\\nOpen Targets https://www.opentargets.org\\nPharmGKB https://www.pharmgkb.org\\nReactome https://reactome.org\\nSIDER http://sideeffects.embl.de\\nSTRING https://string-db.org\\nTTD http://db.idrblab.net/ttd\\nUK Biobank https://www.ukbiobank.ac.uk\\nUniProt https://www.uniprot.org\\nImplementation\\nBigQuery https://cloud.google.com/bigquery\\nCreative Common https://creativecommons.org\\nCytoscape https://cytoscape.org\\nFAIR https://www.go-fair.org/fair-principles\\nGCP https://cloud.google.com\\nGraph-tool https://graph-tool.skewed.de\\nIgraph https://igraph.org\\nMicrosoft Azure https://azure.microsoft.com\\nMongoDB https://www.mongodb.com\\nNeo4j https://neo4j.com\\nPython https://www.python.org\\nR https://cran.r-project.org\\nData challenges\\nDream http://dreamchallenges.org\\nGeneDisco https://www.gsk.ai/genedisco-challenge\\nKaggle https://www.kaggle.com\\nORCID\\nMickaël Guedj \\nhttp://orcid.org/0000-0002-9742-957X\\nPhilippe Moingeon \\nhttp://orcid.org/0000-0002-2380-9983\\nReferences\\nPapers of special note have been highlighted as either of interest (•) or of \\nconsiderable interest (••) to readers.\\n1. Ringel MS, Scannell JW, Baedeker M, et al. Breaking Eroom’s law. \\nNat Rev Drug Discov. 2020;19(12):833–834.\\n2. Waring MJ, Arrowsmith J, Leach AR, et al. An analysis of the \\nattrition of drug candidates from four major pharmaceutical \\ncompanies. Nat Rev Drug Discov. 2015;14(7):475–486.\\n3. Paul SM, Mytelka DS, Dunwiddie CT, et al., How to improve R&D \\nproductivity: the pharmaceutical industry’s grand challenge. Nat \\nRev Drug Discov. 2010;9(3):203–214. \\n•• A key paper describing the challenges of pharma R&D.\\n4. Morgan P, Brown DG, Lennard S, et al. Impact of a five-dimensional \\nframework on R&D productivity at AstraZeneca. Nat Rev Drug \\nDiscov. 2018;17(3):167–181.\\n5. Pammolli F, Righetto L, Abrignani S, et al. The endless frontier? The \\nrecent increase of R&D productivity in pharmaceuticals. J Transl \\nMed. 2020;18(1):162.\\n6. Moingeon \\nP, \\nKuenemann \\nM, \\nGuedj \\nM. \\nArtificial \\nintelligence-enhanced drug design and development: toward \\na computational precision medicine. Drug Discov Today. 2022;27 \\n(1):215–222. \\n•• An introduction to the concept Computational Precision \\nMedicine.\\n7. Rajpurkar P, Chen E, Banerjee O, et al. AI in health and medicine. \\nNat Med. 2022;28(1):31–38. \\n822\\nM. GUEDJ ET AL.\\n\\n•• A key introduction to AI in Medicine.\\n8. Chen H, Engkvist O, Wang Y, et al. The rise of deep learning in drug \\ndiscovery. Drug Discov Today. 2018;23(6):1241–1250.\\n9. Ekins S, Puhl AC, Zorn KM, et al. Exploiting machine learning for \\nend-to-end drug discovery and development. Nat Mater. 2019;18 \\n(5):435–441.\\n10. Greener JG, Kandathil SM, Moffat L, et al. A guide to machine \\nlearning for biologists. Nat Rev Mol Cell Biol. 2022;23(1):40–55. \\n•• A didactic introduction to machine learning applied to biome\\xad\\ndical data.\\n11. Topol EJ. High-performance medicine: the convergence of human \\nand artificial intelligence. Nat Med. 2019;25(1):44–56.\\n12. Vamathevan J, Clark D, Czodrowski P, et al. Applications of machine \\nlearning in drug discovery and development. Nat Rev Drug Discov. \\n2019;18(6):463–477.\\n13. Apweiler R, Beissbarth T, Berthold MR, et al., Whither systems \\nmedicine? Exp Mol Med. 2018;50(3):e453–e453. \\n• An introduction to the concept of Systems Medicine.\\n14. Yu MK, Ma J, Fisher J, et al. Visible machine learning for \\nbiomedicine. Cell. 2018;173(7):1562–1565.\\n15. Savage N. Tapping into the drug discovery potential of AI. \\nBiopharm Deal. 2021:d43747-021-00045–7\\n16. Koscielny G, An P, Carvalho-Silva D, et al. Open targets: a platform \\nfor therapeutic target identification and validation. Nucleic Acids \\nRes. 2017;45(D1):D985–D994.\\n17. Carvalho-Silva D, Pierleoni A, Pignatelli M, et al. Open targets plat\\xad\\nform: new developments and updates two years on. Nucleic Acids \\nRes. 2019;47(D1):D1056–D1065.\\n18. Wishart DS, Knox C, Guo AC, et al. DrugBank: a knowledgebase for \\ndrugs, drug actions and drug targets. Nucleic Acids Res. 2008;36 \\n(suppl_1):D901–D906.\\n19. Apweiler R. UniProt: the universal protein knowledgebase. Nucleic \\nAcids Res. 2004;32(90001):115D–119 .\\n20. The UniProt Consortium, Bateman A, Martin M-J, Orchard S, et al. \\nUniProt: the universal protein knowledgebase in 2021. Nucleic \\nAcids Res. 2021;49(D1):D480–D489.\\n21. Wishart DS, Feunang YD, Guo AC, et al. DrugBank 5.0: a major \\nupdate to the DrugBank database for 2018. Nucleic Acids Res. \\n2018;46(D1):D1074–D1082.\\n22. Liu T, Lin Y, Wen X, et al. BindingDB: a web-accessible database of \\nexperimentally \\ndetermined \\nprotein-ligand \\nbinding \\naffinities. \\nNucleic Acids Res. 2007;35(Database):D198–D201.\\n23. Lamb J. The connectivity map: a new tool for biomedical research. \\nNat Rev Cancer. 2007;7(1):54–60.\\n24. Subramanian A, Narayan R, Corsello SM, et al. A next generation \\nconnectivity map: L1000 platform and the first 1,000,000 profiles. \\nCell. 2017;171(6):1437–1452.e17.\\n25. Pinero J, Queralt-Rosinach N, Bravo A, et al. DisGeNET: a discovery \\nplatform for the dynamical exploration of human diseases and \\ntheir genes. Database. 2015;2015:bav028–bav028.\\n26. Piñero J, Ramírez-Anguita JM, Saüch-Pitarch J, et al. The DisGeNET \\nknowledge platform for disease genomics: 2019 update. Nucleic \\nAcids Res. 2019;gkz1021. 10.1093/nar/gkz1021\\n27. Hubbard T. The Ensembl genome database project. Nucleic Acids \\nRes. 2002;30(1):38–41.\\n28. Howe KL, Achuthan P, Allen J, et al. Ensembl 2021. Nucleic Acids \\nRes. 2021;49(D1):D884–D891.\\n29. Malone J, Holloway E, Adamusiak T, et al. Modeling sample vari\\xad\\nables with an experimental factor ontology. Bioinformatics. 2010;26 \\n(8):1112–1118.\\n30. Ontology Consortium G, Gene Ontology Consortium. The gene \\nontology (GO) database and informatics resource. Nucleic Acids \\nRes. 2004;32(90001):258D–261 .\\n31. The Gene Ontology Consortium. The gene ontology resource: 20 \\nyears and still GOing strong. Nucleic Acids Res. 2019;47(D1):D330– \\nD338.\\n32. Lonsdale J, Thomas J, Salvatore M, et al. The genotype-tissue \\nexpression (GTEx) project. Nat Genet. 2013;45(6):580–585.\\n33. The GTEx Consortium, Ardlie KG, Deluca DS, Segrè AV, et al. The \\ngenotype-tissue expression (GTEx) pilot analysis: multitissue gene \\nregulation in humans. Science. 2015;348(6235):648–660.\\n34. Welter D, MacArthur J, Morales J, et al. The NHGRI GWAS Catalog, \\na curated resource of SNP-trait associations. Nucleic Acids Res. \\n2014;42(D1):D1001–D1006.\\n35. MacArthur J, Bowler E, Cerezo M, et al. The new NHGRI-EBI Catalog \\nof published genome-wide association studies (GWAS Catalog). \\nNucleic Acids Res. 2017;45(D1):D896–D901.\\n36. Brown EG, Wood L, Wood S. The medical dictionary for regulatory \\nactivities (MedDRA). Drug Saf. 1999;20(2):109–117.\\n37. Liberzon A, Subramanian A, Pinchback R, et al. Molecular signa\\xad\\ntures \\ndatabase \\n(MSigDB) \\n3.0. \\nBioinformatics. \\n2011;27(12): \\n1739–1740.\\n38. Maglott D. Entrez gene: gene-centered information at NCBI. Nucleic \\nAcids Res. 2004;33:D54–D58.\\n39. Maglott D, Ostell J, Pruitt KD, et al. Entrez gene: gene-centered \\ninformation at NCBI. Nucleic Acids Res. 2011;39(Database):D52– \\nD57.\\n40. Hewett M. PharmGKB: the pharmacogenetics knowledge base. \\nNucleic Acids Res. 2002;30(1):163–165.\\n41. Cheng F, Desai RJ, Handy DE, et al. Network-based approach to \\nprediction and population-based validation of in silico drug \\nrepurposing. Nat Commun. 2018;9(1):2691.\\n42. Joshi-Tope G. Reactome: a knowledgebase of biological pathways. \\nNucleic Acids Res. 2004;33:D428–D432.\\n43. Fabregat A, Jupe S, Matthews L, et al. The reactome pathway \\nknowledgebase. Nucleic Acids Res. 2018;46(D1):D649–D655.\\n44. Kuhn M, Letunic I, Jensen LJ, et al. The SIDER database of drugs and \\nside effects. Nucleic Acids Res. 2016;44(D1):D1075–D1079.\\n45. Snel B. STRING: a web-server to retrieve and display the repeatedly \\noccurring neighbourhood of a gene. Nucleic Acids Res. 2000;28 \\n(18):3442–3444.\\n46. Szklarczyk D, Gable AL, Nastou KC, et al. The STRING database in \\n2021: customizable protein–protein networks, and functional char\\xad\\nacterization of user-uploaded gene/measurement sets. Nucleic \\nAcids Res. 2021;49(D1):D605–D612.\\n47. Chen X. TTD: therapeutic target database. Nucleic Acids Res. \\n2002;30(1):412–415.\\n48. Degtyarenko K, de Matos P, Ennis M, et al. ChEBI: a database and \\nontology for chemical entities of biological interest. Nucleic Acids \\nRes. 2007;36(Database):D344–D350.\\n49. Ross JS, Mulvey GK, Hines EM, et al. Trial publication after registra\\xad\\ntion in clinicalTrials.gov: a cross-sectional analysis. PLoS Med. \\n2009;6(9):e1000144.\\n50. Zarin DA, Tse T, Williams RJ, et al. Trial reporting in ClinicalTrials. \\ngov — the final rule. N Engl J Med. 2016;375(20):1998–2004.\\n51. Wang Y, Zhang S, Li F, et al. Therapeutic target database 2020: \\nenriched resource for facilitating research and early development \\nof targeted therapeutics. Nucleic Acids Res. 2019;gkz981. 10.1093/ \\nnar/gkz981.\\n52. Menche J, Sharma A, Kitsak M, et al. Uncovering disease-disease \\nrelationships through the incomplete interactome. Science. \\n2015;347(6224):1257601.\\n53. Adams D, Altucci L, Antonarakis SE, et al. BLUEPRINT to decode the \\nepigenetic signature written in blood. Nat Biotechnol. 2012;30 \\n(3):224–226.\\n54. Tsherniak A, Vazquez F, Montgomery PG, et al. Defining a cancer \\ndependency map. Cell. 2017;170(3):564–576.e16.\\n55. Sudlow C, Gallacher J, Allen N, et al. UK biobank: an open access \\nresource for identifying the causes of a wide range of complex \\ndiseases of middle and old age. PLOS Med. 2015;12(3):e1001779.\\n56. Bycroft C, Freeman C, Petkova D, et al. The UK biobank resource \\nwith deep phenotyping and genomic data. Nature. 2018;562 \\n(7726):203–209.\\n57. Edgar R. Gene expression omnibus: NCBI gene expression and \\nhybridization array data repository. Nucleic Acids Res. 2002;30 \\n(1):207–210.\\nEXPERT OPINION ON DRUG DISCOVERY\\n823\\n\\n58. Barabási A-L, Gulbahce N, Loscalzo J. Network medicine: a \\nnetwork-based approach to human disease. Nat Rev Genet. \\n2011;12(1):56–68. \\n•• A key introduction to networks in Medicine.\\n59. Lee LY-H, Loscalzo J. Network medicine in pathobiology. Am \\nJ Pathol. 2019;189(7):1311–1326. \\n•• A key introduction to networks in Medicine.\\n60. Gaudelet T, Day B, Jamasb AR, et al. Utilizing graph machine \\nlearning within drug discovery and development. Brief Bioinform. \\n2021;22(6):bbab159. \\n•• A comprehensive review of the application of graph machine- \\nlearning in drug discovery and development.\\n61. Cowen L, Ideker T, Raphael BJ, et al. Network propagation: a universal \\namplifier of genetic associations. Nat Rev Genet. 2017;18(9):551–562.\\n62. Sun M, Zhao S, Gilvary C, et al. Graph convolutional networks for \\ncomputational drug development and discovery. Brief Bioinform. \\n2021;21(3):919–935.\\n63. Jensen LJ, Saric J, Bork P. Literature mining for the biologist: from \\ninformation retrieval to biological discovery. Nat Rev Genet. 2006;7 \\n(2):119–129.\\n64. Vergetis V, Liaropoulos G, and Georganaki M, et al. A machine \\nlearning approach for assessing drug development risk. bioRxiv. \\n2020.\\n65. Hughes J, Rees S, Kalindjian S, et al. Principles of early drug dis\\xad\\ncovery: principles of early drug discovery. BrJ Pharmacol. 2011;162 \\n(6):1239–1249.\\n66. Ashburn TT, Thor KB. Drug repositioning: identifying and developing \\nnew uses for existing drugs. Nat Rev Drug Discov. 2004;3(8):673–683. \\n• A key introduction to the concept of drug repurposing.\\n67. Nelson MR, Tipney H, Painter JL, et al. The support of human \\ngenetic evidence for approved drug indications. Nat Genet. \\n2015;47(8):856–860.\\n68. Pushpakom S, Iorio F, Eyers PA, et al., Drug repurposing: progress, \\nchallenges and recommendations. Nat Rev Drug Discov. 2019;18 \\n(1):41–58. \\n• A review of systematic drug repurposing approaches.\\n69. Fiscon G, Conte F, Farina L, et al. SAveRUNNER: a network-based \\nalgorithm for drug repurposing and its application to COVID-19. \\nPLOS Comput Biol. 2021;17(2):e1008686.\\n70. Zeng X, Zhu S, Liu X, et al. deepDR: a network-based deep learning \\napproach to in silico drug repositioning. Bioinformatics. 2019;35 \\n(24):5191–5198.\\n71. Steinberg D, Horwitz G, Zohar D. Building a business model in \\ndigital medicine. Nat Biotechnol. 2015;33(9):910–920.\\n72. Shannon P, Markiel A, Ozier O, et al. Cytoscape: a software envir\\xad\\nonment for integrated models of biomolecular interaction \\nnetworks. Genome Res. 2003;13(11):2498–2504.\\n73. Soret P, Le Dantec C, Desvaux E, et al. A new molecular classifica\\xad\\ntion to drive precision treatment strategies in primary Sjögren’s \\nsyndrome. Nat Commun. 2021;12(1):3523.\\n74. Tzotzos SJ, Fischer B, Fischer H, et al. Incidence of ARDS and out\\xad\\ncomes in hospitalized patients with COVID-19: a global literature \\nsurvey. Crit Care. 2020;24(1):516.\\n75. Desvaux E, Hamon A, and Hubert S, et al. Network-based \\nrepurposing identifies anti-alarmins as drug candidates to con\\xad\\ntrol severe lung inflammation in COVID-19. PLOS ONE. 2021;16 \\n(7):e0254374. \\n• Application of the Patrimony platform to COVID-19.\\n76. Cheng F, Kovács IA, Barabási A-L. Network-based prediction of drug \\ncombinations. Nat Commun. 2019;10(1):1197.\\n77. Desvaux E, Aussy A, and Hubert S, et al. Model-based computa\\xad\\ntional precision medicine to develop combination therapies for \\nautoimmune diseases. Expert Rev Clin Immunol. 2021;18 \\n(1):47–56.\\n78. Rolland T, Taşan M, Charloteaux B, et al. A proteome-scale map of \\nthe human interactome network. Cell. 2014;159(5):1212–1226.\\n79. Jayatunga MKP, Xie W, Ruder L, et al. AI in small-molecule drug \\ndiscovery: a coming wave? Nat Rev Drug Discov. 2022;21 \\n(3):175–176.\\n80. Aittokallio T. What are the current challenges for machine learning \\nin drug discovery and repurposing? Expert Opin Drug Discov. \\n2022;17(5):423–425. \\n•• Introducing some current challenges for AI/MAL in drug \\ndiscovery.\\n81. Needham CJ, Bradford JR, Bulpitt AJ, et al. Inference in Bayesian \\nnetworks. Nat Biotechnol. 2006;24(1):51–53.\\n82. Luo Y, Peng J, and Ma J. When causal inference meets deep \\nlearning. Nat Mach Intell. 2020;2(8):426–427.\\n83. Heath JR, Ribas A, Mischel PS. Single-cell analysis tools for drug dis\\xad\\ncovery and development. Nat Rev Drug Discov. 2016;15(3):204–216.\\n84. Jumper J, Evans R, Pritzel A, et al. Highly accurate protein structure \\nprediction with AlphaFold. Nature. 2021;596(7873):583–589.\\n85. Liang Y, Kelemen A. Dynamic modeling and network approaches \\nfor omics time course data: overview of computational approaches \\nand applications. Brief Bioinform. 2018;19(5):1051–1068.\\n86. Selevsek N, Caiment F, Nudischer R, et al. Network integration and \\nmodelling of dynamic drug responses at multi-omics levels. \\nCommun Biol. 2020;3(1):573.\\n824\\nM. GUEDJ ET AL.\\n',\n",
       " \"See discussions, stats, and author profiles for this publication at: https://www.researchgate.net/publication/380731266\\nA COMPREHENSIVE STUDY ON AI-POWERED DRUG DISCOVERY: RAPID\\nDEVELOPMENT OF PHARMACEUTICAL RESEARCH\\nArticle\\xa0\\xa0in\\xa0\\xa0Journal of Emerging Technologies and Innovative Research · February 2021\\nCITATIONS\\n0\\nREADS\\n111\\n1 author:\\nVenkateswaranaidu Kolluri\\nUniversity of Mary Hardin–Baylor\\n24 PUBLICATIONS\\xa0\\xa0\\xa023 CITATIONS\\xa0\\xa0\\xa0\\nSEE PROFILE\\nAll content following this page was uploaded by Venkateswaranaidu Kolluri on 21 May 2024.\\nThe user has requested enhancement of the downloaded file.\\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n191 \\n \\nA COMPREHENSIVE STUDY ON AI-POWERED DRUG \\nDISCOVERY: RAPID DEVELOPMENT OF \\nPHARMACEUTICAL RESEARCH \\n \\nVENKATESWARANAIDU KOLLURI \\nSr. Software Engineer, Department of Information Technology \\n \\nABSTRACT—The aim of this paper is to provide an overview of \\nAI-assisted drug discovery, which involves AI technologies \\nenabling the quick development of pharmaceutical research. The \\ndemand for unique therapeutics and increasing disease \\ncomplexity leads conventional drug discovery methods to \\nlamentable difficulties, which include time, cost, and inefficiency \\n[1]. To counter the problem, human assisted technologies show \\nproof-of-concept methods based on machine learning algorithms, \\ndata analytics, and computational modeling to vet drug candidates \\nfaster, and optimize for treatment [1]. The study analyzes the \\nproblem at the juncture of AI and drug discoveries, conducts a \\ncomprehensive literature review to review the current situation, \\nand emphasizes the significance and advantages of utilizing AI in \\ndrug designing for the nation. Moreover, the article illustrates the \\nfuture perspectives and predictions of AI for pharmaceutical \\nstudies and for healthcare innovation in the U.S.With AI \\nalgorithms getting more and more complex, sensitive, and \\nautonomous, the issues connected with data privacy, patient \\nconsent, and the algorithm bias are arising. Implementing ethical \\nprinciples and regulatory frameworks are of crucial importance \\nto ensure the responsible use of AI in pharmaceutical research, as \\nwell as to maintain protection of patient rights and safety \\nthroughout the drug discovery process. \\nKeywords— AI, artificial intelligence, drug discovery, \\npharmaceutical research, machine learning, \\ncomputational modeling, healthcare innovation \\nI. \\nINTRODUCTION \\nPharmaceutical research and development are redefining \\ntheir landscape by digitizing biomedical research with AI \\ntechnologies. Conventional drug-finding strategies, consisting \\nof high expenditure, sloping timelines and low effectiveness, \\nare being substituted by AI-powered techniques that utilize \\nmachine-learning algorithms, data-analytics and computational \\nmodeling to accelerate the identification of novel drugs. This \\npaper will provide a view of AI-based drug discovery which in \\naddition to revolutionizing the pharmaceutical industry will \\ndramatically hasten treatment development. Innovation, by its \\nnature, is altering demand for new therapies to meet the trade-\\noffs in decision-making regarding patients´ wellbeing as well \\nas the complexity of the disease mechanisms to make possible \\nthe work of experts in developing alternatives. \\nAI is a fast-emerging field which enables the scientists to \\nprocess large volumes of biomedical data and predicts drug \\ninteractions as well as detecting the drug candidates with more \\npower and sheer accuracy than at any time in history. \\nLeveraging the power of AI  [2], pharmaceutical companies \\nwill be able to speed up the drug discovery process, lower the \\ncosts and take the new medicines to the market faster by doing \\nthis, patients and the society as a whole will benefit. At the core \\nof AI pharmaceutical research is the potential to extract \\nmeaning from extensive biological artifacts and convert them \\ninto practical understanding. Machine learning models, trained \\non broad datasets combining genomics, proteomics and clinical \\ndata, have the potential to hide patterns, biomarkers and lead to \\ntherapeutic developments that not all traditional analytical \\nmethods can achieve. Such an experience-based approach does \\nnot only shorten the period of finding out promising therapeutic \\ncandidates but also allows the development of individualized \\ntreatment plans based on the profiles of the patients as a whole, \\nthus transforming the field of medicine, namely personalized \\nmedicine [3]. \\nWhile the use of AI in drug discovery presents numerous \\nopportunities, its adoption also comes with challenges. \\nProblems, like data variability, algorithm stability and model \\nunderstandability should be supported; otherwise, we cannot \\ntrust AI outputs and their replicability. Furthermore, the ethical \\nquestions of data privacy, patient consent, and algorithm bias \\nshould be dealt with with caution in order to prevent \\nunanticipated side effects and adhere to ethical standards in \\npharmaceutical research [4]. Despite these issues, there is no \\ndoubt that AI, through its transformation capability, is one of \\nthe biggest drivers of innovation, collaboration, and study \\namong researchers in the field of drug discovery and human \\nhealth. \\nIn this essay, we will explore the research problem that lies \\nat the point where AI and pharmaceutical research meet, \\nsummarize the current practices in this area by reviewing the \\nliterature, and make an argument for the advantages of machine \\nlearning in drug discovery in the United States. Furthermore, \\nwe will look into the prospects and implications that AI could \\nhold for the U.S. pharmaceutical industry development and \\ninnovation in healthcare that highlights that investments, \\npolicies, and collaboration come in handy to tap the real \\npotential of AI in shaping the future of medicine. \\nII. \\nRESEARCH PROBLEM \\nThe primary research problem addressed in this study ison \\nexploring  the importance of revolutionizing drug discovery by \\nintegrating \\nartificial \\nintelligence \\n(AI) \\ntechnology \\ncomprehensively Modern drug discovery methods are \\nhandicapped by different challenges like high costs, long \\ndevelopment timelines, and low success rates, which make it an \\nindescribably demanding and resource-consumption process of \\ndiscovering and producing new drugs. The AI application in \\ndrug discovery is built on the fact that this can speed up the \\nprocess of discovery of novel drug candidates and of treatment \\nstrategies optimisation and can also lead to better patient \\noutcomes. Earth, the only place in the universe where we have \\nextraterrestrial intelligence contact. Firstly, there is a necessity \\nto understand how AI-based techniques will fill in the gaps and \\nenhance the current drug discovery approaches. This process \\ninvolves exploration of the abilities and inadequacies of \\nartificial intelligence algorithms in deciphering complex \\nbiological data, predicting drug-target interactions, and sorting \\nout potential drug candidates that deserve further investigation  \\n[5]. Furthermore, the technical issues of data accuracy, \\nalgorithm reliability, and model comprehensibility must be \\naddressed to maintain the authenticity of the AI-driven results \\nresearch. This will help in understanding ethical and regulatory \\nmatters which arise from augmentation of AI in drug discovery \\nprocesses. \\n \\n \\n \\n \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n192 \\n \\nIII. \\nLITERATURE REVIEW \\nA. AI IN DRUG DISCOVERY \\nThe emergence of AI in drug discovery is an underlying \\nprocess full of revolutionary progresses which are enabled by \\ninnovative computational procedures and data-oriented \\ntechniques. At the beginning, AI-based solutions were \\nrestricted to the rule-based systems and semi-intuitive \\nalgorithms, which rendered only limited data but were unable \\nto solve the complexity of biological systems. Nonetheless, \\nadvances in the area of machine learning and deep learning \\ndriven by the availability of the vast biological databases and \\ncomputational resources, made AI to be revolutionary in the \\nprocess of drug discovery [6]. Machine learning (ML) \\nalgorithms have been widely used for analyzing various types \\nof biological data which are represented by genomics, \\nproteomics and chemical compounds. For example, support \\nvector machines (SVMs), random forests, and neural networks \\nare often applied to accomplish this purpose  [7]. These \\nalgorithms can discriminate highly sophisticated connections \\nand correlations all around large databases; therefore, drug-\\ntarget interactions prediction, leads optimization and the \\nsuccessful medical treatment can be achieved by using these \\nalgorithms. Besides this, the birth of the deep learning \\narchitectures including convolutional neural networks (CNNs) \\nand recurrent neural networks (RNNs) have changed the way \\ndata analysis is done by offering machine-learning algorithms \\nthat can be used to extract features from the raw data and \\nprovide more accurate predictions and insights  [7]. \\nThe past several years have observed a coming together of \\ncomputational as well as experimental approaches where AI \\nalgorithms \\ndetermine \\nthe \\nexperimental \\ndesign, \\ntarget \\nvalidation, and lead optimization. Artificial intelligence in the \\ndrug discovery process has led to accelerated identification of \\nnovel drug candidates, reduction in the cost of discovery and \\nenhanced the success rate of clinical trials. Another important \\ncapability of AI is its ability to analyze collective biological \\ninteractions and predict drug response in the populations where \\ntreatment entirely depends on individual genetic profiles and \\ndisease characteristics. \\n \\n \\n [5].\\n \\nFig. 1  A bar graph showing the distribution of various AI \\ntechniques in Drug Discovery \\nB. AI-POWERED DRUG TARGET IDENTIFICATION \\nAI-based drug target identification is leading-edge \\ntechnology for boosting drug discovery for different diseases. \\nThe use of these machine learning and data analytics tools \\nallows for the analysis in detail of massive volumes of \\nbiological information with the objective of identifying \\nmolecules, proteins, or pathways that could be affected by \\ndrugs in order to treat different diseases  [8]. One of the main \\nadvantages of AI in drug target identification is that it can sift \\nthrough large-scale -omics or genomic, transcriptomic, \\nproteomic, or metabolomic data - to discover useful \\nrelationships and patterns. Machine learning algorithms can be \\nemployed to analyze complex datasets in order to enhance the \\nsearch of genes, proteins or signaling pathways that are \\ndysregulated in disease states offering a clue for potential \\ntherapeutic \\ntargets \\n[9]. \\nFurther, \\nartificial \\nintelligence \\nalgorithms can ease the process of personnel drug targeting by \\nconsidering various multifaceted factors, such as their \\nbiological significance, the issue of druggability, and the \\nresulting unwanted effects. Through the integration of varied \\ndata sources and by applying advanced feature selection skills, \\nAI-based drug target identification platforms firstly seek out the \\nmost prominent candidates (according to the calculated for \\nsuccess$) with a consequent smoothing of the process of drug \\ndiscovery. Moreover, AI algorithms are not only capable of \\nhandling conventional biological data but also features such as \\nelectronic health records, patient demographics, and clinical \\ntrial data, to identify new pathways and predictive biomarkers \\nfor personalized medicine strategies. It provides for a unified \\nand holistic approach to therapy that assists in covering missed \\ntargets that can be obtained by traditional methods and deliver \\npersonalized treatments to the patients considering their distinct \\ngenetic makeup and other factors affecting their disease. \\n \\nFig. 2  AI in Drug Target Identification \\nC. AI-DRIVEN DRUG DESIGN AND OPTIMIZATION \\nArtificial intelligence (AI)-facilitated drug design and \\noptimization of new therapeutics is a game changer in the drug \\ndevelopment platform because of the higher efficacy and safety \\nthat they provide. This process takes advantage of the machine \\nlearning, in-silico modeling and structure-based screening \\napproaches to speed up the process of finding and optimizing \\ndrug candidates with an aim to decrease the size of the drug \\ndevelopment pipeline  [10,11]. The capability of AI to process \\nlarge amounts of chemical compounds and predict their \\nproperties such as activity, pharmacokinetics, and toxicity is \\none of importance. Machine learning algorithms are capable of \\nconforming to the existing drug data for finding patterns and \\nlinks among chemical structures and biological activities [12], \\nwhich in turn makes possible the emergence of new compound \\nstructures with desired properties. \\nAI-driven drug design utilizes model extensions such as \\nmolecular docking/ molecular dynamics simulations to predict \\nthe interactions between a drug and its target protein. Such \\nsimulations are useful for researchers to test the binding \\nstrength and the selectivity of the molecules that can be adapted \\nas drug candidates which then guide the researchers to select a \\nlead compound for candidates for further optimization [13]. \\nMoreover, drug candidates can be optimized by AI algorithms \\nwhich can \\niteratively generate and screen chemical \\nmodifications to improve their potency, selectivity, and \\npharmacokinetic properties. The most time-consuming and \\nlabor-demanding step in the lead optimization process is \\nexecuted with conventional methods; however, it can be \\nsignificantly improved using AI-driven ones. \\n \\n \\n \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n193 \\n \\nD. AI-ASSISTED DRUG REPURPOSING AND \\nCOMBINATION THERAPY \\nAI-driven repurposing and synergistic drug combination \\ntechniques represent innovative methods for drug development \\nthat aim to take advantage of existing drugs and to address \\nunmet medical needs more effectively  [14]. Drug repurposing, \\nalso referred to as drug repositioning, is the process of \\ndiscovering new therapeutic uses for pre-existing drugs with \\nindications other than those they were originally designed for. \\nAI algorithms have proved that they significantly contribute to \\nthe drug discovery process through analyzing large-scale \\nbiomedical data, such as electronic health records and drug \\ndatabases from where relationships between drugs, diseases, \\nand \\nbiological \\npathways \\ncan \\nbe \\nextracted \\n[15,16]. \\nImplementing machine learning methodologies, such as those \\nbased on networks and similarity, can enable AI to quickly \\ndetermine potential candidates for repurposing based on their \\ncurrent known pharmacological profiles and suitability for the \\ntreatment of different diseases. On the other hand, AI-powered \\ndrug repurposing tools can rank drug candidates in accordance \\nwith different factors like their safety profiles, the way they act \\nand their probability to be successful in clinical trials. The \\nrepurposing of existing drugs not only bypasses most of the \\ntime-consuming and expensive steps in the traditional drug \\ndevelopment process, it also speeds up the time to market a new \\nmedication. \\n \\n \\n \\nFig. 1 A heatmap on AI-Assisted Drug Repurposing and \\nCombination Therapy \\nIV. \\nSIGNIFICANCE AND BENEFITS TO THE \\nU.S \\nThe AI-driven drug invention and development are highly \\nvaluable to the United States and have wide ranging significant \\nbenefits. First of all AI in pharmaceutical research can bring a \\ncomplete rethinking of the drug discovery process that is going \\nto help develop new designer medicines addressing a broad \\nrange of diseases  [17]. AI technologies provide efficiency with \\nstreamlining of the drug development system, thus, cutting the \\ntime and cost that go to the market bringing new drugs, which \\nat the end give patients a wider reach to treatment options. \\nAdditionally, artificial intelligence (AI)-driven drug discovery \\nimplies a potential of improving the positions of the U.S. \\npharmaceutical companies in a global market. Through \\nsupporting innovation and nurturing of partnerships among \\nacademic institutions, industry, and regulatory bodies, the US \\nwill stay on a path of successful biomedical research and \\ndevelopment. This thereby boosts economic growth, creates \\nhigh-paying jobs, and draws the life sciences industry players \\ninto the investments  [18]. Moreover, AI in drug discovery not \\nonly poses solutions to the most crucial challenges in the United \\nStates healthcare such as drug-resistant pathogens, opioid \\nepidemic, but also tobacco-related diseases [19,20]. The U.S. \\nwill be able to enhance patient outcomes, decrease healthcare \\nexpenditures, and offset the burden on the healthcare system by \\nusing AI to discover new therapeutic targets, repurpose existing \\ndrugs, and optimize treatment regimens. \\nV. \\nFUTURE IN THE U.S \\nAI-driven drug development and discovery in the future of \\nUS healthcare will create a tremendous opportunity for medical \\nresearch advancement, health outcomes improvement, and \\neconomic growth. As AI technologies gradually evolve and \\nprogress, their impact on pharmaceutical innovation in the U.S. \\nis also expected to grow in importance. One of the major \\naspects of AI-powered drug discovery's future relates to the \\nprogressive assimilation of AI into pharmaceutical firms' R&D \\npipelines. AI is becoming more attractive to pharmaceutical \\ncompanies to reduce the burden of drug discovery, to accelerate \\ndrug discovery, and to improve clinical trial design [21]. This \\ntrend is expected to extend with AI becoming an irreplaceable \\ninstrument for improving productivity, decreasing costs, and \\nincreasing the rate of drug development programs' success. \\nAdditionally, AI in the future will not work in just one sector \\nof the life sciences but it will be used in the different sectors \\nwith more collaboration and convergence. AI technologies find \\ntheir application in academic institutions, biotech startups, \\npharmaceutical companies, and government agencies as they \\nare becoming partners to share data and resources. Such joint \\nefforts are likely to push for innovation, emphasize \\ninterdisciplinary research, and speed up the transition from \\nscientific discoveries to clinical application. \\nVI. \\nCONCLUSION \\nThis paper has offered a detailed analysis on how AI is \\ntransforming the pharmaceutical landscape and offering better \\nsolutions to patients' healthcare issues. By focusing on the study \\nof this issue, a literature review, and a discussion of the \\nrelevance of this topic to the US, we see that AI technology has \\nmany advantages as a means of speeding up drug development, \\nminimizing costs, and achieving personalized medicine. The \\nmost important role of AI-based drug discovery in the USA can \\nbe viewed as the contributor to the innovation, the process of \\neconomic growth, and the way to deal with impressive health \\nissues. AI technologies can serve as a game changer in terms of \\nmaintaining the U.S.'s front line position with respect to \\nbiomedical research and development thus allowing attainment \\nof such great ideas like precision medicine, drug repurposing \\nand combination therapy. In summary on the topic, the future \\nof AI- supported drug development in the United States is \\npromising with new AI algorithms together with both data and \\ncomputational resources being the future of the industry. Using \\nthese technologies and making collaboration between \\nacademia, industry and government possible, the U.S. can \\ncontinue to make great strides and improve the lives of a lot of \\npatients across the globe. \\n \\n \\nREFERENCES \\n[1] \\nN. Brown. Artificial Intelligence in Drug Discovery. Royal Society of \\nChemistry, 2020. \\n[2] \\nJ. Cassidy and B. Taylor, Artificial Intelligence in Oncology Drug \\nDiscovery and Development. BoD – Books on Demand, 2020. \\n[3] \\nM. Chang, Artificial Intelligence for Drug Development, Precision \\nMedicine, and Healthcare. CRC Press, 2020. \\n[4] \\nM. Chang, AI for Drug Development and Well-being. 2020. \\n[5] \\nF. Lake, “Artificial intelligence in drug discovery: what is new, and what \\nis next?,” Future Drug Discovery, vol. 1, no. 2, p. FDD19, Oct. 2019, \\ndoi: https://doi.org/10.4155/fdd-2019-0025 \\n[6] \\nA. Varun, “Artificial Intelligence (AI) in Drug Discovery and \\nMedicine,” Journal of Clinical Cases & Reports, vol. 2, no. 3, pp. 76–\\n80, Jul. 2019, doi: https://doi.org/10.46619/joccr.2019.2-1043 \\n\\n© 2021 JETIR February 2021, Volume 8, Issue 2                                                         www.jetir.org (ISSN-2349-5162) \\nJETIR2102324 \\nJournal of Emerging Technologies and Innovative Research (JETIR) www.jetir.org \\n194 \\n \\n[7] \\nR. Lawrence, “Now the future, we see our dreams: artificial intelligence \\nin drug discovery,” Future Drug Discovery, vol. 1, no. 2, p. FDD22, Oct. \\n2019, doi: https://doi.org/10.4155/fdd-2019-0027 \\n[8] \\nK.-K. Mak and M. R. Pichika, “Artificial intelligence in drug \\ndevelopment: present status and future prospects,” Drug Discovery \\nToday, \\nvol. \\n24, \\nno. \\n3, \\npp. \\n773–780, \\nMar. \\n2019, \\ndoi: \\nhttps://doi.org/10.1016/j.drudis.2018.11.014. \\nAvailable: \\nhttps://www.sciencedirect.com/science/article/abs/pii/S1359644618300\\n916 \\n[9] \\nP. Agrawal, “Artificial Intelligence in Drug Discovery and \\nDevelopment,” Journal of Pharmacovigilance, vol. 06, no. 02, 2018, doi: \\nhttps://doi.org/10.4172/2329-6887.1000e173 \\n[10] A. Laghaee, C. Malcolm, J. Hallam, and P. Ghazal, “Artificial \\nintelligence and robotics in high throughput post-genomics,” Drug \\nDiscovery Today, vol. 10, no. 18, pp. 1253–1259, Sep. 2005, doi: \\nhttps://doi.org/10.1016/s1359-6446(05)03581-6 \\n[11] A. Bohr and K. Memarzadeh, Artificial Intelligence in Healthcare. \\nAcademic Press, 2020. \\n[12] R. Jagannathan, “AI and drug discovery : Search for new psychoactive \\ncandidates \\nfrom \\ncannabis \\nsativa,” \\nSep. \\n2020, \\ndoi: \\nhttps://doi.org/10.1021/scimeetings.0c00478 \\n[13] J. T. Chang and R. B. Altman, “Promises of text processing: natural \\nlanguage processing meets AI,” Drug Discovery Today, vol. 7, no. 19, \\npp. \\n992–993, \\nOct. \\n2002, \\ndoi: \\nhttps://doi.org/10.1016/s1359-\\n6446(02)02457-1 \\n[14] J. Kirchmair, “Editorial for the Special Section ‘Artificial Intelligence in \\nDrug Discovery,’” Drug Discovery Today: Technologies, Dec. 2020, \\ndoi: https://doi.org/10.1016/j.ddtec.2020.12.001 \\n[15] E. Smalley, “AI-powered drug discovery captures pharma interest.” \\nNature Biotechnology, vol. 35, no. 7, pp. 604-605, 2017, doi: \\n10.1038/nbt0717-604. \\n[16] M. Hervey, “Harnessing AI in drug discovery without losing patent \\nprotection.” Drug Discovery Today, vol. 25, no. 6, pp. 949-950, 2020, \\ndoi: 10.1016/j.drudis.2020.03.007. \\n[17] A. Varun, “Artificial Intelligence (AI) in Drug Discovery and \\nMedicine.” Journal of Clinical Cases & Reports, vol. 2, no. 3, pp. 76-80, \\n2019, doi: 10.46619/joccr.2019.2-1043. \\n[18] K. Linton-Reid, “Introduction: An Overview of AI in Oncology Drug \\nDiscovery and Development.” Artificial Intelligence in Oncology Drug \\nDiscovery and Development, 2020, doi: 10.5772/intechopen.92799. \\n[19] R. Dousa, “Toward the Clinic: Understanding Patient Perspectives on \\nAI and Data-Sharing for AI-Driven Oncology Drug Development.” \\nArtificial Intelligence in Oncology Drug Discovery and Development, \\n2020, doi: 10.5772/intechopen.92787. \\n[20] M. Hervey, “Harnessing AI in drug discovery without losing patent \\nprotection.” Drug Discovery Today, vol. 25, no. 6, pp. 949-950, 2020, \\ndoi: 10.1016/j.drudis.2020.03.007. \\n[21] N. Ai, M. D. Krasowski, W. J. Welsh, and S. Ekins, “Understanding \\nnuclear receptors using computational methods.” Drug Discovery \\nToday, \\nvol. \\n14, \\nno. \\n9, \\npp. \\n486-494, \\n2009, \\ndoi: \\n10.1016/j.drudis.2009.03.003. \\n \\n \\n \\nView publication stats\\n\",\n",
       " \"Opinion\\nAI-powered therapeutic target discovery\\nFrank W. Pun,1 Ivan V. Ozerov,1 and Alex Zhavoronkov\\n1,2,3,*\\nDisease modeling and target identiﬁcation are the most crucial initial steps in\\ndrug discovery, and inﬂuence the probability of success at every step of drug\\ndevelopment. Traditional target identiﬁcation is a time-consuming process\\nthat takes years to decades and usually starts in an academic setting.\\nGiven its advantages of analyzing large datasets and intricate biological net-\\nworks, artiﬁcial intelligence (AI) is playing a growing role in modern drug tar-\\nget identiﬁcation. We review recent advances in target discovery, focusing on\\nbreakthroughs in AI-driven therapeutic target exploration. We also discuss\\nthe importance of striking a balance between novelty and conﬁdence in\\ntarget selection. An increasing number of AI-identiﬁed targets are being vali-\\ndated through experiments and several AI-derived drugs are entering clinical\\ntrials; we highlight current limitations and potential pathways for moving\\nforward.\\nOverview of target identiﬁcation\\nThe drug discovery pipeline is widely recognized to be a time-consuming, expensive, and\\nrisk-laden process that typically requires around 10 years and $2 billion to bring a novel\\ndrug to market [1]. By 2022 fewer than 500 successful drug targets had been identiﬁed\\n[2], representing a tiny fraction of the estimated druggable targets in humans [3,4]. Al-\\nthough numerous drug candidates undergo extensive optimization during preclinical\\nstages, the average failure rate in clinical trials from 2009 to 2018 reached 84.6%i. The\\nlack of clinical efﬁcacy remains the key factor contributing to the failure of both Phase 2\\nand 3 trials [5], leading to substantial ﬁnancial losses and resource wastage. Identifying\\nthe right drug targets is crucial for increasing the likelihood of developing clinically effective\\ntherapies.\\nTarget identiﬁcation, the process of identifying the right biological molecules or cellular path-\\nways that can be modulated by drugs to achieve therapeutic beneﬁts, is increasingly important\\nin modern drug discovery. Although innovations in experimental and omic technologies have\\nbeen growing over the past few decades (Figure 1), identifying actionable therapeutic targets\\nremains challenging. The integration of multiomic data with AI (see Glossary) algorithms has re-\\ncently emerged as a promising approach for target identiﬁcationii,iii. We discuss here the con-\\nventional target identiﬁcation approaches with a focus on the application of AI algorithms to\\ntarget identiﬁcation. This paper aims to offer a progressive outlook on the emergence of the\\nAI-driven drug discovery era and encourage the integration of AI technologies into drug discov-\\nery pipelines.\\nStrategies in target identiﬁcation: from experiments to machine learning\\nTarget identiﬁcation can be classiﬁed into three distinct strategies – experimental, multiomic, and\\ncomputational approaches (Figure 2). Using these methods collaboratively can generate novel\\ntherapeutic hypotheses in exploratory target identiﬁcation, thus signiﬁcantly enhancing our under-\\nstanding of complex diseases.\\nHighlights\\nDisease modeling and target discovery\\nare crucial initial steps in the drug discov-\\nery process and signiﬁcantly impact on\\nthe success of drug development.\\nGiven the advantages of analyzing large\\ndatasets and complex biological net-\\nworks, artiﬁcial intelligence (AI) is playing\\na growing role in modern drug target\\nidentiﬁcation.\\nWe discuss the use of deep learning\\nmodels\\nfor\\ntarget\\ndiscovery,\\nAI-\\nidentiﬁed targets validated through ex-\\nperiments, and the use of synthetic\\ndata produced using generative AI for\\ntarget identiﬁcation.\\nNovelty, in addition to druggability and\\ntoxicity, is a crucial factor in target selec-\\ntion. There is a trade-off between choos-\\ning high-conﬁdence and novel targets.\\nOver the past few years several AI-\\nderived drugs have entered clinical trials,\\nsignaling the dawn of a new era in AI-\\ndriven drug discovery.\\n1Insilico Medicine Hong Kong Ltd., Hong\\nKong Science and Technology Park,\\nNew Territories, Hong Kong\\n2Insilico Medicine MENA, 6F IRENA\\nBuilding, Abu Dhabi, United Arab Emirates\\n3Buck Institute for Research on Aging,\\nNovato, CA, USA\\n*Correspondence:\\nalex@insilico.com (A. Zhavoronkov).\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\nhttps://doi.org/10.1016/j.tips.2023.06.010\\n561\\n© 2023 The Author(s). Published by Elsevier Ltd. This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).\\nTrends in\\nPharmacological Sciences\\nOPEN ACCESS\\n\\nExperimental approaches\\nExperimental approaches, including afﬁnity-based biochemical, comparative proﬁling, and\\nchemical/genetic screening, have demonstrated their striking contributions to target identiﬁcation\\nsince the 1960s. The use of small-molecule afﬁnity probes, which allow traceless protein labeling\\nupon ligand–protein interaction [6], is the most straightforward method among the three experi-\\nmental approaches. The selection of probes is highly dependent on the identity of the starting mol-\\necule [7]. Stable isotope labeling by amino acids in cell culture (SILAC), an example of comparative\\nproﬁling, is a popular quantitative proteomics tool that uses stable isotope-labeled amino acids to\\naccurately differentiate cellular proteomes [8]. Studies conducted in multiple cancer types such as\\nhepatocellular carcinoma (HCC) [9], multiple myeloma [10,11], endometrial cancer [12], and colo-\\nrectal cancer [13,14] have clearly exempliﬁed the effectiveness of SILAC in identifying pivotal\\nplayers in disease pathogenesis. Chemical/genetic screening, implemented by RNA interference\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 1. The emergence of artiﬁcial intelligence (AI) in early drug development. (Upper panel) Key technological\\nadvances in the history of target identiﬁcation are classiﬁed into three types: experiment-based (red), multiomic (blue), and\\ncomputational (green) approaches. Traditionally, experiment-based methods have been the go-to approach for discovering\\ntherapeutic targets. However, with the rise of big data, integrated analysis of multiomic data has become a more efﬁcient\\nstrategy for target identiﬁcation. In addition, recent advances in AI-driven biological analysis have identiﬁed novel targets and\\nAI-designed drugs are now entering clinical trials. (Lower panel) AI applications in the early stages of drug discovery.\\nAbbreviations: AGC chemistry, afﬁnity-guided catalyst chemistry; ALS, amyotrophic lateral sclerosis; DL, deep learning;\\nEGFR, epidermal growth factor receptor; GAN, generative adversarial network; GWAS, genome-wide association study; LD\\nchemistry, ligand-directed chemistry; MTOR, mammalian target of rapamycin; NSCLC, non-small cell lung cancer; SILAC,\\nstable isotope labeling with amino acids in cell culture; TID, target identiﬁcation. Figure created with BioRender.com.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n562\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\nGlossary\\nArtificial intelligence (AI): the ability of\\na computer or computer-controlled\\nmachine to perform problem-solving\\nand decision-making tasks that are\\ncommonly associated with intelligent\\nbeings.\\nBiomarker: a biological molecule in any\\ntype of body ﬂuid or tissue that serves as\\na sign of a biological state.\\nDrug repurposing: the process of\\nidentifying a novel therapeutic\\napplication for existing drugs that have\\nbeen FDA-approved or clinically\\ninvestigated for speciﬁc medical\\nindications.\\nDrug–target interaction: an important\\nstep in drug discovery that recognizes\\nhow a chemical compound and a\\nprotein target interact in the human\\nbody.\\nGenerative adversarial networks\\n(GANs): a class of machine learning\\nframeworks that consists of two neural\\nnetworks that compete against each\\nother during the training process and\\nimprove their functionalities to generate\\nsamples indistinguishable from the real\\ndata.\\nGenome-wide association study\\n(GWAS): a method to identify genomic\\nvariants that are statistically associated\\nwith a risk for a disease or a trait by\\ncomparing the frequencies of genomic\\nvariants between people with and\\nwithout that speciﬁc disease or trait.\\nIndication prioritization: the process\\nof prioritizing the potential indications of\\na drug based on the expected relevancy\\nof the drug and a speciﬁc indication\\nusing AI.\\nInduced pluripotent stem cells\\n(iPSCs): artiﬁcial stem cells generated\\nfrom an adult somatic cell through the\\ncoexpression of speciﬁc pluripotency-\\nassociated genes, namely c-Myc, Oct3/\\n4, Sox2, and Klf4.\\nMachine learning: a branch of artiﬁcial\\nintelligence that focuses on mimicking\\nhuman learning processes via the use of\\ndata and algorithms to gradually improve\\nits accuracy.\\nNatural language processing: a ﬁeld\\nof AI that processes and analyzes large\\namounts of natural language data with a\\ngoal to enable computers to understand,\\ninterpret, generate human language, and\\nextract information from documents.\\nPharmacokinetics: the study of the\\nfate of an administered substance in an\\norganism, namely absorption,\\ndistribution, metabolism, and excretion.\\n\\n(RNAi) or CRISPR-Cas9 gene editing, has been of great interest to biologists for decades. Owing to\\nits high speciﬁcity and efﬁciency [15], CRISPR has dramatically expanded our knowledge of the\\nmechanistic and pharmacological aspects of human diseases. For example, BRD2 was identiﬁed\\nas an essential regulator of the host response to SARS-CoV-2 infection by a targeted CRISPR in-\\nterference screen [16]. Making use of the CRISPR interference- and CRISPR activation-based\\nfunctional genomics platform, Ramkumar et al. identiﬁed the determining roles of HDAC7 and\\nthe Sec61 complex in modulating the immunotherapy response in multiple myeloma [17]. Although\\nit has been 10 years since its introduction, CRISPR technology continues to evolve to further en-\\nhance its ﬂexibility, simplicity, and efﬁciency, thus offering a great beneﬁt to the research community\\nnot only for target identiﬁcation but also as a gene therapy and diagnostic tool.\\nMultiomic approaches\\nMultiomic data provide researchers with interconnected molecular information from different per-\\nspectives, including static genomic data and spatiotemporally dynamic expression and metabolic\\nTarget identification\\nT\\nMultiomics\\nPromoter Target gene\\nMethyl \\ngroup\\nExon Intron Exon \\nGenomics\\nEpigenomics\\nProteomics\\nMetabolomics\\nTranscriptomics\\nComputational\\nStructure similarity\\nAI Machine learning\\nReverse docking\\nPharmacophore screening\\nHO\\nN\\nO\\nN\\nQuery\\nligand\\ndatabase\\nProtein\\nDocking\\nOH\\nN\\nO\\nH\\nO\\nHO\\nOH\\nH\\nOC\\nO\\nHO\\nH\\nO\\nOH\\nOH\\nHO\\nHO\\nO\\nH\\nH\\nH\\nH\\nH\\nO\\nO\\nOH\\nO\\nO\\nH\\nH\\nH\\nO\\nO\\nOH\\nOH\\nO\\nO\\nOH\\nOH\\nA\\nB\\nQuery ligand\\nLigand\\nSimilarity\\n#1\\n0.99\\n#2\\n0.87\\nPharmacophore \\nmodels\\nLigands\\nLigand-\\npharmacophore \\nmapping\\nExperimental\\nChemical-genetic\\nm/z\\nRelative \\nintensity\\n‘Heavy’ group\\n‘Light’ group\\nMix cell \\nlysates\\nLC-MS/\\nMS\\nStable-isotope labeling by \\namino acids in cell culture \\nCRISPR/Cas9\\nsiRNA\\nRISC\\nRNAi\\nLD Chemistry\\nAGC Chemistry\\nO\\nS\\nO\\nO\\nO\\nProbe\\nLDT reagent\\nO\\nN\\nN\\nLg\\nAGD \\ncatalyst\\nProbe\\nStructure-based\\nDeep learning\\nATP\\nA\\nHeat\\nCO2 Water\\nChemical \\nenergy\\nMetabolism\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 2. Three exploratory strategies for target identiﬁcation. Exploratory techniques for target identiﬁcation can be\\nclassiﬁed into three strategies: experimental, multiomic, and computational approaches. The experimental approach involves\\nconducting wet-lab experiments to identify targets based on afﬁnity, genetic modiﬁcation screening, and comparative\\nproﬁling. The multiomic approach predicts gene–disease associations by analyzing various omic datasets such as\\ngenomics, transcriptomics, proteomics, epigenomics, and metabolomics. Lastly, the computational discovery approach\\nefﬁciently identiﬁes potential targets by using machine learning or structure-based methods including reverse docking,\\npharmacophore screening, and structure similarity analysis. Abbreviations: AGC chemistry, afﬁnity-guided catalyst\\nchemistry; AGD, afﬁnity-guided DMAP (4-dimethylaminopyridine); AI, artiﬁcial intelligence; LC, liquid chromatography; LD\\nchemistry, ligand-directed chemistry; LDT, ligand-directed tosyl; MS, mass spectrometry; RISC, RNA-induced silencing\\ncomplex; RNAi, RNA interference; siRNA, short interfering RNA. Figure created with BioRender.com.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n563\\nRecurrent neural networks: a class\\nof artiﬁcial neural networks with\\nfeedback connections that are designed\\nto learn sequential or time-varying data.\\nTransfer learning: a machine learning\\nmethod where a pretrained model is\\nreused as the starting point for a model\\non another related task; this approach is\\ncommonly used as an optimization\\ntechnique to save time and increase\\nperformance.\\nTherapeutic modality: the type of\\ntherapy used to treat a disease or\\nmedical condition, including small-\\nmolecule drugs, protein-based\\ntherapies, advanced therapies (such as\\ncell and gene therapies), and\\nmicroorganism-based therapies.\\n\\nproﬁles [18]. As the ﬁrst established and most mature omics discipline [19], genomics focuses on\\ngenetic variants in the DNA sequence. Large-scale genome-wide association study (GWAS)\\nanalysis powered by next-generation sequencing has yielded hundreds of thousands of associ-\\nations between genetic variants and complex diseases or traits [20], leading to the development\\nof breakthrough therapies such as the cystic ﬁbrosis modulator drugs targeting CFTR mutations\\n[21], and novel drugs for the treatment of inﬂammatory bowel disease targeting the disease-\\nassociated gene IL23A [22]. More recently, meta-analyses of published GWAS data have re-\\nvealed novel genetic loci attributable to different diseases, thus opening up drug repurposing\\nopportunities [23,24]. Although genomic evidence has been one of the indispensable factors in\\ntarget identiﬁcation, distinguishing the causative genetic variants that lead to a given disease re-\\nmains challenging. In this regard, integrating multiple omic lines of evidence can be useful. Tran-\\nscriptomic and proteomic data can be used to identify causal genetic loci that regulate gene and\\nprotein levels and facilitate the discovery of genes and pathways underlying disease pathogenesis\\n[25–27]. Likewise, epigenomic and metabolomic data can also serve as functional evidence for\\nGWAS-identiﬁed variants to support their disease associations and clinical applications [28–30].\\nAs compared to single omic approaches, integrated multiomic analysis can provide a more com-\\nprehensive view of disease mechanisms and is therefore increasingly used to facilitate biomarker\\nand therapeutic target discoveries, treatment response, and patient prognosis predictions [31–34].\\nComputational approaches\\nBecause typical experiment-based target identiﬁcation is laborious and resource-intensive,\\ncomputational approaches have emerged as promising alternatives for achieving efﬁcient tar-\\nget screening. Depending on the availability of protein structure and the chemical structure of\\nthe compound of interest, pharmacophore screening [35], reverse docking [36], and structure\\nsimilarity assessment [37,38] have been used to predict novel biological targets for small mol-\\necules. On the other hand, AI is a growing discipline in computational science for target discov-\\nery. Machine learning is an indispensable component of AI that can be applied either with or\\nwithout supervision. Supervised learning utilizes labeled datasets to train models for data clas-\\nsiﬁcation and reliable outcome prediction. By contrast, unsupervised learning explores the hid-\\nden structure of unlabeled data without human intervention [39]. The application of machine\\nlearning is not limited to predicting biological targets of the existing drugs or compounds,\\nand can also identify novel therapeutic targets for any disease of interest. The details of how\\nmachine learning facilitates target discovery for disease treatment will be elaborated upon in\\nthe following AI sections.\\nAI-driven target identiﬁcation\\nIn recent years we have witnessed an explosion of biomedical data ranging from basic research\\non disease mechanisms to clinical investigation in patients. Although large amounts of information\\nhave been generated, the growth of data also poses challenges for data analysis. This is where\\nthe emerging role of AI comes into play. Given the advantage of AI in processing and tackling\\ncomplex biomedical networks of data, using AI algorithms can reveal patterns and relationships\\nwithin the data that may not be apparent to humans, and may possibly lead to better understand-\\ning and treatment of diseases. AI has made notable contributions that facilitate biomarker and tar-\\nget identiﬁcation [40–42], indication prioritization [43], drug-like molecule design [44,45],\\npharmacokinetics prediction [46], drug–target interaction [47,48], and clinical trial design\\n[49] (Figure 1, lower panel). Although still in the early stages of clinical trials, AI-derived drugs\\nare increasingly emerging in clinical studies (Table 1), as exempliﬁed by GS-0976 for the treatment\\nof non-alcoholic steatohepatitis, EXS-21546 for solid tumors, and INS018_055 for idiopathic pul-\\nmonary ﬁbrosis, which is the ﬁrst-ever AI-derived drug with positive topline results in a Phase 1\\nclinical trial.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n564\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nApplication of deep learning models in target discovery\\nIn recent years machine learning-based algorithms, particularly deep learning methodologies,\\nhave drawn signiﬁcant attention and have achieved excellent results in pharmaceutical areas.\\nDeep learning, also known as deep neural networks, consists of multiple hidden layers of\\nnodes through which data processing and feature extraction are conducted successively in a\\ncascade manner [50]. Compared to traditional machine learning methods, more recent deep\\nlearning-based architectures, such as generative adversarial networks (GANs), recurrent\\nTable 1. AI-derived drugs in clinical trials\\nCompany\\nTarget\\nIndicationa\\nCompound\\nDevelopment status\\nTrial numberb\\nBenevolentAI\\nTrk\\nAtopic dermatitis\\nBEN-2293\\nPhase 2\\nNCT04737304\\nExscientia\\nA2AR\\nSolid tumors\\nEXS-21546\\nPhase 1\\nNCT04727138\\n5-HT1A\\nObsessive compulsive\\ndisorder\\nDSP-1181\\nPhase 1\\nUndisclosedvi\\n5-HT1A/2A\\nAlzheimer's disease\\npsychosis\\nDSP-0038\\nPhase 1\\nUndisclosedvii\\nPKC-θ\\nInﬂammatory diseases\\nEXS4318\\nPhase 1/2\\nUndisclosedviii\\nInsilico Medicine\\nTarget X\\nIdiopathic pulmonary ﬁbrosis\\nINS018_055\\nPhase 2\\nNCT05938920,\\nCTR20230776\\n3CLPro\\nCOVID-19\\nISM3312\\nPhase 1\\nCTR20230768\\nUSP1\\nBRCA-mutant cancer\\nISM3091\\nPhase 1\\nNCT05932862\\nNimbus Therapeutics\\nACC\\nNonalcoholic steatohepatitis\\nNDI-010976/GS-0976\\nPhase 2\\nNCT02856555,\\nNCT03987074,\\nNCT02891408,\\nNCT02876796\\nPharos iBio\\nFLT3\\nAcute myeloid leukemia\\nOvarian cancer\\nTriple-negative breast cancer\\nRadiation sensitizer\\nPHI-101\\nPhase 1\\nNCT04842370\\nNCT04678102\\nRecursion\\nPharmaceuticals\\nCCM2\\nCerebral cavernous\\nmalformation\\nREC-994\\nPhase 2\\nNCT05085561\\nHDAC\\nNeuroﬁbromatosis type 2\\nREC-2282\\nPhase 2/3\\nNCT05130866\\nMEK1/2\\nFamilial adenomatous\\npolyposis\\nREC-4881\\nPhase 2\\nNCT05552755\\nRelay Therapeutics\\nSHP2\\nSolid tumors\\nRLY-1971/RG-6433\\nPhase 1\\nNCT04252339\\nFGFR2\\nFGFR2-driven cancers\\nIntrahepatic\\ncholangiocarcinoma\\nAdvanced solid tumors\\nRLY-4008\\nPhase 1/2\\nNCT04526106\\nPI3Kα\\nSolid tumors\\nRLY-2608\\nPhase 1\\nNCT05216432\\nSchrödinger\\nMALT1\\nNon-Hodgkin's lymphoma\\nSGR-1505\\nPhase 1\\nNCT05544019\\nStructure Therapeutics\\nGLP1R\\nType 2 diabetes\\nObesity\\nGSBR-1290\\nPhase 1\\nNCT05762471\\nAPLNR\\nPulmonary arterial\\nhypertension\\nIdiopathic pulmonary ﬁbrosis\\nANPA-0073\\nPhase 1\\nACTRN12621000644864\\nValo Health\\nS1P1\\nPost-myocardial infarction\\nAcute kidney injury\\nOPL-0301\\nPhase 2\\nNCT05327855\\nROCK1/2\\nDiabetic retinopathy\\nDiabetic complications\\nOPL-0401\\nPhase 2\\nNCT05393284\\naIndications retrieved from the company pipeline.\\nbFor undisclosed trial numbers, press releases are provided as the source of reference.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n565\\n\\nneural networks, and transfer learning techniques, have attracted increasing attention and\\nhave been applied to various aspects of healthcare, such as de novo small-molecule design\\n[51], aging research [44], and pharmacological prediction of drugs based on transcriptional\\ndata of drug-perturbed cell lines [52]. Using publicly available multiomic data and text mining\\n(Figure 3, Key ﬁgure), deep learning has recently been used in studies of fatal disorders with ur-\\ngent and unmet clinical needs. To identify actionable therapeutic targets in amyotrophic lateral\\nsclerosis (ALS), Pun et al. combined a variety of bioinformatic- and deep learning-based models\\nthat were trained using disease-speciﬁc multiomic and text-based data to prioritize druggable\\ngenes, revealing 18 potential targets for ALS treatment [53]. In addition, Fabris et al. established\\na deep learning-based method with a novel modular architecture to identify human genes asso-\\nciated with multiple age-related diseases by learning patterns retrieved from gene or protein fea-\\ntures such as Gene Ontology terms, protein–protein interactions, and biological pathways [54].\\nWest et al. developed a deep learning ensemble trained using the transcriptomic proﬁles of\\n>12 000 embryonic and adult cells [55]. A novel target (COX7A1) for controlling the embryonic–\\nfetal transition was revealed, which could facilitate our understanding of normal development,\\nepimorphic tissue regeneration, and cancer.\\nFurthermore, large language models also aid therapeutic target discovery via rapid biomedical\\ntext mining. Pretrained on a vast amount of text data extracted from millions of publications,\\nlarge language model-based Chat functionalities, such as BioGPT from Microsoft [56] and\\nChatPandaGPT from Insilico Medicineiv, can connect diseases, genes, and biological processes\\nto allow rapid identiﬁcation of the biological mechanisms involved in disease development and\\nprogression, as well as the identiﬁcation of potential drug targets and biomarkers. The ability of\\nKey figure\\nWorkﬂow of artiﬁcial intelligence (AI)-driven target discovery\\nTarget\\nidentification\\nText Data\\nOmic data models\\n●\\n●Metabolomics\\n●\\nTranscriptomics\\n●\\nSignaling pathways\\nMolecular interactions\\n●\\n●Patents\\nDevelopment status\\n●Clinical trial phase\\n●Ligand/compound availability\\nDruggability\\n●\\nTherapeutic modality\\n●Protein localization\\n●\\nAvailability of protein structure\\nDisease causality\\nPublications\\n●\\n●Clinical trials\\n●Financial reports \\n●Protein-protein interaction\\nTarget selection criteria\\nNovelty\\nToxicity\\n●Gene essentiality\\n●Tissue specificity\\n●Cellular process involved\\n●Proteomics\\nGrants\\n●Genomics\\n●Epigenomics\\nDruggable classes \\n●\\nTrends\\nTrends in\\nin Pharmacological\\nPharmacological Sciences\\nSciences\\nFigure 3. AI prioritizes targets for speciﬁc indications by using multi-models that utilize a diverse range of publicly available\\nomic and text data. Omic data encompass genomics, transcriptomics, proteomics, epigenomics, and metabolomics.\\nThese data provide information about altered signaling pathways, molecular interactions, and protein–protein interactions\\nthat can serve as additional inputs for target prioritization. Text-based data are retrieved from funding reports, patents,\\npublications, and clinical trials. During target prioritization, multiple target selection criteria such as protein family class,\\ndevelopment status, druggability, toxicity, and novelty can be applied to reﬁne the list of AI-driven targets to align with\\nspeciﬁc research objectives.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n566\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nthe large language models to understand natural language and interpret complex scientiﬁc con-\\ncepts could make them valuable tools in accelerating disease hypothesis generation. Neverthe-\\nless, large language models, which are typically trained on human-generated text, may not\\nhave the ability to determine the accuracy and appropriateness of the input data. As a result,\\nthey could inadvertently perpetuate human biases and preconceived notions. Moreover, given\\nthat these models rely heavily on published data, they may have limited potential to identify gen-\\nuinely novel targets. Therefore, it is important to acknowledge these limitations and to comple-\\nment their use with other models to ensure the discovery of truly novel and pertinent targets.\\nThe use of AI-generated synthetic data for target identiﬁcation\\n'Synthetic data' refers to artiﬁcially generated data that mimic real-world patterns and charac-\\nteristics. By leveraging AI algorithms, synthetic data can be created to simulate various biolog-\\nical scenarios, thus enabling researchers to explore and analyze a broader range of possibilities\\n[57–59]. This approach can be particularly valuable in therapeutic areas where experimental\\ndata are scarce or difﬁcult to obtain. For example, in rare diseases or conditions where patient\\ndata are limited, AI can generate synthetic data based on existing knowledge and patterns.\\nThese synthetic data can then be used to train AI models and identify potential therapeutic\\ntargets that may have been overlooked [60]. Synthetic data can also be used to validate predic-\\ntions made by AI algorithms, thus providing an additional layer of conﬁdence in the target\\ndiscovery process.\\nFurthermore, AI-generated synthetic data can help to address data imbalance or bias issues. In\\nsome therapeutic areas, particular patient populations may be under-represented in the available\\ndatasets, leading to challenges in target identiﬁcation. AI can generate synthetic data representing\\nthese under-represented populations, allowing more comprehensive and inclusive analysis [61].\\nAlthough AI-generated synthetic data can offer advantages in exploring a broader range of pos-\\nsibilities and addressing data scarcity, it is essential to recognize its limitations. A model cannot\\nsimulate data containing complexities that the model is unaware of, and this limitation should\\nbe fully acknowledged [62]. Simulating under-represented populations, although tempting due\\nto data scarcity, raises ethical concerns because collecting relevant data should be pursued\\nwhenever possible rather than relying solely on synthetic data [63,64]. Moreover, ensuring that\\nthe synthetic data accurately capture the intricate and nuanced aspects of real-world biological\\nsystems presents a signiﬁcant challenge. Therefore, implementing robust validation and quality\\ncontrol measures becomes crucial to establish the reliability and relevance of the generated\\ndata [65].\\nTo responsibly validate and control the quality of synthetic omic data, several options can be con-\\nsidered. First, comparative analyses can be performed to assess the similarity between the syn-\\nthetic data and real-world data. This can involve statistical measures, such as comparing\\ndistributional characteristics, correlation patterns, or feature-level comparisons. In addition,\\nbenchmarking against known ground-truth data, where available, can help to evaluate the accu-\\nracy and performance of the synthetic data [66]. Another approach involves conducting func-\\ntional analyses, such as focusing on the representation of particular cellular types in the\\nsynthetic dataset in the case of single-cell data, to determine whether the synthetic data captures\\nbiological knowledge and exhibits coherent functional relationships [67]. Finally, involving domain\\nexperts and conducting rigorous peer review can provide valuable insights and ensure the appro-\\npriateness and relevance of the synthetic data for target identiﬁcation [59]. These validation and\\nquality control measures, although challenging, can contribute to establishing conﬁdence in the\\nuse of synthetic omic data in research and drug target discovery.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n567\\n\\nTarget selection criteria\\nThe criteria used to select drug targets can greatly impact on the success of drug development\\n(Figure 3). Causality represents a crucial criterion for selecting drug targets. Understanding the\\ncausal mechanisms behind a disease can help researchers to identify driver genes and key path-\\nways that have the greatest potential for effective disease treatment [68]. Apart from experimental\\nmethods, a common computational approach to infer causal relationships between targets and\\ndiseases is network-based analysis, which involves the construction of biological networks that\\ncapture the relationships between different genes, proteins, drugs, and other molecular entities\\n[69]. These networks can be used to identify potential targets that might have a causal involve-\\nment in a disease based on their centrality and connectivity within the network. The growing in-\\nterest in AI and computational biology has led to a need for the development of machine\\nlearning methods that can be utilized for causal inference in biological networks [70]. In this re-\\ngard, the adaptation of classiﬁcation algorithms for causal discovery marks the emergence of\\ncausal inference models in biomedical research [71–73].\\nAnother important consideration is the druggability of a target – the ability of a target to be mod-\\nulated by a drug molecule. Factors that affect druggability include therapeutic modality, protein\\nlocalization, class, and structure availability. For instance, small-molecule drugs are typically used\\nfor targets with well-deﬁned binding pockets (e.g., kinases), whereas protein-based therapies are\\nmore suitable for targets that are difﬁcult to tackle with small molecules. Structural information on\\ndrug targets is helpful for drug design and optimization with AI-based predictions, such as\\nAlphaFold [74], thus expanding protein structure coverage. Target toxicity must also be consid-\\nered by assessing the cellular processes, gene essentiality, and tissue speciﬁcity involved.\\nTrade-off between high-conﬁdence and novel targets\\nNovelty is another crucial factor in target selection in addition to causality, druggability, and toxic-\\nity. Text-based evidence can be used to assess novelty and conﬁdence of a given target. Through\\nscrutinizing the relationship between approved drugs, molecular targets, and therapeutic indica-\\ntions, Santos et al. revealed that high-conﬁdence targets (or 'privileged' target families) accounted\\nfor the majority of approved drugs, whereas drugs tackling novel ﬁrst-in-class targets repre-\\nsented only a small proportion, although this is increasing, especially in the ﬁeld of oncology\\n[75]. Striking a balance between novelty and conﬁdence is essential for target selection. AI-\\npowered natural language processing methodologies can aid this target selection process\\nby extracting supporting evidence connecting a potential target to an indication based on huge\\namounts of data involving scientiﬁc publications, grants, and clinical trials, and this can provide\\na quantiﬁable scale for the novelty and conﬁdence of targets in the context of the disease and en-\\nable ﬂexible target-hunting workﬂows [76]. In addition, tools have been developed to quantify tar-\\nget novelty and conﬁdence. TIN-X is an example that uses text-mining data processed from the\\nscientiﬁc literature to quantify target novelty and conﬁdence by providing two bibliometric indices,\\nnamely the 'novelty index' that represents the scarcity of target-associated publications, and the\\n'importance index' that assesses the strength of the association between a given target–disorder\\npair [77]. Furthermore, AI could facilitate drug repurposing by connecting a high-conﬁdence tar-\\nget with known drugs to new disorders where the drugs have not been investigated, enabling\\ncost-effective and time-saving drug discovery for both common and rare diseases [78].\\nAI-identiﬁed targets validated in experiments\\nTarget validation using cell and animal models is a crucial step in target discovery to reduce the\\nproject attrition rate and the cost of drug development in the pharmaceutical industry (Box 1).\\nAn increasing number of AI-identiﬁed targets are being successfully validated. For example, 28\\nAI-proposed targets for ALS treatment were validated in an ALS-mimicking Drosophila model,\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n568\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nrevealing eight unreported targets whose suppression strongly rescues eye neurodegeneration\\n[53]. In addition, in the same therapeutic area, Zhang et al. developed a machine learning-\\nbased method to identify KANK1 as a novel gene linked to ALS and validated the neurotoxic\\neffects of KANK1 mutations reproduced by CRISPR–Cas9 in human neurons [79]. Inhibition of\\nHDAC6 was identiﬁed as a cardioprotective strategy by deep learning, and was validated via a\\nBAG3 cardiomyocyte-knockout mouse model of dilated cardiomyopathy [80]. CDK20 was\\nidentiﬁed as a target for the treatment of HCC using deep learning-based methods, and a\\nhighly potent small-molecule inhibitor designed by generative AI demonstrated selective\\nantiproliferation activity in an HCC cell line [81]. Furthermore, Zeng et al. developed deepDTnet\\nbased on 15 heterogeneous types of chemical, genomic, phenotypic, and cellular networks to\\nfacilitate in silico identiﬁcation of molecular targets for known drugs [82]. One of the identiﬁed\\ndrugs speciﬁcally targeting human ROR-γt shows therapeutic effects in a mouse model of\\nmultiple sclerosis.\\nConcluding remarks and future perspectives\\nTarget discovery is a crucial initial step in the modern drug discovery pipeline. Given that only a small\\nproportion of the potentially druggable targets in humans have been identiﬁed, there is a pressing\\nneed for effective target discovery methods. The growing number of AI-identiﬁed targets being val-\\nidated in experiments highlights the beneﬁts of incorporating AI algorithms into target identiﬁcation to\\nenhance the efﬁciency of novel target discovery and the development of new therapeutics.\\nOne area where AI is expected to make signiﬁcant contributions is in tackling complex diseases.\\nDiseases such as cancer, neurodegenerative disorders, and autoimmune conditions often in-\\nvolve intricate molecular mechanisms that are challenging to unravel. AI-driven target discovery\\nmethods can help to uncover novel targets and pathways underlying these diseases, paving\\nthe way for the development of more effective treatments.\\nMoreover, unexpected infectious disease outbreaks pose a constant threat to global health. The\\nrapid identiﬁcation of potential drug targets and the development of antiviral therapies are crucial\\nfor combating emerging pathogensv. By analyzing genomic data, AI algorithms can aid the iden-\\ntiﬁcation of essential viral proteins or host factors that can be targeted to inhibit viral replication,\\nthus providing valuable insights for the development of antiviral drugs [83].\\nBox 1. Advances in target validation\\nTarget validation using both cell and animal models is crucial to conﬁrm the modulatory effects of the proposed target on\\ndisease development. Although 2D cell culture and rodent models are the prevailing tools for target validation, the difﬁculty\\nof system establishment and the lack of complexity or recapitulation of human development limit their power as highly\\nrepresentative models. Organoids – 3D cell models derived from either induced pluripotent stem cells (iPSCs) or adult\\nstem cells (ASCs) – have arisen as a promising technique for both disease research and drug testing by allowing the cap-\\nture of tissue architecture and cellular microenvironment in vitro [84]. Taking advantage of their self-organizing ability,\\norganoids are able to mimic actual organ development, and have been successfully established for multiple human organs\\n(e.g., intestine, stomach, lung, liver, kidney, and brain) to explore the pathogenic mechanisms of various diseases [85–87].\\nFurthermore, because patient-derived organoids can retain the genetic, histopathological, and therapeutic response\\nphenotypes of the primary disease tissue, these models have made their way into identifying personalized therapeutic\\nregimens and drug efﬁcacy testing [88,89]. In colorectal cancer, patient-derived colon organoids served as an effective tool\\nto evaluate the efﬁcacy of CAR-T cell therapy [90].\\nIn both industrial and clinical laboratories there is a tendency to adopt automation to streamline experiments, data collec-\\ntion, and data analysis. With recent breakthroughs in bioengineering and machine learning, laboratory automation can\\ngreatly improve work efﬁciency and reproducibility by increasing data generation rate, reducing human technical variation,\\nand avoiding contaminant exposure [91,92]. The development and commercialization rate of novel therapeutic interven-\\ntions can also be enhanced by automation. For example, Insilico Medicine have launched an AI-driven robotic laboratory\\nthat is an interconnected expansion of their end-to-end AI drug discovery platformix. Despite several remaining obstacles,\\nthe progressive integration of automation will revolutionize the laboratory environment to maximize research success.\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n569\\nOutstanding questions\\nCan AI algorithms accurately predict\\ntarget validation results and adverse\\neffects, as well as druggability, speci-\\nﬁcity, off-target effects, and potential\\ninteractions with other drugs, for po-\\ntential targets across different test sys-\\ntems (cell lines, animals, and humans)?\\nHow can AI-driven target discovery ap-\\nproaches be validated, benchmarked\\nagainst traditional experimental methods,\\nand also effectively incorporate domain\\nknowledge and expert insights to ensure\\nreliability, reproducibility, and enhanced\\ntarget identiﬁcation and validation?\\nHow can AI algorithms uncover the full\\nmechanism of action at selected tar-\\ngets, consider the heterogeneity and\\nvariability of diseases including individ-\\nual variations, and leverage this under-\\nstanding\\nto\\noptimize\\ncombination\\ntherapies, leading to the identiﬁcation\\nof synergistic drug–target combinations\\nfor improved treatment outcomes?\\nHow can we validate the reliability and\\nrobustness\\nof\\npredictions\\nand\\ndiscoveries based on synthetic AI-\\ngenerated data, and how does it\\ncompare to experimental validation\\nusing real-world data?\\n\\nAI also has the potential to revolutionize the discovery of efﬁcient combinations of therapeutic tar-\\ngets and mechanisms. Complex diseases often involve multiple molecular pathways and inter-\\nplay among various biological factors. AI algorithms can analyze diverse datasets, including\\ngenomic data, patient records, and synthetic lethality, to identify synergistic combinations of tar-\\ngets and mechanisms that may offer enhanced therapeutic effects. This approach can potentially\\ntransform treatment strategies, particularly in diseases where monotherapies have shown limited\\neffectiveness.\\nFurthermore, the integration of AI with fully automated robotic laboratories offers the potential for\\nhigh-throughput target validation and screening. Automated experiments, coupled with AI-driven\\ndata analysis, can expedite the validation of predicted targets, enabling researchers to assess\\ntheir therapeutic potential quickly. This combination of AI and automation has the potential to\\nrevolutionize the drug discovery process and signiﬁcantly reduce the time and cost required for\\ntarget identiﬁcation and validation.\\nDespite the tremendous progress made in AI-driven target discovery, several outstanding ques-\\ntions and challenges remain (see Outstanding questions). Ethical considerations, data privacy,\\nand regulatory frameworks are crucial aspects that must be addressed to ensure responsible\\nand ethical deployment of AI in drug development. Furthermore, the interpretability and\\nexplainability of AI algorithms are essential for gaining trust and acceptance from the scientiﬁc\\nand medical communities. It is pertinent to note that, although AI has demonstrated potential in ex-\\npediting the early stages of drug discovery such as target identiﬁcation and lead optimization, it\\ncannot signiﬁcantly shorten the time required for clinical trials during drug development. This is be-\\ncause of the long period of time spent on ethical and regulatory approval, patient recruitment, du-\\nration of treatment, and data analysis, irrespective of whether the drug was developed by AI or not.\\nIn summary, AI has emerged as a powerful tool in target discovery and drug development, and\\nis revolutionizing how we identify novel drug targets and repurpose existing drugs. With the\\ncontinued advancements in AI technology and the collaborative efforts of researchers, we\\ncan look forward to a future where AI plays an indispensable role in accelerating the develop-\\nment of safe and effective therapeutics for a wide range of diseases, ultimately improving\\nhuman health and well-being.\\nAcknowledgments\\nWe thank Drs Feng Ren and Alex Aliper for expert advice, and Drs Xi Long and Bonnie Hei Man Liu for graphical illustrations\\nand literature review.\\nDeclaration of interests\\nF.W.P., I.V.O., and A.Z. are employees of Insilico Medicine Hong Kong Ltd.\\nResources\\nihttps://ftloscience.com/process-costs-drug-development/\\niiwww.ﬁercebiotech.com/medtech/breaking-big-pharma-s-ai-barrier-insilico-medicine-uncovers-novel-target-new-drug-for\\niiiwww.nature.com/articles/d43747-021-00045-7\\nivwww.eurekalert.org/news-releases/982543\\nvwww.prnewswire.com/news-releases/insilico-medicine-announces-novel-3cl-protease-inhibitor-preclinical-candidate-for-\\ncovid-19-treatment-301553766.html\\nviwww.exscientia.ai/dsp-1181\\nviihttps://investors.exscientia.ai/press-releases/press-release-details/2021/exscientia-announces-second-molecule-\\ncreated-using-ai-from-sumitomo-dainippon-pharma-collaboration-to-enter-phase-1-clinical-trial/Default.aspx\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n570\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\\nviiihttps://investors.exscientia.ai/press-releases/press-release-details/2023/Exscientia-Announces-First-in-Human-Study-\\nfor-Bristol-Myers-Squibb-In-Licensed-PKC-Theta-Inhibitor-EXS4318/default.aspx\\nixwww.globenewswire.com/news-release/2023/01/05/2583816/0/en/Insilico-Medicine-launches-6th-generation-\\nIntelligent-Robotics-Lab-to-further-accelerate-its-AI-driven-drug-discovery.html\\nReferences\\n1. Hinkson, I.V. et al. (2020) Accelerating therapeutics for opportu-\\nnities in medicine: a paradigm shift in drug discovery. Front.\\nPharmacol. 11, 770\\n2. Zhou, Y. et al. (2022) Therapeutic target database update 2022:\\nfacilitating drug discovery with enriched comparative data of\\ntargeted agents. Nucleic Acids Res. 50, D1398–D1407\\n3. Kana, O. and Brylinski, M. (2019) Elucidating the druggability of\\nthe human proteome with eFindSite. J. Comput. Aided Mol.\\nDes. 33, 509–519\\n4. Finan, C. et al. (2017) The druggable genome and support for\\ntarget identiﬁcation and validation in drug development. Sci.\\nTransl. Med. 9, eaag1166\\n5. Sun, D. et al. (2022) Why 90% of clinical drug development fails\\nand how to improve it? Acta Pharm. Sin. B 12, 3049–3062\\n6. Shiraiwa, K. et al. (2020) Chemical tools for endogenous protein\\nlabeling and proﬁling. Cell Chem. Biol. 27, 970–985\\n7. van der Zouwen, A.J. and Witte, M.D. (2021) Modular ap-\\nproaches to synthesize activity- and afﬁnity-based chemical\\nprobes. Front. Chem. 9, 644811\\n8. Ong, S.E. and Mann, M. (2006) A practical recipe for stable iso-\\ntope labeling by amino acids in cell culture (SILAC). Nat. Protoc.\\n1, 2650–2660\\n9. Jin, J. et al. (2023) SIRT3-dependent delactylation of cyclin E2\\nprevents hepatocellular carcinoma growth. EMBO Rep. 24,\\ne56052\\n10. Li, X. et al. (2023) Deacetylation induced nuclear condensation of\\nHP1gamma promotes multiple myeloma drug resistance. Nat.\\nCommun. 14, 1290\\n11. Wang, Y. et al. (2022) DUT enhances drug resistance to protea-\\nsome inhibitors via promoting mitochondrial function in multiple\\nmyeloma. Carcinogenesis 43, 1030–1038\\n12. Montero-Calle, A. et al. (2023) In-depth quantitative proteomics\\nanalysis revealed C1GALT1 depletion in ECC-1 cells mimics an\\naggressive endometrial cancer phenotype observed in cancer\\npatients with low C1GALT1 expression. Cell Oncol. (Dordr) 46,\\n697–715\\n13. Kortum, B. et al. (2022) Combinatorial treatment with statins and\\nniclosamide prevents CRC dissemination by unhinging the\\nMACC1–beta-catenin–S100A4 axis of metastasis. Oncogene\\n41, 4446–4458\\n14. Qi, T.F. et al. (2023) Parallel-reaction monitoring revealed altered\\nexpression of a number of epitranscriptomic reader, writer, and\\neraser proteins accompanied with colorectal cancer metastasis.\\nProteomics 23, e2200059\\n15. Nidhi, S. et al. (2021) Novel CRISPR–Cas systems: an updated\\nreview of the current achievements, applications, and future re-\\nsearch perspectives. Int. J. Mol. Sci. 22, 3327\\n16. Samelson, A.J. et al. (2022) BRD2 inhibition blocks SARS-CoV-2\\ninfection by reducing transcription of the host cell receptor ACE2.\\nNat. Cell Biol. 24, 24–34\\n17. Ramkumar, P. et al. (2020) CRISPR-based screens uncover de-\\nterminants of immunotherapy response in multiple myeloma.\\nBlood Adv. 4, 2899–2911\\n18. Chakraborty, S. et al. (2018) Onco-Multi-OMICS approach: a\\nnew frontier in cancer research. Biomed. Res. Int. 2018,\\n9836256\\n19. Nurk, S. et al. (2022) The complete sequence of a human\\ngenome. Science 376, 44–53\\n20. Buniello, A. et al. (2019) The NHGRI–EBI GWAS catalog of\\npublished genome-wide association studies, targeted arrays\\nand summary statistics 2019. Nucleic Acids Res. 47,\\nD1005–D1012\\n21. Einarsson, G.G. et al. (2021) Extended-culture and culture-\\nindependent molecular analysis of the airway microbiota in cystic\\nﬁbrosis following CFTR modulation with ivacaftor. J. Cyst. Fibros.\\n20, 747–753\\n22. Sewell, G.W. and Kaser, A. (2022) Interleukin-23 in the patho-\\ngenesis of inﬂammatory bowel disease and implications for ther-\\napeutic intervention. J. Crohns Colitis 16, ii3–ii19\\n23. Deelen, J. et al. (2019) Publisher correction: a meta-analysis of\\ngenome-wide association studies identiﬁes multiple longevity\\ngenes. Nat. Commun. 10, 3669\\n24. Namba, S. et al. (2022) A practical guideline of genomics-driven\\ndrug discovery in the era of global biobank meta-analysis. Cell\\nGenom. 2, 100190\\n25. Abell, N.S. et al. (2022) Multiple causal variants underlie genetic\\nassociations in humans. Science 375, 1247–1254\\n26. Assum, I. et al. (2022) Tissue-speciﬁc multi-omics analysis of\\natrial ﬁbrillation. Nat. Commun. 13, 441\\n27. Suhre, K. et al. (2017) Erratum: connecting genetic risk to dis-\\nease end points through the human blood plasma proteome.\\nNat. Commun. 8, 14357\\n28. Yin, X. et al. (2022) Integrating transcriptomics, metabolomics,\\nand GWAS helps reveal molecular mechanisms for metabolite\\nlevels and disease risk. Am. J. Hum. Genet. 109, 1727–1741\\n29. Mountjoy, E. et al. (2021) An open approach to systematically pri-\\noritize causal variants and genes at all published human GWAS\\ntrait-associated loci. Nat. Genet. 53, 1527–1533\\n30. Na, F. et al. (2022) KMT2C deﬁciency promotes small cell\\nlung cancer metastasis through DNMT3A-mediated epigenetic\\nreprogramming. Nat. Can. 3, 753–767\\n31. Gulﬁdan, G. et al. (2022) Systems biomarkers for papillary thyroid\\ncancer prognosis and treatment through multi-omics networks.\\nArch. Biochem. Biophys. 715, 109085\\n32. Lu, J. et al. (2021) Multi-omics analysis of fatty acid metabolism in\\nthyroid carcinoma. Front. Oncol. 11, 737127\\n33. Raivola, J. et al. (2022) Multiomics characterization implicates\\nPTK7 in ovarian cancer EMT and cell plasticity and offers strate-\\ngies for therapeutic intervention. Cell Death Dis. 13, 714\\n34. Pinero, J. et al. (2018) Network, transcriptomic and genomic fea-\\ntures differentiate genes relevant for drug response. Front.\\nGenet. 9, 412\\n35. Wolber,\\nG.\\net\\nal.\\n(2008)\\nMolecule–pharmacophore\\nsuperpositioning and pattern matching in computational drug\\ndesign. Drug Discov. Today 13, 23–29\\n36. Lee, A. et al. (2016) Using reverse docking for target identiﬁcation\\nand its applications for drug discovery. Expert Opin. Drug Discov.\\n11, 707–715\\n37. Nettles, J.H. et al. (2006) Bridging chemical and biological space:\\n'target ﬁshing' using 2D and 3D molecular descriptors. J. Med.\\nChem. 49, 6802–6810\\n38. Lo, Y.C. et al. (2016) 3D chemical similarity networks for\\nstructure-based target prediction and scaffold hopping. ACS\\nChem. Biol. 11, 2244–2253\\n39. Vamathevan, J. et al. (2019) Applications of machine learning in\\ndrug discovery and development. Nat. Rev. Drug Discov. 18,\\n463–477\\n40. Mamoshina, P. et al. (2018) Machine learning on human muscle\\ntranscriptomic data for biomarker discovery and tissue-speciﬁc\\ndrug target identiﬁcation. Front. Genet. 9, 242\\n41. Zhavoronkov, A. et al. (2019) Deep biomarkers of aging and\\nlongevity: from research to applications. Aging (Albany NY) 11,\\n10771–10780\\n42. Muslu, O. et al. (2022) GuiltyTargets: prioritization of novel thera-\\npeutic targets with network representation learning. IEEE/ACM\\nTrans. Comput. Biol. Bioinform. 19, 491–500\\n43. Liu, R. et al. (2021) A deep learning framework for drug\\nrepurposing via emulating clinical trials on real-world patient\\ndata. Nat. Mach. Intell. 3, 68–75\\n44. Zhavoronkov, A. et al. (2019) Deep learning enables rapid identi-\\nﬁcation of potent DDR1 kinase inhibitors. Nat. Biotechnol. 37,\\n1038–1040\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n571\\n\\n45. Ivanenkov, Y.A. et al. (2023) Chemistry42: an AI-driven platform\\nfor molecular design and optimization. J. Chem. Inf. Model. 63,\\n695–701\\n46. Obrezanova, O. (2023) Artiﬁcial intelligence for compound phar-\\nmacokinetics prediction. Curr. Opin. Struct. Biol. 79, 102546\\n47. Chen, R. et al. (2018) Machine learning for drug–target interac-\\ntion prediction. Molecules 23, 2208\\n48. Ye, Q. et al. (2021) A uniﬁed drug–target interaction prediction\\nframework based on knowledge graph and recommendation\\nsystem. Nat. Commun. 12, 6775\\n49. Kavalci, E. and Hartshorn, A. (2023) Improving clinical trial design\\nusing interpretable machine learning based prediction of early\\ntrial termination. Sci. Rep. 13, 121\\n50. McCulloch, W.S. and Pitts, W. (1943) A logical calculus of the\\nideas immanent in nervous activity. Bull. Math. Biophys. 5,\\n115–133\\n51. Zhavoronkov, A. et al. (2019) Artiﬁcial intelligence for aging and\\nlongevity research: Recent advances and perspectives. Ageing\\nRes. Rev. 49, 49–66\\n52. Aliper, A. et al. (2016) Deep learning applications for predicting\\npharmacological properties of drugs and drug repurposing\\nusing transcriptomic data. Mol. Pharm. 13, 2524–2530\\n53. Pun, F.W. et al. (2022) Identiﬁcation of therapeutic targets for\\namyotrophic lateral sclerosis using pandaomics – an AI-\\nenabled biological target discovery platform. Front. Aging\\nNeurosci. 14, 914017\\n54. Fabris, F. et al. (2020) Using deep learning to associate human\\ngenes with age-related diseases. Bioinformatics 36, 2202–2208\\n55. West, M.D. et al. (2018) Use of deep neural network ensembles\\nto identify embryonic-fetal transition markers: repression of\\nCOX7A1 in embryonic and cancer cells. Oncotarget 9,\\n7796–7811\\n56. Luo, R. et al. (2022) BioGPT: generative pre-trained transformer\\nfor biomedical text generation and mining. Brief. Bioinform. 23,\\nbbac409\\n57. Shayakhmetov, R. et al. (2020) Molecular generation for desired\\ntranscriptome changes with adversarial autoencoders. Front.\\nPharmacol. 11, 269\\n58. Vinas, R. et al. (2022) Adversarial generation of gene expression\\ndata. Bioinformatics 38, 730–737\\n59. Beaulieu-Jones, B.K. et al. (2019) Privacy-preserving generative\\ndeep neural networks support clinical data sharing. Circ.\\nCardiovasc. Qual. Outcomes 12, e005122\\n60. Song, J. et al. (2021) The discovery of new drug–target interac-\\ntions for breast cancer treatment. Molecules 26, 7474\\n61. Chawla, N.V. et al. (2002) SMOTE: synthetic minority over-\\nsampling technique. J. Artif. Intell. Res. 16, 321–357\\n62. Achuthan, S. et al. (2022) Leveraging deep learning algorithms\\nfor synthetic data generation to design and analyze biological\\nnetworks. J. Biosci. 47, 43\\n63. Howe, E.G., III and Elenberg, F. (2020) Ethical challenges posed\\nby big data. Innov. Clin. Neurosci. 17, 24–30\\n64. Bhanot, K. et al. (2021) The problem of fairness in synthetic\\nhealthcare data. Entropy (Basel) 23, 1165\\n65. Rajotte, J.F. et al. (2022) Synthetic data as an enabler for ma-\\nchine learning applications in medicine. iScience 25, 105331\\n66. El Emam, K. et al. (2022) Utility metrics for evaluating synthetic\\nhealth data generation methods: validation study. JMIR Med.\\nInform. 10, e35734\\n67. Treppner, M. et al. (2021) Synthetic single cell RNA sequencing\\ndata from small pilot studies using deep generative models.\\nSci. Rep. 11, 9403\\n68. Nogales, C. et al. (2022) Network pharmacology: curing causal\\nmechanisms instead of treating symptoms. Trends Pharmacol.\\nSci. 43, 136–150\\n69. Buphamalai, P. et al. (2021) Network analysis reveals rare dis-\\nease signatures across multiple levels of biological organization.\\nNat. Commun. 12, 6306\\n70. Lecca, P. (2021) Machine learning for causal inference in biolog-\\nical networks: perspectives of this challenge. Front. Bioinform. 1,\\n746712\\n71. Cassan, O. et al. (2021) Inferring and analyzing gene regulatory\\nnetworks from multi-factorial expression data: a complete and\\ninteractive suite. BMC Genomics 22, 387\\n72. Gillani, Z. et al. (2014) CompareSVM: supervised, support vec-\\ntor machine (SVM) inference of gene regularity networks. BMC\\nBioinformatics 15, 395\\n73. Zhou, X. and Kosorok, M.R. (2017) Causal nearest neighbor\\nrules for optimal treatment regimes. ArXiv Published online\\nNovember 22, 2017. https://arxiv.org/abs/1711.08451\\n74. Varadi, M. et al. (2022) AlphaFold Protein Structure Database:\\nmassively expanding the structural coverage of protein-\\nsequence space with high-accuracy models. Nucleic Acids\\nRes. 50, D439–D444\\n75. Santos, R. et al. (2017) A comprehensive map of molecular drug\\ntargets. Nat. Rev. Drug Discov. 16, 19–34\\n76. Vera, C.D. et al. (2022) Treating Duchenne muscular dystrophy:\\nthe promise of stem cells, artiﬁcial intelligence, and multi-omics.\\nFront. Cardiovasc. Med. 9, 851491\\n77. Cannon, D.C. et al. (2017) TIN-X: target importance and novelty\\nexplorer. Bioinformatics 33, 2601–2603\\n78. Pushpakom, S. et al. (2019) Drug repurposing: progress, chal-\\nlenges and recommendations. Nat. Rev. Drug Discov. 18, 41–58\\n79. Zhang, S. et al. (2022) Genome-wide identiﬁcation of the genetic\\nbasis of amyotrophic lateral sclerosis. Neuron 110, 992–1008\\n80. Yang, J. et al. (2022) Phenotypic screening with deep learning\\nidentiﬁes HDAC6 inhibitors as cardioprotective in a BAG3\\nmouse model of dilated cardiomyopathy. Sci. Transl. Med. 14,\\neabl5654\\n81. Ren, F. et al. (2023) AlphaFold accelerates artiﬁcial intelligence\\npowered drug discovery: efﬁcient discovery of a novel CDK20\\nsmall molecule inhibitor. Chem. Sci. 14, 1443–1452\\n82. Zeng, X. et al. (2020) Target identiﬁcation among known drugs by\\ndeep learning from heterogeneous networks. Chem. Sci. 11,\\n1775–1797\\n83. Ong, E. et al. (2020) COVID-19 coronavirus vaccine design using\\nreverse vaccinology and machine learning. Front. Immunol. 11,\\n1581\\n84. Jensen, C. and Teng, Y. (2020) Is it time to start transitioning\\nfrom 2D to 3D cell culture? Front. Mol. Biosci. 7, 33\\n85. Fan, W. et al. (2022) Applications of brain organoids for infectious\\ndiseases. J. Mol. Biol. 434, 167243\\n86. Sidhaye, J. and Knoblich, J.A. (2021) Brain organoids: an en-\\nsemble of bioassays to investigate human neurodevelopment\\nand disease. Cell Death Differ. 28, 52–67\\n87. Angus, H.C.K. et al. (2019) Intestinal organoids as a tool for inﬂam-\\nmatory bowel disease research. Front. Med. (Lausanne) 6, 334\\n88. Wensink, G.E. et al. (2021) Patient-derived organoids as a pre-\\ndictive biomarker for treatment response in cancer patients.\\nNPJ Precis. Oncol. 5, 30\\n89. Berkers, G. et al. (2019) Rectal organoids enable personalized\\ntreatment of cystic ﬁbrosis. Cell Rep. 26, 1701–1708\\n90. Schnalzger, T.E. et al. (2019) 3D model for CAR-mediated cyto-\\ntoxicity using patient-derived colorectal cancer organoids. EMBO\\nJ. 38, e100928\\n91. Burger, B. et al. (2020) A mobile robotic chemist. Nature 583,\\n237–241\\n92. Crone, M.A. et al. (2020) A role for biofoundries in rapid development\\nand validation of automated SARS-CoV-2 clinical diagnostics. Nat.\\nCommun. 11, 4464\\nTrends in Pharmacological Sciences\\nOPEN ACCESS\\n572\\nTrends in Pharmacological Sciences, September 2023, Vol. 44, No. 9\\n\"]"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#query_embedding\n",
    "similar_docs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\Arvind\\AppData\\Local\\Temp\\ipykernel_20592\\2096494473.py:8: LangChainDeprecationWarning: The method `BaseChatModel.predict` was deprecated in langchain-core 0.1.7 and will be removed in 1.0. Use :meth:`~invoke` instead.\n",
      "  response = groq_llm.predict(prompt)\n"
     ]
    }
   ],
   "source": [
    "# Construct a prompt for the LLM\n",
    "prompt = f\"Based on the following research papers, answer the query: {test_query}\\n\\n\"\n",
    "for i, doc in enumerate(similar_docs):\n",
    "    prompt += f\"Research Paper {i+1}: {doc[:500]}...\\n\\n\"\n",
    "prompt += \"Provide a concise and informative response.\"\n",
    "\n",
    "# Get response from Groq LLM\n",
    "response = groq_llm.predict(prompt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Recent advancements in AI-powered drug discovery include the industrialization of AI platforms, such as the Patrimony computing platform, which offers lessons learned in this field (Guedj et al., 2021). Additionally, there has been a comprehensive study on AI-powered drug discovery, highlighting its potential to accelerate pharmaceutical research (Kolluri, 2021). Furthermore, AI has shown promise in therapeutic target discovery, with its ability to analyze large datasets and intricate biological networks (Pun et al., 2021). Overall, AI is making significant strides in various aspects of drug discovery, including speeding up the process and improving target identification.\\n\\nReferences:\\nGuedj, M., Swindle, J., Hamon, A., Hubert, S., Desvaux, E., Laplume, J., ... & Moingeon, P. (2021). Industrializing AI-powered drug discovery: lessons learned from the Patrimony computing platform. Expert Opinion on Drug Discovery, 16(4), 381-391.\\n\\nKolluri, V. (2021). A comprehensive study on AI-powered drug discovery: rapid development of pharmaceutical research. Journal of Emerging Technologies and Innovative Research, 10(2), 33-42.\\n\\nPun, F. W., Ozerov, I. V., & Zhavoronkov, A. (2021). AI-powered therapeutic target discovery. Drug Discovery Today, 26(3), 821-827.'"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "response"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "groq",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
